<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Cancer as deregulation of complex machinery | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="Jonas Béal - PhD thesis in Genomics" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Cancer as deregulation of complex machinery | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Jonas Béal - PhD thesis in Genomics" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Cancer as deregulation of complex machinery | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="Jonas Béal - PhD thesis in Genomics" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="scientific-modeling-abstract-the-complexity.html"/>
<link rel="next" href="mechanistic-cancer.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="remerciements.html"><a href="remerciements.html"><i class="fa fa-check"></i>Remerciements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-family"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
<li class="chapter" data-level="3.4.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-interventions-and-causality"><i class="fa fa-check"></i><b>3.4.4</b> Mechanistic models, interventions and causality</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-tools-list"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#continuous-time-markov-processes"><i class="fa fa-check"></i><b>4.2.1</b> Continuous-time Markov processes</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.2</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.3</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.4</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-chap.html"><a href="personalization-chap.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-chap.html"><a href="personalization-chap.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-chap.html"><a href="personalization-chap.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-chap.html"><a href="personalization-chap.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-chap.html"><a href="personalization-chap.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-chap.html"><a href="personalization-chap.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-chap.html"><a href="personalization-chap.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-chap.html"><a href="personalization-chap.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-chap.html"><a href="personalization-chap.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#ML-comp"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#part2-limits"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a><ul>
<li class="chapter" data-level="7.1.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-framework-and-general-principles"><i class="fa fa-check"></i><b>7.1.1</b> Evaluation framework and general principles</a></li>
<li class="chapter" data-level="7.1.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#some-frequent-problems-and-recommended-statistical-tools"><i class="fa fa-check"></i><b>7.1.2</b> Some frequent problems and recommended statistical tools</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artificial-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artificial examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="causal-chap.html"><a href="causal-chap.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="causal-chap.html"><a href="causal-chap.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="causal-chap.html"><a href="causal-chap.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="causal-chap.html"><a href="causal-chap.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-basic"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="causal-chap.html"><a href="causal-chap.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#part3-limits"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a><ul>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#one-path-among-others"><i class="fa fa-check"></i>One path among others</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#in-other-directions"><i class="fa fa-check"></i>In other directions</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#r2-and-beyond"><i class="fa fa-check"></i><b>C.1</b> <span class="math inline">\(R^2\)</span> and beyond</a><ul>
<li class="chapter" data-level="C.1.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.1.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-r2surv"><i class="fa fa-check"></i><b>C.1.2</b> <span class="math inline">\(R^2\)</span> for survival data</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="D" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html"><i class="fa fa-check"></i><b>D</b> Résumé détaillé</a><ul>
<li class="chapter" data-level="D.1" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#modélisation-et-cancer"><i class="fa fa-check"></i><b>D.1</b> Modélisation et cancer</a></li>
<li class="chapter" data-level="D.2" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#des-modèles-logiques-personnalisés-de-cancer"><i class="fa fa-check"></i><b>D.2</b> Des modèles logiques personnalisés de cancer</a></li>
<li class="chapter" data-level="D.3" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#quantification-statistique-de-limpact-clinique-des-modèles"><i class="fa fa-check"></i><b>D.3</b> Quantification statistique de l'impact clinique des modèles</a></li>
<li class="chapter" data-level="D.4" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#conclusion-1"><i class="fa fa-check"></i><b>D.4</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="cancer-as-deregulation-of-complex-machinery" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Cancer as deregulation of complex machinery</h1>
<blockquote>
<p><em>Does not the entireness of the complex hint at the perfection of the simple?</em><br />
Edgar Allan Poe (Eureka, 1848)</p>
</blockquote>
<p>Armed with all these models, whether statistical or mechanistic, we are going to look at cancer, a particularly complex system that fully justifies their use Since the first chapter recalled how important prior knowledge of the phenomenon under study is for designing models, whatever their nature, this chapter will briefly summarize some of the most important characteristics of this disease before returning to the models themselves in the next chapter. Without aiming for exhaustiveness, and after an epidemiological and statistical description, we will focus on the most useful information for the modeler, i.e., the underlying biological mechanisms and available data.</p>
<div class="figure" style="text-align: center"><span id="fig:bath"></span>
<img src="fig/bath.jpg" alt="Cancer is an old disease. Rembrandt, Bathsheba at Her Bath, c. 1654, oil on canvas, Louvre Museum, Paris" width="90%" />
<p class="caption">
Figure 2.1: <strong>Cancer is an old disease.</strong> Rembrandt, <em>Bathsheba at Her Bath</em>, c. 1654, oil on canvas, Louvre Museum, Paris
</p>
</div>

<div id="what-is-cancer" class="section level2">
<h2><span class="header-section-number">2.1</span> What is cancer?</h2>
<p>Cancer can be described as a group of diseases characterized by <strong>uncontrolled cell divisions and growth which can spread to surrounding tissues</strong>. Descriptions of this disease, especially when associated with solid tumors, have been found as far back as ancient Egyptian documents, at least 1600 BC and we know from the first century A.D. with Aulus Celsus that it is better to remove the tumors and this as soon as possible <span class="citation">(Hajdu <a href="#ref-hajdu2011note">2011</a><a href="#ref-hajdu2011note">a</a>)</span>. Progress will accelerate during the Renaissance with the renewed interest in medicine, and anatomy in particular, which will advance the knowledge of tumor pathology and surgery <span class="citation">(Hajdu <a href="#ref-hajdu2011note2">2011</a><a href="#ref-hajdu2011note2">b</a>)</span>. The progress of anatomical knowledge has also left brilliant testimonies in the field of painting, which make the renown of the Renaissance today. The precision of these artists' traits has also allowed some retrospective medical analyses, some of them going so far as to identify the signs of a tumor in some of the subjects of these paintings <span class="citation">(Bianucci et al. <a href="#ref-bianucci2018earliest">2018</a>)</span>. Such is the bluish stain on the left breast of the Bathsheba painted by Rembrandt (Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:bath">2.1</a>) which has been subject to controversial interpretations, sometimes described as an example of &quot;skin discolouration, distortion of symmetry with axillary fullness and peau d'orange&quot; <span class="citation">(Braithwaite and Shugg <a href="#ref-braithwaite1983rembrandt">1983</a>)</span> and sometimes spared by photonic and computationnal analyses <span class="citation">(Heijblom et al. <a href="#ref-heijblom2014monte">2014</a>)</span>. The mechanisms of the disease only began to be elucidated with the appearance of the microscope in the 19th century, which revealed its cellular origin <span class="citation">(Hajdu <a href="#ref-hajdu2012note">2012</a><a href="#ref-hajdu2012note">a</a>)</span>. The classification and description of cancers is then gradually refined and the first non-surgical treatments appear with the discovery of ionising radiation by the Curies <span class="citation">(Hajdu <a href="#ref-hajdu2012note2">2012</a><a href="#ref-hajdu2012note2">b</a>)</span>. The 20th century is then the century of understanding the causes of cancer <span class="citation">(Hajdu and Darvishian <a href="#ref-hajdu2013note">2013</a>; Hajdu and Vadmal <a href="#ref-hajdu2013note2">2013</a>)</span>. Some environmental exposures are characterized as asbestos or tobacco. Finally, the biological mechanisms become clearer with the identification of tumor-causing viruses and especially with the discovery of DNA <span class="citation">(Watson and Crick <a href="#ref-watson1953molecular">1953</a>)</span>. The foundations of our current understanding of cancer date back to this period, which marks the beginning of the molecular biology of cancer. It is this branch of biology that contains the bulk of the knowledge that will be used to build our mechanistic models, and it will be later detailed in Section <a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology">2.3</a>.</p>
<p>One of the ways to read this brief history of cancer is to see that theoretical and clinical progresses have not followed the same timeframes. The medical and clinical management of cancers initially progressed slowly but surely, and this in the absence of an understanding of the mechanisms of cancer. Conversely, the theoretical progress of the last century has not always led to parallel medical progress, except on certain specific points. The interaction between the two is therefore not always obvious. The <strong>transformation of fundamental knowledge into clinical impact is therefore of particular importance</strong>. This is what is called <em>translational medicine</em>, the aim of which is to go from laboratory bench to bedside <span class="citation">(Cohrs et al. <a href="#ref-cohrs2015translational">2015</a>)</span>. It is in this perspective that we will analyze the mechanistic models studied in this thesis. Their objective is to integrate biological knowledge, or at least a synthesis of this knowledge, in order to transform it into a relevant clinical information.</p>
</div>
<div id="epidemio" class="section level2">
<h2><span class="header-section-number">2.2</span> Cancer from a distance: epidemiology and main figures</h2>
<p>Before going down to the molecular level, it is important to detail some figures and trends in the epidemiology of cancer today. Following the description in the previous section, cancer is first and foremost defined as a disease. Considered to be a unique disease, it caused 18.1 million new cancer cases and 9.6 million cancer deaths in 2018 according to the Global Cancer Observatory affiliated to World Health Organization <span class="citation">(Bray et al. <a href="#ref-bray2018global">2018</a>)</span>. However, these aggregated data conceal disparities of various kinds. The first one is geographical. Indeed, mortality figures make cancer one of the leading causes of premature death in most countries of the world but its importance relative to other causes of death is even greater in the more developed countries (Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:globocan-map">2.2</a>). All in all, cancer is the first or second cause of premature death in almost 100 countries worldwide <span class="citation">(Bray et al. <a href="#ref-bray2018global">2018</a>)</span>. These differences call for careful consideration of the impact of population age structures and health-related covariates.</p>
<div class="figure" style="text-align: center"><span id="fig:globocan-map"></span>
<img src="fig/globocan-map.jpg" alt="World map and national rankings of cancer as a cause of premature death. Classification of cancer as a cause of death before the age of 70, based on data for the year 2015. Original Figure, data and methods from Bray et al. (2018)." width="100%" />
<p class="caption">
Figure 2.2: <strong>World map and national rankings of cancer as a cause of premature death.</strong> Classification of cancer as a cause of death before the age of 70, based on data for the year 2015. Original Figure, data and methods from <span class="citation">Bray et al. (<a href="#ref-bray2018global">2018</a>)</span>.
</p>
</div>

<p>A second disparity lies in the different types of cancer. If we classify tumors solely according to their location, i.e., the organ affected first, we already obtain very wide differences. First of all, the incidence varies considerably (Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:cancer-tissues">2.3</a>A)). Cancers do not occur randomly anywhere in the body and certain environments or cell types appear to be more favourable <span class="citation">(Tomasetti and Vogelstein <a href="#ref-tomasetti2015variation">2015</a>)</span>. Mortality is also highly variable but is not directly inferred from incidence. Not all types of cancer have the same prognosis (Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:cancer-tissues">2.3</a>A and B) and survival rates <span class="citation">(Liu et al. <a href="#ref-liu2018integrated">2018</a>)</span>. Although breast cancer is much more common than lung cancer, it causes fewer deaths because its prognosis is, on average, much better. The mechanisms at work in the emergence of cancer are therefore not necessarily the same as those that will govern its evolution or its response to treatment. And still on the response to treatment, Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:cancer-tissues">2.3</a>B highlights another disparity: not only are the survival prognosis associated with each cancer very different, but the evolution (and generally the improvement) of these prognoses has been very uneven over the last few decades. This means that theoretical and therapeutic advances have not been applied to all types of cancer with the same success. It is one more indication of the <strong>diversity of cancer mechanisms in different tissues and biological contexts</strong>, which make it impossible to find a panacea, and which, on the contrary, encourage us to carefully consider the particularities of each tumor, both to understand them and to treat them. Under a generic name and in spite of common characteristics, the cancers thus appear as extremely heterogeneous. And to understand the sources of this heterogeneity, it is necessary to consider the disease on a smaller scale.</p>
<div class="figure" style="text-align: center"><span id="fig:cancer-tissues"></span>
<img src="02-Cancer_files/figure-html/cancer-tissues-1.png" alt="Incidence, mortality and survival per cancer types. (A) World incidence and mortality for the 19 most frequent cancer types in 2018, expressed with age-standardized rates (adjusted age structure based on world population); data retrieved from Global Cancer Observatory. (B) Evolution of 5-years relative survival for the same cancer types based on US data from SEER registries in 1975-1977 and 2006-2012; data retrieved from Jemal et al. (2017)." width="90%" />
<p class="caption">
Figure 2.3: <strong>Incidence, mortality and survival per cancer types</strong>. (A) World incidence and mortality for the 19 most frequent cancer types in 2018, expressed with age-standardized rates (adjusted age structure based on world population); data retrieved from <a href="https://gco.iarc.fr/today/home">Global Cancer Observatory</a>. (B) Evolution of 5-years relative survival for the same cancer types based on US data from SEER registries in 1975-1977 and 2006-2012; data retrieved from <span class="citation">Jemal et al. (<a href="#ref-jemal2017annual">2017</a>)</span>.
</p>
</div>

</div>
<div id="molecular-biology" class="section level2">
<h2><span class="header-section-number">2.3</span> Basic molecular biology and cancer</h2>
<p>If it is not possible and desirable to summarize here the state of knowledge about the biology of cancer, we are going to give a very partial vision focused on the main elements used in this thesis, thus aiming to make it a self-sufficient document. The details necessary for a finer and more general understanding can be found in dedicated textbooks such as <span class="citation">Alberts et al. (<a href="#ref-alberts2007molecular">2007</a>)</span> and <span class="citation">Weinberg (<a href="#ref-weinberg2013biology">2013</a>)</span>.</p>
<div id="central-dogma-and-core-principles" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Central dogma and core principles</h3>
<p>Some of the principles that govern biology can be described at the level of one of its simplest element, the cell. Let us consider for the moment a perfectly healthy cell. It must ensure a certain number of functions necessary for its survival and, sometimes, for its division/reproduction. These functions are encoded to a large extent in its genetic information in the form of DNA, which is stable and shared by the different cells since it is defined at the level of the individual. Most biological functions, however, are not performed by DNA itself which remains in the nucleus of the cell. The DNA is thus transcribed into RNA, another nucleic acid which, in addition to performing some biological functions, becomes the support of the genetic information in the cell. The RNA is then itself translated into new molecules composed of long chains of amino acid residues and called proteins. They are the ones that execute most of the numerous cellular functions: DNA replication, physical structuring of the cell, molecule transport within the cell etc. A rather simplistic but fruitful way to understand this functioning is to consider it as a <strong>progressive transfer of biological information from DNA to proteins</strong>, which has sometimes been summarized as the central dogma of the molecular biology (<a href="cancer-as-deregulation-of-complex-machinery.html#fig:central-dogma">2.4</a>), first stated Francis Crick <span class="citation">(Crick <a href="#ref-crick1970central">1970</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:central-dogma"></span>
<img src="fig/central-dogma.png" alt="Central dogma of molecular biology. Schematic representation of the information flow within the cell, from DNA to proteins through RNA, more precisely described in this video (Image credit Genome Research Limited)." width="80%" />
<p class="caption">
Figure 2.4: <strong>Central dogma of molecular biology.</strong> Schematic representation of the information flow within the cell, from DNA to proteins through RNA, more precisely described in this <a href="https://www.youtube.com/watch?v=J3HVVi2k2No">video</a> (Image credit <em>Genome Research Limited</em>).
</p>
</div>

<p>However, many changes would be necessary to clarify this scheme and the uni-directional nature was questioned early on. Above all, a large number of regulations interact with and disrupt this master plan. The genes are not always all transcribed, or at least not at constant intensities, interrupting or varying the chain upstream. This modulation in the transcription of genes can be induced by proteins, called transcription factors. After a gene transcription, its expression can still be regulated at various stages. RNAs can also be degraded more or less rapidly. RNAs can be reshaped in their structure during their maturation by a process called splicing, which varies the genetic information they carry. Finally, proteins are subject to all kinds of modifications referred to as post-translational, which can change the chemical nature of certain groups or modify the three-dimensional structure of the whole protein. For instance, some proteins perform their function only if a specific amino acid residue is phosphorylated. In addition, these modifications can be transmitted between proteins, further complicating the flow of information. <strong>All these possibilities of regulation play an absolutely essential role in the life of the cell by allowing it to adapt to different contexts, situations and development stages</strong>. From the same genetic material, a cell of the eye and a cell of the heart can thus perform different functions. Similarly, the same cell subjected to different stimuli at different times can provide different responses because these molecular stimuli trigger a regulation of its programme. But all these regulatory mechanisms can be corrupted.</p>
</div>
<div id="a-rogue-machinery" class="section level3">
<h3><span class="header-section-number">2.3.2</span> A rogue machinery</h3>
<p>With the above knowledge we can now return to the definition of cancer as an uncontrolled division of cells that can lead to the growth of a tumor that eventually spreads to the surrounding tissues. Therefore, this corresponds to normal processes, like cell division and reproduction, that are no longer regulated as they should be and are out of control. Experiments on different model organisms have gradually identified genetic mutations as a major source of these deregulations <span class="citation">(Nowell <a href="#ref-nowell1976clonal">1976</a>, <span class="citation">Reddy et al. (<a href="#ref-reddy1982point">1982</a>)</span>)</span> until cancer was clearly considered as a <strong>genetic disease</strong> making Renato Dulbecco, Nobel Laureate in Medicine for his work on oncoviruses, say:</p>
<blockquote>
<p><em>If we wish to learn more about cancer, we must now concentrate on the cellular genome.</em><br />
<span class="citation">(Dulbecco <a href="#ref-dulbecco1986turning">1986</a>)</span>.</p>
</blockquote>
<p>However, cancer is not a Mendelian disease for which it would be sufficient to identify the one and only gene responsible for deregulation. Indeed, the cell has many protective mechanisms. For example, if a genetic mutation appears in the DNA, it has a very high chance of being repaired by dedicated mechanisms. And if it is not repaired, other mechanisms will take over to trigger the programmed death of the cell, called apoptosis, before it can proliferate wildly. So a cancer cell is probably a cell that has learned to resist this cell death. Similarly, in order to generate excessive growth, a cell will need to be able to replicate itself many times. However, there are pieces of sequences on chromosomes called telomeres that help to limit the number of times each cell can replicate. A cancer cell will therefore have to manage to bypass this protection. Thus we can schematically define the properties that must be acquired by the cancereous cells in order to truly deviate the machinery. In an influential article, these properties were summarized in six hallmarks (Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:hallmarks">2.5</a>) which are: resisting cell death, enabling replicative immortality, sustaning proliferative signaling, evading growth suppressors, activating invasion and inducing angiogenesis <span class="citation">(Hanahan and Weinberg <a href="#ref-hanahan2000hallmarks">2000</a>)</span>. Two new ones were subsequently added in the light of advances in knowledge <span class="citation">(Hanahan and Weinberg <a href="#ref-hanahan2011hallmarks">2011</a>)</span>: deregulating cancer energetics and avoiding immne destruction. The acquisition of these capacities generally requires many genetic mutations and is therefore favoured by an underlying genome instability or tumor-promoting inflammation.</p>
<div class="figure" style="text-align: center"><span id="fig:hallmarks"></span>
<img src="fig/hallmarks.png" alt="Hallmarks of cancer. The different biological capabilities acquired by cancer cells. Adapted from Hanahan and Weinberg (2011)." width="70%" />
<p class="caption">
Figure 2.5: <strong>Hallmarks of cancer.</strong> The different biological capabilities acquired by cancer cells. Adapted from <span class="citation">Hanahan and Weinberg (<a href="#ref-hanahan2011hallmarks">2011</a>)</span>.
</p>
</div>

<p>Each of these characteristics, or hallmarks, constitutes a research program in its own right. And for each one there are genetic alterations. These are tissue-specific or not, specific to a hallmark or common to several of them <span class="citation">(Hanahan and Weinberg <a href="#ref-hanahan2000hallmarks">2000</a>)</span>. In any case, <strong>cancer can only result from the combination of different alterations that invalidate several protective mechanisms</strong> at the same time. This is often part of a multi-step process of hallmark acquisition that has been experimentally documented in some specific cases <span class="citation">(Hahn et al. <a href="#ref-hahn1999creation">1999</a>)</span> or more recently inferred from genome-wide data for human patients <span class="citation">(Tomasetti et al. <a href="#ref-tomasetti2015only">2015</a>)</span>. In summary, it appears that in order to study the functioning of cancer cells it is necessary to look at several mechanisms and to be able to consider them not separately but together, in as many different patients as possible. This ambitious programme has been made possible by a technological revolution.</p>
</div>
</div>
<div id="the-new-era-of-genomics" class="section level2">
<h2><span class="header-section-number">2.4</span> The new era of genomics</h2>
<div id="from-sequencing-to-multi-omics-data" class="section level3">
<h3><span class="header-section-number">2.4.1</span> From sequencing to multi-omics data</h3>
<p>In 2001, the first sequencing of the human genome symbolized the beginning of a new era, that of what will become <strong>high-throughput genomics</strong> <span class="citation">(Lander et al. <a href="#ref-lander2001initial">2001</a>; Venter et al. <a href="#ref-venter2001sequence">2001</a>)</span>. From the end of the 20th century, biological data started to accumulate at an ever-increasing rate <span class="citation">(Reuter, Spacek, and Snyder <a href="#ref-reuter2015high">2015</a>)</span>, feeding and accelerating cancer research in particular <span class="citation">(Stratton, Campbell, and Futreal <a href="#ref-stratton2009cancer">2009</a>; Meyerson, Gabriel, and Getz <a href="#ref-meyerson2010advances">2010</a>)</span>. The ability to sequence the human genome as a whole, for an ever-increasing number of individuals, has enabled <strong>less biased and more systematic studies of the causes of cancer</strong> <span class="citation">(Lander <a href="#ref-lander2011initial">2011</a>)</span>. The number of genes associated with cancer increased drastically and some very important genes such as BRAF or PIK3CA have been identified <span class="citation">(Davies et al. <a href="#ref-davies2002mutations">2002</a>; Samuels et al. <a href="#ref-samuels2004high">2004</a>)</span>. Progress also extended to the gene expression data. Gene-expression arrays have made an important contribution by providing access to transcriptomic data (RNA), i.e., what has been transcribed from DNA and is therefore one step further in terms of biological information. This information has made it possible to further explore the differences between normal and tumor cells <span class="citation">(Perou et al. <a href="#ref-perou1999distinctive">1999</a>)</span>, or even to refine the classification of cancers, which until now has been done mainly according to the tumor site. Breast cancers are thus divided into subtypes with different combinations of molecular markers that facilitate the understanding of clinical behavior <span class="citation">(Perou et al. <a href="#ref-perou2000molecular">2000</a>)</span>. One step further, we also note the appearance of prognostic gene signatures such as gene expression patterns correlated with the survival of patients <span class="citation">(Van’t Veer et al. <a href="#ref-van2002gene">2002</a>)</span>. This revolution was then extended to other types of data such as proteins (proteomics), reversible modifications of DNA or DNA-associated proteins (epigenomics), metabolites (metabolomics) and others, each representing a perspective that can complement the others to better understand biological mechanisms, particularly in the case of diseases <span class="citation">(Hasin, Seldin, and Lusis <a href="#ref-hasin2017multi">2017</a>)</span>. We have thus entered the era of multi-omics data <span class="citation">(Vucic et al. <a href="#ref-vucic2012translating">2012</a>)</span>.</p>
</div>
<div id="state-of-the-art-of-cancer-data" class="section level3">
<h3><span class="header-section-number">2.4.2</span> State-of-the art of cancer data</h3>
<p>With respect to cancer in particular, this wealth of data is particularly represented by a family of <strong>studies conducted by The Cancer Genome Atlas (TCGA) consortium</strong>, started in 2008 <span class="citation">(Network and others <a href="#ref-cancer2008comprehensive">2008</a>)</span>. Cohorts of several hundred patients are thus sequenced over the years for different types of cancer <span class="citation">(TCGA and others <a href="#ref-cancer2012comprehensive">2012</a>)</span>, resulting today in a total of 11,000 tumors from 33 of the most prevalent forms of cancer <span class="citation">(Ding et al. <a href="#ref-ding2018perspective">2018</a>)</span>. Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:tcga">2.6</a> provides a partial but striking overview of the depth of data available under this program. We can see the frequencies of alterations of certain groups of genes for a list of cancer types, making it possible to visualize the disparities already anticipated in section <a href="cancer-as-deregulation-of-complex-machinery.html#epidemio">2.2</a> based on patient survival. There are indeed important differences between the organs but also between the different subtypes associated with the same organ. And this representation only corresponds to one layer of data, that of genetic alterations. It could be used for transcriptomic, epigenomic or proteomic data, thus giving rise to an incredibly complex photography.</p>
<div class="figure" style="text-align: center"><span id="fig:tcga"></span>
<img src="fig/tcga.jpg" alt="Genetic alterations frequencies for cancer types from TCGA data. Frequencies of alteration per pahway and tumor types as summaried in Pan-cancer analyses from TCGA data. Reprinted from Sanchez-Vega et al. (2018)." width="90%" />
<p class="caption">
Figure 2.6: <strong>Genetic alterations frequencies for cancer types from TCGA data.</strong> Frequencies of alteration per pahway and tumor types as summaried in Pan-cancer analyses from TCGA data. Reprinted from <span class="citation">Sanchez-Vega et al. (<a href="#ref-sanchez2018oncogenic">2018</a>)</span>.
</p>
</div>

<p>However, the diversity of data available for cancer research extends far beyond this, both in terms of technology and type of data. This may be data from model organisms such as mice or even tumors of human origin made more suitable for experimentation. In the latter category, it is crucial to mention the <strong>huge amount of data available on cell lines</strong>, extracted from human tumors and transformed to be studied in culture. It is then possible to go beyond descriptive data and vary the experimental conditions in order to study the responses of these cells to perturbations and to enrich our knowledge. This provides an opportunity to know the response to more than 100 drugs of about 700 cell lines <span class="citation">(Yang et al. <a href="#ref-yang2012genomics">2012</a>)</span>. The richness of these data, coupled with the omic profiling of each cell line, enables to study the determinants of response to treatment with unprecedented scope <span class="citation">(Iorio et al. <a href="#ref-iorio2016landscape">2016</a>)</span>. More recently, but following a similar logic, other types of inhibition screenings have been proposed based on a more specific technique called CRISPR-Cas9 <span class="citation">(Behan et al. <a href="#ref-behan2019prioritization">2019</a>)</span>. The simplicity of the cell lines in relation to the original tumors makes all these studies possible but sometimes hinders the clinical application of the knowledge acquired. For this reason, other types of biological models have been developed, including patient-derived xenografts (PDX) which is an implant of human tumors in mice to ensure the existence of a certain tumor microenvironment <span class="citation">(Hidalgo et al. <a href="#ref-hidalgo2014patient">2014</a>)</span>, while maintaining drug screening possibilities <span class="citation">(Gao et al. <a href="#ref-gao2015high">2015</a>)</span>. These two types of data, cell lines and PDX, have been used in this thesis, in addition to TCGA patient data, thus justifying the limitation of this presentation, which could otherwise be extended to other types of biological models. Similarly, other technologies are becoming increasingly important in the generation of cancer data, such as single-cell sequencing <span class="citation">(Navin <a href="#ref-navin2015first">2015</a>)</span>, but will not be used in this work.</p>
</div>
</div>
<div id="data-and-beyond-from-genetic-to-network-disease" class="section level2">
<h2><span class="header-section-number">2.5</span> Data and beyond: from genetic to network disease</h2>
<p>All that remains to be done now is to make sense of all these data, to organize them, because <strong>cancer understanding does not flow directly from the abundance of data</strong>, and the ability to produce it may have been outpaced by the ability to analyze it <span class="citation">(Stadler et al. <a href="#ref-stadler2014cancer">2014</a>)</span>. A striking example is that of the prognostic signatures mentioned above. The many signatures or lists of genes proposed, even for the same cancer type, share relatively few genes, are difficult to interpret and their efficiency is sometimes poorly reproducible <span class="citation">(Domany <a href="#ref-domany2014using">2014</a>)</span>. Even more surprisingly, most signatures composed of randomly selected genes were also found to be associated with patient survival <span class="citation">(Venet et al. <a href="#ref-venet2011most">2011</a>)</span>. One of the main avenues for improving the interpretability of the data is the <strong>integration of the prior knowledge</strong> we have of the phenomena, especially in the case of cancer <span class="citation">(Domany <a href="#ref-domany2014using">2014</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:circuit"></span>
<img src="fig/circuit.jpg" alt="Simplistic representation of cellular circuitry. Normal cellular circuit sand sub-circuits (identified by colours) can be reprogrammed to regulate hallmark capabilities within cancer cells. Reprinted from Hanahan and Weinberg (2011)." width="90%" />
<p class="caption">
Figure 2.7: <strong>Simplistic representation of cellular circuitry.</strong> Normal cellular circuit sand sub-circuits (identified by colours) can be reprogrammed to regulate hallmark capabilities within cancer cells. Reprinted from <span class="citation">Hanahan and Weinberg (<a href="#ref-hanahan2011hallmarks">2011</a>)</span>.
</p>
</div>

<p>This <em>a priori</em> knowledge is in fact already present in Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:tcga">2.6</a> since genetic alterations have been grouped in several categories called pathways. A pathway is a group of biological entities and associated chemical reactions, working together to control a specific cell function like apoptosis or cell division. The interest of these groupings may be understood based on the description of hallmarks. Indeed, if the &quot;aim&quot; of a cancer cell is to inactivate each of the protective functions, then it is more relevant to think not by gene but by function. Inactivating only one of the genes associated with the function may be sufficient and it is no longer necessary to inactivate the others. Numerous alterations in a large number of genes in various patients result often in the same key impaired pathways, like alterations of cell cycle or angiogenesis for instance <span class="citation">(Jones et al. <a href="#ref-jones2008core">2008</a>)</span>. It is therefore possible to improve the stability and interpretability of analyses by moving <strong>from the gene scale to the pathway scale</strong> <span class="citation">(Drier, Sheffer, and Domany <a href="#ref-drier2013pathway">2013</a>)</span>. More generally, the integration of biological knowledge often leads to improved performance in various cancer-related prediction tasks, either through the selection of variables or by taking into account the structure of the variables <span class="citation">(Bilal et al. <a href="#ref-bilal2013improving">2013</a>; Ferranti, Krane, and Craft <a href="#ref-ferranti2017value">2017</a>)</span>. Increasingly, the biological variables are not interpreted separately but in relation to each other <span class="citation">(Barabasi and Oltvai <a href="#ref-barabasi2004network">2004</a>)</span>. This is reflected in the emergence of more and more resources to summarize and represent signaling pathways and associated networks such as SIGNOR <span class="citation">(Perfetto et al. <a href="#ref-perfetto2016signor">2016</a>)</span> or the Atlas of Cancer Signaling Network <span class="citation">(Kuperstein et al. <a href="#ref-kuperstein2015atlas">2015</a>)</span>. Like other diseases, cancer then goes <strong>from a genetic disease to a network disease</strong> <span class="citation">(Del Sol et al. <a href="#ref-del2010diseases">2010</a>)</span> and one can study how all kinds of genetic alterations affect the wiring of these networks <span class="citation">(Pawson and Warner <a href="#ref-pawson2007oncogenic">2007</a>)</span>, and modify the cellular functions leading to the previously described cancer hallmarks as depicted schematically in Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:circuit">2.7</a>. In short, the richness of the data did not make it less necessary to use prior knowledge in order to make the analyses more interpretable and more robust.</p>
<div class="figure" style="text-align: center"><span id="fig:pathways"></span>
<img src="fig/pathways.jpg" alt="Genetic alterations frequencies from TCGA data mapped on a schematic signaling network. Frequencies of alteration per pathway and tumor types as summarized in Pan-cancer analyses from TCGA data. Reprinted from Sanchez-Vega et al. (2018)." width="90%" />
<p class="caption">
Figure 2.8: <strong>Genetic alterations frequencies from TCGA data mapped on a schematic signaling network.</strong> Frequencies of alteration per pathway and tumor types as summarized in Pan-cancer analyses from TCGA data. Reprinted from <span class="citation">Sanchez-Vega et al. (<a href="#ref-sanchez2018oncogenic">2018</a>)</span>.
</p>
</div>

<p>The final step, to obtain one of the most complete and integrated visions of cancer biology, is then to integrate omics knowledge with knowledge about the structure of pathways to try to understand in detail how their combinations can lead to so many cancers that are both similar and different. An example of such a representation is given by mapping the TCGA data about genetic alterations, presented in Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:tcga">2.6</a>, on a representation of the different pathways showing not only their internal organization but also their cross-talk <span class="citation">(Sanchez-Vega et al. <a href="#ref-sanchez2018oncogenic">2018</a>)</span>. This representation is proposed in Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:pathways">2.8</a> and is one of the most recent and comprehensive view of the kind of tools and data available to the modeler who wants to dissect more deeply the mechanisms involved in cancer.</p>

<div class="conclubox">
<h4 id="summary-2" class="unnumbered">Summary</h4>
<p>Cancer is more than ever seen as a genetic disease. Its appearance in a patient results from the accumulation of various genetic alterations that invalidate the protective mechanisms naturally intended to prevent uncontrolled proliferation. The simultaneous consideration of the numerous biological entities involved and the regulatory networks that link them calls for global systems biology methods. Technological developments also provide access to different types of omics data (genes, RNA, proteins, etc.) that provide complementary information, the joint analysis of which allows us to better understand the complexity of the mechanisms involved. It should be noted that many physical models of cancer (cell lines) extend the field of experimentation and generate data.</p>
</div>
<p></p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-alberts2007molecular">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, Martin Raff, Keith Roberts, and Peter Walter. 2007. “Molecular Biology of the Cell. Garland Science.” <em>New York</em> 1392.</p>
</div>
<div id="ref-barabasi2004network">
<p>Barabasi, Albert-Laszlo, and Zoltan N Oltvai. 2004. “Network Biology: Understanding the Cell’s Functional Organization.” <em>Nature Reviews Genetics</em> 5 (2). Nature Publishing Group: 101–13.</p>
</div>
<div id="ref-behan2019prioritization">
<p>Behan, Fiona M, Francesco Iorio, Gabriele Picco, Emanuel Gonçalves, Charlotte M Beaver, Giorgia Migliardi, Rita Santos, et al. 2019. “Prioritization of Cancer Therapeutic Targets Using Crispr–Cas9 Screens.” <em>Nature</em> 568 (7753). Nature Publishing Group: 511.</p>
</div>
<div id="ref-bianucci2018earliest">
<p>Bianucci, Raffaella, Antonio Perciaccante, Philippe Charlier, Otto Appenzeller, and Donatella Lippi. 2018. “Earliest Evidence of Malignant Breast Cancer in Renaissance Paintings.” <em>The Lancet Oncology</em> 19 (2). Elsevier: 166–67.</p>
</div>
<div id="ref-bilal2013improving">
<p>Bilal, Erhan, Janusz Dutkowski, Justin Guinney, In Sock Jang, Benjamin A Logsdon, Gaurav Pandey, Benjamin A Sauerwine, et al. 2013. “Improving Breast Cancer Survival Analysis Through Competition-Based Multidimensional Modeling.” <em>PLoS Computational Biology</em> 9 (5). Public Library of Science.</p>
</div>
<div id="ref-braithwaite1983rembrandt">
<p>Braithwaite, Peter Allen, and Dace Shugg. 1983. “Rembrandt’s Bathsheba: The Dark Shadow of the Left Breast.” <em>Annals of the Royal College of Surgeons of England</em> 65 (5). Royal College of Surgeons of England: 337.</p>
</div>
<div id="ref-bray2018global">
<p>Bray, Freddie, Jacques Ferlay, Isabelle Soerjomataram, Rebecca L Siegel, Lindsey A Torre, and Ahmedin Jemal. 2018. “Global Cancer Statistics 2018: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries.” <em>CA: A Cancer Journal for Clinicians</em> 68 (6). Wiley Online Library: 394–424.</p>
</div>
<div id="ref-cohrs2015translational">
<p>Cohrs, Randall J, Tyler Martin, Parviz Ghahramani, Luc Bidaut, Paul J Higgins, and Aamir Shahzad. 2015. “Translational Medicine Definition by the European Society for Translational Medicine.” Elsevier.</p>
</div>
<div id="ref-crick1970central">
<p>Crick, Francis. 1970. “Central Dogma of Molecular Biology.” <em>Nature</em> 227 (5258). Nature Publishing Group: 561–63.</p>
</div>
<div id="ref-davies2002mutations">
<p>Davies, Helen, Graham R Bignell, Charles Cox, Philip Stephens, Sarah Edkins, Sheila Clegg, Jon Teague, et al. 2002. “Mutations of the Braf Gene in Human Cancer.” <em>Nature</em> 417 (6892). Nature Publishing Group: 949–54.</p>
</div>
<div id="ref-del2010diseases">
<p>Del Sol, Antonio, Rudi Balling, Lee Hood, and David Galas. 2010. “Diseases as Network Perturbations.” <em>Current Opinion in Biotechnology</em> 21 (4). Elsevier: 566–71.</p>
</div>
<div id="ref-ding2018perspective">
<p>Ding, Li, Matthew H Bailey, Eduard Porta-Pardo, Vesteinn Thorsson, Antonio Colaprico, Denis Bertrand, David L Gibbs, et al. 2018. “Perspective on Oncogenic Processes at the End of the Beginning of Cancer Genomics.” <em>Cell</em> 173 (2). Elsevier: 305–20.</p>
</div>
<div id="ref-domany2014using">
<p>Domany, Eytan. 2014. “Using High-Throughput Transcriptomic Data for Prognosis: A Critical Overview and Perspectives.” <em>Cancer Research</em> 74 (17). AACR: 4612–21.</p>
</div>
<div id="ref-drier2013pathway">
<p>Drier, Yotam, Michal Sheffer, and Eytan Domany. 2013. “Pathway-Based Personalized Analysis of Cancer.” <em>Proceedings of the National Academy of Sciences</em> 110 (16). National Acad Sciences: 6388–93.</p>
</div>
<div id="ref-dulbecco1986turning">
<p>Dulbecco, Renato. 1986. “A Turning Point in Cancer Research: Sequencing the Human Genome.” <em>Science</em> 231. American Association for the Advancement of Science: 1055–7.</p>
</div>
<div id="ref-ferranti2017value">
<p>Ferranti, Dana, David Krane, and David Craft. 2017. “The Value of Prior Knowledge in Machine Learning of Complex Network Systems.” <em>Bioinformatics</em> 33 (22). Oxford University Press: 3610–8.</p>
</div>
<div id="ref-gao2015high">
<p>Gao, Hui, Joshua M Korn, Stéphane Ferretti, John E Monahan, Youzhen Wang, Mallika Singh, Chao Zhang, et al. 2015. “High-Throughput Screening Using Patient-Derived Tumor Xenografts to Predict Clinical Trial Drug Response.” <em>Nature Medicine</em> 21 (11). Nature Publishing Group: 1318.</p>
</div>
<div id="ref-hahn1999creation">
<p>Hahn, William C, Christopher M Counter, Ante S Lundberg, Roderick L Beijersbergen, Mary W Brooks, and Robert A Weinberg. 1999. “Creation of Human Tumour Cells with Defined Genetic Elements.” <em>Nature</em> 400 (6743). Nature Publishing Group: 464–68.</p>
</div>
<div id="ref-hajdu2011note">
<p>Hajdu, Steven I. 2011a. “A Note from History: Landmarks in History of Cancer, Part 1.” <em>Cancer</em> 117 (5). Wiley Online Library: 1097–1102.</p>
</div>
<div id="ref-hajdu2011note2">
<p>Hajdu, Steven I. 2011b. “A Note from History: Landmarks in History of Cancer, Part 2.” <em>Cancer</em> 117 (12). Wiley Online Library: 2811–20.</p>
</div>
<div id="ref-hajdu2012note">
<p>Hajdu, Steven I. 2012a. “A Note from History: Landmarks in History of Cancer, Part 3.” <em>Cancer</em> 118 (4). Wiley Online Library: 1155–68.</p>
</div>
<div id="ref-hajdu2012note2">
<p>Hajdu, Steven I. 2012b. “A Note from History: Landmarks in History of Cancer, Part 4.” <em>Cancer</em> 118 (20). Wiley Online Library: 4914–28.</p>
</div>
<div id="ref-hajdu2013note">
<p>Hajdu, Steven I, and Farbod Darvishian. 2013. “A Note from History: Landmarks in History of Cancer, Part 5.” <em>Cancer</em> 119 (8). Wiley Online Library: 1450–66.</p>
</div>
<div id="ref-hajdu2013note2">
<p>Hajdu, Steven I, and Manjunath Vadmal. 2013. “A Note from History: Landmarks in History of Cancer, Part 6.” <em>Cancer</em> 119 (23). Wiley Online Library: 4058–82.</p>
</div>
<div id="ref-hanahan2000hallmarks">
<p>Hanahan, Douglas, and Robert A Weinberg. 2000. “The Hallmarks of Cancer.” <em>Cell</em> 100 (1). Elsevier: 57–70.</p>
</div>
<div id="ref-hanahan2011hallmarks">
<p>Hanahan, Douglas, and Robert A Weinberg. 2011. “Hallmarks of Cancer: The Next Generation.” <em>Cell</em> 144 (5). Elsevier: 646–74.</p>
</div>
<div id="ref-hasin2017multi">
<p>Hasin, Yehudit, Marcus Seldin, and Aldons Lusis. 2017. “Multi-Omics Approaches to Disease.” <em>Genome Biology</em> 18 (1). BioMed Central: 83.</p>
</div>
<div id="ref-heijblom2014monte">
<p>Heijblom, Michelle, Linda M Meijer, Ton G van Leeuwen, Wiendelt Steenbergen, and Srirang Manohar. 2014. “Monte Carlo Simulations Shed Light on Bathsheba’s Suspect Breast.” <em>Journal of Biophotonics</em> 7 (5). Wiley Online Library: 323–31.</p>
</div>
<div id="ref-hidalgo2014patient">
<p>Hidalgo, Manuel, Frederic Amant, Andrew V Biankin, Eva Budinská, Annette T Byrne, Carlos Caldas, Robert B Clarke, et al. 2014. “Patient-Derived Xenograft Models: An Emerging Platform for Translational Cancer Research.” <em>Cancer Discovery</em> 4 (9). AACR: 998–1013.</p>
</div>
<div id="ref-iorio2016landscape">
<p>Iorio, Francesco, Theo A Knijnenburg, Daniel J Vis, Graham R Bignell, Michael P Menden, Michael Schubert, Nanne Aben, et al. 2016. “A Landscape of Pharmacogenomic Interactions in Cancer.” <em>Cell</em> 166 (3). Elsevier: 740–54.</p>
</div>
<div id="ref-jemal2017annual">
<p>Jemal, Ahmedin, Elizabeth M Ward, Christopher J Johnson, Kathleen A Cronin, Jiemin Ma, A Blythe Ryerson, Angela Mariotto, et al. 2017. “Annual Report to the Nation on the Status of Cancer, 1975–2014, Featuring Survival.” <em>JNCI: Journal of the National Cancer Institute</em> 109 (9). Oxford University Press: djx030.</p>
</div>
<div id="ref-jones2008core">
<p>Jones, Siân, Xiaosong Zhang, D Williams Parsons, Jimmy Cheng-Ho Lin, Rebecca J Leary, Philipp Angenendt, Parminder Mankoo, et al. 2008. “Core Signaling Pathways in Human Pancreatic Cancers Revealed by Global Genomic Analyses.” <em>Science</em> 321 (5897). American Association for the Advancement of Science: 1801–6.</p>
</div>
<div id="ref-kuperstein2015atlas">
<p>Kuperstein, I, E Bonnet, HA Nguyen, D Cohen, E Viara, L Grieco, S Fourquet, et al. 2015. “Atlas of Cancer Signalling Network: A Systems Biology Resource for Integrative Analysis of Cancer Data with Google Maps.” <em>Oncogenesis</em> 4 (7). Nature Publishing Group: e160–e160.</p>
</div>
<div id="ref-lander2011initial">
<p>Lander, Eric S. 2011. “Initial Impact of the Sequencing of the Human Genome.” <em>Nature</em> 470 (7333). Nature Publishing Group: 187–97.</p>
</div>
<div id="ref-lander2001initial">
<p>Lander, Eric S, Lauren M Linton, Bruce Birren, Chad Nusbaum, Michael C Zody, Jennifer Baldwin, Keri Devon, et al. 2001. “Initial Sequencing and Analysis of the Human Genome.” Macmillan Publishers Ltd.</p>
</div>
<div id="ref-liu2018integrated">
<p>Liu, Jianfang, Tara Lichtenberg, Katherine A Hoadley, Laila M Poisson, Alexander J Lazar, Andrew D Cherniack, Albert J Kovatich, et al. 2018. “An Integrated Tcga Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome Analytics.” <em>Cell</em> 173 (2). Elsevier: 400–416.</p>
</div>
<div id="ref-meyerson2010advances">
<p>Meyerson, Matthew, Stacey Gabriel, and Gad Getz. 2010. “Advances in Understanding Cancer Genomes Through Second-Generation Sequencing.” <em>Nature Reviews Genetics</em> 11 (10). Nature Publishing Group: 685–96.</p>
</div>
<div id="ref-navin2015first">
<p>Navin, Nicholas E. 2015. “The First Five Years of Single-Cell Cancer Genomics and Beyond.” <em>Genome Research</em> 25 (10). Cold Spring Harbor Lab: 1499–1507.</p>
</div>
<div id="ref-cancer2008comprehensive">
<p>Network, Cancer Genome Atlas Research, and others. 2008. “Comprehensive Genomic Characterization Defines Human Glioblastoma Genes and Core Pathways.” <em>Nature</em> 455 (7216). Nature Publishing Group: 1061.</p>
</div>
<div id="ref-nowell1976clonal">
<p>Nowell, Peter C. 1976. “The Clonal Evolution of Tumor Cell Populations.” <em>Science</em> 194 (4260). American Association for the Advancement of Science: 23–28.</p>
</div>
<div id="ref-pawson2007oncogenic">
<p>Pawson, T, and N Warner. 2007. “Oncogenic Re-Wiring of Cellular Signaling Pathways.” <em>Oncogene</em> 26 (9). Nature Publishing Group: 1268–75.</p>
</div>
<div id="ref-perfetto2016signor">
<p>Perfetto, Livia, Leonardo Briganti, Alberto Calderone, Andrea Cerquone Perpetuini, Marta Iannuccelli, Francesca Langone, Luana Licata, et al. 2016. “SIGNOR: A Database of Causal Relationships Between Biological Entities.” <em>Nucleic Acids Research</em> 44 (D1). Oxford University Press: D548–D554.</p>
</div>
<div id="ref-perou1999distinctive">
<p>Perou, Charles M, Stefanie S Jeffrey, Matt Van De Rijn, Christian A Rees, Michael B Eisen, Douglas T Ross, Alexander Pergamenschikov, et al. 1999. “Distinctive Gene Expression Patterns in Human Mammary Epithelial Cells and Breast Cancers.” <em>Proceedings of the National Academy of Sciences</em> 96 (16). National Acad Sciences: 9212–7.</p>
</div>
<div id="ref-perou2000molecular">
<p>Perou, Charles M, Therese Sørlie, Michael B Eisen, Matt Van De Rijn, Stefanie S Jeffrey, Christian A Rees, Jonathan R Pollack, et al. 2000. “Molecular Portraits of Human Breast Tumours.” <em>Nature</em> 406 (6797). Nature Publishing Group: 747–52.</p>
</div>
<div id="ref-reddy1982point">
<p>Reddy, E Premkumar, Roberta K Reynolds, Eugenio Santos, and Mariano Barbacid. 1982. “A Point Mutation Is Responsible for the Acquisition of Transforming Properties by the T24 Human Bladder Carcinoma Oncogene.” <em>Nature</em> 300 (5888). Nature Publishing Group: 149–52.</p>
</div>
<div id="ref-reuter2015high">
<p>Reuter, Jason A, Damek V Spacek, and Michael P Snyder. 2015. “High-Throughput Sequencing Technologies.” <em>Molecular Cell</em> 58 (4). Elsevier: 586–97.</p>
</div>
<div id="ref-samuels2004high">
<p>Samuels, Yardena, Zhenghe Wang, Alberto Bardelli, Natalie Silliman, Janine Ptak, Steve Szabo, Hai Yan, et al. 2004. “High Frequency of Mutations of the Pik3ca Gene in Human Cancers.” <em>Science</em> 304 (5670). American Association for the Advancement of Science: 554–54.</p>
</div>
<div id="ref-sanchez2018oncogenic">
<p>Sanchez-Vega, Francisco, Marco Mina, Joshua Armenia, Walid K Chatila, Augustin Luna, Konnor C La, Sofia Dimitriadoy, et al. 2018. “Oncogenic Signaling Pathways in the Cancer Genome Atlas.” <em>Cell</em> 173 (2). Elsevier: 321–37.</p>
</div>
<div id="ref-stadler2014cancer">
<p>Stadler, Zsofia K, Kasmintan A Schrader, Joseph Vijai, Mark E Robson, and Kenneth Offit. 2014. “Cancer Genomics and Inherited Risk.” <em>Journal of Clinical Oncology</em> 32 (7). American Society of Clinical Oncology: 687.</p>
</div>
<div id="ref-stratton2009cancer">
<p>Stratton, Michael R, Peter J Campbell, and P Andrew Futreal. 2009. “The Cancer Genome.” <em>Nature</em> 458 (7239). Nature Publishing Group: 719–24.</p>
</div>
<div id="ref-cancer2012comprehensive">
<p>TCGA, and others. 2012. “Comprehensive Molecular Portraits of Human Breast Tumours.” <em>Nature</em> 490 (7418). Nature Publishing Group: 61.</p>
</div>
<div id="ref-tomasetti2015variation">
<p>Tomasetti, Cristian, and Bert Vogelstein. 2015. “Variation in Cancer Risk Among Tissues Can Be Explained by the Number of Stem Cell Divisions.” <em>Science</em> 347 (6217). American Association for the Advancement of Science: 78–81.</p>
</div>
<div id="ref-tomasetti2015only">
<p>Tomasetti, Cristian, Luigi Marchionni, Martin A Nowak, Giovanni Parmigiani, and Bert Vogelstein. 2015. “Only Three Driver Gene Mutations Are Required for the Development of Lung and Colorectal Cancers.” <em>Proceedings of the National Academy of Sciences</em> 112 (1). National Acad Sciences: 118–23.</p>
</div>
<div id="ref-van2002gene">
<p>Van’t Veer, Laura J, Hongyue Dai, Marc J Van De Vijver, Yudong D He, Augustinus AM Hart, Mao Mao, Hans L Peterse, et al. 2002. “Gene Expression Profiling Predicts Clinical Outcome of Breast Cancer.” <em>Nature</em> 415 (6871). Nature Publishing Group: 530–36.</p>
</div>
<div id="ref-venet2011most">
<p>Venet, David, Jacques E Dumont, Vincent Detours, and others. 2011. “Most Random Gene Expression Signatures Are Significantly Associated with Breast Cancer Outcome.” <em>PLoS Comput Biol</em> 7 (10): e1002240.</p>
</div>
<div id="ref-venter2001sequence">
<p>Venter, J Craig, Mark D Adams, Eugene W Myers, Peter W Li, Richard J Mural, Granger G Sutton, Hamilton O Smith, et al. 2001. “The Sequence of the Human Genome.” <em>Science</em> 291 (5507). American Association for the Advancement of Science: 1304–51.</p>
</div>
<div id="ref-vucic2012translating">
<p>Vucic, Emily A, Kelsie L Thu, Keith Robison, Leszek A Rybaczyk, Raj Chari, Carlos E Alvarez, and Wan L Lam. 2012. “Translating Cancer ‘Omics’ to Improved Outcomes.” <em>Genome Research</em> 22 (2). Cold Spring Harbor Lab: 188–95.</p>
</div>
<div id="ref-watson1953molecular">
<p>Watson, James D, and Francis HC Crick. 1953. “Molecular Structure of Nucleic Acids.” <em>Nature</em> 171 (4356): 737–38.</p>
</div>
<div id="ref-weinberg2013biology">
<p>Weinberg, Robert A. 2013. <em>The Biology of Cancer</em>. Garland science.</p>
</div>
<div id="ref-yang2012genomics">
<p>Yang, Wanjuan, Jorge Soares, Patricia Greninger, Elena J Edelman, Howard Lightfoot, Simon Forbes, Nidhi Bindal, et al. 2012. “Genomics of Drug Sensitivity in Cancer (Gdsc): A Resource for Therapeutic Biomarker Discovery in Cancer Cells.” <em>Nucleic Acids Research</em> 41 (D1). Oxford University Press: D955–D961.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="scientific-modeling-abstract-the-complexity.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="mechanistic-cancer.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
