<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 9 Causal inference for precision medicine | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 9 Causal inference for precision medicine | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 9 Causal inference for precision medicine | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="clinical-evidence-generation-and-causal-inference.html"/>
<link rel="next" href="appendix-datasets.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#introducing-the-diversity-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive-models"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#tools-for-logical-modeling"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.1</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.2</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.3</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#comparison-of-the-mechanistic-approach-with-machine-learning-methods"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#limitations-and-perspectives"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.1</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.1.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information-in-information-out"><i class="fa fa-check"></i><b>7.1.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.1.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artifical-examples"><i class="fa fa-check"></i><b>7.1.2</b> Emergence of information in artifical examples</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>7.2</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.2.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.2.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-inference-methods-to-leverage-data"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#identification-of-causal-effects"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#an-illustration-with-patient-derived-xenografts"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a></li>
<li class="chapter" data-level="9.3" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#causal-inference-under-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>9.3</b> Causal inference under multiple versions of treatment</a></li>
<li class="chapter" data-level="9.4" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a></li>
<li class="chapter" data-level="9.5" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="causal-inference-for-precision-medicine.html"><a href="causal-inference-for-precision-medicine.html#limitations-and-perspective"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspective</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="causal-inference-for-precision-medicine" class="section level1">
<h1><span class="header-section-number">Chapter 9</span> Causal inference for precision medicine</h1>
<blockquote>
<p><em>Felix qui potuit rerum cognoscere causas.</em><br/> Virgil (Georgics, 29 BC)</p>
</blockquote>
<p>Throughout this manuscript, we first described the complexity of cancer mechanisms, through the diversity of genetic alterations or non-linear signaling pathways. This complexity naturally led to the choice of systemic modelling approaches and in particular mechanistic models whose explicit nature facilitates the study of the effects of new molecular perturbations such as treatments. The simple study of the response to BRAF inhibitors has thus required the consideration of many other genes and pathways.</p>
<p>This chapter proposes to take the complexity a step further by considering different treatments. The diversity of patients’ molecular profiles suggests that the best treatment is not necessarily the same for all patients: this is what is known as precision medicine. This is already a clinical reality in oncology that could be reinforced in the future by the emergence of new computational models of cancer, whether mechanistic or not. How then can we assess the relevance of these models in their ability to guide patient treatment?</p>

<div class="summarybox">
<h4 id="scientific-content-5" class="unnumbered">Scientific content</h4>
<p>This chapter presents an extension of the causal inference framework to quantify the value of precision medicine strategies. This work is currently under revision and is available as a pre-print <span class="citation">Béal and Latouche (<a href="#ref-beal2020causal">2020</a>)</span>.</p>
</div>
<p></p>
<div id="precision-medicine-in-oncology" class="section level2">
<h2><span class="header-section-number">9.1</span> Precision medicine in oncology</h2>
<p>It is first important to understand what is meant by the concept of precision medicine in the treatment of cancer patients in order to place subsequent questions in a plausible clinical framework.</p>
<div id="an-illustration-with-patient-derived-xenografts" class="section level3">
<h3><span class="header-section-number">9.1.1</span> An illustration with patient-derived xenografts</h3>
<p>Precision medicine is based on the diversity of treatment responses observed in different tumors. It has already been observed in previous chapters that different cell lines respond differently to a particular treatment, the BRAF inhbitors. A broader analysis of the data shows that the same is true for the vast majority of treatments. It would be possible to illustrate this using the same data from cell lines extended to other drugs. However, because of the more directly clinical impact of the issues discussed in this chapter, the analyses presented below will focus on another type of data that is closer to patient data: patient-derived xenografts (PDX).</p>
<p>A PDX is a tumor tissue that has been removed from a patient and implanted into immunodeficient mice <span class="citation">(Hidalgo et al. <a href="#ref-hidalgo2014patient">2014</a>)</span>. Unlike cell lines, which are <em>in vitro</em> models, PDXs are <em>in vivo</em> models that allow cancer cells to evolve in a more realistic microenvironment. In the same way as for cell lines, PDX can be used for drug screening in patients who cannot be screened directly. The data used in this chapter come from a study by <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span> which contains several hundred tumors and more than fifty drugs. Not all drugs having been tested for all tumours and details of the drugs and types of cancer tested are available in the appendix <a href="appendix-datasets.html#appendix-PDX">A.2</a>. This dataset was generated following the “one animal per model per treatment” approach (<span class="math inline">\(1 \times 1 \times 1\)</span>), the principles of which are summarized in Figure <a href="causal-inference-for-precision-medicine.html#fig:PDX-principles">9.1</a>A. It should be noted that different drug response metrics are computed in the source data, two of which will be used in the analyses. The first one is continuous and called <em>Best Average Response</em> in the data, it is based on the variation of the tumor volume after treatment, the lower values (and especially negative) corresponding to better responses. The second one is originally categorical and based on a modified Response Evaluation Criteria In Solid Tumors (RECIST) criteria. It was binarized for this study so that the responders have a score of <span class="math inline">\(1\)</span> and non-responders <span class="math inline">\(0\)</span>. The details of the definition and distribution of these metrics are given in appendix <a href="appendix-datasets.html#appendix-PDX">A.2</a></p>
<div class="figure" style="text-align: center"><span id="fig:PDX-principles"></span>
<img src="fig/PDX.png" alt="Principles of PDX screening. (A) Schematic pipeline for PDX screening with tumour biopsies from one patient divided in several pieces later implanted in similar immunodeficient mice. Each mouse is then treated with a different drug; the collection of mice that have received tumour samples from the same patient but have been treated with different drugs therefore gives access to several outcomes for the same tumor of origin. (B) Corresponding counterfactual variables." width="90%" />
<p class="caption">
Figure 9.1: <strong>Principles of PDX screening.</strong> (A) Schematic pipeline for PDX screening with tumour biopsies from one patient divided in several pieces later implanted in similar immunodeficient mice. Each mouse is then treated with a different drug; the collection of mice that have received tumour samples from the same patient but have been treated with different drugs therefore gives access to several outcomes for the same tumor of origin. (B) Corresponding counterfactual variables.
</p>
</div>

<p>In order to illustrate the diversity of response to treatment, the database is momentarily restricted to the 4 most widely tested drugs and the 180 tumours (or PDX models) that were evaluated for all four drugs. The four chosen drugs target different pathways: binimetinib (MAPK inhibitor), BKM120 (PIK inhibitor), HDM201 (MDM2 inhibitor) and LEE011 (CDK inhibitor). In Figure <a href="causal-inference-for-precision-medicine.html#fig:PDX-dense">9.2</a>A the 4 treatments show a high variability of response, with a slight advantage for BKM120 and binimetinib on average over all tumours. However, each of the treatments was found to be the most effective of the 4 for a significant proportion of patients (Figure <a href="causal-inference-for-precision-medicine.html#fig:PDX-dense">9.2</a>B), with binimetinib and BKM being the best treatment for one-third of tumours each and LEE011/HDM201 sharing the remaining one-third of tumours. It thus appears that in view of the molecular diversity of tumours and the increasing number of treatments available, it does not seem advisable, according to these preclinical data, to treat all tumours with the same gold standard treatment. Furthermore, the tissue of origin of the tumors in this example does not appear to be the main determinant of tumour preference for certain treatments.</p>
<div class="figure" style="text-align: center"><span id="fig:PDX-dense"></span>
<img src="09-Precision_files/figure-html/PDX-dense-1.png" alt="Differences in drug response for 4 drugs and 180 tumors a call for precision medicine. (A) Distribution of treatment response for the 4 different drugs, each with all 180 tumours. (B) Number of times each of the 4 drugs is the most effective for a given tumour, distribution by tissue of origin." width="90%" />
<p class="caption">
Figure 9.2: <strong>Differences in drug response for 4 drugs and 180 tumors a call for precision medicine.</strong> (A) Distribution of treatment response for the 4 different drugs, each with all 180 tumours. (B) Number of times each of the 4 drugs is the most effective for a given tumour, distribution by tissue of origin.
</p>
</div>

</div>
<div id="clinical-trials-and-treatment-algorithms" class="section level3">
<h3><span class="header-section-number">9.1.2</span> Clinical trials and treatment algorithms</h3>
<p>These remarks can be extended to patients. Thus, precision medicine (PM) consists in assigning the most appropriate treatment to each patient according to his or her characteristics, usually genomic alterations for cancer patients <span class="citation">(Friedman et al. <a href="#ref-friedman2015precision">2015</a>; De Gramont et al. <a href="#ref-de2015pragmatic">2015</a>)</span>. At the individual level, targeted treatments have provided relevant solutions for patients with specific mutations <span class="citation">(Abou-Jawde et al. <a href="#ref-abou2003overview">2003</a>)</span>. Putting together these various treatments, some precision medicine strategies can be defined. Based on the genomic profile of the patient, the treatment most likely to be successful is chosen. If the information available is reliable, precision medicine can thus be reduced to a treatment algorithm that takes as input the molecular characteristics of the patient’s tumor and outputs a recommendation of treatment. An example of such a treatment algorithm from the SHIVA clinical trial by <span class="citation">Le Tourneau et al. (<a href="#ref-le2015molecularly">2015</a>)</span> is shown in Figure <a href="causal-inference-for-precision-medicine.html#fig:SHIVA">9.3</a> where different treatments are associated with different alterations. In this case, the treatment algorithm can be considered as an aggregation of the medical knowledge accumulated on the individual biomarkers.</p>
<div class="figure" style="text-align: center"><span id="fig:SHIVA"></span>
<img src="fig/SHIVA_algo.png" alt="An example of a precision medicine treatment algorithm: the SHIVA clinical trial. Specific molecular alterations and their associated treatments, as proposed in the SHIVA clinical trial (Le Tourneau et al. 2015)." width="80%" />
<p class="caption">
Figure 9.3: <strong>An example of a precision medicine treatment algorithm: the SHIVA clinical trial.</strong> Specific molecular alterations and their associated treatments, as proposed in the SHIVA clinical trial <span class="citation">(Le Tourneau et al. <a href="#ref-le2015molecularly">2015</a>)</span>.
</p>
</div>

</div>
<div id="computational-models-to-assign-cancer-treatments" class="section level3">
<h3><span class="header-section-number">9.1.3</span> Computational models to assign cancer treatments</h3>
<p>But the treatment algorithm example in Figure A could be more complex. Indeed, previous chapters have stressed, for example, that being mutated for the BRAF gene is not the only predictor of response to an inhibitor of BRAF (here Vemurafenib). The same is true for most treatments that could benefit from more global and systemic analyses, taking into account more variables and their interactions. This complexity would require the use of computational methods.</p>
<p>It is on this point that this chapter links to the previous ones. Some of the cancer models studied throughout this thesis, or their future developments, could be interpreted as treatment algorithms. Indeed, a model capable of predicting the response to a single treatment does not necessarily allow the inference of precision medicine strategies. On the other hand, a model capable of predicting a patient’s response to different treatments is also capable of indicating which one is the best. Such models would then move from systems biology to systems therapeutics <span class="citation">(Hansen and Iyengar <a href="#ref-hansen2013computation">2013</a>)</span>, taking patients’ geneomic features as inputs and outputting a treatment recommendation. In theory, mechanistic models seem to be suitable for this purpose since their explicit representation of genes and proteins makes it possible to simulate the effect of different therapeutic interventions. However, the feasibility of designing and calibrating such a model has yet to be demonstrated. Other types of models are being studied to provide a computational tool for predicting the sensitivity of cell lines to different treatments, in particular using deep learning methods <span class="citation">(Manica et al. <a href="#ref-manica2019toward">2019</a>)</span>.</p>
<p>In short, if no computational model is sufficiently developed to date to replace the clinician, the emergence of this type of tool is likely in the medium term. This raises the question of how to assess the clinical value of the precision medicine strategies (and corresponding treatment algorithms) derived from these models. For the sake of generality, this question will be addressed more broadly in the following without reference to models as a possible source of the treatment algorithm: how to evaluate the clinical impact of a precision medicine strategy and the treatment algorithm? The methods presented will indeed be the same, whether the algorithm evaluated comes from a model or from the knowledge of clinicians as in Figure <a href="causal-inference-for-precision-medicine.html#fig:SHIVA">9.3</a>. In the spirit of this thesis, the question nevertheless finds its origin in the first hypothesis related to models.</p>
</div>
</div>
<div id="emulating-clinical-trials-to-evaluate-precision-medicine-algorithms" class="section level2">
<h2><span class="header-section-number">9.2</span> Emulating clinical trials to evaluate precision medicine algorithms</h2>
<p>The question then arises of how to quantify the clinical benefit provided by these treatment algorithms. Some clinical trials have been proposed, demonstrating both the feasibility of collecting information about mutations <span class="citation">(Le Tourneau et al. <a href="#ref-le2015molecularly">2015</a>)</span> or RNA <span class="citation">(Rodon et al. <a href="#ref-rodon2019genomic">2019</a>)</span> in real-time and the clinical benefit that can be expected from these approaches for some patients <span class="citation">(Coyne, Takebe, and Chen <a href="#ref-coyne2017defining">2017</a>)</span>. However, the increasing abundance of genomic data and biological knowledge make it progressively easier to establish new algorithms for precision medicine, either directly based on physician knowledge or provided by computational models <span class="citation">(Hansen and Iyengar <a href="#ref-hansen2013computation">2013</a>)</span>. For practical reasons it is not possible to propose a real clinical trial for each new precision medicine algorithm or for any variants, comparing standard of care with new algorithm-based therapeutic strategies.</p>
</div>
<div id="causal-inference-under-multiple-versions-of-treatment" class="section level2">
<h2><span class="header-section-number">9.3</span> Causal inference under multiple versions of treatment</h2>
</div>
<div id="application-to-simulated-data" class="section level2">
<h2><span class="header-section-number">9.4</span> Application to simulated data</h2>
</div>
<div id="application-to-pdx" class="section level2">
<h2><span class="header-section-number">9.5</span> Application to PDX</h2>
</div>
<div id="limitations-and-perspective" class="section level2">
<h2><span class="header-section-number">9.6</span> Limitations and perspective</h2>
<div class="figure" style="text-align: center"><span id="fig:Shiny"></span>
<iframe src="http://jonasbeal.shinyapps.io/application_causal_pm/?showcase=0" width="90%" height="600px">
</iframe>
<p class="caption">
Figure 9.4: <strong>Schematic extension of PROFILE-personalized logical models to drug investigation.</strong>
</p>
</div>


</div>
</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-abou2003overview">
<p>Abou-Jawde, Rony, Toni Choueiri, Carlos Alemany, and Tarek Mekhail. 2003. “An Overview of Targeted Treatments in Cancer.” <em>Clinical Therapeutics</em> 25 (8). Elsevier: 2121–37.</p>
</div>
<div id="ref-beal2020causal">
<p>Béal, Jonas, and Aurélien Latouche. 2020. “Causal Inference with Multiple Versions of Treatment and Application to Personalized Medicine.” <em>ArXiv Preprint ArXiv:2005.12427</em>.</p>
</div>
<div id="ref-coyne2017defining">
<p>Coyne, Geraldine O’Sullivan, Naoko Takebe, and Alice P Chen. 2017. “Defining Precision: The Precision Medicine Initiative Trials Nci-Mpact and Nci-Match.” <em>Current Problems in Cancer</em> 41 (3). Elsevier: 182–93.</p>
</div>
<div id="ref-de2015pragmatic">
<p>De Gramont, Armand, Sarah Watson, Lee M Ellis, Jordi Rodón, Josep Tabernero, Aimery De Gramont, and Stanley R Hamilton. 2015. “Pragmatic Issues in Biomarker Evaluation for Targeted Therapies in Cancer.” <em>Nature Reviews Clinical Oncology</em> 12 (4). Nature Publishing Group: 197.</p>
</div>
<div id="ref-friedman2015precision">
<p>Friedman, Adam A, Anthony Letai, David E Fisher, and Keith T Flaherty. 2015. “Precision Medicine for Cancer with Next-Generation Functional Diagnostics.” <em>Nature Reviews Cancer</em> 15 (12). Nature Publishing Group: 747–56.</p>
</div>
<div id="ref-gao2015high">
<p>Gao, Hui, Joshua M Korn, Stéphane Ferretti, John E Monahan, Youzhen Wang, Mallika Singh, Chao Zhang, et al. 2015. “High-Throughput Screening Using Patient-Derived Tumor Xenografts to Predict Clinical Trial Drug Response.” <em>Nature Medicine</em> 21 (11). Nature Publishing Group: 1318.</p>
</div>
<div id="ref-hansen2013computation">
<p>Hansen, J, and R Iyengar. 2013. “Computation as the Mechanistic Bridge Between Precision Medicine and Systems Therapeutics.” <em>Clinical Pharmacology &amp; Therapeutics</em> 93 (1). Wiley Online Library: 117–28.</p>
</div>
<div id="ref-hidalgo2014patient">
<p>Hidalgo, Manuel, Frederic Amant, Andrew V Biankin, Eva Budinská, Annette T Byrne, Carlos Caldas, Robert B Clarke, et al. 2014. “Patient-Derived Xenograft Models: An Emerging Platform for Translational Cancer Research.” <em>Cancer Discovery</em> 4 (9). AACR: 998–1013.</p>
</div>
<div id="ref-le2015molecularly">
<p>Le Tourneau, Christophe, Jean-Pierre Delord, Anthony Gonçalves, Céline Gavoille, Coraline Dubot, Nicolas Isambert, Mario Campone, et al. 2015. “Molecularly Targeted Therapy Based on Tumour Molecular Profiling Versus Conventional Therapy for Advanced Cancer (Shiva): A Multicentre, Open-Label, Proof-of-Concept, Randomised, Controlled Phase 2 Trial.” <em>The Lancet Oncology</em> 16 (13). Elsevier: 1324–34.</p>
</div>
<div id="ref-manica2019toward">
<p>Manica, Matteo, Ali Oskooei, Jannis Born, Vigneshwari Subramanian, Julio Sáez-Rodríguez, and María Rodríguez Martínez. 2019. “Toward Explainable Anticancer Compound Sensitivity Prediction via Multimodal Attention-Based Convolutional Encoders.” <em>Molecular Pharmaceutics</em>. ACS Publications.</p>
</div>
<div id="ref-rodon2019genomic">
<p>Rodon, Jordi, Jean-Charles Soria, Raanan Berger, Wilson H Miller, Eitan Rubin, Aleksandra Kugel, Apostolia Tsimberidou, et al. 2019. “Genomic and Transcriptomic Profiling Expands Precision Cancer Medicine: The Winther Trial.” <em>Nature Medicine</em> 25 (5). Nature Publishing Group: 751.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="clinical-evidence-generation-and-causal-inference.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="appendix-datasets.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
