<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 8 Clinical evidence generation and causal inference | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 8 Clinical evidence generation and causal inference | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 8 Clinical evidence generation and causal inference | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="information-flows-in-mechanistic-models-of-cancer.html"/>
<link rel="next" href="chapter-precision.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#introducing-the-diversity-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#tools-for-logical-modeling"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.1</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.2</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.3</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#comparison-of-the-mechanistic-approach-with-machine-learning-methods"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#limitations-and-perspectives"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information-in-information-out"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artifical-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artifical examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-inference-methods-to-leverage-data"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#limitations-and-perspective"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspective</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="clinical-evidence-generation-and-causal-inference" class="section level1">
<h1><span class="header-section-number">Chapter 8</span> Clinical evidence generation and causal inference</h1>
<blockquote>
<p><em>Maudit<br />
soit le père de l’épouse<br />
du forgeron qui forgea le fer de la cognée<br />
avec laquelle le bûcheron abattit le chêne<br />
dans lequel on sculpta le lit<br />
où fut engendré l’arrière-grand-père<br />
de l’homme qui conduisit la voiture<br />
Wdans laquelle ta mère<br />
rencontra ton père!.</em><br/> Robert Desnos (La colombe de l’arche, 1923)</p>
</blockquote>
<p>The previous chapter introduced some tools to evaluate and quantify the value of mechanistic models, and in particular their outputs, with simple statistical tools. The latter, such as <span class="math inline">\(R^2\)</span>, are by no means specific to medical applications. One of the particularities of mechanistic cancer models, on the other hand, is the possibility of simulating treatments that imitate therapeutic interventions. Before tackling more precise questions, this chapter will therefore introduce certain clinical or statistical methods used to evaluate the effect of different types of treatments on patients. A more specific issue related to the evaluation of mechanistic models will be explored in the next chapter using these methods.</p>

<div class="summarybox">
<h4 id="scientific-content-5" class="unnumbered">Scientific content</h4>
<p>This short chapter introduces the framework of causal inference based on the literature and the description of causal inference in the preprint <span class="citation">Béal and Latouche (<a href="#ref-beal2020causal">2020</a>)</span>.</p>
</div>
<p></p>
<div id="clinical-trials-and-beyond" class="section level2">
<h2><span class="header-section-number">8.1</span> Clinical trials and beyond</h2>
<div id="randomized-clinical-trials-as-gold-standards" class="section level3">
<h3><span class="header-section-number">8.1.1</span> Randomized clinical trials as gold standards</h3>
<p>When it comes to evaluating the effect of a therapeutic intervention, the reference method in most cases in modern medicine is the randomized clinical trial, which will be described now in its simplest version. Without loss of generality, the rationale for this approach can be detailed for one drug, which will be referred to as <span class="math inline">\(A\)</span> in the remainder of the chapter (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:trials">8.1</a>). The patients who can benefit from this drug, and therefore those eligible for the clinical trial, are first of all defined (specific disease, characteristics, etc.). Then, they are randomly separated into two distinct groups, one receiving the new treatment to be evaluated (<span class="math inline">\(A=1\)</span>) and the other generally receiving the treatment considered as standard of care, or a placebo if no validated treatment is available (<span class="math inline">\(A=0\)</span>). A predefined treatment response criterion <span class="math inline">\(Y\)</span> (viral load, tumor size, etc.) is then compared for the two groups to quantify the average treatment effect (ATE):</p>
<p><span class="math display">\[ATE= E[Y|A=1]-E[Y|A=0]\]</span></p>
<p>Thus it will be possible to say, for example, that “compared to patients who received the standard treatment, those treated with the new drug have a 20% lower tumor volume”. In this example, <strong>randomly choosing how the two groups of patients, treated and untreated, are constituted ensures <em>a priori</em> that the two groups are comparable</strong>. Indeed, it should be verified that the untreated patients were not on average suffering from more advanced cancers that are more likely to proliferate and grow. In this case, the difference in outcome between the groups could simply come from a difference in initial composition and not from a difference derived from therapeutic interventions. Random assignment of treatments therefore offers minimum guarantees concerning the characteristics of the two subgroups.</p>
<div class="figure" style="text-align: center"><span id="fig:trials"></span>
<img src="fig/trials.png" alt="Principles of randomized clinical trials.. This trial evaluates the impact of treatment \(A\)." width="90%" />
<p class="caption">
Figure 8.1: <strong>Principles of randomized clinical trials.</strong>. This trial evaluates the impact of treatment <span class="math inline">\(A\)</span>.
</p>
</div>

</div>
<div id="observational-data-and-confounding-factors" class="section level3">
<h3><span class="header-section-number">8.1.2</span> Observational data and confounding factors</h3>
<p>The problem of comparability between the two groups is reinforced when the data used does not come from a randomized clinical trial. In the remainder of this thesis these data will be called <strong>observational data</strong>. This means that in the available data, some patients were treated with the new drug (<span class="math inline">\(A=1\)</span>) and others received the reference treatment (<span class="math inline">\(A=0\)</span>). However, the assignment of treatment was not decided by the observer. This assignment was therefore made according to a protocol unknown to the observer which has no guarantee that the two groups are in fact comparable.</p>
<p>The situation can be illustrated with a simple simulated example involving a confounding variable <span class="math inline">\(C\)</span> in addition to the treatment variable <span class="math inline">\(A\)</span> and the outcome variable <span class="math inline">\(Y\)</span>. If <span class="math inline">\(Y\)</span> represents tumor volume and <span class="math inline">\(A\)</span> the treatment to be evaluated, <span class="math inline">\(C\)</span> could be a biomarker of cancer agressiveness. 1000 patients have been simulated for all variables in two different settings represented in Figures <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example">8.2</a> and <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example2">8.3</a>. In the first case (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example">8.2</a>), the outcome <span class="math inline">\(Y\)</span> is positively correlated to <span class="math inline">\(C\)</span> (more agressive tumors have bigger volume) and decreased when <span class="math inline">\(A=1\)</span> (treatment decreases tumor volume). <strong><span class="math inline">\(C\)</span> has no influence on <span class="math inline">\(A\)</span></strong>. The causal relationships between the variables and the associated coefficients used to simulate data are summarized in the directed acyclic graphs (DAG) in Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example">8.2</a>A. The observed relations between variables in simulated data are shown in Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example">8.2</a>B, C and D. In particular, the <strong>theoretical influence of <span class="math inline">\(A\)</span> on <span class="math inline">\(Y\)</span> is recovered in the observed data</strong> since <span class="math inline">\(E[Y|A=1]-E[Y|A=0]=-5.05\)</span></p>
<div class="figure" style="text-align: center"><span id="fig:causality-example"></span>
<img src="08-Causal_files/figure-html/causality-example-1.png" alt="Analysis on observed data without confounder. (A) Directed acyclic graphs with causal relations between variables and parameters used to simulate data. (B) Influence of \(C\) on \(A\) in observed simulated data. (C) Same with \(C\) and \(Y\). (D) Same with \(A\) and \(Y\)." width="90%" />
<p class="caption">
Figure 8.2: <strong>Analysis on observed data without confounder.</strong> (A) Directed acyclic graphs with causal relations between variables and parameters used to simulate data. (B) Influence of <span class="math inline">\(C\)</span> on <span class="math inline">\(A\)</span> in observed simulated data. (C) Same with <span class="math inline">\(C\)</span> and <span class="math inline">\(Y\)</span>. (D) Same with <span class="math inline">\(A\)</span> and <span class="math inline">\(Y\)</span>.
</p>
</div>

<p>In the second case (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example2">8.3</a>), <span class="math inline">\(C\)</span> has an influence on <span class="math inline">\(Y\)</span>: the more aggressive the tumor, the more likely the patient is to be treated with the new drug. In this case the <strong>simultaneous influence of <span class="math inline">\(C\)</span> on <span class="math inline">\(A\)</span> and <span class="math inline">\(Y\)</span> makes it a real confounder</strong>. The direct observation of the differences in outcomes between treated and untreated patients reveals only a small benefit of the new treatment which does not correspond to the underlying reality used in these simulations since the theoretical causal influence of <span class="math inline">\(A\)</span> on <span class="math inline">\(Y\)</span> remained the same as in the previous case. The <strong>confounding factor prevents the nature of the causal link between A and Y from being simply inferred</strong>.</p>
<div class="figure" style="text-align: center"><span id="fig:causality-example2"></span>
<img src="08-Causal_files/figure-html/causality-example2-1.png" alt="Analysis on observed data with confounder. (A) Directed acyclic graphs with causal relations between variables and parameters used to simulate data. (B) Influence of \(C\) on \(A\) in observed simulated data. (C) Same with \(C\) and \(Y\). (D) Same with \(A\) and \(Y\)." width="90%" />
<p class="caption">
Figure 8.3: <strong>Analysis on observed data with confounder.</strong> (A) Directed acyclic graphs with causal relations between variables and parameters used to simulate data. (B) Influence of <span class="math inline">\(C\)</span> on <span class="math inline">\(A\)</span> in observed simulated data. (C) Same with <span class="math inline">\(C\)</span> and <span class="math inline">\(Y\)</span>. (D) Same with <span class="math inline">\(A\)</span> and <span class="math inline">\(Y\)</span>.
</p>
</div>

</div>
</div>
<div id="causal-inference-methods-to-leverage-data" class="section level2">
<h2><span class="header-section-number">8.2</span> Causal inference methods to leverage data</h2>
<p>Despite these difficulties, some statistical methods have been developed to derive estimates with a causal interpretation from observational data, under precise assumptions. This work will focus on the potential outcomes framework <span class="citation">(Rubin <a href="#ref-rubin1974estimating">1974</a>)</span>. We will first describe briefly the fundamentals of this framework and different methods that are part of it.</p>
<div id="notations-in-potential-outcomes-framework" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Notations in potential outcomes framework</h3>
<p>First of all, the notations used in this and the next chapter are defined as follows. We will use <span class="math inline">\(j=1,...,N\)</span> to index the individuals in the population. <span class="math inline">\(A_j\)</span> and <span class="math inline">\(Y_j\)</span> correspond respectively to the actual treatment received by individual <span class="math inline">\(j\)</span> and the outcome. In the most simple case, treatment takes values in <span class="math inline">\(\mathcal{A}=\{0, 1\}\)</span>, <span class="math inline">\(1\)</span> denoting the treated patients and <span class="math inline">\(0\)</span> the control ones. <span class="math inline">\(Y_j\)</span> corresponds to the patient’s response to treatment. In the case of cancer it may be a continuous value (<em>e.g.</em>, size of tumor), a binary value (<em>e.g.</em>, status or event indicator), or even a time-to-event (<em>e.g.</em>, time to relapse or death). Only the first two cases will be discussed later. Finally, it is necessary to take into account the possible presence of confounders influencing both <span class="math inline">\(A\)</span> and <span class="math inline">\(Y\)</span> and denoted <span class="math inline">\(C_j\)</span> for individual <span class="math inline">\(j\)</span>.</p>
<p>The <strong>potential outcomes framework is also described as counterfactual</strong> because it defines variables like <span class="math inline">\(Y_j(a)\)</span> to denote the potential outcome of individual <span class="math inline">\(j\)</span> in case he/she has been treated by <span class="math inline">\(A=a\)</span> which may be different from what we observe if <span class="math inline">\(A_j\neq a\)</span>. These counterfactual variables make it possible to write the causal estimands. For instance, in this context, we can easily compute the difference in outcome between treated patients and control patients (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality">8.4</a>, left part): <span class="math inline">\(E[Y | A=1] - E[Y | A=0].\)</span> However, this difference has no causal interpretation as it does not offer any guarantees as to the confounding factor, as an unbalanced distribution of <span class="math inline">\(C\)</span> can induce biases. Thus we define another estimate: <span class="math inline">\(E[Y(1)] - E[Y(0)].\)</span> In this case, we compare between two ideal cohorts (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality">8.4</a>, right part), one in which all patients have been treated (possibly contrary to the fact) and one in which all patients have been left in the control arm (once again, possibly contrary to the fact).</p>
<div class="figure" style="text-align: center"><span id="fig:causality"></span>
<img src="fig/causality.png" alt="Association, causation and their associated cohorts. Association analyses are based on observed cohorts and conditional probabilities. Causation analyses are based on counterfactual variables and cohorts." width="90%" />
<p class="caption">
Figure 8.4: <strong>Association, causation and their associated cohorts.</strong> Association analyses are based on observed cohorts and conditional probabilities. Causation analyses are based on counterfactual variables and cohorts.
</p>
</div>

</div>
<div id="causal-identification-simple" class="section level3">
<h3><span class="header-section-number">8.2.2</span> Identification of causal effects</h3>
<p>The next question is whether it is possible to estimate the counterfactual variables <span class="math inline">\(Y(A)\)</span> and under what conditions. The potential outcomes framework explicits <strong>assumptions of consistency, positivity and conditional exchangeability to estimate these counterfactual variables</strong> and therefore infer causal estimates from observational (non-randomized) data <span class="citation">(Rubin <a href="#ref-rubin1974estimating">1974</a>, <span class="citation">Hernán and Robins (<a href="#ref-hernan2020causal">2020</a>)</span>)</span>.</p>
<p><em>Consistency</em> means that values of treatment under comparison represent well-defined interventions which themselves correspond to the treatments in the data: <span class="math inline">\(\textrm{if} \: A_j=a, \textrm{then} \: Y_j(a)=Y_j.\)</span></p>
<p><em>Exchangeability</em> means that treated and control patients are exchangeable, <em>i.e.</em>, if the treated patients had not been treated they would have had the same outcomes as the controls, and conversely. Since we usually observe some confounders we define conditional exchangeability to hold if cohorts are exchangeable for same values of confounding <span class="math inline">\(C\)</span>. Therefore conditional exchangeability will hold if there are no unmeasured confounding: <span class="math inline">\(Y(a) {\perp \!\!\! \perp}A | C.\)</span></p>
<p><em>Positivity</em> assumption states that the probability of being administered a certain version of treatment conditional on <span class="math inline">\(C\)</span> is greater than zero: <span class="math inline">\(\textrm{if} \: P[C=c] \neq 0, P[A=a | C=c] &gt;0.\)</span> Intuitively, this positivity condition is required to ensure that the defined counterfactual variables make sense and do not represent something that cannot exist.</p>
<p>Under these three assumptions, there are different methods and estimators available to evaluate causal effects from observational data. Two of them will be described and applied to the same example as above: the description of the example and the failure of the direct methods are recalled in Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>A and B and two causal inference methods are illustrated in Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>C, D and E.</p>
<div id="std-classic" class="section level4">
<h4><span class="header-section-number">8.2.2.1</span> Standardization or parametric g-formula</h4>
<p>The first method is called standardization or parametric g-formula and it is the one that will be described in more detail in this chapter and the following one. It is based on the following equations:</p>
<span class="math display">\[\begin{equation*}
\begin{aligned}
  E[Y(a)] &amp; = \sum_{c} E[Y(a)|c] \times P[c] \\
          &amp; = \sum_{c} E[Y(a)|a,c] \times P[c,w]
          &amp;&amp;\text{ (exchangeability } Y(a) \perp \!\!\! \perp A | C \text{ )} \\
          &amp; = \sum_{c} E[Y|a,c] \times P[c]
          &amp;&amp;\text{ (consistency)}
\end{aligned}
\end{equation*}\]</span>
<p>Thus the average effect of treatment on the entire cohort can be written with standardized means:</p>
<span class="math display">\[\begin{equation}
\begin{aligned}
  E[Y(A=1)] - &amp; E[Y(A=0)] = \\ 
   \sum_{c} \Big( &amp; E[Y | A=1, C=c]-E[Y | A=0, C=c]\Big) \times P[C=c]
\end{aligned}
\end{equation}\]</span>
<p>Computationally, non-parametric estimation of <span class="math inline">\(E[Y | A=a, C=c]\)</span> is usually out of reach. Thus, on real-world dataset, <span class="math inline">\(E[Y | A=a, C=c]\)</span> is estimated through <strong>outcome modeling</strong> and explicit computation <span class="math inline">\(P[C=c]\)</span> is replaced by its empirical estimate. The nature of the statistical model used will be specified in the various applications presented. In the simple example depicted in Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>A, a linear model of the outcome (<span class="math inline">\(Y\sim C+A\)</span>) is fitted on observed data. This model is then used to infer <span class="math inline">\(E[Y | A=1, C=c]\)</span> and <span class="math inline">\(E[Y | A=0, C=c]\)</span> for each patient with covariate <span class="math inline">\(C=c\)</span> (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>C). By averaging these values over the whole cohort the confounding effect is corrected and the estimator is much closer to the true value <span class="math inline">\(-5\)</span> than the naive estimates (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>D and B).</p>
<div class="figure" style="text-align: center"><span id="fig:causality-example3"></span>
<img src="08-Causal_files/figure-html/causality-example3-1.png" alt="Causal inference methods on a simple example. (A) Directed acyclic graphs with causal relations between variables and parameters used to simulate data. (B) Association between \(A\) and \(Y\) from observed data. (C) Some simulated samples/patients with their original variables (\(C\), \(A\) and \(Y\)), variables from outcome model (\(E[Y|A=0,c)]\), \(E[Y|A=1,c)]\)) and weights from treatment model (\(W^A\)). (D) Standardized causal effect of \(A\) on \(Y\) based on and outcome modeling. (E) IPW causal effect of \(A\) on \(Y\) based on weights derived from treatment modeling; in this panel weights are taken into account in boxplots and estimations." width="90%" />
<p class="caption">
Figure 8.5: <strong>Causal inference methods on a simple example.</strong> (A) Directed acyclic graphs with causal relations between variables and parameters used to simulate data. (B) Association between <span class="math inline">\(A\)</span> and <span class="math inline">\(Y\)</span> from observed data. (C) Some simulated samples/patients with their original variables (<span class="math inline">\(C\)</span>, <span class="math inline">\(A\)</span> and <span class="math inline">\(Y\)</span>), variables from outcome model (<span class="math inline">\(E[Y|A=0,c)]\)</span>, <span class="math inline">\(E[Y|A=1,c)]\)</span>) and weights from treatment model (<span class="math inline">\(W^A\)</span>). (D) Standardized causal effect of <span class="math inline">\(A\)</span> on <span class="math inline">\(Y\)</span> based on and outcome modeling. (E) IPW causal effect of <span class="math inline">\(A\)</span> on <span class="math inline">\(Y\)</span> based on weights derived from treatment modeling; in this panel weights are taken into account in boxplots and estimations.
</p>
</div>

</div>
<div id="IPW-classic" class="section level4">
<h4><span class="header-section-number">8.2.2.2</span> Inverse probability weighting (IPW) and propensity scores</h4>
<p>Based on the same counterfactual framework, it is possible to build another class of models, called marginal structural models <span class="citation">(Robins, Hernan, and Brumback <a href="#ref-robins2000marginal">2000</a>)</span>, from which we derive estimators different from the standardized estimators called inverse-probability-of-treatment weighted (IPW) estimators <span class="citation">(Cole and Hernán <a href="#ref-cole2008constructing">2008</a>)</span>. IP weighting is equivalent to creating a <strong>pseudo-population where the link between covariates and treatment is cancelled</strong>. In the case of binary treatment <span class="math inline">\(A \in {0 ,1}\)</span>, weights are defined for each patient as the inverse of the probability to have received the version of treatment he or she actually received, knowing his or her covariates:</p>
<p><span class="math display">\[W^A=\dfrac{1}{f[A|C]} \text{ with } f[a|c]=P[A=a|C=c],\]</span></p>
<p><span class="math inline">\(f[a|c]\)</span> being called the propensity score, <em>i.e.</em>, the probability to have received the treatment <span class="math inline">\(A=a\)</span>, given the covariates <span class="math inline">\(C=c\)</span>. Again, propensity scores will be estimated in later examples using a parametric model. In this case with a binary treatment <span class="math inline">\(A\)</span>, a logistic <strong>treatment model</strong> is used (<span class="math inline">\(A\sim C\)</span>) to derive the weights <span class="math inline">\(W^A\)</span> (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>C). Note that propensity scores are also useful for positivity investigations since values very close to <span class="math inline">\(0\)</span> or <span class="math inline">\(1\)</span> may indicate (quasi-)violations of positivity. Under the same hypothesis of exchangeability, positivity and consistency, we can derive the modified Horvitz-Thompson estimator <span class="citation">(Horvitz and Thompson <a href="#ref-horvitz1952generalization">1952</a>; Hernán and Robins <a href="#ref-hernan2020causal">2020</a>)</span>:</p>
<span class="math display" id="eq:ipweq2">\[\begin{equation}
E[Y(a)]=\dfrac{\hat{E}[I(A=a)W^{A}Y]}{\hat{E}[I(A=a)W^A]},
\tag{8.1}
\end{equation}\]</span>
<p><span class="math inline">\(I\)</span> being the indicator function. Once again, this method brings estimates closer to the true causal effect by correcting for the influence of the confounder (Figure <a href="clinical-evidence-generation-and-causal-inference.html#fig:causality-example3">8.5</a>E).</p>
</div>
<div id="limitations-and-additional-methods" class="section level4">
<h4><span class="header-section-number">8.2.2.3</span> Limitations and additional methods</h4>
<p>These causal inference methods therefore allow to control for the effect of possible biases and confounders, at the cost of strong hypotheses that it is not possible to verify. The plausibility of these hypotheses, and therefore of the resulting estimates, requires a good knowledge of the context. Furthermore, the estimates are largely based on statical models of outcome or treatment. The correct specification of these models is therefore imperative to ensure the relevance of the estimates. In order to limit the risks of misspecification, some <strong>doubly robust</strong> approaches have also been developed. They require estimating both an outcome model and a treatment model, but the resulting estimates are asymptotically correct if at least one of the two models is correctly specified. One of these methodologies among others, called Targeted Maximum Likelihood Estimation (TMLE), will be mentioned in the next section and is detailed in appendix <a href="about-statistics.html#appendix-TMLE">C.2.4</a>.</p>
<p>In summary, evaluating the effect of a treatment requires isolating its impact from that of all confounding factors. This can be done in a randomized clinical trial designed for this purpose. However, there is a great amount of other data available that may not have been generated in this rigorous framework. It is nevertheless possible to draw causal interpretations from them, under certain hypotheses, thus offering insights for <em>a posteriori</em> statistical evaluation of specific therapeutic strategies.</p>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-beal2020causal">
<p>Béal, Jonas, and Aurélien Latouche. 2020. “Causal Inference with Multiple Versions of Treatment and Application to Personalized Medicine.” <em>ArXiv Preprint ArXiv:2005.12427</em>.</p>
</div>
<div id="ref-cole2008constructing">
<p>Cole, Stephen R, and Miguel A Hernán. 2008. “Constructing Inverse Probability Weights for Marginal Structural Models.” <em>American Journal of Epidemiology</em> 168 (6). Oxford University Press: 656–64.</p>
</div>
<div id="ref-hernan2020causal">
<p>Hernán, MA, and JM Robins. 2020. “Causal Inference: What If.” <em>Boca Raton: Chapman &amp; Hill/CRC</em>.</p>
</div>
<div id="ref-horvitz1952generalization">
<p>Horvitz, Daniel G, and Donovan J Thompson. 1952. “A Generalization of Sampling Without Replacement from a Finite Universe.” <em>Journal of the American Statistical Association</em> 47 (260). Taylor &amp; Francis Group: 663–85.</p>
</div>
<div id="ref-robins2000marginal">
<p>Robins, James M, Miguel Angel Hernan, and Babette Brumback. 2000. “Marginal Structural Models and Causal Inference in Epidemiology.” LWW.</p>
</div>
<div id="ref-rubin1974estimating">
<p>Rubin, Donald B. 1974. “Estimating Causal Effects of Treatments in Randomized and Nonrandomized Studies.” <em>Journal of Educational Psychology</em> 66 (5). American Psychological Association: 688.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="information-flows-in-mechanistic-models-of-cancer.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chapter-precision.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
