<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Computational modeling of cancer: from complex disease to systems biology | From mechanistic modeling of cancer to clinical interpretation and impact evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Computational modeling of cancer: from complex disease to systems biology | From mechanistic modeling of cancer to clinical interpretation and impact evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Computational modeling of cancer: from complex disease to systems biology | From mechanistic modeling of cancer to clinical interpretation and impact evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="cancer-as-deregulation-of-complex-machinery.html"/>
<link rel="next" href="test.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#a-tale-of-prey-and-predators"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html"><a href="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html"><i class="fa fa-check"></i><b>3</b> Computational modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html"><a href="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html#diversity-and-expansion-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Diversity and expansion of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html"><a href="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html#focus-on-molecular-signaling"><i class="fa fa-check"></i><b>3.2</b> Focus on molecular signaling</a></li>
<li class="chapter" data-level="3.3" data-path="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html"><a href="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html#theory-features-of-complex-systems-emergence-non-linearity-feedback"><i class="fa fa-check"></i><b>3.3</b> Theory –&gt; Features of complex systems: emergence, non-linearity, feedback</a></li>
<li class="chapter" data-level="3.4" data-path="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html"><a href="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html#applications-impact-of-data"><i class="fa fa-check"></i><b>3.4</b> Applications –&gt; impact of data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="test.html"><a href="test.html"><i class="fa fa-check"></i><b>4</b> Test part</a><ul>
<li class="chapter" data-level="4.1" data-path="test.html"><a href="test.html#abc"><i class="fa fa-check"></i><b>4.1</b> Abc</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From mechanistic modeling of cancer to clinical interpretation and impact evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="computational-modeling-of-cancer-from-complex-disease-to-systems-biology" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Computational modeling of cancer: from complex disease to systems biology</h1>
<blockquote>
<p><em>“How remarkable is life? The answer is: very. Those of us who deal in networks of chemical reactions know of nothing like it… How could a chemical sludge become a rose, even with billions of years to try.”.</em><br/> George Whitesides</p>
</blockquote>
<p>The previous chapter identified the need to organize cancer knowledge and data. The integration of biological knowledge, particularly in the form of networks, is a first step in this direction. The deepening of knowledge, however, requires the ability to manipulate objects even more, to experiment, to dissect their behaviour in an infinite number of situations, such as the astronomer with his orrery or physicians with their old anatomical models (Figure <a href="computational-modeling-of-cancer-from-complex-disease-to-systems-biology.html#fig:anatomy">3.1</a>). Is it then possible to create mechanistic models of cancer in the same way?</p>
<div class="figure" style="text-align: center"><span id="fig:anatomy"></span>
<img src="fig/anatomy.jpg" alt="Dissecting disease using non-computational models. Rembrandt, The Anatomy Lesson of Dr Nicolaes Tulp, 1634, oil on canvas, Mauritshuis museum, The Hague" width="90%" />
<p class="caption">
Figure 3.1: <strong>Dissecting disease using non-computational models.</strong> Rembrandt, <em>The Anatomy Lesson of Dr Nicolaes Tulp</em>, 1634, oil on canvas, Mauritshuis museum, The Hague
</p>
</div>

<div id="diversity-and-expansion-of-mechanistic-models-of-cancer" class="section level2">
<h2><span class="header-section-number">3.1</span> Diversity and expansion of mechanistic models of cancer</h2>
<p>Modeling cancer is not a new idea. While methods and terminologies may have changed, there are nevertheless traces of modelling as early as the 1950s. We then speak rather of mathematical modelling with a meaning that is nevertheless intermediate between what we have defined as mechanistic models and statistical models. Armitage and Doll thus theorized the initiation of tumorigenesis by biologically justifying their mathematical expressions <span class="citation">(Armitage and Doll <a href="#ref-armitage1954age">1954</a>)</span> <span class="citation">(Knudson <a href="#ref-knudson1971mutation">1971</a>)</span></p>
<p>Timeline of mathematics and cancer <span class="citation">(Byrne <a href="#ref-byrne2010dissecting">2010</a>)</span> Integrative and scales. Many mathematical models are cell-based <span class="citation">(Anderson and Quaranta <a href="#ref-anderson2008integrative">2008</a>)</span></p>
<p>Many reviews . Mainly focused on angiogenesis, carconigenesis and tumour growth. Different formalisms depending on scales review <span class="citation">(Bellomo, Li, and Maini <a href="#ref-bellomo2008foundations">2008</a>)</span>. Focus on cellular scale and bigger.</p>
<p>PKPD and regimens <span class="citation">(Barbolosi et al. <a href="#ref-barbolosi2016computational">2016</a>)</span></p>
<p>Tumour growth <span class="citation">(Benzekry et al. <a href="#ref-benzekry2014classical">2014</a>)</span></p>
<p>review <span class="citation">(Altrock, Liu, and Michor <a href="#ref-altrock2015mathematics">2015</a>)</span></p>
<p>PubMed trends</p>
<p>Different kinds of models: scales –&gt; focus to signaling. We will model a cell</p>
</div>
<div id="focus-on-molecular-signaling" class="section level2">
<h2><span class="header-section-number">3.2</span> Focus on molecular signaling</h2>
<p>With two main formalisms: ODE and logical <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span></p>
</div>
<div id="theory-features-of-complex-systems-emergence-non-linearity-feedback" class="section level2">
<h2><span class="header-section-number">3.3</span> Theory –&gt; Features of complex systems: emergence, non-linearity, feedback</h2>
<p>critical transitions in multiple fields <span class="citation">(Trefois et al. <a href="#ref-trefois2015critical">2015</a>)</span> bistable and switches <span class="citation">(Markevich, Hoek, and Kholodenko <a href="#ref-markevich2004signaling">2004</a>)</span> emergence <span class="citation">(Helikar et al. <a href="#ref-helikar2008emergent">2008</a>)</span> tyson ode examples breast <span class="citation">(Tyson et al. <a href="#ref-tyson2011dynamic">2011</a>)</span> and general <span class="citation">(Tyson, Chen, and Novak <a href="#ref-tyson2003sniffers">2003</a>)</span></p>
<p>network and complex elements. Examples of loops <span class="citation">(Kitano <a href="#ref-kitano2002computational">2002</a><a href="#ref-kitano2002computational">a</a>)</span> <span class="citation">(Kitano <a href="#ref-kitano2002systems">2002</a><a href="#ref-kitano2002systems">b</a>)</span></p>
</div>
<div id="applications-impact-of-data" class="section level2">
<h2><span class="header-section-number">3.4</span> Applications –&gt; impact of data</h2>
<p>Boris Kholodenko With data also <span class="citation">(Bouhaddou et al. <a href="#ref-bouhaddou2018mechanistic">2018</a>)</span></p>
<p>Cancer systems biology can be understood as the study of how perturbations of the normal and healthy genome leads to tumorigenesis. This interdisciplinary fields combine mathematics, statistics, physics, and molecular biology in order to decipher or to begin to understand the effect of these alterations [16]. The cell can be seen as a system composed of genes, proteins, small molecules, enzymes, etc. which interact and are involved in many different intertwined signalling pathways. Cancer is characterised by an accumulation of genetic and epigenetic alterations and regulatory networks, which deregulate the molecular machinery of normal cells. Cancer is often referred to as a network disease [1], because the genes that are mutated or altered affect or even rewire the original network leading to different outcomes [17]. An alteration of one of these genes may have an impact or not on the cellular fate; the outcome is often unpredictable and unexpected.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-altrock2015mathematics">
<p>Altrock, Philipp M, Lin L Liu, and Franziska Michor. 2015. “The Mathematics of Cancer: Integrating Quantitative Models.” <em>Nature Reviews Cancer</em> 15 (12). Nature Publishing Group: 730–45.</p>
</div>
<div id="ref-anderson2008integrative">
<p>Anderson, Alexander RA, and Vito Quaranta. 2008. “Integrative Mathematical Oncology.” <em>Nature Reviews Cancer</em> 8 (3). Nature Publishing Group: 227–34.</p>
</div>
<div id="ref-armitage1954age">
<p>Armitage, Peter, and Richard Doll. 1954. “The Age Distribution of Cancer and a Multi-Stage Theory of Carcinogenesis.” <em>British Journal of Cancer</em> 8 (1). Nature Publishing Group: 1.</p>
</div>
<div id="ref-barbolosi2016computational">
<p>Barbolosi, Dominique, Joseph Ciccolini, Bruno Lacarelle, Fabrice Barlési, and Nicolas André. 2016. “Computational Oncology—mathematical Modelling of Drug Regimens for Precision Medicine.” <em>Nature Reviews Clinical Oncology</em> 13 (4). Nature Publishing Group: 242.</p>
</div>
<div id="ref-bellomo2008foundations">
<p>Bellomo, Nicola, NK Li, and Ph K Maini. 2008. “On the Foundations of Cancer Modelling: Selected Topics, Speculations, and Perspectives.” <em>Mathematical Models and Methods in Applied Sciences</em> 18 (04). World Scientific: 593–646.</p>
</div>
<div id="ref-benzekry2014classical">
<p>Benzekry, Sébastien, Clare Lamont, Afshin Beheshti, Amanda Tracz, John ML Ebos, Lynn Hlatky, and Philip Hahnfeldt. 2014. “Classical Mathematical Models for Description and Prediction of Experimental Tumor Growth.” <em>PLoS Comput Biol</em> 10 (8). Public Library of Science: e1003800.</p>
</div>
<div id="ref-bouhaddou2018mechanistic">
<p>Bouhaddou, Mehdi, Anne Marie Barrette, Alan D Stern, Rick J Koch, Matthew S DiStefano, Eric A Riesel, Luis C Santos, Annie L Tan, Alex E Mertz, and Marc R Birtwistle. 2018. “A Mechanistic Pan-Cancer Pathway Model Informed by Multi-Omics Data Interprets Stochastic Cell Fate Responses to Drugs and Mitogens.” <em>PLoS Computational Biology</em> 14 (3). Public Library of Science: e1005985.</p>
</div>
<div id="ref-byrne2010dissecting">
<p>Byrne, Helen M. 2010. “Dissecting Cancer Through Mathematics: From the Cell to the Animal Model.” <em>Nature Reviews Cancer</em> 10 (3). Nature Publishing Group: 221–30.</p>
</div>
<div id="ref-helikar2008emergent">
<p>Helikar, Tomáš, John Konvalina, Jack Heidel, and Jim A Rogers. 2008. “Emergent Decision-Making in Biological Signal Transduction Networks.” <em>Proceedings of the National Academy of Sciences</em> 105 (6). National Acad Sciences: 1913–8.</p>
</div>
<div id="ref-kitano2002computational">
<p>Kitano, Hiroaki. 2002a. “Computational Systems Biology.” <em>Nature</em> 420 (6912). Nature Publishing Group: 206–10.</p>
</div>
<div id="ref-kitano2002systems">
<p>Kitano, Hiroaki. 2002b. “Systems Biology: A Brief Overview.” <em>Science</em> 295 (5560). American Association for the Advancement of Science: 1662–4.</p>
</div>
<div id="ref-knudson1971mutation">
<p>Knudson, Alfred G. 1971. “Mutation and Cancer: Statistical Study of Retinoblastoma.” <em>Proceedings of the National Academy of Sciences</em> 68 (4). National Acad Sciences: 820–23.</p>
</div>
<div id="ref-le2015quantitative">
<p>Le Novere, Nicolas. 2015. “Quantitative and Logic Modelling of Molecular and Gene Networks.” <em>Nature Reviews Genetics</em> 16 (3). Nature Publishing Group: 146–58.</p>
</div>
<div id="ref-markevich2004signaling">
<p>Markevich, Nick I, Jan B Hoek, and Boris N Kholodenko. 2004. “Signaling Switches and Bistability Arising from Multisite Phosphorylation in Protein Kinase Cascades.” <em>The Journal of Cell Biology</em> 164 (3). Rockefeller University Press: 353–59.</p>
</div>
<div id="ref-trefois2015critical">
<p>Trefois, Christophe, Paul MA Antony, Jorge Goncalves, Alexander Skupin, and Rudi Balling. 2015. “Critical Transitions in Chronic Disease: Transferring Concepts from Ecology to Systems Medicine.” <em>Current Opinion in Biotechnology</em> 34. Elsevier: 48–55.</p>
</div>
<div id="ref-tyson2011dynamic">
<p>Tyson, John J, William T Baumann, Chun Chen, Anael Verdugo, Iman Tavassoly, Yue Wang, Louis M Weiner, and Robert Clarke. 2011. “Dynamic Modelling of Oestrogen Signalling and Cell Fate in Breast Cancer Cells.” <em>Nature Reviews Cancer</em> 11 (7). Nature Publishing Group: 523–32.</p>
</div>
<div id="ref-tyson2003sniffers">
<p>Tyson, John J, Katherine C Chen, and Bela Novak. 2003. “Sniffers, Buzzers, Toggles and Blinkers: Dynamics of Regulatory and Signaling Pathways in the Cell.” <em>Current Opinion in Cell Biology</em> 15 (2). Elsevier: 221–31.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="cancer-as-deregulation-of-complex-machinery.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="test.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
