<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Conclusion | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Conclusion | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Conclusion | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chapter-precision.html"/>
<link rel="next" href="appendix-datasets.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="remerciements.html"><a href="remerciements.html"><i class="fa fa-check"></i>Remerciements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-family"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
<li class="chapter" data-level="3.4.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-interventions-and-causality"><i class="fa fa-check"></i><b>3.4.4</b> Mechanistic models, interventions and causality</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-tools-list"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#continuous-time-markov-processes"><i class="fa fa-check"></i><b>4.2.1</b> Continuous-time Markov processes</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.2</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.3</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.4</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-chap.html"><a href="personalization-chap.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-chap.html"><a href="personalization-chap.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-chap.html"><a href="personalization-chap.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-chap.html"><a href="personalization-chap.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-chap.html"><a href="personalization-chap.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-chap.html"><a href="personalization-chap.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-chap.html"><a href="personalization-chap.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-chap.html"><a href="personalization-chap.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-chap.html"><a href="personalization-chap.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#ML-comp"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#part2-limits"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a><ul>
<li class="chapter" data-level="7.1.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-framework-and-general-principles"><i class="fa fa-check"></i><b>7.1.1</b> Evaluation framework and general principles</a></li>
<li class="chapter" data-level="7.1.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#some-frequent-problems-and-recommended-statistical-tools"><i class="fa fa-check"></i><b>7.1.2</b> Some frequent problems and recommended statistical tools</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artificial-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artificial examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="causal-chap.html"><a href="causal-chap.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="causal-chap.html"><a href="causal-chap.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="causal-chap.html"><a href="causal-chap.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="causal-chap.html"><a href="causal-chap.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-basic"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="causal-chap.html"><a href="causal-chap.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#part3-limits"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a><ul>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#one-path-among-others"><i class="fa fa-check"></i>One path among others</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#in-other-directions"><i class="fa fa-check"></i>In other directions</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#r2-and-beyond"><i class="fa fa-check"></i><b>C.1</b> <span class="math inline">\(R^2\)</span> and beyond</a><ul>
<li class="chapter" data-level="C.1.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.1.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-r2surv"><i class="fa fa-check"></i><b>C.1.2</b> <span class="math inline">\(R^2\)</span> for survival data</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="D" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html"><i class="fa fa-check"></i><b>D</b> Résumé détaillé</a><ul>
<li class="chapter" data-level="D.1" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#modélisation-et-cancer"><i class="fa fa-check"></i><b>D.1</b> Modélisation et cancer</a></li>
<li class="chapter" data-level="D.2" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#des-modèles-logiques-personnalisés-de-cancer"><i class="fa fa-check"></i><b>D.2</b> Des modèles logiques personnalisés de cancer</a></li>
<li class="chapter" data-level="D.3" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#quantification-statistique-de-limpact-clinique-des-modèles"><i class="fa fa-check"></i><b>D.3</b> Quantification statistique de l'impact clinique des modèles</a></li>
<li class="chapter" data-level="D.4" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#conclusion-1"><i class="fa fa-check"></i><b>D.4</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="conclusion" class="section level1 unnumbered">
<h1>Conclusion</h1>
<div id="one-path-among-others" class="section level2 unnumbered">
<h2>One path among others</h2>
<p>The aim of this thesis was to trace <strong>a path to link the biological knowledge of cancer to the clinical impact through mechanistic models</strong>. At each of the stages that separate the creation of a mechanistic model of cancer from its possible clinical application, choices were made that determined the obstacles encountered and the final shape of this work. Before elaborating on the subject, it is worth summarizing one last time the thread followed during this thesis.</p>
<p>Among the many possible orientations, it was chosen to take the opposite side of the data-intensive machine learning methods. The main approach proposed uses a qualitative logical formalism and <strong>integrates the data by interpreting them</strong> rather than by optimizing the parameters with respect to a particular objective. As a result, the resulting personalized mechanistic models have proven to be <strong>more of an interpretive than a predictive tool</strong>. Their versatility and low data requirements nevertheless allow them to be applied to a wide range of questions, particularly concerning the response to treatments that their mechanistic nature facilitates. This seemingly limitless versatility can, however, prove to be a trap because, while all kinds of applications are theoretically possible, the need to rely on detailed biological knowledge and appropriate data limits its scope.</p>
<p>In the case of mechanistic molecular signaling models, this interpretive nature of the models is confirmed by statistical analyses. The main value of these models is to provide an understandable framework for extracting relevant biological information in the context of current biological knowledge. The <strong>ability of these models to detect emerging non-linear information is also proven, but is rarer and of relatively smaller magnitude</strong>. Given the influx of biological knowledge and data, computational models of cancer, with various formalisms, are nevertheless multiplying, particularly with medical aims. In the context of cancer, their use to recommend personalised treatment for each patient is a possible horizon. The evaluation of these models could then become increasingly acute. This thesis proposes the <strong>adaptation of causal inference methods in order to simulate their evaluation in clinical trials</strong> and to come as close as possible to medical evaluation standards.</p>
</div>
<div id="in-other-directions" class="section level2 unnumbered">
<h2>In other directions</h2>
<p>Along this path, many forks are conceivable, as well as the further pursuit of some of the paths that have been explored. Without going back over the technical limits and perspectives of parts II and III, already mentioned in sections <a href="logical-drug-chap.html#part2-limits">6.4</a> and <a href="chapter-precision.html#part3-limits">9.6</a> respectively, it is possible to paint a broader picture of some of the obstacles and alternatives.</p>
<p>If we question the different choices and presuppositions of this thesis we must first mention that <strong>mechanistic models of cancer are not limited to the modeling of molecular networks</strong>. Other fields have been explored, and some applications are already more mature and closer to direct use in clinical oncology. This is essentially the case for pharmacological approaches focusing on the pharmacokinetics or pharmacodynamics of certain compounds and the modeling of tumor size <span class="citation">(Benzekry <a href="#ref-benzekry2020artificial">2020</a>)</span>. The two approaches answer different questions from those raised in this thesis, but they are nevertheless closely linked and we must consider the enrichments they can bring. For example, pharmacokinetic and pharmacodynamic modeling represents a very clear blind spot of the approaches presented in chapter <a href="logical-drug-chap.html#logical-drug-chap">6</a> where all BRAF inhibitors are modeled in an identical and crude way: quantitative parameters of BRAF inhibition are not translated. All these aspects constitute the core of mechanistic pharmacokinetic/pharmacodynamic models that are biochemically more accurate and fine-grained. On the other hand, tumor size models present a more macroscopic approach that is easier to relate to patient-level clinical interpretations.</p>
<p>All these considerations thus incite us not to consider a single-layer model and to try to make it cover the entire path, from the molecule to the patient, as proposed in this thesis. Rather than building huge mechanistic models covering all scales, it seems appropriate to work on <strong>linking different models in a modular multi-scale approach</strong> making it easier to cooperate between fields of modeling that often require different knowledge. As an example, it is possible to combine MaBoSS and its molecular network models with PhysiCell, a 3D multicellular system simulator <span class="citation">(Ghaffarizadeh et al. <a href="#ref-ghaffarizadeh2018physicell">2018</a>)</span>. The PhysiBoss framework schematically proposes to encapsulate a logical model in each cell to simulate its internal functions and then let PhysiCell model the three-dimensional interactions between multiple cells of potentially different types <span class="citation">(Letort et al. <a href="#ref-letort2019physiboss">2019</a>)</span>. Probably the most promising prospects for mechanistic models lie in synergies of this kind, which, however, require broader collaborations and longer design times to truly enable the different modeling scales to communicate. The idea of proposing multi-scale models encapsulated in a hierarchical way to ensure the modeling from the genetic scale to the phenotype is also found in intermediate approaches between mechanistic and deep learning models that force neural networks to follow this architecture in order to ease <em>post hoc</em> interpretation <span class="citation">(Ma et al. <a href="#ref-ma2018using">2018</a>)</span>.</p>
<p>The previous mention of quantitative pharmacology models also underlines the <strong>intrinsic limitations of qualitative mechanistic models</strong>, <em>a fortiori</em> for clinical applications. Thus the outputs of the personalized logical models can only be interpreted in relation to each other; it is difficult to give each one an absolute interpretation. Without proper calibration, what does a <em>Proliferation</em> score of 0.2 or 0.8 mean? On first reading, they have only a relative significance that precludes direct clinical interpretation. However, it seems difficult to make these logical models much more quantitative. The very nature of formalism limits the precise translation of biological mechanisms retrieved from literature. Likewise, the partial integration of some omics data only in logical models, presented in chapter <a href="personalization-chap.html#personalization-chap">5</a>, required a specific methodology to allow the data to be adapted to the discrete and qualitative formalism. Obtaining truly quantitative and well-calibrated mechanistic models has until now always required intrinsically more quantitative formalisms such as ordinary differential equations <span class="citation">(Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>, or Bayesian frameworks allowing a more flexible representation of entities and their relationships <span class="citation">(Jastrzebski et al. <a href="#ref-jastrzebski2018integrative">2018</a>)</span>. In these two cases, as in most others, the <strong>quantitative quality of the predictions relies on model training and parameter fitting</strong>, contrary to the approach proposed in this thesis.</p>
<p>These methods therefore depend very heavily on the available data to train their models. One of their limitations is to use data from cell line screening. Schematically it is necessary to have the response to different treatments for each line in order to obtain personalized models or interpretations. The abundance of this type of screening data makes them the preferred methods for these applications where they greatly outperform qualitative models without learning. On the other hand, in a more prospective way, it is interesting to ask the question of the availability of patient data with the view to one day apply these models as close as possible to the clinic. Today it is inconceivable to obtain such rich data for patients. This would imply being able to develop cell lines for each patient, then screen all of them for drugs and use the results. This is a prospect whose feasibility is being studied <span class="citation">(Lee et al. <a href="#ref-lee2018pharmacogenomic">2018</a>)</span>, but it is not a routine clinical procedure today. If it were possible, the models would probably be replaced by the experiments themselves. More realistically, it is hoped that once the model has been sufficiently trained, the data required to make predictions for a new patient will be more modest. However, this new patient still needs to remain within the training data field. Conversely, logical models personalized without learning are based on other assumptions, mainly related to the correct mathematical translation of biological knowledge and interpreted omics data. Continued growth in the quantity and quality of data is likely to give an overall advantage in the medium term to the above mentioned quantitative mechanistic methods, more data-intensive but more accurate. <strong>Qualitative models such as those presented in chapters <a href="personalization-chap.html#personalization-chap">5</a> and <a href="logical-drug-chap.html#logical-drug-chap">6</a> may nonetheless remain a complementary approach, used beforehand or in more data-poor cases that do not allow for learning</strong>, as discussed when comparing the results with machine learning approaches in section <a href="logical-drug-chap.html#ML-comp">6.2.5</a>.</p>
<p>In such a world of abundant data, is it only then relevant to continue to propose mechanistic models instead of biologically agnostic machine learning models? The latter have indeed become much more frequent with a particular emphasis in recent years on deep learning approaches <span class="citation">(Angermueller et al. <a href="#ref-angermueller2016deep">2016</a>; Eraslan et al. <a href="#ref-eraslan2019deep">2019</a>)</span>. The comparison between quantitative mechanistic approaches and machine learning approaches then uses terms similar to the previous comparison between qualitative and quantitative mechanistic approaches. Machine learning approaches can be even more flexible because their degrees of freedom are not constrained by an <em>a priori</em> biological structure, but they consequently require larger amounts of data. The loss of any internal mechanistic structure can however hamper the interpretability of the results, although many methods have been developed to facilitate this interpretation of machine learning models in biology <span class="citation">(Azuaje <a href="#ref-azuaje2019artificial">2019</a>; Manica et al. <a href="#ref-manica2019toward">2019</a>)</span>. The loss of the mechanistic nature in most machine learning approaches also makes it impossible to test perturbations or hypotheses that are clearly outside the scope of training data. If one can imagine the predominance of well-calibrated machine learning models for routine decisions, <strong>more exploratory and prospective investigations could however favour mechanistic models more capable of extrapolation</strong>. Moreover, while the question of the acceptability of computational decision support models in oncology arises <span class="citation">(Vollmer et al. <a href="#ref-vollmer2020machine">2020</a>)</span>, the explicit representation of the internal mechanisms may present a reassuring aspect. It is simpler for both the patient and the physician to imagine the functioning of a model that replicates human bodies or cells than a more exclusively mathematical black box. However, this argument only seems valid in the case of very similar performances.</p>
<p>Finally, most of the remaining questions that arise concern all computational models of cancer in the same way, whatever their nature. <span class="citation">Wiens et al. (<a href="#ref-wiens2019no">2019</a>)</span> thus propose a <strong>roadmap for the responsible use of the models in the clinic</strong> that targets a number of challenges. One of them is to know how computational models can be seamlessly integrated into medical practice while leaving the various stakeholders, from medical experts to patients, in the loop. Physicians, for example, need to be informed in a relevant way about the nature and limitations of these models, which can facilitate the decision but whose technical details may escape them. The presentation of these models to them is therefore particularly crucial in order to ensure their proper use, as <span class="citation">Sendak et al. (<a href="#ref-sendak2020presenting">2020</a>)</span> point out by proposing standardized methods of presentation and communication. Numerous ethical or data privacy issues also arise when precise genetic information about patients needs to be processed.</p>
<p>One of the challenges mentioned by <span class="citation">Wiens et al. (<a href="#ref-wiens2019no">2019</a>)</span> which resonates particularly with this thesis is that of the rigorous evaluation of the clinical contribution of these models, mechanistic or not. The choice of metrics is therefore very important and must be made to optimize clinical information. For example, while the discrimination capability of models is frequently measured, the calibration performance of risk prediction models is also crucial to inform clinical practice <span class="citation">(Van Calster et al. <a href="#ref-van2019calibration">2019</a>)</span>. In general, although computational cancer models are changing the cancer research landscape, <strong>proven standards from epidemiology or clinical statistics for evaluation should be maintained</strong> to allow comparison and quantification of improvements beyond scientific trends <span class="citation">(Christodoulou et al. <a href="#ref-christodoulou2019systematic">2019</a>)</span>. These evaluations require real clinical expertise to measure the extent to which a gain in algorithmic performance can be translated into clinical improvement depending on the patients concerned and the resources mobilized. Concerning the mechanistic models in particular, for example, their capacity to extrapolate and possibly reason about new treatments for which data are missing or few have been underlined several times. However, the clinical evaluation of their impact always requires data at the end of the day. The methods of causal inference applied in chapter <a href="chapter-precision.html#chapter-precision">9</a> make it possible to correct the effects and bring them closer to the standards of clinical trials, but they can only rearrange clinical strategies on the basis of existing data. <strong>Mechanistic models are therefore there to guide or enlighten experiments, but are not intended to replace them</strong>.</p>
<p>To conclude in a word, cancer models still have a bright future ahead of them. Mechanistic models will continue to be attractive because of their ability not only to predict but, more importantly, to explain. However, the transparency of their mechanisms should not prevent them from being rigorously evaluated statistically. It is not enough for them to explain, they must also be well understood.</p>

</div>
</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-angermueller2016deep">
<p>Angermueller, Christof, Tanel Pärnamaa, Leopold Parts, and Oliver Stegle. 2016. “Deep Learning for Computational Biology.” <em>Molecular Systems Biology</em> 12 (7): 878.</p>
</div>
<div id="ref-azuaje2019artificial">
<p>Azuaje, Francisco. 2019. “Artificial Intelligence for Precision Oncology: Beyond Patient Stratification.” <em>NPJ Precision Oncology</em> 3 (1). Nature Publishing Group: 1–5.</p>
</div>
<div id="ref-benzekry2020artificial">
<p>Benzekry, Sébastien. 2020. “Artificial Intelligence and Mechanistic Modeling for Clinical Decision Making in Oncology.” <em>Clinical Pharmacology &amp; Therapeutics</em>. Wiley Online Library.</p>
</div>
<div id="ref-christodoulou2019systematic">
<p>Christodoulou, Evangelia, Jie Ma, Gary S Collins, Ewout W Steyerberg, Jan Y Verbakel, and Ben Van Calster. 2019. “A Systematic Review Shows No Performance Benefit of Machine Learning over Logistic Regression for Clinical Prediction Models.” <em>Journal of Clinical Epidemiology</em> 110. Elsevier: 12–22.</p>
</div>
<div id="ref-eraslan2019deep">
<p>Eraslan, Gökcen, Žiga Avsec, Julien Gagneur, and Fabian J Theis. 2019. “Deep Learning: New Computational Modelling Techniques for Genomics.” <em>Nature Reviews Genetics</em> 20 (7). Nature Publishing Group: 389–403.</p>
</div>
<div id="ref-frohlich2018efficient">
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6). Elsevier: 567–79.</p>
</div>
<div id="ref-ghaffarizadeh2018physicell">
<p>Ghaffarizadeh, Ahmadreza, Randy Heiland, Samuel H Friedman, Shannon M Mumenthaler, and Paul Macklin. 2018. “PhysiCell: An Open Source Physics-Based Cell Simulator for 3-d Multicellular Systems.” <em>PLoS Computational Biology</em> 14 (2). Public Library of Science: e1005991.</p>
</div>
<div id="ref-jastrzebski2018integrative">
<p>Jastrzebski, Katarzyna, Bram Thijssen, Roelof JC Kluin, Klaas de Lint, Ian J Majewski, Roderick L Beijersbergen, and Lodewyk FA Wessels. 2018. “Integrative Modeling Identifies Key Determinants of Inhibitor Sensitivity in Breast Cancer Cell Lines.” <em>Cancer Research</em> 78 (15). AACR: 4396–4410.</p>
</div>
<div id="ref-lee2018pharmacogenomic">
<p>Lee, Jin-Ku, Zhaoqi Liu, Jason K Sa, Sang Shin, Jiguang Wang, Mykola Bordyuh, Hee Jin Cho, et al. 2018. “Pharmacogenomic Landscape of Patient-Derived Tumor Cells Informs Precision Oncology Therapy.” <em>Nature Genetics</em> 50 (10). Nature Publishing Group: 1399–1411.</p>
</div>
<div id="ref-letort2019physiboss">
<p>Letort, Gaelle, Arnau Montagud, Gautier Stoll, Randy Heiland, Emmanuel Barillot, Paul Macklin, Andrei Zinovyev, and Laurence Calzone. 2019. “PhysiBoSS: A Multi-Scale Agent-Based Modelling Framework Integrating Physical Dimension and Cell Signalling.” <em>Bioinformatics</em> 35 (7). Oxford University Press: 1188–96.</p>
</div>
<div id="ref-ma2018using">
<p>Ma, Jianzhu, Michael Ku Yu, Samson Fong, Keiichiro Ono, Eric Sage, Barry Demchak, Roded Sharan, and Trey Ideker. 2018. “Using Deep Learning to Model the Hierarchical Structure and Function of a Cell.” <em>Nature Methods</em> 15 (4). Nature Publishing Group: 290.</p>
</div>
<div id="ref-manica2019toward">
<p>Manica, Matteo, Ali Oskooei, Jannis Born, Vigneshwari Subramanian, Julio Sáez-Rodríguez, and María Rodríguez Martínez. 2019. “Toward Explainable Anticancer Compound Sensitivity Prediction via Multimodal Attention-Based Convolutional Encoders.” <em>Molecular Pharmaceutics</em>. ACS Publications.</p>
</div>
<div id="ref-sendak2020presenting">
<p>Sendak, Mark P, Michael Gao, Nathan Brajer, and Suresh Balu. 2020. “Presenting Machine Learning Model Information to Clinical End Users with Model Facts Labels.” <em>NPJ Digital Medicine</em> 3 (1). Nature Publishing Group: 1–4.</p>
</div>
<div id="ref-van2019calibration">
<p>Van Calster, Ben, David J McLernon, Maarten Van Smeden, Laure Wynants, and Ewout W Steyerberg. 2019. “Calibration: The Achilles Heel of Predictive Analytics.” <em>BMC Medicine</em> 17 (1). BioMed Central: 1–7.</p>
</div>
<div id="ref-vollmer2020machine">
<p>Vollmer, Sebastian, Bilal A Mateen, Gergo Bohner, Franz J Király, Rayid Ghani, Pall Jonsson, Sarah Cumbers, et al. 2020. “Machine Learning and Artificial Intelligence Research for Patient Benefit: 20 Critical Questions on Transparency, Replicability, Ethics, and Effectiveness.” <em>Bmj</em> 368. British Medical Journal Publishing Group.</p>
</div>
<div id="ref-wiens2019no">
<p>Wiens, Jenna, Suchi Saria, Mark Sendak, Marzyeh Ghassemi, Vincent X Liu, Finale Doshi-Velez, Kenneth Jung, et al. 2019. “Do No Harm: A Roadmap for Responsible Machine Learning for Health Care.” <em>Nature Medicine</em> 25 (9). Nature Publishing Group: 1337–40.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="chapter-precision.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="appendix-datasets.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
