<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Information flows in mechanistic models of cancer | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Information flows in mechanistic models of cancer | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Information flows in mechanistic models of cancer | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="logical-drug-chap.html"/>
<link rel="next" href="causal-chap.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="remerciements.html"><a href="remerciements.html"><i class="fa fa-check"></i>Remerciements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-family"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
<li class="chapter" data-level="3.4.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-interventions-and-causality"><i class="fa fa-check"></i><b>3.4.4</b> Mechanistic models, interventions and causality</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-tools-list"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#continuous-time-markov-processes"><i class="fa fa-check"></i><b>4.2.1</b> Continuous-time Markov processes</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.2</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.3</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.4</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-chap.html"><a href="personalization-chap.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-chap.html"><a href="personalization-chap.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-chap.html"><a href="personalization-chap.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-chap.html"><a href="personalization-chap.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-chap.html"><a href="personalization-chap.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-chap.html"><a href="personalization-chap.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-chap.html"><a href="personalization-chap.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-chap.html"><a href="personalization-chap.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-chap.html"><a href="personalization-chap.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#ML-comp"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#part2-limits"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a><ul>
<li class="chapter" data-level="7.1.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-framework-and-general-principles"><i class="fa fa-check"></i><b>7.1.1</b> Evaluation framework and general principles</a></li>
<li class="chapter" data-level="7.1.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#some-frequent-problems-and-recommended-statistical-tools"><i class="fa fa-check"></i><b>7.1.2</b> Some frequent problems and recommended statistical tools</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artificial-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artificial examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="causal-chap.html"><a href="causal-chap.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="causal-chap.html"><a href="causal-chap.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="causal-chap.html"><a href="causal-chap.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="causal-chap.html"><a href="causal-chap.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-basic"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="causal-chap.html"><a href="causal-chap.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#part3-limits"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a><ul>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#one-path-among-others"><i class="fa fa-check"></i>One path among others</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#in-other-directions"><i class="fa fa-check"></i>In other directions</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#r2-and-beyond"><i class="fa fa-check"></i><b>C.1</b> <span class="math inline">\(R^2\)</span> and beyond</a><ul>
<li class="chapter" data-level="C.1.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.1.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-r2surv"><i class="fa fa-check"></i><b>C.1.2</b> <span class="math inline">\(R^2\)</span> for survival data</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="D" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html"><i class="fa fa-check"></i><b>D</b> Résumé détaillé</a><ul>
<li class="chapter" data-level="D.1" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#modélisation-et-cancer"><i class="fa fa-check"></i><b>D.1</b> Modélisation et cancer</a></li>
<li class="chapter" data-level="D.2" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#des-modèles-logiques-personnalisés-de-cancer"><i class="fa fa-check"></i><b>D.2</b> Des modèles logiques personnalisés de cancer</a></li>
<li class="chapter" data-level="D.3" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#quantification-statistique-de-limpact-clinique-des-modèles"><i class="fa fa-check"></i><b>D.3</b> Quantification statistique de l'impact clinique des modèles</a></li>
<li class="chapter" data-level="D.4" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#conclusion-1"><i class="fa fa-check"></i><b>D.4</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="information-flows-in-mechanistic-models-of-cancer" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Information flows in mechanistic models of cancer</h1>
<blockquote>
<p><em>Et l'effet qui s'en va nous decouvre les causes.</em><br/> Alfred de Musset (Poésies nouvelles, 1843)</p>
</blockquote>
<p>The mechanistic models of cancer presented in the previous section have allowed us to integrate the omics data, to &quot;make them speak&quot; in order to better understand the clinical characteristics of cell lines or patients. But beyond their undeniable intellectual and scientific interest, do they have a direct clinical utility? Given the abundance and complexity of patient data available to physicians, the use of computer tools and mathematical models is inevitable and increasingly frequent. Because of their explicit representation of phenomena, mechanistic models can provide a more easily understood alternative for physicians or patients. Is it therefore desirable and relevant to use these models in support of medical decision making? And how can their clinical validity and impact be rigorously measured?</p>
<p>First of all, the purpose of this chapter is to outline some of the limitations of the previously presented evaluations of mechanistic models, together with some recommended statistical tools. These evaluations answered the question: do the models have any clinical utility? We will show that an additional question could be: <strong>do mechanistic models have an incremental clinical utility</strong>, in comparison to the direct use of the data used to construct or specify them? This chapter is intended as a statistical introduction for systems biologists to some of the problems encountered in model evaluation.</p>

<div class="summarybox">
<h4 id="scientific-content-4" class="unnumbered">Scientific content</h4>
<p>This chapter is relying on literature for the first section and unpublished content for the second. The exploratory analyses presented below have helped to clarify considerations expressed qualitatively in previous chapters and formed the starting point for subsequent chapters on the clinical impact of cancer models.</p>
</div>
<p></p>
<div id="evaluation-of-models-as-biomarkers" class="section level2">
<h2><span class="header-section-number">7.1</span> Evaluation of models as biomarkers</h2>
<div id="evaluation-framework-and-general-principles" class="section level3">
<h3><span class="header-section-number">7.1.1</span> Evaluation framework and general principles</h3>
<p>First of all, mechanistic models of cancer should be considered as biomarkers among others, and therefore evaluated as such. This means focusing on the clinical information provided by the model outputs. In the previous examples, these outputs would be for example the <span class="math inline">\(H\)</span>/<span class="math inline">\(K_{50}\)</span>/<span class="math inline">\(A\)</span> biomarkers from Fey's model (described in section <a href="mechanistic-cancer.html#prognostic">3.4.2</a>) or the personalized <em>Proliferation</em> scores from the mechanistic models in the examples in sections <a href="personalization-chap.html#validation-METABRIC">5.2.2</a> or <a href="logical-drug-chap.html#diff-BRAF">6.2.4.1</a>. The prognostic or predictive value of model outputs can then be evaluated according to the methods and recommendations present in the literature on prognostic or predictive biomarkers. Without going into too much detail, guidelines in this area are quite numerous and detailed, both for prognostic biomarkers <span class="citation">(McShane et al. <a href="#ref-mcshane2005reporting">2005</a>; Sauerbrei et al. <a href="#ref-sauerbrei2018reporting">2018</a>)</span> and predictive biomarkers <span class="citation">(Janes et al. <a href="#ref-janes2014approach">2014</a>)</span>. Most of the points mentioned in these articles should apply identically for the particular type of biomarker that are the outputs of mechanistic models of cancer. The purpose of this thesis is not to exhaustively list these recommendations for the evaluation of biomarkers, so we will simply highlight the <strong>most salient issues identified in the systems biology literature</strong>.</p>
</div>
<div id="some-frequent-problems-and-recommended-statistical-tools" class="section level3">
<h3><span class="header-section-number">7.1.2</span> Some frequent problems and recommended statistical tools</h3>
<p>Concerning continuous and prognostic model outputs/biomarkers, they are sometimes confronted with naturally binary data (<em>e.g.</em>, event or not). In this case, many methods exist, among which the <strong>Area Under the receiver operating Curve (ROC), usually denoted as AUC</strong> <span class="citation">(Søreide <a href="#ref-soreide2009receiver">2009</a>)</span>. With a continuous biomarker <span class="math inline">\(X\)</span> and a binary outcome to predict <span class="math inline">\(D\)</span>, the ROC curve plots sensitivity, <span class="math inline">\(P(X &gt; c | D = 1)\)</span>, against <span class="math inline">\((1 -~\text{specificity})= (1-P(X\leq c|D= 0))\)</span>, for all possible values <span class="math inline">\(c\)</span>. the AUC is then simply computed as the area under this curve. The resulting AUC is computed as the area under this curve and measures the ability of the biomarker to discriminate between the two classes of interest and is a common tool for the evaluation of biomarkers.</p>
<p>However, prognostic validation often requires time-to-envent data such as survival data. Very schematically, if we study patients suffering from cancer and we are interested in the <em>death</em> event from a time <span class="math inline">\(t_0\)</span>, which we define to be common to all patients, different cases are possible. Some patients have died and we therefore know their status and the time of their death <span class="math inline">\(t_1\)</span>. Others are still alive at the time <span class="math inline">\(t_{max}\)</span> when the study stops (administrative censorship) or have withdrawn from the study at time <span class="math inline">\(t_2\)</span> so that their fate is then unknown. These patients are said to be right-censored. These data are extremely frequent and require specific methods such as Cox's proportional risk model <span class="citation">(Cox <a href="#ref-cox1972regression">1972</a>)</span> or accelerated failure time models. The <strong>reasoned use of these dedicated survival models, and the associated assumptions, should be preferred to the forced binarization of survival data</strong>, sometimes encountered in an apparent concern for simplification. In general, the validation of prognostic biomarkers using survival data therefore requires specific metrics such as <strong>time dependent AUC for censored survival data</strong> <span class="citation">(Heagerty, Lumley, and Pepe <a href="#ref-heagerty2000time">2000</a>)</span>. This measure, however, requires a more complex definition of sensitivity and specificity to accommodate censored data <span class="citation">(Heagerty and Zheng <a href="#ref-heagerty2005survival">2005</a>)</span> and to be applicable to real biomarker validation data <span class="citation">(Buyse et al. <a href="#ref-buyse2006validation">2006</a>)</span>. Despite fairly frequent use <span class="citation">(Ching, Zhu, and Garmire <a href="#ref-ching2018cox">2018</a>)</span>, the use of another metric called c-index is not recommended for assessing a model's ability to predict risk over a given time horizon <span class="citation">(Blanche, Kattan, and Gerds <a href="#ref-blanche2019c">2019</a>)</span>. However, the clinical interpretation of AUC values is not straightforward and the complementarity of other approaches that focus on <strong>the usefulness of risk models at the population level</strong> rather than the ability to discriminate has been highlighted <span class="citation">(Pepe et al. <a href="#ref-pepe2008integrating">2008</a>)</span>. The costs and benefits of prognostic models with various AUC values have been studied in an applied context by <span class="citation">Gail and Pfeiffer (<a href="#ref-gail2018breast">2018</a>)</span>.</p>
<p>Another frequent issue, already encountered in the examples from previous chapters, is the discretization of continuous markers. This is often done in order to classify patients into high and low risk groups for example for prognostic biomarkers. In the first place, although discretization may be required clinically, it is not necessary to evaluate the clinical value of the biomarker beforehand. Secondly, the choice of thresholds is crucial. In particular, in the case of biomarkers derived from mechanistic models, the artificial nature of the markers often makes difficult a binarization based on an <em>a priori</em> interpretation of the values. Choosing the cut-off point in order to maximise the significance or separation of the survival curves, as proposed in <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span> and presented in Figure <a href="mechanistic-cancer.html#fig:fey">3.6</a>, is however not recommended <span class="citation">(Altman et al. <a href="#ref-altman1994dangers">1994</a>)</span>, among other things because it can be interpreted as uncorrected multiple testing. Such practices may thus contribute to the low clinical reproducibility of the contribution of certain biomarkers <span class="citation">(Hilsenbeck, Clark, and McGuire <a href="#ref-hilsenbeck1992so">1992</a>)</span>. For this problem in particular, tools have been proposed in the literature on clinical biomarkers, such as the <strong>predictivenesse curve</strong> <span class="citation">(Mboup, Blanche, and Latouche <a href="#ref-mboup2020evaluating">2020</a>)</span>. Similarly but in a more general framework, <span class="citation">Janes et al. (<a href="#ref-janes2014approach">2014</a>)</span> propose the use of <strong>risk curves</strong> to better evaluate predictive biomarkers beyond the crude computation of statistical interaction between the biomarker and the treatment in randomized clinical trial.</p>
<p>Another potential issue, particularly important in subsequent analyses (see section <a href="information-flows-in-mechanistic-models-of-cancer.html#information">7.2.1</a>), is the <strong>incremental value of biomarkers</strong>. For instance, in the context of prognostic biomarkers, it is of course necessary to present univariable analyses showing the relationship between the marker and the outcome, which is almost systematically done, but also to question the value of this biomarker compared to other prognostic factors already known: does it add information or is it redundant? It is theoretically possible to consider the potential increase in AUC resulting from the addition of the new biomarker to the model. The increase in AUC, however, requires the addition of very strong markers <span class="citation">(Gail and Pfeiffer <a href="#ref-gail2018breast">2018</a>)</span>, which has prompted the emergence of popular alternative metrics (Net Reclassification Index NRI, integrated discrimination improvement IDI) to evaluate the added predictive ability of a new marker <span class="citation">(Pencina et al. <a href="#ref-pencina2008evaluating">2008</a>)</span>. However, these metrics have been criticized as being unsafe since they can improve with the addition of non-informative markers <span class="citation">(Hilden and Gerds <a href="#ref-hilden2014note">2014</a>; Pepe, Janes, and Li <a href="#ref-pepe2014net">2014</a>)</span>.</p>
<p>All in all, the first step in a good evaluation of mechanistic models would be that the <strong>standards recommended for the evaluation of biomarkers can be applied in the same way to mechanistic models</strong> that have certain applications or validation based on prognostic or predictive values. As these topics are well covered in the relevant literature, we will subsequently focus on a <strong>specific vision of incremental value of biomarkers that is more specific to mechanistic models</strong>.</p>
</div>
</div>
<div id="processing-of-biological-information" class="section level2">
<h2><span class="header-section-number">7.2</span> Processing of biological information</h2>
<p>Mechanistic models, and their outputs in particular, have so far been considered and evaluated as biomarkers. A comprehensive appreciation requires that they be seen as <strong>information processing tools</strong> in relation to the biological data they use. In this section, we will focus on a toy example to introduce some concepts. We will thus speak in general terms of the clinical value of this model, understood in the sense of a prognostic or predictive value depending on the application. The next section (<a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis">7.3</a>) will extend the same analyses to published models. The purpose of these two sections is to question the way in which mechanistic models process information. These <strong>qualitative questions have been written essentially for those who design mechanistic models</strong>. For the sake of technical simplification, the statistical tools chosen for illustration are therefore simpler than those presented in the previous section.</p>
<div id="information" class="section level3">
<h3><span class="header-section-number">7.2.1</span> Information in, information out</h3>
<p>Indeed, the mechanistic models presented in this thesis (Figures <a href="mechanistic-cancer.html#fig:fey">3.6</a>, <a href="personalization-chap.html#fig:PROFILE-METABRIC-Survival">5.9</a> and <a href="logical-drug-chap.html#fig:BRAF-results">6.6</a>) can be schematically represented by Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:box-mech">7.1</a>: inputs <span class="math inline">\(X\)</span> (often omics data) are processed through a mechanistic model (here the grey box) to result in an output <span class="math inline">\(Y\)</span>. These models can thus be assimilated to a mathematical transformation, often non-linear, of <span class="math inline">\(X\)</span> in <span class="math inline">\(Y\)</span>. Thus, when validating the biological or clinical relevance of <span class="math inline">\(Y\)</span>, either by calculating a correlation with the ground truth or by using it to stratify survival curves, only the univariate value of <span class="math inline">\(Y\)</span> is checked. This is an important step and a prerequisite for a well-constructed model. On the other hand, it is not sufficient information to understand how the model works. Indeed, the inputs <span class="math inline">\(X\)</span> can themselves be prognostic biomarkers, and this from the outset and before being transformed into Y: <em>e.g.</em>, if the mechanistic model uses different inputs, each of which has a prognostic value, the fact that the output also has a prognostic value does not necessarily indicate the relevance of the model in its ability to make sense of the information. In short, <strong>measuring only the output value of the model does not necessarily reveal the model's ability to make sense of the data it uses</strong>. Put more figuratively, it is important to know whether the model turns lead into gold by giving a clinical interpretation (<span class="math inline">\(Y\)</span>) based on information that was devoid of it (<span class="math inline">\(X\)</span>), or whether it simply turns gold into gold by refactoring information that is already clinically relevant. In the latter case, the interest of the model may lie in the way it better summarizes or makes the information understandable, but its incremental clinical value is low.</p>
<div class="figure" style="text-align: center"><span id="fig:box-mech"></span>
<img src="fig/box-mech.png" alt="Evaluation of a mechanistic model. Adapted from Figure 1.6." width="80%" />
<p class="caption">
Figure 7.1: <strong>Evaluation of a mechanistic model.</strong> Adapted from Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>.
</p>
</div>

<p>Therefore, the question of the incremental value of the model can be explained as follows: what does the output of the model represent in relation to the inputs? If we restrict ourselves to cases where the absolute biological/clinical value of <span class="math inline">\(Y\)</span> is positive, we can then identify two families of situations. First we can imagine a situation where the mechanistic model has &quot;improved&quot; the value of the inputs: the output would then have a higher value than the inputs (better biological validation, better pronostic value etc.), or in any case a complementary value, a value not present in the inputs. This would correspond to the <strong>capture by the model of emerging or non-linear effects</strong>. For the sake of simplification, we will here assimilate the two in the sense that a non-linear effect resulting from the interaction between certain variables was indeed not predictable from the components taken individually, and therefore emergent. Note, for example, that the identification and capture by statistical models of non-linear components of treatment response is important in the ability to generalize findings from preclinical models to human tumours <span class="citation">(Mourragui et al. <a href="#ref-mourragui2020precise">2020</a>)</span>. In the second situation, the output does not capture emergent properties but summarizes, totally or partially, the information present in the inputs. This would correspond to a <strong>knowledge-informed dimensionality-reduction</strong>. Even in the latter case, the scientific value of the model as a tool for understanding is not necessarily questioned. The analyses presented below are simply intended to supplement the understanding of models and how they process information.</p>
</div>
<div id="emergence-of-information-in-artificial-examples" class="section level3">
<h3><span class="header-section-number">7.2.2</span> Emergence of information in artificial examples</h3>
<p>These questions can be illustrated using a very simple artificial model represented in Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:model-simulation">7.2</a>. On the one hand there are two latent biological variables called <em>Proliferation (P)</em> and <em>Apoptosis (A)</em> resulting in our biological ground truth, <em>Growth</em>. On the other hand, the modeler has access to three different random variables <span class="math inline">\(N_1\)</span>, <span class="math inline">\(N_2\)</span> and <span class="math inline">\(N_3\)</span> respectively associated with the sign of <em>P</em>, the absolute value of <em>P</em> and the value of <em>A</em>. Two mechanistic models are defined, one linear (with its output <span class="math inline">\(O_{linear}\)</span>) and one non-linear (with its output <span class="math inline">\(O_{non-linear}\)</span>). We note that the two outputs are sufficiently well defined to be correlated with <em>Growth</em> but only the non-linear model makes use of <span class="math inline">\(N_2\)</span> by multiplying it with <span class="math inline">\(N_1\)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:model-simulation"></span>
<img src="fig/model-simulation.png" alt="Evaluation of a mechanistic model. Adapted from Figure 1.6." width="80%" />
<p class="caption">
Figure 7.2: <strong>Evaluation of a mechanistic model.</strong> Adapted from Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>.
</p>
</div>

<p>The ability of models to use inputs to create or summarize information through outputs will be studied using the <strong>explained variation metric <span class="math inline">\(R^2\)</span></strong>. If a linear model is defined as <span class="math inline">\(y_i=\beta_0+\beta_1x_i+e_i\)</span>, linear coefficients <span class="math inline">\(\beta\)</span> are estimated by minimizing the sum of squared differences between predicted and real values of <span class="math inline">\(y\)</span>. The fitted model is written <span class="math inline">\(\hat{y_i}=\hat{\beta_0}+\hat{\beta_1}x_i\)</span> and <span class="math inline">\(R^2\)</span> also called coefficient of determination is defined as:</p>
<p><span class="math display">\[R^2=1-\dfrac{\sum_{i=1}^{n} (\hat{y_i}-\bar{y_i})^2}{\sum_{i=1}^{n}  (y_i-\bar{y})^2}\]</span></p>
<p>Therefore <span class="math inline">\(R^2\)</span> measures the proportion of variation in <span class="math inline">\(y\)</span> that is explained by the regressors. A different way of expressing this is to say that <span class="math inline">\(R^2\)</span> compares the null model without covariate (observations are compared to their mean) to the linear model with covariates. By extension, it has been proposed to use R2 to assess the effect of adding a new biomarker to a previously established model<a href="#fn8" class="footnoteRef" id="fnref8"><sup>8</sup></a> <span class="citation">(Schemper <a href="#ref-schemper2003predictive">2003</a>)</span>. In order to avoid overfitting, it is possible to calculate the adjusted <span class="math inline">\(R^2\)</span> that corrects with the number of regressors or to fit the model on training data and calculate the <span class="math inline">\(R^2\)</span> on validation data. The latter option was chosen using cross validation and averaging over the <span class="math inline">\(R^2\)</span> obtained in the different folds. Metrics with an interpretation similar to <span class="math inline">\(R^2\)</span> have been defined for logistic regressions or survival analysis <span class="citation">(Choodari-Oskooei, Royston, and Parmar <a href="#ref-choodari2012simulation">2012</a>)</span>. In the case of regressions with several variables <span class="math inline">\(x_i\)</span>, it is possible to <strong>decompose <span class="math inline">\(R^2\)</span> into different components associated with each of the variables</strong>. This decomposition is carried out here by averaging over orderings according to the method proposed by <span class="citation">Lindeman (<a href="#ref-lindeman1980introduction">1980</a>)</span> and applied in R code by <span class="citation">Grömping and others (<a href="#ref-gromping2006relative">2006</a>)</span>. The precise formulas are detailed in appendix <a href="about-statistics.html#appendix-decomp">C.1.1</a>.</p>
<p>Here is an example of schematic reasoning that can be carried out with <span class="math inline">\(R^2\)</span> about the two models in Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:model-simulation">7.2</a>. We will <strong>denote <span class="math inline">\(R^2_{X_1+X_2}\)</span> the <span class="math inline">\(R^2\)</span> corresponding to the linear model <span class="math inline">\(Growth=\beta_0+\beta_1 X_1+\beta_2 X_2\)</span></strong> (written more compactly <span class="math inline">\(Y\sim X_1+X_2\)</span>, by analogy to its implementation in <em>R</em>). Using only the outputs of the models to predict <em>Growth</em>, explained variations are <span class="math inline">\(R^2_{O_{non-linear}}=0.455\)</span> and <span class="math inline">\(R^2_{O_{linear}}=0.379\)</span>. The mechanistic models are thus correctly defined since the mechanistic output partly recover the biological read-out. However, the inputs of the model also have an important predictive value since <span class="math inline">\(R^2_{N_1+N_2+N_3}=0.514\)</span>. How can we understand the relationship between these values? First, the model including the <span class="math inline">\(N_i\)</span> inputs and the output <span class="math inline">\(O_{linear}\)</span> as regressors show identical performances with</p>
<p><span class="math display">\[R^2_{N_1+N_2+N_3+O_{linear}}=0.514=R^2_{N_1+N_2+N_3},\]</span></p>
which means that <span class="math inline">\(O_{linear}\)</span> has no incremental value compared to a linear combination of the inputs. This is perfectly obvious from a statistical point of view since the two models are equivalent:
<span class="math display">\[\begin{equation*}
\begin{aligned}
Growth &amp;=\beta_0+\beta_1 N_1+\beta_2 N_2 + \beta_3 N_3+\beta_4 O_{linear}\\
       &amp;=\beta_0+(\beta_1+2\beta_4) N_1+\beta_2 N_2 + (\beta_3 - \beta_4) N_3
\end{aligned}
\end{equation*}\]</span>
<p>The purpose of this example is to explicitly underline what is done implicitly in the study of certain mechanistic models. The complexity of the described mechanisms sometimes hides more or less linear combinations of inputs that may make it possible to obtain meaningful biomarkers but without incremental value by construction. On the other hand, <span class="math inline">\(O_{non-linear}\)</span> has allowed to extract an emergent information which improves the global prediction when combined linearly with the inputs: <span class="math display">\[R^2_{N_1+N_2+N_3+O_{non-linear}}=0.586&gt;R^2_{N_1+N_2+N_3}.\]</span></p>
<p>We can go further in understanding by breaking down the <span class="math inline">\(R^2\)</span>. In Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:R2-artificial">7.3</a>A and B (left columns), <span class="math inline">\(R^2\)</span> of the inputs' models (<span class="math inline">\(Growth = \beta_0+\beta_1 N_1+\beta_2 N_2 + \beta_3 N_3\)</span>) are decomposed to show that <span class="math inline">\(N_1\)</span> and <span class="math inline">\(N_3\)</span> contribute most to the prediction in a linear model. By using the same strategies for decomposing the <span class="math inline">\(R_2\)</span> and calculating the incremental <span class="math inline">\(R^2\)</span>, it is also possible to <strong>decompose the <span class="math inline">\(R^2\)</span> of <span class="math inline">\(O_{linear}\)</span> and <span class="math inline">\(O_{non-linear}\)</span> according to its origin: its component <span class="math inline">\(N_1\)</span> (<span class="math inline">\(0.22\)</span> in Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:R2-artificial">7.3</a>A) is the proportion of <span class="math inline">\(R^2\)</span> that is also explained by <span class="math inline">\(N_1\)</span>, so it can be interpreted as being the part of the value of <span class="math inline">\(N_1\)</span> captured by <span class="math inline">\(O\)</span></strong>. In the non-linear case, we can see in the decomposition that <span class="math inline">\(O_{non-linear}\)</span> has an additional created component (<span class="math inline">\(0.07\)</span>), it is the non-linear component that is not shared with any of the inputs.</p>
<div class="figure" style="text-align: center"><span id="fig:R2-artificial"></span>
<img src="07-Evaluation_files/figure-html/R2-artificial-1.png" alt="Decomposition of \(R^2\) for inputs and output of example models. (A) Results for the non-linear model inputs and output \(O_{non-linear}\) as defined in Figure 7.2: the left column represents the \(R^2\) decomposition of model \(Growth = \beta_0+\beta_1 N_1+\beta_2 N_2 + \beta_3 N_3\) and the right column the \(R^2\) decomposition of \(Growth = \beta_0+ \beta_4 O_{non-linear}\). (B) Same with the linear model and the corresponding \(O_{linear}\). For both (A) and (B), colors represent the origin of \(R^2\) contribution according to the decomposition. In particular, for right colums (model\(Y\sim O\)), the red share represent the proportion of the \(R^2\) of the regressor \(O\) that does not come linearly from the inputs, and therefore its emerging part. The horizontal reference line corresponds to the maximal \(R^2\) obtained from the model \(Growth = \beta_0+\beta_1 N_1+\beta_2 N_2 + \beta_3 N_3 + \beta_4O\)" width="90%" />
<p class="caption">
Figure 7.3: <strong>Decomposition of <span class="math inline">\(R^2\)</span> for inputs and output of example models.</strong> (A) Results for the non-linear model inputs and output <span class="math inline">\(O_{non-linear}\)</span> as defined in Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:model-simulation">7.2</a>: the left column represents the <span class="math inline">\(R^2\)</span> decomposition of model <span class="math inline">\(Growth = \beta_0+\beta_1 N_1+\beta_2 N_2 + \beta_3 N_3\)</span> and the right column the <span class="math inline">\(R^2\)</span> decomposition of <span class="math inline">\(Growth = \beta_0+ \beta_4 O_{non-linear}\)</span>. (B) Same with the linear model and the corresponding <span class="math inline">\(O_{linear}\)</span>. For both (A) and (B), colors represent the origin of <span class="math inline">\(R^2\)</span> contribution according to the decomposition. In particular, for right colums (model<span class="math inline">\(Y\sim O\)</span>), the red share represent the proportion of the <span class="math inline">\(R^2\)</span> of the regressor <span class="math inline">\(O\)</span> that does not come linearly from the inputs, and therefore its emerging part. The horizontal reference line corresponds to the maximal <span class="math inline">\(R^2\)</span> obtained from the model <span class="math inline">\(Growth = \beta_0+\beta_1 N_1+\beta_2 N_2 + \beta_3 N_3 + \beta_4O\)</span>
</p>
</div>

<p>In conclusion, if these two models generate meaningful outputs that are correlated with the biological read-out <em>Growth</em>, the analysis of their information processing classifies them into two different categories outlined in the previous sub-section. The linear model summarizes some of the information present in the inputs, without creating any. It can be likened to a relevant dimensionality reduction. The output of the non-linear model also fails to avoid some information losses, but at the same time it extracts new non-linear information. Thus, in combination with the inputs, it provides incremental value measured by the increase in total <span class="math inline">\(R^2\)</span>. Note that <span class="math inline">\(R^2\)</span> is used here as one tool among others to illustrate the reflection on personalized mechanistic models as information processing tools. The point to remember is not technical but rather methodological: these <strong>mechanistic models using on omics data cannot be evaluated for themselves but must be evaluated in comparison with the data they use in order to better explain the way they process information</strong>. Following this rationale of model selection, other tools such as the Akaike Information Criterion (AIC) have been proposed and could allow to quantify if the reduction of dimension carried out by the models (from many omics inputs to one mechanistic output) allows a more parsimonious description of biology than the direct use of inputs <span class="citation">(Kirk, Thorne, and Stumpf <a href="#ref-kirk2013model">2013</a>)</span>.</p>
</div>
</div>
<div id="reanalysis" class="section level2">
<h2><span class="header-section-number">7.3</span> Reanalysis of mechanistic models of cancer</h2>
<p>Using the tools presented above, it is possible to deepen the analysis of some mechanistic models already presented in this thesis.</p>
<div id="ode-model-of-jnk-pathway-by-fey2015signaling" class="section level3">
<h3><span class="header-section-number">7.3.1</span> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></h3>
<p>One of the first applications of personalized mechanistic models to cancer is the one proposed by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span> regarding JNK pathways in patients with neuroblastomas. This work has been described in section <a href="mechanistic-cancer.html#prognostic">3.4.2</a> and is recalled in Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:fey2">7.4</a>. The evaluation of the mechanistic models in the original paper was performed by assessing the clinical value of the inputs (RNA levels of ZAK, MKK4, MKK7, JNK and AKT genes) and outputs (<span class="math inline">\(H\)</span>, <span class="math inline">\(A\)</span> and <span class="math inline">\(K_{50}\)</span>) separately by comparing them with survival data. The outputs were binarized to optimize the separation between the curves in a log-rank test. In this section we propose to <strong>quantify the value of the output in relation to those of the inputs</strong>, leaving the output continuous, using the tools described in the previous section. In the context of survival data, different measures called <span class="math inline">\(R^2\)</span> by analogy have been described in the literature. The one used thereafter was described by <span class="citation">Royston and Sauerbrei (<a href="#ref-royston2004new">2004</a>)</span>, its detailed definition is given in Appendix <a href="about-statistics.html#appendix-r2surv">C.1.2</a> and its properties have been studied and validated in previous studies using simulated data <span class="citation">(Choodari-Oskooei, Royston, and Parmar <a href="#ref-choodari2012simulation">2012</a>)</span>. <span class="math inline">\(R^2\)</span> is not the preferred tool for survival data and is only used here to allow a qualitative description in line with the previous ones without introducing new tools. A formal and rigorous analysis should favour the tools presented at the beginning of the chapter.</p>
<div class="figure" style="text-align: center"><span id="fig:fey2"></span>
<img src="fig/fey.png" alt="Mechanistic modeling of JNK pathway and survival of neuroblastoma patients, as described by Fey et al. (2015). (A) Schematic representation (as a process description (Le Novere 2015)) for the ODE model of JNK pathway. (B) Response curve (phosphorylated JNK) as a function of the input stimulus (Stress) and characterization of the corresponding sigmoidal function with maximum amplitude \(A\), Hill exponent \(H\) and activation threshold \(K_{50}\). (C) Survival curves for neuroblastoma patients based on binarized \(A\), \(K_{50}\) and \(H\); binarization thresholds having been defined based on optimization screening on calibration cohort." width="90%" />
<p class="caption">
Figure 7.4: <strong>Mechanistic modeling of JNK pathway and survival of neuroblastoma patients, as described by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span>.</strong> (A) Schematic representation (as a process description <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>) for the ODE model of JNK pathway. (B) Response curve (phosphorylated JNK) as a function of the input stimulus (Stress) and characterization of the corresponding sigmoidal function with maximum amplitude <span class="math inline">\(A\)</span>, Hill exponent <span class="math inline">\(H\)</span> and activation threshold <span class="math inline">\(K_{50}\)</span>. (C) Survival curves for neuroblastoma patients based on binarized <span class="math inline">\(A\)</span>, <span class="math inline">\(K_{50}\)</span> and <span class="math inline">\(H\)</span>; binarization thresholds having been defined based on optimization screening on calibration cohort.
</p>
</div>

<p>Thus, the <span class="math inline">\(R^2\)</span> of the output <span class="math inline">\(H\)</span> is <span class="math inline">\(0.39\)</span> while that of the combined inputs is <span class="math inline">\(0.60\)</span>. We can see from the decompositions that <span class="math inline">\(H\)</span> derives most of its the value from ZAK, MKK4 and AKT (Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:R2-Fey">7.5</a>A, right column), which were already the largest contributors in the combined evaluation of the inputs (Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:R2-Fey">7.5</a>A, left column). However, <span class="math inline">\(H\)</span> also includes an emerging non-linear share (<span class="math inline">\(R^2=0.08\)</span>) that was not explained by the linear combination of inputs. Thus, incorporating <span class="math inline">\(H\)</span> with the inputs in a survival prediction model does indeed allow to observe an added value with a global <span class="math inline">\(R^2\)</span> of <span class="math inline">\(0.68\)</span>. In addition, the authors in the original study stressed the importance of positive feedback from JNK to MKK7 (Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:fey2">7.4</a>A). In its absence, we find that the value of <span class="math inline">\(H\)</span> is almost reduced to zero, since not only its non-linear part (Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:R2-Fey">7.5</a>, red share), but also its parts derived from inputs, disappear. Analyzing the other outputs of the model (<span class="math inline">\(A\)</span> and <span class="math inline">\(K_{50}\)</span>) reveals similar but less dramatic trends underlining the importance of this feedback which allows the model to capture a clinically relevant behaviour, assimilated by the authors to the capacity of cells to trigger apoptosis in case of stress. In the case of this model, the analyses provide a better understanding of how the model works with respect to survival prediction: <strong>the outputs partly summarize clinical information already present in the inputs but also reveal relevant emerging information</strong>.</p>
<div class="figure" style="text-align: center"><span id="fig:R2-Fey"></span>
<img src="07-Evaluation_files/figure-html/R2-Fey-1.png" alt="Decomposition of \(R^2\) for inputs and output for ODE model in Fey et al. (2015). (A) Results for the Fey model inputs and output \(H\) as defined in Figure 7.4A and B. (B) Same using the model without positive feedback between JNK and MKK7. Colors represent the origine of \(R^2\) contribution. In particular, for right colums (model\(Y\sim H\)), the red share represent the proportion of the \(R^2\) of the output \(H\) that does not come linearly from the inputs, and therefore its emerging part." width="90%" />
<p class="caption">
Figure 7.5: <strong>Decomposition of <span class="math inline">\(R^2\)</span> for inputs and output for ODE model in <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span>.</strong> (A) Results for the Fey model inputs and output <span class="math inline">\(H\)</span> as defined in Figure <a href="information-flows-in-mechanistic-models-of-cancer.html#fig:fey2">7.4</a>A and B. (B) Same using the model without positive feedback between JNK and MKK7. Colors represent the origine of <span class="math inline">\(R^2\)</span> contribution. In particular, for right colums (model<span class="math inline">\(Y\sim H\)</span>), the red share represent the proportion of the <span class="math inline">\(R^2\)</span> of the output <span class="math inline">\(H\)</span> that does not come linearly from the inputs, and therefore its emerging part.
</p>
</div>

</div>
<div id="personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers" class="section level3">
<h3><span class="header-section-number">7.3.2</span> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</h3>
<p>Similarly, it is appropriate to assess the relevance of the personalized logical models presented so far. Unlike the models of the previous sub-section, however, they integrate a much larger number of variables and the decomposition of <span class="math inline">\(R^2\)</span> is no longer accessible, because of its computational cost, which increases exponentially with the number of variables. If we focus on the example best suited to these models, that of BRAF inhibition sensistivity, we can however reformulate the question more simply. Given that the most important predictor of the answer is the status of the BRAF mutation itself, <strong>do the personalized models allow us to do better or provide additional information?</strong> In the case of CRISPR data, the <span class="math inline">\(R^2\)</span> of BRAF alone is <span class="math inline">\(0.75\)</span>, the <span class="math inline">\(R^2\)</span> of the personalized scores from the models is <span class="math inline">\(0.73\)</span>, while the combination of the two increases the <span class="math inline">\(R^2\)</span> to <span class="math inline">\(0.83\)</span>. In the absence of a precise decomposition, this gain can come either from the contribution of the other variables used in the model (the RNA levels of CRAF for example) or from the emergence of non-linear effects. In both cases, these figures are another way of expressing the remarks in section <a href="logical-drug-chap.html#diff-BRAF">6.2.4.1</a>: thanks to the integration of other data and their organization in a framework based on literature knowledge, the model provides a more precise and complete vision of the response mechanisms. As positive as it is, this increase in <span class="math inline">\(R^2\)</span> remains modest, illustrating that the <strong>main interest of these models is not necessarily a pure gain in predictive performance. Rather, it lies in their explanatory capacity and in their ability to support the investigation of mechanisms</strong> such as in section <a href="logical-drug-chap.html#diff-BRAF">6.2.4.1</a>. In a complementary way, one could imagine extending these analyses to other nodes of the model and not only to its output in order to dissect even more precisely the information processing within the model.</p>

<div class="conclubox">
<h4 id="summary-7" class="unnumbered">Summary</h4>
<p>The use of certain mechanistic models of cancer for clinical purposes reinforces the importance of a rigorous evaluation of their performance. One of the main recommendations is to consider the outputs of these models as classical biomarkers, ignoring initially the mechanistic architecture that generated them. Numerous methods and metrics then exist to quantify the clinical information they provide, whether prognostic or predictive. In a complementary way, it is then insightful to examine the particularities of this kind of biomarker which does not come out of nowhere but often results from the knowledge-based transformation of biological data, including possibly other biomarkers. The prognostic or predictive value of mechanistic models must then be interpreted against the value of the biological data used to build or customize the models.</p>
</div>
<p></p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-altman1994dangers">
<p>Altman, Douglas G, Berthold Lausen, Willi Sauerbrei, and Martin Schumacher. 1994. “Dangers of Using ‘Optimal’ Cutpoints in the Evaluation of Prognostic Factors.” <em>JNCI: Journal of the National Cancer Institute</em> 86 (11). Oxford University Press: 829–35.</p>
</div>
<div id="ref-blanche2019c">
<p>Blanche, Paul, Michael W Kattan, and Thomas A Gerds. 2019. “The c-Index Is Not Proper for the Evaluation of-Year Predicted Risks.” <em>Biostatistics</em> 20 (2). Oxford University Press: 347–57.</p>
</div>
<div id="ref-buyse2006validation">
<p>Buyse, Marc, Sherene Loi, Laura Van’t Veer, Giuseppe Viale, Mauro Delorenzi, Annuska M Glas, Mahasti Saghatchian d’Assignies, et al. 2006. “Validation and Clinical Utility of a 70-Gene Prognostic Signature for Women with Node-Negative Breast Cancer.” <em>Journal of the National Cancer Institute</em> 98 (17). Oxford University Press: 1183–92.</p>
</div>
<div id="ref-ching2018cox">
<p>Ching, Travers, Xun Zhu, and Lana X Garmire. 2018. “Cox-Nnet: An Artificial Neural Network Method for Prognosis Prediction of High-Throughput Omics Data.” <em>PLoS Computational Biology</em> 14 (4). Public Library of Science: e1006076.</p>
</div>
<div id="ref-choodari2012simulation">
<p>Choodari-Oskooei, Babak, Patrick Royston, and Mahesh KB Parmar. 2012. “A Simulation Study of Predictive Ability Measures in a Survival Model I: Explained Variation Measures.” <em>Statistics in Medicine</em> 31 (23). Wiley Online Library: 2627–43.</p>
</div>
<div id="ref-cox1972regression">
<p>Cox, David R. 1972. “Regression Models and Life-Tables.” <em>Journal of the Royal Statistical Society: Series B (Methodological)</em> 34 (2). Wiley Online Library: 187–202.</p>
</div>
<div id="ref-fey2015signaling">
<p>Fey, Dirk, Melinda Halasz, Daniel Dreidax, Sean P Kennedy, Jordan F Hastings, Nora Rauch, Amaya Garcia Munoz, et al. 2015. “Signaling Pathway Models as Biomarkers: Patient-Specific Simulations of Jnk Activity Predict the Survival of Neuroblastoma Patients.” <em>Sci. Signal.</em> 8 (408). American Association for the Advancement of Science: ra130–ra130.</p>
</div>
<div id="ref-gail2018breast">
<p>Gail, Mitchell H, and Ruth M Pfeiffer. 2018. “Breast Cancer Risk Model Requirements for Counseling, Prevention, and Screening.” <em>JNCI: Journal of the National Cancer Institute</em> 110 (9). Oxford University Press: 994–1002.</p>
</div>
<div id="ref-gromping2006relative">
<p>Grömping, Ulrike, and others. 2006. “Relative Importance for Linear Regression in R: The Package Relaimpo.” <em>Journal of Statistical Software</em> 17 (1): 1–27.</p>
</div>
<div id="ref-heagerty2005survival">
<p>Heagerty, Patrick J, and Yingye Zheng. 2005. “Survival Model Predictive Accuracy and Roc Curves.” <em>Biometrics</em> 61 (1). Wiley Online Library: 92–105.</p>
</div>
<div id="ref-heagerty2000time">
<p>Heagerty, Patrick J, Thomas Lumley, and Margaret S Pepe. 2000. “Time-Dependent Roc Curves for Censored Survival Data and a Diagnostic Marker.” <em>Biometrics</em> 56 (2). Wiley Online Library: 337–44.</p>
</div>
<div id="ref-hilden2014note">
<p>Hilden, Jørgen, and Thomas A Gerds. 2014. “A Note on the Evaluation of Novel Biomarkers: Do Not Rely on Integrated Discrimination Improvement and Net Reclassification Index.” <em>Statistics in Medicine</em> 33 (19). Wiley Online Library: 3405–14.</p>
</div>
<div id="ref-hilsenbeck1992so">
<p>Hilsenbeck, Susan Galloway, Gary M Clark, and William L McGuire. 1992. “Why Do so Many Prognostic Factors Fail to Pan Out?” <em>Breast Cancer Research and Treatment</em> 22 (3). Springer: 197–206.</p>
</div>
<div id="ref-janes2014approach">
<p>Janes, Holly, Marshall D Brown, Ying Huang, and Margaret S Pepe. 2014. “An Approach to Evaluating and Comparing Biomarkers for Patient Treatment Selection.” <em>The International Journal of Biostatistics</em> 10 (1). De Gruyter: 99–121.</p>
</div>
<div id="ref-kirk2013model">
<p>Kirk, Paul, Thomas Thorne, and Michael PH Stumpf. 2013. “Model Selection in Systems and Synthetic Biology.” <em>Current Opinion in Biotechnology</em> 24 (4). Elsevier: 767–74.</p>
</div>
<div id="ref-le2015quantitative">
<p>Le Novere, Nicolas. 2015. “Quantitative and Logic Modelling of Molecular and Gene Networks.” <em>Nature Reviews Genetics</em> 16 (3). Nature Publishing Group: 146–58.</p>
</div>
<div id="ref-lindeman1980introduction">
<p>Lindeman, Richard Harold. 1980. <em>Introduction to Bivariate and Multivariate Analysis</em>. Glenview, Ill: Scott, Foresman.</p>
</div>
<div id="ref-mboup2020evaluating">
<p>Mboup, Bassirou, Paul Blanche, and Aurélien Latouche. 2020. “On Evaluating How Well a Biomarker Can Predict Treatment Response with Survival Data.” <em>Pharmaceutical Statistics</em>. Wiley Online Library.</p>
</div>
<div id="ref-mcshane2005reporting">
<p>McShane, Lisa M, Douglas G Altman, Willi Sauerbrei, Sheila E Taube, Massimo Gion, and Gary M Clark. 2005. “Reporting Recommendations for Tumor Marker Prognostic Studies (Remark).” <em>Journal of the National Cancer Institute</em> 97 (16). Oxford University Press: 1180–4.</p>
</div>
<div id="ref-mourragui2020precise">
<p>Mourragui, Soufiane, Marco Loog, Daniel J Vis, Kat Moore, Anna Gonzalez Manjon, Mark A van de Wiel, Marcel JT Reinders, and Lodewyk FA Wessels. 2020. “PRECISE+ Predicts Drug Response in Patients by Non-Linear Subspace-Based Transfer from Cell Lines and Pdx Models.” <em>BioRxiv</em>. Cold Spring Harbor Laboratory.</p>
</div>
<div id="ref-pencina2008evaluating">
<p>Pencina, Michael J, Ralph B D’Agostino Sr, Ralph B D’Agostino Jr, and Ramachandran S Vasan. 2008. “Evaluating the Added Predictive Ability of a New Marker: From Area Under the Roc Curve to Reclassification and Beyond.” <em>Statistics in Medicine</em> 27 (2). Wiley Online Library: 157–72.</p>
</div>
<div id="ref-pepe2008integrating">
<p>Pepe, Margaret S, Ziding Feng, Ying Huang, Gary Longton, Ross Prentice, Ian M Thompson, and Yingye Zheng. 2008. “Integrating the Predictiveness of a Marker with Its Performance as a Classifier.” <em>American Journal of Epidemiology</em> 167 (3). Oxford University Press: 362–68.</p>
</div>
<div id="ref-pepe2014net">
<p>Pepe, Margaret S, Holly Janes, and Christopher I Li. 2014. “Net Risk Reclassification P Values: Valid or Misleading?” <em>Journal of the National Cancer Institute</em> 106 (4). Oxford University Press US: dju041.</p>
</div>
<div id="ref-royston2004new">
<p>Royston, Patrick, and Willi Sauerbrei. 2004. “A New Measure of Prognostic Separation in Survival Data.” <em>Statistics in Medicine</em> 23 (5). Wiley Online Library: 723–48.</p>
</div>
<div id="ref-sauerbrei2018reporting">
<p>Sauerbrei, Willi, Sheila E Taube, Lisa M McShane, Margaret M Cavenagh, and Douglas G Altman. 2018. “Reporting Recommendations for Tumor Marker Prognostic Studies (Remark): An Abridged Explanation and Elaboration.” <em>JNCI: Journal of the National Cancer Institute</em> 110 (8). Oxford University Press: 803–11.</p>
</div>
<div id="ref-schemper2003predictive">
<p>Schemper, Michael. 2003. “Predictive Accuracy and Explained Variation.” <em>Statistics in Medicine</em> 22 (14). Wiley Online Library: 2299–2308.</p>
</div>
<div id="ref-soreide2009receiver">
<p>Søreide, Kjetil. 2009. “Receiver-Operating Characteristic Curve Analysis in Diagnostic, Prognostic and Predictive Biomarker Research.” <em>Journal of Clinical Pathology</em> 62 (1). BMJ Publishing Group: 1–5.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="8">
<li id="fn8"><p>An unpublished note by Frank Harrell details and illustrates the possibilities and limitations of <span class="math inline">\(R^2\)</span> for this type of analysis (<a href="https://www.fharrell.com/post/addvalue/">link</a>)<a href="information-flows-in-mechanistic-models-of-cancer.html#fnref8">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="logical-drug-chap.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="causal-chap.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
