<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 Logical modeling principles and data integration | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 Logical modeling principles and data integration | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Logical modeling principles and data integration | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="mechanistic-cancer.html"/>
<link rel="next" href="personalization-chap.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="remerciements.html"><a href="remerciements.html"><i class="fa fa-check"></i>Remerciements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-family"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
<li class="chapter" data-level="3.4.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-interventions-and-causality"><i class="fa fa-check"></i><b>3.4.4</b> Mechanistic models, interventions and causality</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-tools-list"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#continuous-time-markov-processes"><i class="fa fa-check"></i><b>4.2.1</b> Continuous-time Markov processes</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.2</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.3</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.4</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-chap.html"><a href="personalization-chap.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-chap.html"><a href="personalization-chap.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-chap.html"><a href="personalization-chap.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-chap.html"><a href="personalization-chap.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-chap.html"><a href="personalization-chap.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-chap.html"><a href="personalization-chap.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-chap.html"><a href="personalization-chap.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-chap.html"><a href="personalization-chap.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-chap.html"><a href="personalization-chap.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#ML-comp"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="logical-drug-chap.html"><a href="logical-drug-chap.html#part2-limits"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a><ul>
<li class="chapter" data-level="7.1.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-framework-and-general-principles"><i class="fa fa-check"></i><b>7.1.1</b> Evaluation framework and general principles</a></li>
<li class="chapter" data-level="7.1.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#some-frequent-problems-and-recommended-statistical-tools"><i class="fa fa-check"></i><b>7.1.2</b> Some frequent problems and recommended statistical tools</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artificial-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artificial examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="causal-chap.html"><a href="causal-chap.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="causal-chap.html"><a href="causal-chap.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="causal-chap.html"><a href="causal-chap.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="causal-chap.html"><a href="causal-chap.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-basic"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="causal-chap.html"><a href="causal-chap.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="causal-chap.html"><a href="causal-chap.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#part3-limits"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a><ul>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#one-path-among-others"><i class="fa fa-check"></i>One path among others</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html#in-other-directions"><i class="fa fa-check"></i>In other directions</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#r2-and-beyond"><i class="fa fa-check"></i><b>C.1</b> <span class="math inline">\(R^2\)</span> and beyond</a><ul>
<li class="chapter" data-level="C.1.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.1.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-r2surv"><i class="fa fa-check"></i><b>C.1.2</b> <span class="math inline">\(R^2\)</span> for survival data</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="D" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html"><i class="fa fa-check"></i><b>D</b> Résumé détaillé</a><ul>
<li class="chapter" data-level="D.1" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#modélisation-et-cancer"><i class="fa fa-check"></i><b>D.1</b> Modélisation et cancer</a></li>
<li class="chapter" data-level="D.2" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#des-modèles-logiques-personnalisés-de-cancer"><i class="fa fa-check"></i><b>D.2</b> Des modèles logiques personnalisés de cancer</a></li>
<li class="chapter" data-level="D.3" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#quantification-statistique-de-limpact-clinique-des-modèles"><i class="fa fa-check"></i><b>D.3</b> Quantification statistique de l'impact clinique des modèles</a></li>
<li class="chapter" data-level="D.4" data-path="résumé-détaillé.html"><a href="résumé-détaillé.html#conclusion-1"><i class="fa fa-check"></i><b>D.4</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="logical-modeling-principles-and-data-integration" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> Logical modeling principles and data integration</h1>
<blockquote>
<p><em>Je suis l’halluciné de la forêt des Nombres.<br />
Ils me fixent, avec leurs yeux de leurs problèmes ;<br />
Ils sont, pour éternellement rester : les mêmes.<br />
Primordiaux et définis,<br />
Ils tiennent le monde entre leurs infinis ;<br />
Ils expliquent le fond et l’essence des choses,<br />
Puisqu’à travers les temps planent leurs causes.</em><br />
Émile Verhaeren (Les nombres, in Les Flambeaux noirs, 1891)</p>
</blockquote>
<p>Another way of ordering the diversity of mechanistic models presented above is to consider their relationship to biological data. Those that make little use of these data are essentially theoretical scope models that describe the general functioning of signaling pathways and associated systems <span class="citation">(Calzone et al. <a href="#ref-calzone2010mathematical">2010</a>)</span>. Other models propose more quantitative models but require much more data, either from databases or experimental data generated for this purpose in order to fit the parameters. In the latter case, the necessary data is usually perturbation data: how does my system react to this or that inhibition or activation? For a single cell line this already corresponds to a large amount of data <span class="citation">(Razzaq et al. <a href="#ref-razzaq2018computational">2018</a>)</span>. And if we want to extend these approaches to many cell lines, the amount of data becomes massive <span class="citation">(Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>. For patient-specific models, access to this perturbation data is even more difficult.</p>
<p>Between theoretical models that are not very demanding in terms of data but not very applicable clinically and models with a clinical focus but very demanding in terms of data, an intermediate alternative is missing. <strong>Can patient-specific mechanistic models be developed that would provide qualitative clinical interpretation with a small amount of data, accessible even in patients?</strong> In this part, composed of three chapters, a middle way will be described to answer positively to this question. This methodology will be based on a historically qualitative mathematical formalism already presented in the previous chapter under the name of logical modeling. Logical modeling in general will be detailed in this chapter before describing an original personalized approach in the next two chapters.</p>

<div class="summarybox">
<h4 id="scientific-content-1" class="unnumbered">Scientific content</h4>
<p>This chapter presents the theoretical bases of logical modeling and the tools used thereafter. The content is taken entirely from the literature and the personal presentation that is made is the one that was used to introduce article <span class="citation">Béal et al. (<a href="#ref-beal2019personalization">2019</a>)</span> and to organize review <span class="citation">Béal, Rémy, and Calzone (<a href="#ref-beal2020modelisation">2020</a>)</span>.</p>
</div>
<p></p>
<div id="logical-modeling-paradigms-for-qualitative-description" class="section level2">
<h2><span class="header-section-number">4.1</span> Logical modeling paradigms for qualitative description</h2>
<p>Mathematical models serve as tools to answer a biological question in a formal way, to detect blind spots and thus better understand a system, to organize, into a consensual and compact manner, information dispersed in different articles. In the light of this definition, logical formalism may seem one of the closest to natural language in that it <strong>can translate quite directly the statements present in the literature</strong> such as &quot;protein A activates protein B&quot; or &quot;the expression of gene C requires the joint presence of factors D and E&quot;. Indeed, shortly after the first descriptions of control circuits by <span class="citation">Jacob and Monod (<a href="#ref-jacob1961genetic">1961</a>)</span>, the interest of logical models to describe biological systems was put forward by <span class="citation">Kauffman (<a href="#ref-kauffman1969homeostasis">1969</a>)</span> and <span class="citation">Thomas (<a href="#ref-thomas1973boolean">1973</a>)</span>. Since then, studies have multiplied <span class="citation">(Thomas and d’Ari <a href="#ref-thomas1990biological">1990</a>)</span>, varying the fields of biological applications and also the mathematical and computational implementations <span class="citation">(Naldi, Hernandez, Levy, et al. <a href="#ref-naldi2018colomoto">2018</a>)</span>. The two subsections below summarize the characteristics common to most of the logical formalisms, before detailing the implementation chosen in this thesis in section <a href="logical-modeling-principles-and-data-integration.html#maboss-section">4.2</a>. A review of the use of data in logic models will finally be proposed in section <a href="logical-modeling-principles-and-data-integration.html#logical-data-section">4.3</a>.</p>
<div id="regulatory-graph-and-logical-rules" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Regulatory graph and logical rules</h3>
<p>A logical model is based on a network called <strong>regulatory graph</strong> (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical">4.1</a>), where each <strong>node</strong> represents a component (e.g. genes, proteins, complexes, phenotypes or processes), and is associated with discrete levels of activity (<span class="math inline">\(0\)</span>, <span class="math inline">\(1\)</span>, or more when justified). The use of a discrete formalism in molecular network modeling relies on the highly non-linear nature of regulation, and thus on the existence of a regulatory threshold. Assuming that each variable represents a level of expression, it will take the value <span class="math inline">\(0\)</span> if the level of expression of the entity is below the regulation threshold, i.e., insufficient to carry out the regulation; and the value <span class="math inline">\(1\)</span> if it is above the threshold and regulation is possible. In other words, the control threshold discretizes the state space, here the expression levels. It is therefore possible to distinguish several thresholds for the same variable, corresponding to distinct controls that do not take place at the same expression levels. The variable is then multivalued. This extension greatly enriches the formalism, because it allows to distinguish situations that are qualitatively different and that would be confused with Boolean variables. In the continuation of this thesis, we will consider by default that the activity levels are binary, <span class="math inline">\(0\)</span> corresponding to an inactive entity and <span class="math inline">\(1\)</span> to an active entity. The <strong>edges</strong> of this regulatory graph correspond to influences, either positive or negative, which illustrate the possible interactions between two entities (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical">4.1</a>). Positive edges can represent the formation of active complexes, mediation of synthesis, catalysis, etc. and they will be later depicted as green arrows (<span class="math inline">\(\leftarrow\)</span>). Negative edges on the other hand can represent inhibition of synthesis, degradation, inhibiting (de)phosphorylation, etc. and they will be depicted as red turnstiles (<span class="math inline">\(\vdash\)</span>).</p>
<div class="figure" style="text-align: center"><span id="fig:logical"></span>
<img src="fig/logical.png" alt="A simple example of a logical model. Regulatory graph on the left with positive (green) and negative regulations (red); a set of possible corresponding logical rules on the right." width="50%" />
<p class="caption">
Figure 4.1: <strong>A simple example of a logical model.</strong> Regulatory graph on the left with positive (green) and negative regulations (red); a set of possible corresponding logical rules on the right.
</p>
</div>

<p>Then, each node of the regulatory graph has a corresponding Boolean variable associated to it. The variables can take two values: <span class="math inline">\(0\)</span> for absent or inactive (OFF), and <span class="math inline">\(1\)</span> for present or active (ON). These variables change their value according to a logical rule assigned to them. The state of a variable will thus depend on its <strong>logical rule</strong>, which is based on logical statements, i.e., on a function of the node regulators linked with logical connectors AND (<span class="math inline">\(\&amp;\)</span>), OR (<span class="math inline">\(|\)</span>) and NOT (<span class="math inline">\(!\)</span>). These operators can account for what is known about the biology behind these edges. If two input nodes are needed for the activation of the target node, they will be linked by an AND gate; to list different means of activation of a node, an OR gate will be used; and negative influences will rely on NOT gates. The rules corresponding to the toy model in Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical">4.1</a> could be interpreted literally like this: A is activated to 1 if B is active; B is updated to 1 in the absence of A and the presence/activity of C; C is an input of the model and therefore not regulated. It can be noted that the logical rules cannot be deduced only from the regulatory graph, which is less precise and ambiguous. One could thus imagine that B is activated if C is, OR if A is not, thus changing the behavior of the model.</p>
</div>
<div id="state-transition-graph-and-updates" class="section level3">
<h3><span class="header-section-number">4.1.2</span> State transition graph and updates</h3>
<p>In a Boolean framework, the variables associated to each node can take two values, either <span class="math inline">\(0\)</span> or <span class="math inline">\(1\)</span>. We define a model state as a vector of all node states. All the possible transitions from any model state to another are dependent on the set of logical rules that define the model. These transitions can be viewed into a graph called a <strong>state transition graph</strong> (STG), where nodes are model states and edges are the transitions from one model state to another. STG nodes will be later depicted with rounded squares instead of circles in order to emphasize the difference with regulatory graphs. That way, trajectories from an initial condition to all the final states can be determined. In a model with <span class="math inline">\(n\)</span> nodes, the STG can contain up to <span class="math inline">\(2^n\)</span> model state nodes; thus, if <span class="math inline">\(n\)</span> is too big, the construction and the visualization of the graph become difficult.</p>
<div class="figure" style="text-align: center"><span id="fig:synchronous"></span>
<img src="fig/synchronous.png" alt="State transition graph and synchronous updates. Stable state (A) and limit cycle (B) attractors obtained for the example logical model with synchronous updates (all possible updates simultaneously). Figures above/below STG edges correspond to the number of nodes updated in each transition." width="90%" />
<p class="caption">
Figure 4.2: <strong>State transition graph and synchronous updates.</strong> Stable state (A) and limit cycle (B) attractors obtained for the example logical model with synchronous updates (all possible updates simultaneously). Figures above/below STG edges correspond to the number of nodes updated in each transition.
</p>
</div>

<p>Based the simple logical model of Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical">4.1</a> it is nevertheless possible to represent the STG comprehensively. The idea for this is to start from a state of the system and track the successive states defined by the logical rules and the corresponding updates. The first strategy to construct this STG is to change simultaneously at each time step all the variables that can be changed (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:synchronous">4.2</a>). This method is referred to as a <strong>synchronous updating strategy</strong>. In the second method, referred to as a <strong>asynchronous updating strategy</strong>, variables are changed one at a time (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:asynchronous">4.3</a>) and therefore each state has as many successors as there are components whose state must be changed according to logical rules (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:asynchronous">4.3</a>A). Performing only one transition at a time implies having to choose this transition between the different possible options, which can be done according to pre-established rules or stochastically as explained in section <a href="logical-modeling-principles-and-data-integration.html#maboss-section">4.2</a>. The latter asynchronous method will be used exclusively in the work presented thereafter.</p>
<div class="figure" style="text-align: center"><span id="fig:asynchronous"></span>
<img src="fig/asynchronous.png" alt="State transition graph and asynchronous updates. Stable state (A) and limit cycle (B) attractors obtained for the example logical model with asynchronous updates (one update at a time). Figures above/below STG edges correspond to the number of nodes updated in each transition." width="90%" />
<p class="caption">
Figure 4.3: <strong>State transition graph and asynchronous updates.</strong> Stable state (A) and limit cycle (B) attractors obtained for the example logical model with asynchronous updates (one update at a time). Figures above/below STG edges correspond to the number of nodes updated in each transition.
</p>
</div>

<p>We then define attractors of the model as long-term asymptotic behaviors of the system. Two types of attractors are identified: stable states, when the system has reached a model state whose successor in the transition graph is the model state itself; and cyclic attractors, when trajectories in the transition graph lead to a group of model states that are cycling. For both synchronous and asynchronous updating strategies, the toy model shows the existence of <strong>two types of attractors: a stable steady state and a limit cycle</strong>, depending on the initial value of <span class="math inline">\(C\)</span>. There are two disconnected components of the STG for this example that correspond to the two possible values for the input <span class="math inline">\(C\)</span>. If <span class="math inline">\(C\)</span> is initially equal to 0 (inactive), then there exists only one stable state: <span class="math inline">\(A=B=C=0\)</span>. All the trajectories in the state transition graph lead to a single final model state. If <span class="math inline">\(C\)</span> is initially equal to 1, then the attractor is a limit cycle. The path in the STG cycles for any initial model state of this connected component. Note that for the asynchronous and synchronous graphs, the precise paths or limit cycles may differ. To conclude, it is important to emphasize and illustrate the characteristics of asynchronous updates in this toy example. In Figure <a href="logical-modeling-principles-and-data-integration.html#fig:asynchronous">4.3</a>A, the transition from the initial state (<span class="math inline">\(A=C=0;B=1\)</span>) suggests two distinct possibilities, so it is necessary to <strong>define additional rules or heuristics to choose between possible transitions</strong>. We will come back to this by specifying the logical modeling implementation chosen in this thesis in section <a href="logical-modeling-principles-and-data-integration.html#maboss-section">4.2</a>, in which a stochastic exploration of these alternatives is proposed.</p>
</div>
<div id="logical-tools-list" class="section level3">
<h3><span class="header-section-number">4.1.3</span> Tools for logical modeling</h3>
<p>Numerous tools have been developed to build logical models and study the dynamics of the systems under investigation, each with its own specificity. They allow, for example, to represent regulation networks; to edit, modify or infer logical rules; to identify stable states; to reduce models; to visualize graphs of synchronous or asynchronous transitions. Some also allow to integrate temporal data; to discretize expression data; to simulate the model stochastically or to integrate delays; to identify existing models, etc. Among them, we can cite GINsim <span class="citation">(Naldi, Hernandez, Abou-Jaoudé, et al. <a href="#ref-naldi2018logical">2018</a>)</span>, BoolNet <span class="citation">(Müssel, Hopfensitz, and Kestler <a href="#ref-mussel2010boolnet">2010</a>)</span>, pyBoolNet <span class="citation">(Klarner, Streck, and Siebert <a href="#ref-klarner2016pyboolnet">2016</a>)</span>, BooleanNet <span class="citation">(Albert et al. <a href="#ref-albert2008boolean">2008</a>)</span>, CellCollective <span class="citation">(Helikar et al. <a href="#ref-helikar2012cell">2012</a>)</span>, bioLQM <span class="citation">(Naldi <a href="#ref-naldi2018biolqm">2018</a>)</span>, MaBoSS <span class="citation">(Stoll et al. <a href="#ref-stoll2012continuous">2012</a>; Stoll et al. <a href="#ref-stoll2017maboss">2017</a>)</span>, PINT <span class="citation">(Paulevé <a href="#ref-Pint">2017</a>)</span>, CaspoTS <span class="citation">(Ostrowski et al. <a href="#ref-ostrowski2016boolean">2016</a>)</span>, or CellNOptR <span class="citation">(Terfve et al. <a href="#ref-terfve2012cellnoptr">2012</a>)</span>. The interaction between all these tools, their interoperability and complementarity are highlighted in the form of a notebook Jupyter <span class="citation">(Naldi, Hernandez, Levy, et al. <a href="#ref-naldi2018colomoto">2018</a>)</span>, and some of them are described in more details in section <a href="logical-modeling-principles-and-data-integration.html#logical-data-section">4.3</a>.</p>
</div>
</div>
<div id="maboss-section" class="section level2">
<h2><span class="header-section-number">4.2</span> The MaBoSS framework for logical modeling</h2>
<p>In the present study, all simulations have been performed with MaBoSS, a <strong>Ma</strong>rkovian <strong>Bo</strong>olean <strong>S</strong>tochastic <strong>S</strong>imulator whose design is summarized in Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a> and precisely described by <span class="citation">Stoll et al. (<a href="#ref-stoll2012continuous">2012</a>)</span> and <span class="citation">Stoll et al. (<a href="#ref-stoll2017maboss">2017</a>)</span>. This framework is based on an asynchronous update scheme combined with a continuous time feature obtained with Gillespie algorithm <span class="citation">(Gillespie <a href="#ref-gillespie1976general">1976</a>)</span>, allowing simulations to be continuous in time despite the discrete nature of logical modeling.</p>
<div class="figure" style="text-align: center"><span id="fig:maboss"></span>
<img src="fig/maboss.png" alt="Main principles of MaBoSS simulation framework and Gillespie algorithm. (A) A logical model with regulatory graph, logical rules and transition rates. (B) A subset of the corresponding state transition graph with two possible transitions in asynchronous update for a given initial state; the probability probability associated with each transition comes from Gillespie algorithm. (C) Schematic diagram of the Gillespie algorithm applied to the asynchronous transition depicted in (B); random selection of a specific transition and time by the algorithm from two uniform random variables. (D) Schematic representation of a logical model simulation with MaBoSS: average trajectory obtained from the mean of many individual stochastic trajectories, each resulting from the succession of transitions as represented in (C)." width="100%" />
<p class="caption">
Figure 4.4: <strong>Main principles of MaBoSS simulation framework and Gillespie algorithm.</strong> (A) A logical model with regulatory graph, logical rules and transition rates. (B) A subset of the corresponding state transition graph with two possible transitions in asynchronous update for a given initial state; the probability probability associated with each transition comes from Gillespie algorithm. (C) Schematic diagram of the Gillespie algorithm applied to the asynchronous transition depicted in (B); random selection of a specific transition and time by the algorithm from two uniform random variables. (D) Schematic representation of a logical model simulation with MaBoSS: average trajectory obtained from the mean of many individual stochastic trajectories, each resulting from the succession of transitions as represented in (C).
</p>
</div>

<div id="continuous-time-markov-processes" class="section level3">
<h3><span class="header-section-number">4.2.1</span> Continuous-time Markov processes</h3>
<p>The implementation of asynchronous updates proposed by the MaBoSS software is based on <strong>continuous time Markov processes applied on a Boolean state space</strong> and is therefore part of the large family of stochastic models of cancer outlined in section <a href="mechanistic-cancer.html#mechanistic-family">3.1</a>. The precise relations of this software with the generic results on Markovian processes, and the associated demonstrations, are available in the original publication by <span class="citation">Stoll et al. (<a href="#ref-stoll2012continuous">2012</a>)</span> and detailed in particular in the related supplementary document<a href="#fn4" class="footnoteRef" id="fnref4"><sup>4</sup></a>. The essence of the formalism, as set out in this article, is outlined below.</p>
<p>Before returning to the simplified example of Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>, it is possible to describe the general scheme of Markov processes and Gillespie algorithm for simulating logical models. We restrict ourselves here to a Boolean model (binary variables), composed of <span class="math inline">\(n\)</span> variables. The network state of the system is a vector <span class="math inline">\(S\)</span> of binary values such as <span class="math inline">\(S_i \in \{0, 1\}\)</span>, with <span class="math inline">\(i=1,...,n\)</span> and <span class="math inline">\(S_i\)</span> representing the state of the node <span class="math inline">\(i\)</span>. The network state space of all possible network states is called <span class="math inline">\(\Omega\)</span>. The evolution of the state of the system can be represented by a stochastic process <span class="math inline">\(s:t \rightarrow s(t)\)</span> defined on <span class="math inline">\(t \in I \subset \mathbb{R}\)</span> applied on the network state space. For each time <span class="math inline">\(t \in I \subset \mathbb{R}\)</span>, <span class="math inline">\(s(t)\)</span> represents a random variable applied on the network state space. Thus, <span class="math inline">\(P[s(t)=S]\in[0, 1], \forall S \in \Omega\)</span> and <span class="math inline">\(\sum_{S \in \Omega} P[s(t)=S] = 1\)</span>. This <strong>stochastic process verifies the Markov property which stipulates the absence of memory</strong>, <em>i.e.</em>, the conditional probability distribution of future states of the process depends only on the present state, not on the sequence of past states. The resulting stochastic process, called Markov process is defined by an initial condition <span class="math inline">\(P[s(0)=S], \forall S \in \Omega\)</span> and the conditional probabilities <span class="math inline">\(P[s(t)=S|s(t&#39;)=S&#39;], \forall S,S&#39; \in \Omega, \forall t,t&#39; \in I, t&#39;&lt;t\)</span>. In this work we will focus on continuous time Markov processes where it has been shown that all conditional probabilities are functions of transition rates <span class="math inline">\(\rho_{(S&#39; \rightarrow S)} (t) \in [0, \infty[\)</span> <span class="citation">(Van Kampen <a href="#ref-van2004stochastic">2004</a>)</span>. Only the case of <strong>time independent transition rates</strong> (and Markov processes) will be explored below.</p>
<p>It is then possible to re-state the description of the logical models in this formalism. One of the first representations of the model is the regulatory network with a set of directed arrows linking the <span class="math inline">\(n\)</span> nodes (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>A, left column). For each node <span class="math inline">\(i\)</span>, a logical rule <span class="math inline">\(L_i(S)\)</span> is defined based only on the nodes <span class="math inline">\(j\)</span> for which there exists an arrow from node <span class="math inline">\(j\)</span> to <span class="math inline">\(i\)</span>, as represented in Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>A middle column, <em>e.g.</em>, <span class="math inline">\(L_B = (\text{NOT}~A)~\text{AND}~(C)\)</span> with <span class="math inline">\(L_B\)</span> the logical rule of node <span class="math inline">\(B\)</span>. The notion of <strong>asynchronous transition</strong> can then be defined as a pair of network states <span class="math inline">\((S, S&#39;) \in \Omega\)</span>, written <span class="math inline">\((S \rightarrow S&#39;)\)</span> such that:</p>
<p><span class="math display">\[S&#39;_j=L_B(S) \text{ for a given } j\]</span> <span class="math display">\[S&#39;_i=S_i \text{ for } i \neq j\]</span></p>
<p>This means that during an asynchronous transition, only one node changes state, among those whose logical rule allows it. In Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>B, two possible asynchronous transitions from the same initial state are represented. Then, transition rates <span class="math inline">\(\rho_{(S \rightarrow S&#39;)}\)</span> are non-zero only if <span class="math inline">\(S\)</span> and <span class="math inline">\(S&#39;\)</span> differ by only one node. In that case, each Boolean logic <span class="math inline">\(L_i(S)\)</span> is replaced by two functions <span class="math inline">\(k_{0 \rightarrow 1 / 1 \rightarrow 0}^{i}(S) \in [0, \infty[\)</span>. The transition rates are defined as follows: if <span class="math inline">\(i\)</span> is the node that differs from <span class="math inline">\(S\)</span> and <span class="math inline">\(S&#39;\)</span>, then:</p>
<p><span class="math display">\[\rho_{(S \rightarrow S&#39;)}=k_{0 \rightarrow 1}^{i}(S) \text{ if } S_i=0\]</span> <span class="math display">\[\rho_{(S \rightarrow S&#39;)}=k_{1 \rightarrow 0}^{i}(S) \text{ if } S_i=1\]</span></p>
<p>Therefore, the <strong>continuous Markov process is completely defined by all these <span class="math inline">\(k^i\)</span> and an initial condition</strong>. The state transition graph can also be re-defined as a graph in <span class="math inline">\(\Omega\)</span>, with an edge between <span class="math inline">\(S\)</span> and <span class="math inline">\(S&#39;\)</span> if and only if <span class="math inline">\(\rho_{S \rightarrow S&#39;} &gt; 0\)</span>.</p>
</div>
<div id="gillespie-algorithm" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Gillespie algorithm</h3>
<p>Although the Markov process is completely defined, the cost of its theoretical resolution increases exponentially with the number of nodes since the transition matrix of the system is of size <span class="math inline">\(2^n \times 2^n\)</span>. Thus, although the exact resolution of these Markov processes from their master equation has been well described <span class="citation">(Van Kampen <a href="#ref-van2004stochastic">2004</a>)</span>, their computational cost quickly becomes too high. As an example, a recent method for the exact resolution of these stochastic processes applied to Boolean models is limited to models of about 20 nodes <span class="citation">(Koltai et al. <a href="#ref-koltai2020exact">2020</a>)</span>. One solution consists in <strong>sampling the probability space by simulating time trajectories in the state transition graph</strong>, which is what MaBoSS performs through Gillespie algorithm, also called kinetic Monte-Carlo. The principle of the algorithm is to compute a finite set of individual stochastic trajectories of the Markov process and to use them to infer probabilities for the given Markov process (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>D). For the sake of simplicity the remainder of this section will describe the simulation of a single individual stochastic trajectory, first with a focus on the formal description of the algorithm and then with a more qualitative explanation.</p>
<p>A stochastic trajectory <span class="math inline">\(\hat{S}(t)\)</span> is a function from a predefined time interval <span class="math inline">\([ 0, t_{max}]\)</span> to <span class="math inline">\(\Omega\)</span>. The iterative computation of the trajectory is obtained as follows:</p>
<ol style="list-style-type: decimal">
<li>From a state <span class="math inline">\(S\)</span> at <span class="math inline">\(t_0\)</span>, sum the rates of all possible transitions for leaving this state: <span class="math inline">\(\rho_{total}=\sum_{S&#39;}{\rho_{(S \rightarrow S&#39;)}}\)</span></li>
<li>Draw two uniform random numbers <span class="math inline">\(u,u&#39; \in [0,1]\)</span></li>
<li>Compute the transition time <span class="math inline">\(\delta t=-log(u)/\rho_{total}\)</span></li>
<li>Order the potential target states <span class="math inline">\(S&#39;^{(j)}\)</span> and their corresponding transition rates <span class="math inline">\(\rho^{(j)}=\rho_{(S \rightarrow S&#39;^{j})}\)</span></li>
<li>Choose the new state <span class="math inline">\(S&#39;=S&#39;^{(k)}\)</span> such that <span class="math inline">\(\sum_{j=0}^{k-1} \rho^{(j)} &lt; u&#39;\rho_{total} &lt; \sum_{j=0}^{k} \rho^{(j)}\)</span>, with <span class="math inline">\(\rho^{(0)}=0\)</span></li>
<li>Define the trajectory <span class="math inline">\(\hat{S}(t)\)</span> such as <span class="math inline">\(\hat{S}(t)=S\)</span> for <span class="math inline">\(t \in [t_0, t_0 + \delta t[\)</span> and <span class="math inline">\(\hat{S}(t_0 + \delta t) = S&#39;\)</span></li>
<li>Repeat iteratively from <span class="math inline">\(S&#39;\)</span> until <span class="math inline">\(t_{max}\)</span> is reached</li>
</ol>
<p>Thus, Gillespie algorithm provides a <strong>stochastic way to choose a specific transition among several possible ones</strong> and to infer a corresponding time for this transition. To achieve this, transition rates seen as qualitative activation or inactivation rates, must be specified for each node (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>A). They can be set either all to the same value by default, in the absence of any indication, or in various levels reflecting different orders of magnitude: post-translational modifications are quicker than transcriptions for instance. These transition rates are translated as <strong>transition probabilities</strong> in order to determine the actual transition (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>B). Indeed, the probability for each possible transition to be chosen for the next update is the ratio of its transition rate to the sum of rates of all possible transitions. Higher rates correspond to transitions that will take place with greater probability, or in other words more quickly. At each update, the Gillespie algorithm performs the procedure described above and depicted schematically in Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>C. Two uniform random variables <span class="math inline">\(u\)</span> and <span class="math inline">\(u&#39;\)</span> are drawn and used respectively to select the transition among the different possibilities (with <span class="math inline">\(u\)</span>) and to infer the corresponding time (with <span class="math inline">\(u&#39;\)</span>). Based on the described formula, time <span class="math inline">\(\delta t\)</span> follows an exponential law whose average is equal to the inverse of the sum of all possible transition rates (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>C). In the present work, except otherwise stated, all transition states will be initially assigned to 1.</p>
</div>
<div id="a-stochastic-exploration-of-model-behaviours" class="section level3">
<h3><span class="header-section-number">4.2.3</span> A stochastic exploration of model behaviours</h3>
<p>Since MaBoSS computes stochastic trajectories, it is relevant to compute several trajectories in order to get an insight of the average behavior by generating a <strong>population of stochastic trajectories</strong> over the asynchronous state transition graph (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:maboss">4.4</a>D). The aggregation of stochastic trajectories can also be interpreted as a <strong>description of a heterogeneous population</strong>. Thus, in all the examples in next chapters, all simulations have consisted of thousands of computed trajectories. The larger the model, the larger the space of possibilities and the more trajectories are required to explore it. Since several trajectories are simulated, initial values of each node can be defined with a continuous value between 0 and 1 representing the probability for the node to be defined to 1 for each new trajectory. For instance, a node with a <span class="math inline">\(0.6\)</span> initial condition will be set to <span class="math inline">\(1\)</span> in <span class="math inline">\(60\%\)</span> of simulated trajectories and to <span class="math inline">\(0\)</span> in <span class="math inline">\(40\%\)</span> of the cases.</p>
<p>In the present work, we will focus on the <em>asymptotic</em> state of these simulations instead of transient dynamics and we will call <strong>node scores</strong> the asymptotic agregated score obtained by averaging all trajectories at a given final time point. Indeed, asymptotic states are more closely related to logical model attractors than transient dynamics and are therefore less dependent on updating stochasticity and more biologically meaningful <span class="citation">(Huang, Ernberg, and Kauffman <a href="#ref-huang2009cancer">2009</a>)</span>. Note that the simulation time should be chosen carefully to ensure that the asymptotic state is achieved, and the term <em>final state</em> may be considered as safer. All in all this modeling framework is at the intersection of logical modeling and continuous dynamic modeling. If the definition of time remains rather abstract and difficult to interpret experimentally, the stochastic exploration of trajectories makes it possible to refine the purely binary interpretation of the variables. Finally, it is important to point out that while the following chapters will focus on the MaBoSS tool, it is not exclusive but on the contrary very complementary to the other logical model analysis tools mentioned in section <a href="logical-modeling-principles-and-data-integration.html#logical-tools-list">4.1.3</a>, or even to the methods specific to Markov processes in particular. The choice of the tools will be made according to the biological questions and the associated analyses: estimation of attractors or basins of attraction, exact resolution, study of transient dynamics, etc.</p>
</div>
<div id="from-theoretical-models-to-data-models" class="section level3">
<h3><span class="header-section-number">4.2.4</span> From theoretical models to data models?</h3>
<p>To sum up, logical formalism makes it possible to design qualitative models that reflect <em>a priori</em> knowledge of the phenomena being studied. Thus, they allow answering questions for which there is little quantitative information on the precise mechanisms involved in a disease, for instance a lack of data related to the expression of genes, the quantity of key proteins or the speed of certain processes. Logical models can confirm that a network is a good illustration of the underlying biological question. However, the construction of the model from the literature often results in a generic consensus network that cannot explain the differences observed between certain patients with different molecular profiles. In order to propose a patient-specific mechanistic approach, it seems crucial to <strong>use the biological data</strong> available. How is this possible in a formalism that is by definition quite abstract?</p>
</div>
</div>
<div id="logical-data-section" class="section level2">
<h2><span class="header-section-number">4.3</span> Data integration and semi-quantitative logical modeling</h2>
<p>The higher level of abstraction of the logical formalism sometimes makes the necessary back and forth between theoretical modeling and experimental or clinical data less easy. However, many theoretical approaches have been developed over the years to enable this dialogue at all stages, from the construction to the validation of a logical model, as summarized in Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical-data">4.5</a>. This section summarizes some of these approaches to show <strong>how the use of biological data enriches logical models and brings them closer to clinical applications</strong> in precision medicine. The purpose of this presentation is also to better contextualize the original approach presented in the following chapter. It should be noted that the methods presented below are all applicable to logical models, and illustrated with such examples where possible. However, some methods are not specific to this formalism and can be applied to other modeling frameworks.</p>
<div class="figure" style="text-align: center"><span id="fig:logical-data"></span>
<img src="fig/logical-data.png" alt="Data integration in logical modeling. The main types of data used are shown on the left; the essential steps of the logical modeling are shown linearly on the right." width="90%" />
<p class="caption">
Figure 4.5: <strong>Data integration in logical modeling.</strong> The main types of data used are shown on the left; the essential steps of the logical modeling are shown linearly on the right.
</p>
</div>

<div id="build-the-regulatory-graph" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Build the regulatory graph</h3>
<p>Faced with a biological question (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical-data">4.5</a>, first step), it is crucial to identify the main actors in the process in order to <strong>define the outline of the model</strong> (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical-data">4.5</a>, second step). A first approach relies on the existing scientific literature on the topic: which biological species and which interactions have been identified as relevant to my problem? In a more automatic way, it is possible to extract information from different databases in order to establish an initial list of biological entities and interactions associated with a biological phenomenon or even a gene of interest <span class="citation">(Kanehisa et al. <a href="#ref-kanehisa2012kegg">2012</a>; Perfetto et al. <a href="#ref-perfetto2016signor">2016</a>)</span>. As an example, starting from the study of E2F1 gene as the hub of many regulatory mechanisms, <span class="citation">Khan et al. (<a href="#ref-khan2017unraveling">2017</a>)</span> have reconstructed a dense network of interactions in the vicinity of E2F1, which will be used for the construction of their subsequent model. The main difficulty here is to choose and select the relevant biological information adapted to the context of the model to be created, depending for example on the type of cancer studied or the desired level of precision.</p>
<p>But if the literature can be considered as processed data, it is also possible to use directly experimental data related to the problem under study. Key actors of biological processes identified by statistical analysis, such as differentially expressed genes or the most frequently mutated genes in a patient cohort, are selected and used as a starting point for the construction of the model <span class="citation">(Remy et al. <a href="#ref-remy2015modeling">2015</a>)</span>. More comprehensive approaches can use differential analysis tools on signaling pathways, rather than individual genes, to choose the relevant processes to include by contrasting different groups of patients based on their grades, metstatic status, resistance to treatments etc. <span class="citation">(Martignetti et al. <a href="#ref-martignetti2016roma">2016</a>; Montagud et al. <a href="#ref-montagud2017conceptual">2017</a>)</span>. Similarly, the study of regulatory networks involving transcription factors may justify the use of ChIP-seq data to identify possible new transcriptional regulations not previously listed <span class="citation">(Collombet et al. <a href="#ref-collombet2017logical">2017</a>)</span>.</p>
<p>Once the main actors have been identified, it is necessary to <strong>infer the links</strong> between them (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical-data">4.5</a>, third and fourth step). However, starting from a list of genes and proteins of interest, how can we ensure that the regulatory relationships are complete and relevant? While a careful reading of the literature can provide locally interesting information, the use of omics data is also a resource that can be broken down into different levels of precision. The major interest of these methods, assuming that the data are adequate and sufficiently massive, is to be able to extract information as large as the dataset, potentially on hundreds of entities, and above all specific to the object of study: a cancer subtype or a particular cell line can thus generate their own interaction network <span class="citation">(Lefebvre et al. <a href="#ref-lefebvre2010human">2010</a>)</span>. Inference methods extract biological knowledge hidden in large databases, summarize it and represent it via networks. Many methods construct coexpression networks, which are non-oriented graphs, with different metrics and methods <span class="citation">(Margolin et al. <a href="#ref-margolin2006aracne">2006</a>; Vert, Qiu, and Noble <a href="#ref-vert2007new">2007</a>)</span>. Other approaches seek to infer causal relations between components, allowing the reconstruction of directed graphs where the links between entities are oriented, and sometimes even signed as activating (positive) or inhibiting (negative) regulations. These methods often make use of time series <span class="citation">(Hill et al. <a href="#ref-hill2016inferring">2016</a>)</span> or perturbation data <span class="citation">(Meinshausen et al. <a href="#ref-meinshausen2016methods">2016</a>)</span>, but also more recently from observational data <span class="citation">(Verny et al. <a href="#ref-verny2017learning">2017</a>)</span>. The information extracted from the data is then directly readable in the form of activity flows as described in the SBGN standards <span class="citation">(Novère et al. <a href="#ref-novere2009systems">2009</a>)</span>, thus providing a representation adapted to the construction of qualitative models and <em>a fortiori</em> of logical models <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>. Closer to the objective of defining logical models, certain methods allow the study and inference of co-regulation expressed with logical operators <span class="citation">(Elati et al. <a href="#ref-elati2007licorn">2007</a>)</span>, thus facilitating the passage from the definition of an interaction network to the construction of a true logical model.</p>
</div>
<div id="define-the-logical-rules" class="section level3">
<h3><span class="header-section-number">4.3.2</span> Define the logical rules</h3>
<p>Precision must then be taken further by defining the logical rules that complete the network (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical-data">4.5</a>, fifth step). The first source of aggregated data to define logical rules is the scientific literature. The modeler looks for the state of knowledge on a given regulatory mechanism and translates it into a <strong>local logical rule</strong>, according to the desired level of precision. For example, it has been observed that the protein kinase AKT can stabilize the oncogene MDM2 by phosphorylation, which leads to the degradation of p53 by forming a complex with it: this example can be translated by a simple inhibition relationship of AKT on p53 if this level of precision is considered sufficient or else intermediate species such as MDM2 can be used <span class="citation">(Cohen et al. <a href="#ref-cohen2015mathematical">2015</a>)</span>. Then, the effect of inhibition must be defined: can MDM2 alone inhibit p53 or does the presence of other activators outweigh this effect? This kind of considerations allows to define the logical combinations between the different inputs of a network node. In some cases, experimental data can be used to answer such questions: is a single activator sufficient or is the presence of all activators necessary? Which of the activator or inhibitor prevails in the case of simultaneous presence? While this information is often found in the literature, one should generate one's own experimental data to ensure an answer tailored to the study context, using a variety of experimental molecular biology techniques. For example, in order to elucidate the relationship between Foxo1 and Cebpa in a model of differentiation of myeloid and lymphoid cells, <span class="citation">Collombet et al. (<a href="#ref-collombet2017logical">2017</a>)</span> first established the physical relationship between these species by ChIP-seq before determining the nature of this relationship using an ectopic expression experiment of Foxo1 in macrophage cells.</p>
<p>Other, more global approaches have been developed in recent years, driven by the influx of data from high-throughput sequencing techniques. Based on this rich and complex data, it has become possible to <strong>infer entire logical models, with precisely defined rules and interactions</strong> <span class="citation">(Ostrowski et al. <a href="#ref-ostrowski2016boolean">2016</a>)</span>. The algorithms CellNOpt <span class="citation">(Terfve et al. <a href="#ref-terfve2012cellnoptr">2012</a>)</span> and caspo <span class="citation">(Videla et al. <a href="#ref-videla2017caspo">2017</a>)</span> provide two examples of these approaches, and more recently the SCNS tool described a graphical interface to infer logical models from single cell data <span class="citation">(Woodhouse et al. <a href="#ref-woodhouse2018scns">2018</a>)</span>. This model-inference goes beyond simpler structure-inference by defining the logical rules, but it is generally based on a predefined topological structure to which time series or perturbation data are added. These data provide access to the response dynamics of a system. By questioning the way the system reacts, these data are therefore richer than a snapshot and thus facilitate the transition from correlation to causality, and thus the inference of logical rules. In practice, the use of proteomic or phospho-proteomic data is often recommended because these data account for the activity of the protein and are in fact the closest to the cellular response <span class="citation">(Ostrowski et al. <a href="#ref-ostrowski2016boolean">2016</a>; Terfve et al. <a href="#ref-terfve2012cellnoptr">2012</a>; Terfve et al. <a href="#ref-terfve2015largescale">2015</a>)</span>. In spite of the richness of this type of data, model inference is sometimes still an under-determined problem that can lead to a large number of models with different logical rules equally compatible with the data. In such situations, it is then a matter of choosing the model on the basis of biological relevance criteria or of accepting to use families of models, or ensemble models, instead of limiting oneself to a single model <span class="citation">(Videla et al. <a href="#ref-videla2017caspo">2017</a>)</span>. In all cases, constructing logical rules directly from data specific to the problem can make it possible to obtain logical rules that are also specific to the context or the system under study <span class="citation">(Saez-Rodriguez, Alexopoulos, et al. <a href="#ref-saezrodriguez2011comparing">2011</a><a href="#ref-saezrodriguez2011comparing">b</a>)</span>. For example, the inference of logical models specific to one or some cancer cell lines is a powerful tool to study their particularities <span class="citation">(Razzaq et al. <a href="#ref-razzaq2018computational">2018</a>)</span>.</p>
</div>
<div id="validate-the-model" class="section level3">
<h3><span class="header-section-number">4.3.3</span> Validate the model</h3>
<p>Finally, the data can be used to validate the <strong>biological or clinical relevance of the models</strong> (Figure <a href="logical-modeling-principles-and-data-integration.html#fig:logical-data">4.5</a>, sixth step). Compared to a system of differential equations, logical modeling has the particularity of being more abstract and therefore less directly reliable to an experimental reality for its validation. A system of differential equations can be compared to the chemical kinetics of the biological system under study. Compared to continuous formalisms, the dynamics of logic model simulation is more difficult to take into account but it is possible to verify it qualitatively, for example by validating the cyclic nature of activation trajectories for a model simulating the cell cycle <span class="citation">(Fauré et al. <a href="#ref-faure2006dynamical">2006</a>)</span> or cellular decisions as a function of the activation signal <span class="citation">(Calzone et al. <a href="#ref-calzone2010mathematical">2010</a>)</span>. A second, more frequent approach consists in looking at the model's steady states and associating them with physiological conditions <span class="citation">(Weinstein et al. <a href="#ref-weinstein2017network">2017</a>; Cohen et al. <a href="#ref-cohen2015mathematical">2015</a>)</span>. A third strategy focuses on the asymptotic state reached during the stochastic simulation of the model(s), a state representing a mixture of the different steady states according to the probability that the model has of reaching them. It is also possible, in some model checking frameworks, to study the ability of models to reach certains states, interpreted as cellular types, from given initial conditions <span class="citation">(Abou-Jaoudé et al. <a href="#ref-abou2015model">2015</a>)</span>.</p>
<p>In many models, to facilitate the analysis, <strong>nodes representing phenotypes have been added as &quot;read-out&quot; of the activity of certain entities</strong>. Thus, if a model includes a node named <em>Proliferation</em>, it will then be simpler to draw interpretations from the simulations performed with the model that will be linked to experimental observations of tumor growth or cell proliferation <span class="citation">(Grieco et al. <a href="#ref-grieco2013integrative">2013</a>; Steinway et al. <a href="#ref-steinway2015combinatorial">2015</a>)</span>. To validate these models, the activity of phenotypes when forcing some node activity to <span class="math inline">\(0\)</span> or <span class="math inline">\(1\)</span> is compared with the results of gene mutations reported in experiments carried out on mice or cell lines <span class="citation">(Fauré et al. <a href="#ref-faure2006dynamical">2006</a>; Cohen et al. <a href="#ref-cohen2015mathematical">2015</a>)</span>. Another similar method for validating the relevance of a logical model is based on the analysis of the effects of different therapeutic molecules. The mechanistic nature of logical modeling makes it possible to simulate the effect of these molecules, at least for targeted treatments with known mechanisms of action. It is then possible to simulate the effect of an inhibitory molecule by forcing the activity of its target to 0 and to compare with data <span class="citation">(Zañudo, Scaltriti, and Albert <a href="#ref-zanudo2017network">2017</a>; Iorio et al. <a href="#ref-iorio2016landscape">2016</a>; Knijnenburg et al. <a href="#ref-knijnenburg2016logic">2016</a>)</span>.</p>
<p>Beyond validation, some studies have predicted <strong>new therapeutic targets</strong> based on logical models, for instance by pointing out weaknesses in the topology of a regulatory system <span class="citation">(Sahin et al. <a href="#ref-sahin2009modeling">2009</a>)</span>. Taking advantage of the versatility of the formalism to study combinations of therapeutic molecules, logical modeling has also proved fruitful in predicting the best therapeutic combinations and their synergies, in the context of gastric cancers for example <span class="citation">(Flobak et al. <a href="#ref-flobak2015discovery">2015</a>)</span>. Experimental confirmation of the predictions resulting from the modeling is then the ultimate stage in the validation of a logical model, completing the fruitful round trip between models and data.</p>

<div class="conclubox">
<h4 id="summary-4" class="unnumbered">Summary</h4>
<p>Logical models represent a qualitative formalism where biological entities are represented by discrete entities linked together by logical rules. The evolution of the biological system thus modeled can be described as a Markov process, as in the MaBoSS software that stochastically explores the space of possibilities of the model thus defined through a Gillespie algorithm. This approach allows both to avoid the exponential computational cost of exact resolutions and to translate the stochastic diversity of heterogeneous cell populations. Finally, although discrete and theoretical in nature, logical models allow the use of quantitative experimental data, in order to support or validate their biological relevance.</p>
</div>
<p></p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-abou2015model">
<p>Abou-Jaoudé, Wassim, Pedro T Monteiro, Aurélien Naldi, Maximilien Grandclaudon, Vassili Soumelis, Claudine Chaouiya, and Denis Thieffry. 2015. “Model Checking to Assess T-Helper Cell Plasticity.” <em>Frontiers in Bioengineering and Biotechnology</em> 2. Frontiers: 86.</p>
</div>
<div id="ref-albert2008boolean">
<p>Albert, István, Juilee Thakar, Song Li, Ranran Zhang, and Réka Albert. 2008. “Boolean Network Simulations for Life Scientists.” <em>Source Code for Biology and Medicine</em> 3 (1). BioMed Central: 16.</p>
</div>
<div id="ref-beal2019personalization">
<p>Béal, Jonas, Arnau Montagud, Pauline Traynard, Emmanuel Barillot, and Laurence Calzone. 2019. “Personalization of Logical Models With Multi-Omics Data Allows Clinical Stratification of Patients.” <em>Frontiers in Physiology</em>. Frontiers Media S.A. doi:<a href="https://doi.org/10.3389/fphys.2018.01965">10.3389/fphys.2018.01965</a>.</p>
</div>
<div id="ref-beal2020modelisation">
<p>Béal, Jonas, Elizabeth Rémy, and Laurence Calzone. 2020. “Modélisation Logique et Données Omiques : De La Construction Des Modèles à La Médecine Personnalisée.” In <em>Approche Symbolique de La Modélisation et de L’analyse Des Systèmes Biologiques</em>, edited by Elisabeth Rémy and Cédric Lhoussaine. ISTE.</p>
</div>
<div id="ref-calzone2010mathematical">
<p>Calzone, Laurence, Laurent Tournier, Simon Fourquet, Denis Thieffry, Boris Zhivotovsky, Emmanuel Barillot, and Andrei Zinovyev. 2010. “Mathematical Modelling of Cell-Fate Decision in Response to Death Receptor Engagement.” <em>PLoS Computational Biology</em> 6 (3). Public Library of Science.</p>
</div>
<div id="ref-cohen2015mathematical">
<p>Cohen, David P. A., Loredana Martignetti, Sylvie Robine, Emmanuel Barillot, Andrei Zinovyev, and Laurence Calzone. 2015. “Mathematical Modelling of Molecular Pathways Enabling Tumour Cell Invasion and Migration.” <em>PLOS Computational Biology</em> 11 (11): e1004571. doi:<a href="https://doi.org/10.1371/journal.pcbi.1004571">10.1371/journal.pcbi.1004571</a>.</p>
</div>
<div id="ref-collombet2017logical">
<p>Collombet, Samuel, Chris van Oevelen, Jose Luis Sardina Ortega, Wassim Abou-Jaoude, Bruno Di Stefano, Morgane Thomas-Chollier, Thomas Graf, and Denis Thieffry. 2017. “Logical Modeling of Lymphoid and Myeloid Cell Specification and Transdifferentiation.” <em>Proceedings of the National Academy of Sciences</em> 114 (23): 5792–9. doi:<a href="https://doi.org/10.1073/pnas.1610622114">10.1073/pnas.1610622114</a>.</p>
</div>
<div id="ref-elati2007licorn">
<p>Elati, Mohamed, Pierre Neuvial, Monique Bolotin-Fukuhara, Emmanuel Barillot, François Radvanyi, and Céline Rouveirol. 2007. “LICORN: Learning Cooperative Regulation Networks from Gene Expression Data.” <em>Bioinformatics</em> 23 (18): 2407–14. doi:<a href="https://doi.org/10.1093/bioinformatics/btm352">10.1093/bioinformatics/btm352</a>.</p>
</div>
<div id="ref-faure2006dynamical">
<p>Fauré, Adrien, Aurélien Naldi, Claudine Chaouiya, and Denis Thieffry. 2006. “Dynamical Analysis of a Generic Boolean Model for the Control of the Mammalian Cell Cycle.” <em>Bioinformatics</em> 22 (14). Oxford University Press: e124–e131.</p>
</div>
<div id="ref-flobak2015discovery">
<p>Flobak, Åsmund, Anaïs Baudot, Elisabeth Remy, Liv Thommesen, Denis Thieffry, Martin Kuiper, and Astrid Lægreid. 2015. “Discovery of Drug Synergies in Gastric Cancer Cells Predicted by Logical Modeling.” <em>PLoS Computational Biology</em> 11 (8). Public Library of Science.</p>
</div>
<div id="ref-frohlich2018efficient">
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6). Elsevier: 567–79.</p>
</div>
<div id="ref-gillespie1976general">
<p>Gillespie, Daniel T. 1976. “A General Method for Numerically Simulating the Stochastic Time Evolution of Coupled Chemical Reactions.” <em>Journal of Computational Physics</em> 22 (4): 403–34. doi:<a href="https://doi.org/10.1016/0021-9991(76)90041-3">10.1016/0021-9991(76)90041-3</a>.</p>
</div>
<div id="ref-grieco2013integrative">
<p>Grieco, Luca, Laurence Calzone, Isabelle Bernard-Pierrot, François Radvanyi, Brigitte Kahn-Perles, and Denis Thieffry. 2013. “Integrative Modelling of the Influence of Mapk Network on Cancer Cell Fate Decision.” <em>PLoS Computational Biology</em> 9 (10). Public Library of Science: e1003286.</p>
</div>
<div id="ref-helikar2012cell">
<p>Helikar, Tomáš, Bryan Kowal, Sean McClenathan, Mitchell Bruckner, Thaine Rowley, Alex Madrahimov, Ben Wicks, Manish Shrestha, Kahani Limbu, and Jim A Rogers. 2012. “The Cell Collective: Toward an Open and Collaborative Approach to Systems Biology.” <em>BMC Systems Biology</em> 6 (1). BioMed Central: 96.</p>
</div>
<div id="ref-hill2016inferring">
<p>Hill, Steven M, Laura M Heiser, Thomas Cokelaer, Michael Unger, Nicole K Nesser, Daniel E Carlin, Yang Zhang, et al. 2016. “Inferring Causal Molecular Networks: Empirical Assessment Through a Community-Based Effort.” <em>Nature Methods</em> 13 (4). Nature Publishing Group: 310.</p>
</div>
<div id="ref-huang2009cancer">
<p>Huang, Sui, Ingemar Ernberg, and Stuart Kauffman. 2009. “Cancer Attractors: A Systems View of Tumors from a Gene Network Dynamics and Developmental Perspective” 20 (7). Elsevier: 869–76.</p>
</div>
<div id="ref-iorio2016landscape">
<p>Iorio, Francesco, Theo A Knijnenburg, Daniel J Vis, Graham R Bignell, Michael P Menden, Michael Schubert, Nanne Aben, et al. 2016. “A Landscape of Pharmacogenomic Interactions in Cancer.” <em>Cell</em> 166 (3). Elsevier: 740–54.</p>
</div>
<div id="ref-jacob1961genetic">
<p>Jacob, François, and Jacques Monod. 1961. “Genetic Regulatory Mechanisms in the Synthesis of Proteins.” <em>Journal of Molecular Biology</em> 3 (3). Elsevier: 318–56.</p>
</div>
<div id="ref-kanehisa2012kegg">
<p>Kanehisa, Minoru, Susumu Goto, Yoko Sato, Miho Furumichi, and Mao Tanabe. 2012. “KEGG for Integration and Interpretation of Large-Scale Molecular Data Sets.” <em>Nucleic Acids Research</em> 40 (D1). Oxford University Press: D109–D114.</p>
</div>
<div id="ref-kauffman1969homeostasis">
<p>Kauffman, Stuart. 1969. “Homeostasis and Differentiation in Random Genetic Control Networks.” <em>Nature</em> 224 (5215). Springer: 177–78.</p>
</div>
<div id="ref-khan2017unraveling">
<p>Khan, Faiz M, Stephan Marquardt, Shailendra K Gupta, Susanne Knoll, Ulf Schmitz, Alf Spitschak, David Engelmann, Julio Vera, Olaf Wolkenhauer, and Brigitte M Pützer. 2017. “Unraveling a Tumor Type-Specific Regulatory Core Underlying E2f1-Mediated Epithelial-Mesenchymal Transition to Predict Receptor Protein Signatures.” <em>Nature Communications</em> 8 (1). Nature Publishing Group: 1–15.</p>
</div>
<div id="ref-klarner2016pyboolnet">
<p>Klarner, Hannes, Adam Streck, and Heike Siebert. 2016. “PyBoolNet: A Python Package for the Generation, Analysis and Visualization of Boolean Networks.” <em>Bioinformatics</em> 33 (5). Oxford University Press: 770–72.</p>
</div>
<div id="ref-knijnenburg2016logic">
<p>Knijnenburg, Theo A, Gunnar W Klau, Francesco Iorio, Mathew J Garnett, Ultan McDermott, Ilya Shmulevich, and Lodewyk FA Wessels. 2016. “Logic Models to Predict Continuous Outputs Based on Binary Inputs with an Application to Personalized Cancer Therapy.” <em>Scientific Reports</em> 6 (1). Nature Publishing Group: 1–14.</p>
</div>
<div id="ref-koltai2020exact">
<p>Koltai, Mihály, Vincent Noel, Andrei Zinovyev, Laurence Calzone, and Emmanuel Barillot. 2020. “Exact Solving and Sensitivity Analysis of Stochastic Continuous Time Boolean Models.” <em>BMC Bioinformatics</em> 21 (1). Springer: 1–22.</p>
</div>
<div id="ref-le2015quantitative">
<p>Le Novere, Nicolas. 2015. “Quantitative and Logic Modelling of Molecular and Gene Networks.” <em>Nature Reviews Genetics</em> 16 (3). Nature Publishing Group: 146–58.</p>
</div>
<div id="ref-lefebvre2010human">
<p>Lefebvre, Celine, Presha Rajbhandari, Mariano J. Alvarez, Pradeep Bandaru, Wei Keat Lim, Mai Sato, Kai Wang, et al. 2010. “A Human B-Cell Interactome Identifies MYB and FOXM1 as Master Regulators of Proliferation in Germinal Centers.” <em>Molecular Systems Biology</em> 6 (June): 377. doi:<a href="https://doi.org/10.1038/msb.2010.31">10.1038/msb.2010.31</a>.</p>
</div>
<div id="ref-margolin2006aracne">
<p>Margolin, Adam A., Ilya Nemenman, Katia Basso, Chris Wiggins, Gustavo Stolovitzky, Riccardo Dalla Favera, and Andrea Califano. 2006. “ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context.” <em>BMC Bioinformatics</em> 7 (1): S7. doi:<a href="https://doi.org/10.1186/1471-2105-7-S1-S7">10.1186/1471-2105-7-S1-S7</a>.</p>
</div>
<div id="ref-martignetti2016roma">
<p>Martignetti, Loredana, Laurence Calzone, Eric Bonnet, Emmanuel Barillot, and Andrei Zinovyev. 2016. “ROMA: Representation and Quantification of Module Activity from Target Expression Data.” <em>Frontiers in Genetics</em> 7. Frontiers: 18.</p>
</div>
<div id="ref-meinshausen2016methods">
<p>Meinshausen, Nicolai, Alain Hauser, Joris M Mooij, Jonas Peters, Philip Versteeg, and Peter Bühlmann. 2016. “Methods for Causal Inference from Gene Perturbation Experiments and Validation.” <em>Proceedings of the National Academy of Sciences</em> 113 (27). National Acad Sciences: 7361–8.</p>
</div>
<div id="ref-montagud2017conceptual">
<p>Montagud, Arnau, Pauline Traynard, Loredana Martignetti, Eric Bonnet, Emmanuel Barillot, Andrei Zinovyev, and Laurence Calzone. 2017. “Conceptual and Computational Framework for Logical Modelling of Biological Networks Deregulated in Diseases.” <em>Briefings in Bioinformatics</em>. doi:<a href="https://doi.org/10.1093/bib/bbx163">10.1093/bib/bbx163</a>.</p>
</div>
<div id="ref-mussel2010boolnet">
<p>Müssel, Christoph, Martin Hopfensitz, and Hans A Kestler. 2010. “BoolNet?an R Package for Generation, Reconstruction and Analysis of Boolean Networks.” <em>Bioinformatics</em> 26 (10). Oxford University Press: 1378–80.</p>
</div>
<div id="ref-naldi2018biolqm">
<p>Naldi, Aurélien. 2018. “BioLQM: A Java Toolkit for the Manipulation and Conversion of Logical Qualitative Models of Biological Networks.” <em>Frontiers in Physiology</em> 9. Frontiers: 1605.</p>
</div>
<div id="ref-naldi2018logical">
<p>Naldi, Aurélien, Céline Hernandez, Wassim Abou-Jaoudé, Pedro T Monteiro, Claudine Chaouiya, and Denis Thieffry. 2018. “Logical Modeling and Analysis of Cellular Regulatory Networks with Ginsim 3.0.” <em>Frontiers in Physiology</em> 9. Frontiers Media SA.</p>
</div>
<div id="ref-naldi2018colomoto">
<p>Naldi, Aurélien, Céline Hernandez, Nicolas Levy, Gautier Stoll, Pedro T Monteiro, Claudine Chaouiya, Tomáš Helikar, et al. 2018. “The Colomoto Interactive Notebook: Accessible and Reproducible Computational Analyses for Qualitative Biological Networks.” <em>Frontiers in Physiology</em> 9. Frontiers: 680.</p>
</div>
<div id="ref-novere2009systems">
<p>Novère, Nicolas Le, Michael Hucka, Huaiyu Mi, Stuart Moodie, Falk Schreiber, Anatoly Sorokin, Emek Demir, et al. 2009. “The Systems Biology Graphical Notation.” <em>Nature Biotechnology</em> 27 (8): 735–41. doi:<a href="https://doi.org/10.1038/nbt.1558">10.1038/nbt.1558</a>.</p>
</div>
<div id="ref-ostrowski2016boolean">
<p>Ostrowski, M., L. Paulevé, T. Schaub, A. Siegel, and C. Guziolowski. 2016. “Boolean Network Identification from Perturbation Time Series Data Combining Dynamics Abstraction and Logic Programming.” <em>Biosystems</em>, Selected papers from the Computational Methods in Systems Biology 2015 conference, 149 (November): 139–53. doi:<a href="https://doi.org/10.1016/j.biosystems.2016.07.009">10.1016/j.biosystems.2016.07.009</a>.</p>
</div>
<div id="ref-Pint">
<p>Paulevé, Loïc. 2017. “Pint: A Static Analyzer for Transient Dynamics of Qualitative Networks with IPython Interface.” In <em>CMSB 2017 - 15th conference on Computational Methods for Systems Biology</em>, 10545:370–16. Lecture Notes in Computer Science. Springer. doi:<a href="https://doi.org/10.1007/978-3-319-67471-1\_20">10.1007/978-3-319-67471-1\_20</a>.</p>
</div>
<div id="ref-perfetto2016signor">
<p>Perfetto, Livia, Leonardo Briganti, Alberto Calderone, Andrea Cerquone Perpetuini, Marta Iannuccelli, Francesca Langone, Luana Licata, et al. 2016. “SIGNOR: A Database of Causal Relationships Between Biological Entities.” <em>Nucleic Acids Research</em> 44 (D1). Oxford University Press: D548–D554.</p>
</div>
<div id="ref-razzaq2018computational">
<p>Razzaq, Misbah, Loïc Paulevé, Anne Siegel, Julio Saez-Rodriguez, Jérémie Bourdon, and Carito Guziolowski. 2018. “Computational Discovery of Dynamic Cell Line Specific Boolean Networks from Multiplex Time-Course Data.” <em>PLoS Computational Biology</em> 14 (10). Public Library of Science: e1006538.</p>
</div>
<div id="ref-remy2015modeling">
<p>Remy, Elisabeth, Sandra Rebouissou, Claudine Chaouiya, Andrei Zinovyev, François Radvanyi, and Laurence Calzone. 2015. “A Modeling Approach to Explain Mutually Exclusive and Co-Occurring Genetic Alterations in Bladder Tumorigenesis.” <em>Cancer Research</em> 75 (19). AACR: 4042–52.</p>
</div>
<div id="ref-saezrodriguez2011comparing">
<p>Saez-Rodriguez, Julio, Leonidas G. Alexopoulos, MingSheng Zhang, Melody K. Morris, Douglas A. Lauffenburger, and Peter K. Sorger. 2011b. “Comparing Signaling Networks Between Normal and Transformed Hepatocytes Using Discrete Logical Models.” <em>Cancer Research</em> 71 (16): 5400–5411. doi:<a href="https://doi.org/10.1158/0008-5472.CAN-10-4453">10.1158/0008-5472.CAN-10-4453</a>.</p>
</div>
<div id="ref-sahin2009modeling">
<p>Sahin, Özgür, Holger Fröhlich, Christian Löbke, Ulrike Korf, Sara Burmester, Meher Majety, Jens Mattern, et al. 2009. “Modeling Erbb Receptor-Regulated G1/S Transition to Find Novel Targets for de Novo Trastuzumab Resistance.” <em>BMC Systems Biology</em> 3 (1). BioMed Central: 1.</p>
</div>
<div id="ref-steinway2015combinatorial">
<p>Steinway, Steven Nathaniel, Jorge Gomez Tejeda Zañudo, Paul J Michel, David J Feith, Thomas P Loughran, and Reka Albert. 2015. “Combinatorial Interventions Inhibit Tgf<span class="math inline">\(\beta\)</span>-Driven Epithelial-to-Mesenchymal Transition and Support Hybrid Cellular Phenotypes.” <em>NPJ Systems Biology and Applications</em> 1. Nature Publishing Group: 15014.</p>
</div>
<div id="ref-stoll2017maboss">
<p>Stoll, Gautier, Barthélémy Caron, Eric Viara, Aurélien Dugourd, Andrei Zinovyev, Aurélien Naldi, Guido Kroemer, Emmanuel Barillot, and Laurence Calzone. 2017. “MaBoSS 2.0: An Environment for Stochastic Boolean Modeling.” <em>Bioinformatics</em> 33 (14). Oxford University Press: 2226–8.</p>
</div>
<div id="ref-stoll2012continuous">
<p>Stoll, Gautier, Eric Viara, Emmanuel Barillot, and Laurence Calzone. 2012. “Continuous Time Boolean Modeling for Biological Signaling: Application of Gillespie Algorithm.” <em>BMC Systems Biology</em> 6 (1). BioMed Central: 116.</p>
</div>
<div id="ref-terfve2015largescale">
<p>Terfve, Camille D. A., Edmund H. Wilkes, Pedro Casado, Pedro R. Cutillas, and Julio Saez-Rodriguez. 2015. “Large-Scale Models of Signal Propagation in Human Cells Derived from Discovery Phosphoproteomic Data.” <em>Nature Communications</em> 6 (September): 8033. doi:<a href="https://doi.org/10.1038/ncomms9033">10.1038/ncomms9033</a>.</p>
</div>
<div id="ref-terfve2012cellnoptr">
<p>Terfve, Camille, Thomas Cokelaer, David Henriques, Aidan MacNamara, Emanuel Goncalves, Melody K. Morris, Martijn van Iersel, Douglas A. Lauffenburger, and Julio Saez-Rodriguez. 2012. “CellNOptR: A Flexible Toolkit to Train Protein Signaling Networks to Data Using Multiple Logic Formalisms.” <em>BMC Systems Biology</em> 6 (1): 133. doi:<a href="https://doi.org/10.1186/1752-0509-6-133">10.1186/1752-0509-6-133</a>.</p>
</div>
<div id="ref-thomas1973boolean">
<p>Thomas, René. 1973. “Boolean Formalization of Genetic Control Circuits.” <em>Journal of Theoretical Biology</em> 42 (3). Elsevier: 563–85.</p>
</div>
<div id="ref-thomas1990biological">
<p>Thomas, René, and Richard d’Ari. 1990. <em>Biological Feedback</em>. CRC press.</p>
</div>
<div id="ref-van2004stochastic">
<p>Van Kampen, NG. 2004. “Stochastic Processes in Physics and Chemistry. 5th.” Elsevier Amsterdam:</p>
</div>
<div id="ref-verny2017learning">
<p>Verny, Louis, Nadir Sella, Séverine Affeldt, Param Priya Singh, and Hervé Isambert. 2017. “Learning Causal Networks with Latent Variables from Multivariate Information in Genomic Data.” <em>PLOS Computational Biology</em> 13 (10): e1005662. doi:<a href="https://doi.org/10.1371/journal.pcbi.1005662">10.1371/journal.pcbi.1005662</a>.</p>
</div>
<div id="ref-vert2007new">
<p>Vert, Jean-Philippe, Jian Qiu, and William S. Noble. 2007. “A New Pairwise Kernel for Biological Network Inference with Support Vector Machines.” <em>BMC Bioinformatics</em> 8 (Suppl 10): S8.</p>
</div>
<div id="ref-videla2017caspo">
<p>Videla, Santiago, Julio Saez-Rodriguez, Carito Guziolowski, and Anne Siegel. 2017. “Caspo: A Toolbox for Automated Reasoning on the Response of Logical Signaling Networks Families.” <em>Bioinformatics</em> 33 (6): 947–50. doi:<a href="https://doi.org/10.1093/bioinformatics/btw738">10.1093/bioinformatics/btw738</a>.</p>
</div>
<div id="ref-weinstein2017network">
<p>Weinstein, Nathan, Luis Mendoza, Isidoro Gitler, and Jaime Klapp. 2017. “A Network Model to Explore the Effect of the Micro-Environment on Endothelial Cell Behavior During Angiogenesis.” <em>Frontiers in Physiology</em> 8. Frontiers: 960.</p>
</div>
<div id="ref-woodhouse2018scns">
<p>Woodhouse, Steven, Nir Piterman, Christoph M Wintersteiger, Berthold Göttgens, and Jasmin Fisher. 2018. “SCNS: A Graphical Tool for Reconstructing Executable Regulatory Networks from Single-Cell Genomic Data.” <em>BMC Systems Biology</em> 12 (1). BioMed Central: 59.</p>
</div>
<div id="ref-zanudo2017network">
<p>Zañudo, Jorge Gómez Tejeda, Maurizio Scaltriti, and Réka Albert. 2017. “A Network Modeling Approach to Elucidate Drug Resistance Mechanisms and Predict Combinatorial Drug Treatments in Breast Cancer.” <em>Cancer Convergence</em> 1 (1). Springer: 5.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="4">
<li id="fn4"><p>Additional information on Markov processes and MaBoSS in the following <a href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-6-116/MediaObjects/12918_2012_969_MOESM1_ESM.pdf">document</a><a href="logical-modeling-principles-and-data-integration.html#fnref4">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="mechanistic-cancer.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="personalization-chap.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
