<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Mechanistic modeling of cancer: from complex disease to systems biology | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Mechanistic modeling of cancer: from complex disease to systems biology | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Mechanistic modeling of cancer: from complex disease to systems biology | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="cancer-as-deregulation-of-complex-machinery.html"/>
<link rel="next" href="logical-modeling-principles-and-data-integration.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#introducing-the-diversity-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive-models"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#tools-for-logical-modeling"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.1</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.2</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.3</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#comparison-of-the-mechanistic-approach-with-machine-learning-methods"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#limitations-and-perspectives"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.1</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.1.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information-in-information-out"><i class="fa fa-check"></i><b>7.1.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.1.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artifical-examples"><i class="fa fa-check"></i><b>7.1.2</b> Emergence of information in artifical examples</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>7.2</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.2.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.2.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-inference-methods-to-leverage-data"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#limitations-and-perspective"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspective</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mechanistic-cancer" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Mechanistic modeling of cancer: from complex disease to systems biology</h1>
<blockquote>
<p><em>“How remarkable is life? The answer is: very. Those of us who deal in networks of chemical reactions know of nothing like it… How could a chemical sludge become a rose, even with billions of years to try.”</em><br/> George Whitesides</p>
</blockquote>
<p>The previous chapter identified the need to organize cancer knowledge and data. The integration of biological knowledge, particularly in the form of networks, is a first step in this direction. The deepening of knowledge, however, requires the ability to manipulate objects even more, to experiment, to dissect their behaviour in an infinite number of situations, such as the astronomer with his orrery or physicians with their old anatomical models (Figure <a href="mechanistic-cancer.html#fig:anatomy">3.1</a>). Is it then possible to create mechanistic models of cancer in the same way?</p>
<div class="figure" style="text-align: center"><span id="fig:anatomy"></span>
<img src="fig/anatomy.jpg" alt="Dissecting a biological phenomenon using a non-computational model. Rembrandt, The Anatomy Lesson of Dr Nicolaes Tulp, 1634, oil on canvas, Mauritshuis museum, The Hague" width="90%" />
<p class="caption">
Figure 3.1: <strong>Dissecting a biological phenomenon using a non-computational model.</strong> Rembrandt, <em>The Anatomy Lesson of Dr Nicolaes Tulp</em>, 1634, oil on canvas, Mauritshuis museum, The Hague
</p>
</div>

<div id="introducing-the-diversity-of-mechanistic-models-of-cancer" class="section level2">
<h2><span class="header-section-number">3.1</span> Introducing the diversity of mechanistic models of cancer</h2>
<p>Modeling cancer is not a new idea. And the diversity of biological phenomena involved in cancer has given rise to an equally important diversity of models and formalisms, which we seek here to give a brief overview in order to better identify the specific models that we will focus on later. One way to order this diversity is to consider the scales of these models (Figure <a href="mechanistic-cancer.html#fig:multiscale">3.2</a>). Indeed, <strong>cancer can be read at different levels, from the molecular level of DNA and proteins, to the cellular level, to the level of tissues and organisms</strong> <span class="citation">(Anderson and Quaranta <a href="#ref-anderson2008integrative">2008</a>)</span>. Models have been proposed at all these scales, using different formalisms <span class="citation">(Bellomo, Li, and Maini <a href="#ref-bellomo2008foundations">2008</a>)</span> and answering different questions.</p>
<p>Consistent with the evolution of knowledge and data, the early models were at the <strong>macroscopic level</strong>. While methods and terminologies may have changed, there are nevertheless traces of these models as early as the 1950s. We then speak rather of mathematical modeling with a meaning that is nevertheless intermediate between what we have defined as mechanistic models and statistical models <span class="citation">(Byrne <a href="#ref-byrne2010dissecting">2010</a>)</span>. First, the initiation of tumorigenesis was theorized with biologically-supported mathematical expressions in order to make sense of cancer incidence statistics <span class="citation">(Armitage and Doll <a href="#ref-armitage1954age">1954</a>, <span class="citation">Knudson (<a href="#ref-knudson1971mutation">1971</a>)</span>)</span>. These models, however, remained relatively descriptive in that they did not shed any particular light on the biological mechanisms involved and focused on gross characteristics of tumours. The integration of more advanced knowledge as well as the progressive refinement of mathematical formalisms has nevertheless allowed these models to proliferate while gaining in interpretability, with for instance mechanistic models of metastatic relapse <span class="citation">(Nicolò et al. <a href="#ref-nicolo2020machine">2020</a>)</span>. Always on a macroscopic scale, the study of tumor growth has also been the playground of many mathematicians [<span class="citation">Araujo and McElwain (<a href="#ref-araujo2004history">2004</a>)</span>; byrne2010dissecting], even predicting invasion or response to surgical treatments using spatial modeling <span class="citation">(Swanson et al. <a href="#ref-swanson2003virtual">2003</a>)</span>. This line of research is still quite active today and provides a mathematical basis for comparison with tumour experimental growth <span class="citation">(Benzekry et al. <a href="#ref-benzekry2014classical">2014</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:multiscale"></span>
<img src="fig/multiscale.png" alt="The different scales of cancer modeling. Cancer can be approached at different scales, from molecules to organs, using different data (dark blue), but often with the direct or indirect objective of contributing to the study of clinically interpretable phenomena (yellow boxes), in particular by studying the influence of anticancer agents (pale blue). Reprinted from Barbolosi et al. (2016)." width="90%" />
<p class="caption">
Figure 3.2: <strong>The different scales of cancer modeling.</strong> Cancer can be approached at different scales, from molecules to organs, using different data (dark blue), but often with the direct or indirect objective of contributing to the study of clinically interpretable phenomena (yellow boxes), in particular by studying the influence of anticancer agents (pale blue). Reprinted from <span class="citation">Barbolosi et al. (<a href="#ref-barbolosi2016computational">2016</a>)</span>.
</p>
</div>

<p>Taking it down a step further, it is also possible to model cancer at the <strong>cellular level</strong>, for example by looking at the clonal evolution of cancer <span class="citation">(Altrock, Liu, and Michor <a href="#ref-altrock2015mathematics">2015</a>)</span>. The aim is then to understand the impact of the processes of mutation, selection, expansion and cohabitation of different populatons of cells, at specifc rates. The accumulation of a mutation in a population of cells can thus be studied <span class="citation">(Bozic et al. <a href="#ref-bozic2010accumulation">2010</a>)</span>. Modeling at the cellular level is well suited to the study of interactions between cells, between cancer cells and their environment or with the immune system. Similar to other kinds of studies of population dynamics, formalisms based on differential equations are quite common <span class="citation">(Bellomo, Li, and Maini <a href="#ref-bellomo2008foundations">2008</a>)</span>; but there are many other methods such partial differential equations or agent-based modeling <span class="citation">(Letort et al. <a href="#ref-letort2019physiboss">2019</a>)</span>.</p>
<p>Finally, at an even smaller scale, it is possible to model the <strong>molecular networks</strong> at work in cells <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>. The aim is then to simulate mathematically how the different genes and molecules regulate each other, transmit information and, in the case of cancer, end up being deregulated <span class="citation">(Calzone et al. <a href="#ref-calzone2010mathematical">2010</a>)</span>. These models will be the subject of the thesis and will therefore be defined more precisely and used to detail the concepts and tools of systems biology in the following sections. It can already be noted that while these models can integrate the most fundamental biological mechanisms of living organisms, one of the most burning questions is whether it is possible to link them to the larger scales that are clinically more interesting (tissues, organs etc.). Can these models tell us something about the molecular nature of cancer? About patient survival? Their response to treatment? These questions apply to all of the above models, whatever their scales (Figure <a href="mechanistic-cancer.html#fig:multiscale">3.2</a>), but are more difficult to answer for models defined at molecular scale that are further from the clinical data of interest. The aim of this thesis is to provide potential answers to these questions. One of the ways of approaching these issues has been to propose multi-scale models, which are nevertheless very complex <span class="citation">(Anderson and Quaranta <a href="#ref-anderson2008integrative">2008</a>; Powathil, Swat, and Chaplain <a href="#ref-powathil2015systems">2015</a>)</span>. We will focus here on the use of models defined almost exclusively at the molecular scale, which is assumed to be prominent, to study what can be inferred on the larger scales.</p>
</div>
<div id="cell-circuitry-and-the-need-for-cancer-systems-biology" class="section level2">
<h2><span class="header-section-number">3.2</span> Cell circuitry and the need for cancer systems biology</h2>
<p>Most biological systems, and certainly cells, fall into the category of <strong>complex systems</strong>. These are systems made up of many interacting elements. While these systems can be found in many different scientific fields, the cell as a complex system is characterized by the diversity and multifunctionality of its constituent elements (genes, proteins, small molecules, enzymes), which nevertheless contribute to organized and a priori non-chaotic behaviour <span class="citation">(Kitano <a href="#ref-kitano2002computational">2002</a>)</span>. Thus, the role of a protein such as the p53 tumour suppressor can only be understood by taking into account the interplay between its relationships with transcription factors and biochemical modifications of the molecule itself <span class="citation">(Kitano <a href="#ref-kitano2002computational">2002</a>)</span>. In a cell, as in any complex system, the multiplication of components and interactions can make the response or behaviour of the system unexpected or unpredictable. Non-linear responses, such as abrupt changes in the state of a system, called critical transitions, can be observed in response to a moderate change in the signal <span class="citation">(Trefois et al. <a href="#ref-trefois2015critical">2015</a>)</span>. Generally speaking, it is possible to observe <strong>emergent behaviours</strong>, i.e., behaviours of the system as a whole that were not trivially deducible from the individual behaviours of its components. This has been documented, through experiments and simulations, in the study of cell signalling pathways and the resulting biological decisions <span class="citation">(Bhalla and Iyengar <a href="#ref-bhalla1999emergent">1999</a>; Helikar et al. <a href="#ref-helikar2008emergent">2008</a>)</span>. These considerations have thus given rise to <strong>system-level or holistic approaches that aim to integrate data and knowledge into more comprehensive representations, often called systems biology</strong>.</p>
<p>What is true for the cell in general is just as true for cancer in particular. Understanding the intertwining of signaling pathways is necessary to study their contributions to different cancer hallmarks, as shown in Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:circuit">2.7</a>. The concepts described above can thus be transposed to <strong>cancer systems biology</strong> <span class="citation">(Hornberg et al. <a href="#ref-hornberg2006cancer">2006</a>; Kreeger and Lauffenburger <a href="#ref-kreeger2010cancer">2010</a>; Barillot et al. <a href="#ref-barillot2012computational">2012</a>)</span>. Indeed, it is often a question of understanding or predicting the impact of perturbations on cellular networks. Understanding how a single genetic mutation disrupts and reprograms networks, or even predicting the responses triggered by a drug on a presumably promising molecular target, makes little sense without integrated approaches. In addition, cancers are characterized by the accumulation of numerous mutations and alterations over time that must be considered concomitantly. These points of view of biologists and modellers reinforce the observation already made in the previous chapter of cancer as a network disease, as a system disease (Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:pathways">2.8</a>).</p>
<p>Finally, to conclude this general presentation, it is important to understand that while small molecular network modeling is not recent, the rise and multiplication of wide range systems biology approaches is very much related to the production of biological data <span class="citation">(De Jong <a href="#ref-de2002modeling">2002</a>)</span>. The last few decades have seen the emergence of high-throughput data that have made it possible to identify and link hundreds of genes or proteins involved in cancer. Exploring the interaction and back and forth between these models and the data they use or predict is therefore of utmost importance. In addition, the now ** massive amount of data has also imposed mathematical or computational approaches as a central element in the management of this profusion** and more and more modeling approaches are focused on data integration or inference <span class="citation">(Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>; Bouhaddou et al. <a href="#ref-bouhaddou2018mechanistic">2018</a>)</span>. More generally, Figure <a href="mechanistic-cancer.html#fig:pubmed">3.3</a> shows that while the number of scientific articles devoted to cancer has increased drastically since the 1950s (panel A), the proportion of these same articles mentioning <em>models</em>, <em>networks</em> or <em>computational</em> approaches has also increased (panel B), illustrating a change in paradigms.</p>
<div class="figure" style="text-align: center"><span id="fig:pubmed"></span>
<img src="03-Mechanistic_files/figure-html/pubmed-1.png" alt="PubMed trends in cancer studies. (A) PubMed articles with the word Cancer in either title or abstract from 1950 to 2019. (B) Proportion of the Cancer articles with additional keywords expressed as PubMed logical queries." width="90%" />
<p class="caption">
Figure 3.3: <strong>PubMed trends in cancer studies.</strong> (A) PubMed articles with the word <em>Cancer</em> in either title or abstract from 1950 to 2019. (B) Proportion of the <em>Cancer</em> articles with additional keywords expressed as PubMed logical queries.
</p>
</div>

</div>
<div id="mechanistic-models-of-molecular-signaling" class="section level2">
<h2><span class="header-section-number">3.3</span> Mechanistic models of molecular signaling</h2>
<p>Once the context has been defined, both biologically and methodologically, it is possible to begin the exploration of the models that will constitute the core of this thesis: the <strong>mechanistic models of molecular networks</strong> and signaling pathways. Before describing and illustrating some of the existing mathematical formalisms, it is possible to describe the common fundamental elements of this family of approaches.</p>
<div id="networks-and-data" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Networks and data</h3>
<p>The first step is to identify the relevant biological entities from a question or system of interest (e.g. tumor suppressor genes, signaling cascades of proteins) and then to model their interactions, the regulatory relationships that link them. At this stage the model can generally be represented by a network but this word can cover different realities <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>. The simplest network just represents undirected interactions between entities, which therefore only establishes relationships and not causal mechanisms. But modeling requires more precise definitions, in particular concerning the direction of the interaction (is it A that acts on B or the opposite) and its nature (type of chemical reaction, activation/inhibition etc.). This is usually summarized as <strong>activity flows (or influence diagrams) with activation and inhibition arrows</strong> as in Figure <a href="cancer-as-deregulation-of-complex-machinery.html#fig:circuit">2.7</a> or Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>A. These arrows emphasize the transormation of static networks into dynamic objects that can be manipulated and interpreted mechanistically. This work can be taken further by writing bipartite graphs, known as process descriptions, which explicitly show the different states of each variable (first type of nodes), depending on their phosphorylation sate for instance, and the reactions that link them (second type of nodes) as in Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>B. A more precise description of these different representations and their meanings can be found in <span class="citation">Le Novere (<a href="#ref-le2015quantitative">2015</a>)</span>. <strong>Once the network structure of the model has been defined, it is possible write the corresponding mathematical formalism</strong> and potentially to refine certain parameters. Finally, the model is often confronted with new data to check its consistency with the biological behaviour studied or possibly make new predictions.</p>
<p>However, all these steps are not linear and sequential, but rather iterative and cyclical. This <strong>modeling cycle</strong>, with back and forth to the data, is not specific to molecular network models, but it is possible to specify it in this case (Figure <a href="mechanistic-cancer.html#fig:cycle">3.4</a>). The names of the key players involved in the question of interest are thus first extracted from adapted data or from the literature. A first mathematical translation of the relationships between the entities is then proposed before verifying the compatibility of this model with the observations, whether qualitative or quantitative. If the compatibility is not good, we come back to the definition or the parameterization of the model. If compatibility is correct, the model can be used to make new predictions or study phenomena that go beyond the initial data set. Ideally, these predictions will be tested afterwards. This cyclic approach with two successive checks is analogous to the use of validation and test data in the evaluation of most learning algorithms. This analogy can sometimes be masked by the qualitative nature of the predictions or by the lack of explicit fitting of the parameters.</p>
<div class="figure" style="text-align: center"><span id="fig:cycle"></span>
<img src="fig/cycle.png" alt="Modeling a biological network: an iterative and cyclical process. Reprinted from (Béal, Rémy, and Calzone 2020). A different and simpler version of this cycle is described in (Le Novere 2015)." width="70%" />
<p class="caption">
Figure 3.4: <strong>Modeling a biological network: an iterative and cyclical process.</strong> Reprinted from <span class="citation">(Béal, Rémy, and Calzone <a href="#ref-beal2020modelisation">2020</a>)</span>. A different and simpler version of this cycle is described in <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>.
</p>
</div>

</div>
<div id="different-formalisms-for-different-applications" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Different formalisms for different applications</h3>
<p>Beyond these similarities in the construction and representation of models, the precise mathematical formalism that underlies them varies according to the type of question and the data <span class="citation">(De Jong <a href="#ref-de2002modeling">2002</a>)</span>. For the sake of simplicity, and without exhaustiveness, we propose to divide into quantitative and qualitative formalisms which will be essentially illustrated respectively by <strong>ordinary differential equation (ODE)</strong> models and <strong>logical (or Boolean) models</strong> for which a graphical and schematic comparison is proposed in Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:toyraf"></span>
<img src="fig/toyraf.png" alt="Schematic example of logical and ODE modeling around MAPK signaling. (A) Activity flow diagram of a small part of MAPK signaling, each node representing a gene or protein, with an example of logical rule for MEK node for the corresponding logical model. (B) Process description of the same diagram with BRAF and CRAF merged in RAF for the sake of simplicity; each square representing a reaction and the correspondong rate; an example differential equation is provided for the phosphorylated (active) form of MEK." width="90%" />
<p class="caption">
Figure 3.5: <strong>Schematic example of logical and ODE modeling around MAPK signaling.</strong> (A) Activity flow diagram of a small part of MAPK signaling, each node representing a gene or protein, with an example of logical rule for MEK node for the corresponding logical model. (B) Process description of the same diagram with BRAF and CRAF merged in RAF for the sake of simplicity; each square representing a reaction and the correspondong rate; an example differential equation is provided for the phosphorylated (active) form of MEK.
</p>
</div>

<p>One of the most frequent approaches is the use of <strong>chemical kinetics</strong> equations to construct ODE systems which are a fairly natural translation of the process descritption networks described in the previous section <span class="citation">(Polynikis, Hogan, and Bernardo <a href="#ref-polynikis2009comparing">2009</a>)</span>. Each biological interaction is treated as a reaction governed by the law of mass action and, under certain hypotheses, as a differential equation (Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>B); the set of reactions in the system then generates a set of differential equations with coupled variables, in an analogous way to the Lotka Volterra system presented in section <a href="scientific-modeling-abstract-the-complexity.html#lotkasection">1.2.2</a>. Thus the variables generally represent quantities of molecular species, for example concentrations of RNA or proteins, and the stoichiometric coefficients and reaction rates are used to define the system parameters. Approximations are sometimes made to simplify the equations, for example by assuming that they can be written as Michaelis-Menten’s enzymatic reactions, which have a simple and well known behaviour. However, the theoretical accuracy of quantitative models has a cost since <strong>each differential equation requires parameters</strong>, such as reaction constants or initial conditions, to which the system is very sensitive <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>. The biochemical interpretation of the parameters sometimes allow to find their value in the literature, if the reactions are well characterized, even if possible variations in a given biological or physical context are often unknown. Since knowledge of the values of these parameters is often limited or even non-existent, it may require a very large volume of data (including time series) to fit the many missing parameters which can be difficult if the number of parameters is large <span class="citation">(Villaverde and Banga <a href="#ref-villaverde2014reverse">2014</a>)</span>. However, recent work has demonstrated the feasibility and scalability of this type of inference with sufficiently rich data <span class="citation">(Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>.</p>
<p>At the same time, more qualitative approaches to modeling biological networks have been proposed with discrete variables linked together by rules expressed as logical statements <span class="citation">(Abou-Jaoudé et al. <a href="#ref-abou2016logical">2016</a>)</span>. These models are both more abstract since variables do not have a direct biological interpretation (e.g. concentration of a species) but are more versatile since they can unify different biological realities under the same formalism (e.g. activation of a gene or phosphorlation of a protein). The discrete nature of the variables can then be seen as an asymptotic case of the sigmoidal (e.g. Hill function) relationships often found in biology <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>)</span>. The step function thus obtained can keep a natural interpretation in the context of biological phenomena: genes activated or not, protein present or absent etc. Similarly, interactions between species are not quantified but are based on a qualitative statements (e.g. A will be active if B and C are active), drastically reducing the number of parameters (Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>A). If the theoretical interest of this formalism to study biological mechanisms was proposed quite early <span class="citation">(Kauffman <a href="#ref-kauffman1969homeostasis">1969</a>; Thomas <a href="#ref-thomas1973boolean">1973</a>)</span>, many concrete applications have also been developed over the years, particularly in cancer research <span class="citation">(Saez-Rodriguez, Alexopoulos, et al. <a href="#ref-saez2011comparing">2011</a><a href="#ref-saez2011comparing">a</a>; Remy et al. <a href="#ref-remy2015modeling">2015</a>)</span>. This <strong>logical formalism will constitute the core of the work presented in Part II</strong>, where it will therefore be discussed in greater detail.</p>
<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:odelogic">Table 3.1: </span><strong>Features of quantitative and qualitative modeling applied to biological molecular networks</strong> (adapted from <span class="citation">Le Novere (<a href="#ref-le2015quantitative">2015</a>)</span>)
</caption>
<thead>
<tr>
<th style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;text-align: center;">
</th>
<th style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;text-align: center;">
Quantitative modeling
</th>
<th style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;text-align: center;">
Qualitative modeling
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;border-right:1px solid;">
Example formalism
</td>
<td style="text-align:left;">
Ordinary differential equation (ODE) models
</td>
<td style="text-align:left;">
Logical models
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;border-right:1px solid;">
Type of variables
</td>
<td style="text-align:left;">
Direct translation of biological quantities, usually continuous
</td>
<td style="text-align:left;">
Abstract representation of activity levels, usually discrete
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;border-right:1px solid;">
Objective
</td>
<td style="text-align:left;">
Quantitatively accurate and temporal simulation of an experimental phenomenon
</td>
<td style="text-align:left;">
Coarse-grained simulation of qualitative phenotypes
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;border-right:1px solid;">
Advantages
</td>
<td style="text-align:left;">
Direct confrontation with experimental data; precise; linear representation of time
</td>
<td style="text-align:left;">
Faster design; easy translation of literature-based assertions; simulation of perturbations
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;border-right:1px solid;">
Drawbacks
</td>
<td style="text-align:left;">
Difficulty determining or fitting parameters
</td>
<td style="text-align:left;">
More difficult to link to data; lower precision
</td>
</tr>
</tbody>
</table>

<p>These two formalisms, which are among the most frequent for modelling biological networks, share many similarities, in particular the propensity to be built according to bottom-up strategies based on knowledge of the elementary parts of the model, i.e., biological entities and reactions. However, they differ in their implementation and objectives, <strong>one aiming at the most accurate representation possible, the other seeking to capture the essence of the system’s dynamics in a parsimonious way</strong> (Table <a href="mechanistic-cancer.html#tab:odelogic">3.1</a>). The opposition is not irrevocable, as illustrated by the numerous hybrid formalisms that lie within the spectrum delimited by these two extremes such as fuzzy logic or discrete-time differential equations <span class="citation">(Le Novere <a href="#ref-le2015quantitative">2015</a>; Calzone, Barillot, and Zinovyev <a href="#ref-calzone2018logical">2018</a>)</span>. To conclude, a comparison between the two approaches applied to the same problem is proposed by <span class="citation">Calzone, Barillot, and Zinovyev (<a href="#ref-calzone2018logical">2018</a>)</span>, studying the epithelio-mesenchymal transition (EMT, a biological process involved in cancer), to illustrate in concrete terms their complementarity.</p>
</div>
<div id="some-examples-of-complex-features" class="section level3">
<h3><span class="header-section-number">3.3.3</span> Some examples of complex features</h3>
<p>With the help of these models, both qualitative and quantitative, many complex behaviours have been identified. Benefiting from the knowledge accumulated in the study of dynamic systems, a whole zoo of patterns with complex and non-intuitive behaviours such as non-linearities have been highlighted <span class="citation">(Tyson, Chen, and Novak <a href="#ref-tyson2003sniffers">2003</a>)</span>. The MAPK pathway, coarsely described in Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>, and often simplified as a rather unidirectional cascade, shows switch or bistability behaviors generated by the complexity of its multiple phosphorylation sites <span class="citation">(Markevich, Hoek, and Kholodenko <a href="#ref-markevich2004signaling">2004</a>)</span>. These models have also been put at the service of understanding cancer and the erroneous decision-making by cells resulting from impaired signaling pathways. Thus, <span class="citation">Tyson et al. (<a href="#ref-tyson2011dynamic">2011</a>)</span> summarize superbly well the complexity that can be hidden in the dynamics of smallest molecular networks as soon as they contain more than two entites and crossed regulations or feedback loops. Logical models have also made it possible to better dissect some complex phenomena at play in the cell such as emergent behaviours <span class="citation">(Helikar et al. <a href="#ref-helikar2008emergent">2008</a>)</span> or mechanisms behind mutation patterns in cancer <span class="citation">(Remy et al. <a href="#ref-remy2015modeling">2015</a>)</span>.</p>
</div>
</div>
<div id="from-mechanistic-models-to-clinical-impact" class="section level2">
<h2><span class="header-section-number">3.4</span> From mechanistic models to clinical impact?</h2>
<p>Mechanistic models have therefore undeniably led to a better understanding of the complex molecular machinery of signalling pathways. But beyond the interest that this understanding represents, do these models also have a clinical utility? In other words, <strong>are they of clinical or only scientific value?</strong></p>
<div id="a-new-class-of-biomarkers" class="section level3">
<h3><span class="header-section-number">3.4.1</span> A new class of biomarkers</h3>
<p>Throughout this thesis, the clinical value of mechanical models will often be analyzed by analogy to that of biomarkers. Throughout this thesis, the clinical value of mechanical models will often be analyzed by analogy to that of biomarkers. Biomarkers are usually defined as measurable indicators of patient status or disease progression, such as prostate-specific antigen (PSA) for prostate cancer screening or BRCA1 mutation for breast cancer risk <span class="citation">(Henry and Hayes <a href="#ref-henry2012cancer">2012</a>)</span>. Biomarkers also encompass multivariate signatures that identify more complex patterns with clinical significance. Taking the logic even further, it was therefore proposed that mechanistic models, which also reveal complex molecular behaviours, could be considered as biomarkers, capturing perhaps even dynamic information <span class="citation">(Fey et al. <a href="#ref-fey2015signaling">2015</a>)</span>.</p>
<p>Like oncology biomarkers, the models will be divided into two categories according to their clinical objectives: <strong>prognostic models and predictive models</strong> <span class="citation">(Oldenhuis et al. <a href="#ref-oldenhuis2008prognostic">2008</a>)</span>. Prognostic biomarkers and models are those that provide information on the evolution of cancer independently of treatment. They are therefore generally confronted with survival or relapse data. The protein Ki-67 for example, encoded by the MKI67 gene, is known to be indicative of the level of proliferation and high levels of expression are thus associated with a poorer prognosis in many cancers <span class="citation">(Sawyers <a href="#ref-sawyers2008cancer">2008</a>)</span>. Predictive biomarkers and models, on the other hand, give an indication of the effect of a therapeutic strategy. The simplest example, but not the only one, concerns biomarkers that are themselves the target of treatment: treatments based on monoclonal antibodies directed against HER2 receptors in breast cancer are only effective if the HER2 receptor has been detected in the patient <span class="citation">(Sawyers <a href="#ref-sawyers2008cancer">2008</a>)</span>. Without attempting to be exhaustive, some logical and ODE models, with either prognostic or predictive claims, will be described.</p>
</div>
<div id="prognostic" class="section level3">
<h3><span class="header-section-number">3.4.2</span> Prognostic models</h3>
<p>One of the first mechanical models of cell signalling to have been explicitly presented as a prognostic biomarker is the one proposed by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span> and describing c-Jun N-terminal kinase (JNK) pathway in neuroblastoma cells. A summary of the study is provided in Figure <a href="mechanistic-cancer.html#fig:fey">3.6</a>). The model is an ODE translation of the process description network of Figure <a href="mechanistic-cancer.html#fig:fey">3.6</a>)A, further determined and calibrated with molecular biology experimental data obtained using neuroblastoma cell lines. We thus observe the non-linear switch-like dynamics of JNK activation as a function of cellular stress (Figure <a href="mechanistic-cancer.html#fig:fey">3.6</a>)B). The precise characteristics of this sgmoidal response can, however, vary from one individual to another as captured by the network output descriptors <span class="math inline">\(A\)</span>, <span class="math inline">\(K_{50}\)</span> and <span class="math inline">\(H\)</span>. Fey et al. proposed to perform neuroblastoma patient–specific simulations of the model, using patient gene expressions for ZAK, MKK4, MKK7, JNK and AKT genes to specify the initial conditions of the ODE system. Since JNK activation induces cell death through apoptosis, the patient-specific <span class="math inline">\(A\)</span>, <span class="math inline">\(K_{50}\)</span> and <span class="math inline">\(H\)</span> derived from patient-specifc models are then analyzed as prognostic biomarkers (Figure <a href="mechanistic-cancer.html#fig:fey">3.6</a>)C). Readers are invited to refer to the original article for details on model calibration or binarization of network descriptors <span class="citation">(Fey et al. <a href="#ref-fey2015signaling">2015</a>)</span>. The authors also showed that in the absence of positive feedback from <span class="math inline">\(JNK^{**}\)</span> to <span class="math inline">\(^PMKK7\)</span>, an important component of non-linearity, the prognostic value is drastically decreased. All in all, this pipeline from ODE model to survival curves, thus provides a <strong>paradigmatic example of the clinical interpretation of mechanistic models of molecular networks</strong> that will be reused in later chapters for illustration purposes. Other ODE models following a similar rationale have been proposed by the same group for colorectal cancer <span class="citation">(Hector et al. <a href="#ref-hector2012clinical">2012</a>; Salvucci et al. <a href="#ref-salvucci2017stepwise">2017</a>)</span> or glioblastoma [<span class="citation">Murphy et al. (<a href="#ref-murphy2013activation">2013</a>)</span>; salvucci2019system]. Machine learning approaches have also been proposed to ease the clinical implementation of this kind of prognostic models by dealing with the potential lack of patient data needed to personalize them <span class="citation">(Salvucci, Rahman, et al. <a href="#ref-salvucci2019machine">2019</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:fey"></span>
<img src="fig/fey.png" alt="Mechanistic modeling of JNK pathway and survival of neuroblastima patients, as described by Fey et al. (2015). (A) Schematic representation, as a process description, for the ODE model of JNK pathway. (B) Response curve (phosphorylated JNK) as a function of the input stimulus (Stress) and characterization of the corresponding sigmoidal function with maximal amplitude \(A\), Hill exponent \(H\) and activation threshold \(K_{50}\). (C) Survival curves for neuroblastoma patients based on binarized \(A\), \(K_{50}\) and \(H\); binarization thresholds having been defined based on optimization screening on calibration cohort." width="90%" />
<p class="caption">
Figure 3.6: <strong>Mechanistic modeling of JNK pathway and survival of neuroblastima patients, as described by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span>.</strong> (A) Schematic representation, as a process description, for the ODE model of JNK pathway. (B) Response curve (phosphorylated JNK) as a function of the input stimulus (Stress) and characterization of the corresponding sigmoidal function with maximal amplitude <span class="math inline">\(A\)</span>, Hill exponent <span class="math inline">\(H\)</span> and activation threshold <span class="math inline">\(K_{50}\)</span>. (C) Survival curves for neuroblastoma patients based on binarized <span class="math inline">\(A\)</span>, <span class="math inline">\(K_{50}\)</span> and <span class="math inline">\(H\)</span>; binarization thresholds having been defined based on optimization screening on calibration cohort.
</p>
</div>

<p>On the logical modeling side, there are also studies including prognostic value validation. Thus, <span class="citation">Khan et al. (<a href="#ref-khan2017unraveling">2017</a>)</span> proposed two logical models of epitelio-mesenchymal transition (EMT) in bladder and breast cancers. These models are inferred from prior mechanisms knowledge and data analysis with particular attention to potential feedback loops. Using these models, it is possible to study the behaviour of them for all combinations of model inputs (growth factors and receptor proteins) and derive subsequent signatures for good or bad prognosis. These signatures are later validated with cohorts of patients. In this case, the mechanistic model does not seek to capture a dynamic behavior but to facilitate and <strong>make understandable the exploration of combinations of input signals that grow exponentially with the number of inputs considered</strong>. Other formalisms, called pathway activity analysis and following the same activity flows principles (Figure <a href="mechanistic-cancer.html#fig:toyraf">3.5</a>A), have been analysed in the light of their prognostic value. Their greater flexibility enables the direct use of networks of several hundred or thousands of genes, such as those present in the KEGG database <span class="citation">(Kanehisa et al. <a href="#ref-kanehisa2012kegg">2012</a>)</span>. The benefit of mechanistic modeling is then to organize high-dimensional data and to facilitate the <em>a posteriori</em> analysis of the results.</p>
</div>
<div id="predictive-models" class="section level3">
<h3><span class="header-section-number">3.4.3</span> Predictive models</h3>
<p>But the explicit representation of biological entities in mechanistic models makes them particularly <strong>suitable for the study of well-defined perturbations such as drug effects</strong>. Indeed, by assuming that the mechanism of action of a drug is at least partially known, it is possible to integrate this mechanism into the model if it contains the target of the drug (Figure <a href="mechanistic-cancer.html#fig:netdrug">3.7</a>). One can therefore simulate the effect of one drug or even compare several. These strategies have already been implemented in a qualitative way with logical models used to explain resistance to certain treatments of breast cancer <span class="citation">(Zañudo, Scaltriti, and Albert <a href="#ref-zanudo2017network">2017</a>)</span> or even highlight the synergy of certain combinations of treatments in gastric cancer <span class="citation">(Flobak et al. <a href="#ref-flobak2015discovery">2015</a>)</span>. The value of these models, however, is more scientific than clinical in that they focus on a single cell line or a restricted group of cell lines. The possibility to personalize the predictions or recommendations for different molecular profiles of cell lines or patients is therefore not obvious. Still within the context of logical formalism, <span class="citation">Knijnenburg et al. (<a href="#ref-knijnenburg2016logic">2016</a>)</span> proposed a broader approach: if their model needs to be trained, it can nevertheless provide an analytical framework for several hundred cell lines, while remaining within the scope of the training data to ensure the validity of predictions.</p>
<div class="figure" style="text-align: center"><span id="fig:netdrug"></span>
<img src="fig/netdrug.png" alt="Network model of oncogenic signal transduction in ER+ breast cancer, including some drugs and their targets. Reprinted from Zañudo, Scaltriti, and Albert (2017)." width="90%" />
<p class="caption">
Figure 3.7: <strong>Network model of oncogenic signal transduction in ER+ breast cancer, including some drugs and their targets.</strong> Reprinted from <span class="citation">Zañudo, Scaltriti, and Albert (<a href="#ref-zanudo2017network">2017</a>)</span>.
</p>
</div>

<p>Conceptually comparable strategies can be found on the side of differential equations where large mechanical models of cell signalling are also trained to predict the response to different treatments <span class="citation">(Bouhaddou et al. <a href="#ref-bouhaddou2018mechanistic">2018</a>; Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>. A calibrated model can then predict the response to a combination of treatments not tested in the training data, thereby proving the ability of mechanistic models to extend their predictive value beyond the data <span class="citation">(Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>. As with prognostic models, mechanical approaches other than logical formalisms and ODEs have been proposed and validated <span class="citation">(Jastrzebski et al. <a href="#ref-jastrzebski2018integrative">2018</a>)</span>. What can be learned from these predictive models is that they require <strong>significant training data to be able to go beyond qualitative predictions and dissect treatment response mechanisms of many cell lines simultaneously</strong>. For obvious practical and ethical reasons, the validation of these models is for the moment limited to preclinical data since they require data for many uncertain therapeutic interventions.</p>
<p>This first bridge between mechanistic models of cell signalling and clinical applications concludes this introductory part. The next part will be devoted to the definition of new methods to establish this connection based on logical formalism, before the third part proposes a more statistical evaluation of the prognostic and predictive values of the models presented in the previous parts.</p>

</div>
</div>
</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-abou2016logical">
<p>Abou-Jaoudé, Wassim, Pauline Traynard, Pedro T Monteiro, Julio Saez-Rodriguez, Tomáš Helikar, Denis Thieffry, and Claudine Chaouiya. 2016. “Logical Modeling and Dynamical Analysis of Cellular Networks.” <em>Frontiers in Genetics</em> 7. Frontiers: 94.</p>
</div>
<div id="ref-altrock2015mathematics">
<p>Altrock, Philipp M, Lin L Liu, and Franziska Michor. 2015. “The Mathematics of Cancer: Integrating Quantitative Models.” <em>Nature Reviews Cancer</em> 15 (12). Nature Publishing Group: 730–45.</p>
</div>
<div id="ref-anderson2008integrative">
<p>Anderson, Alexander RA, and Vito Quaranta. 2008. “Integrative Mathematical Oncology.” <em>Nature Reviews Cancer</em> 8 (3). Nature Publishing Group: 227–34.</p>
</div>
<div id="ref-araujo2004history">
<p>Araujo, Robyn P, and DL Sean McElwain. 2004. “A History of the Study of Solid Tumour Growth: The Contribution of Mathematical Modelling.” <em>Bulletin of Mathematical Biology</em> 66 (5). Elsevier: 1039–91.</p>
</div>
<div id="ref-armitage1954age">
<p>Armitage, Peter, and Richard Doll. 1954. “The Age Distribution of Cancer and a Multi-Stage Theory of Carcinogenesis.” <em>British Journal of Cancer</em> 8 (1). Nature Publishing Group: 1.</p>
</div>
<div id="ref-barbolosi2016computational">
<p>Barbolosi, Dominique, Joseph Ciccolini, Bruno Lacarelle, Fabrice Barlési, and Nicolas André. 2016. “Computational Oncology—mathematical Modelling of Drug Regimens for Precision Medicine.” <em>Nature Reviews Clinical Oncology</em> 13 (4). Nature Publishing Group: 242.</p>
</div>
<div id="ref-barillot2012computational">
<p>Barillot, Emmanuel, Laurence Calzone, Philippe Hupe, Jean-Philippe Vert, and Andrei Zinovyev. 2012. <em>Computational Systems Biology of Cancer</em>. CRC Press.</p>
</div>
<div id="ref-bellomo2008foundations">
<p>Bellomo, Nicola, NK Li, and Ph K Maini. 2008. “On the Foundations of Cancer Modelling: Selected Topics, Speculations, and Perspectives.” <em>Mathematical Models and Methods in Applied Sciences</em> 18 (04). World Scientific: 593–646.</p>
</div>
<div id="ref-benzekry2014classical">
<p>Benzekry, Sébastien, Clare Lamont, Afshin Beheshti, Amanda Tracz, John ML Ebos, Lynn Hlatky, and Philip Hahnfeldt. 2014. “Classical Mathematical Models for Description and Prediction of Experimental Tumor Growth.” <em>PLoS Comput Biol</em> 10 (8). Public Library of Science: e1003800.</p>
</div>
<div id="ref-beal2020modelisation">
<p>Béal, Jonas, Elizabeth Rémy, and Laurence Calzone. 2020. “Modélisation Logique et Données Omiques : De La Construction Des Modèles à La Médecine Personnalisée.” In <em>Approche Symbolique de La Modélisation et de L’analyse Des Systèmes Biologiques</em>, edited by Elisabeth Rémy and Cédric Lhoussaine. ISTE.</p>
</div>
<div id="ref-bhalla1999emergent">
<p>Bhalla, Upinder S, and Ravi Iyengar. 1999. “Emergent Properties of Networks of Biological Signaling Pathways.” <em>Science</em> 283 (5400). American Association for the Advancement of Science: 381–87.</p>
</div>
<div id="ref-bouhaddou2018mechanistic">
<p>Bouhaddou, Mehdi, Anne Marie Barrette, Alan D Stern, Rick J Koch, Matthew S DiStefano, Eric A Riesel, Luis C Santos, Annie L Tan, Alex E Mertz, and Marc R Birtwistle. 2018. “A Mechanistic Pan-Cancer Pathway Model Informed by Multi-Omics Data Interprets Stochastic Cell Fate Responses to Drugs and Mitogens.” <em>PLoS Computational Biology</em> 14 (3). Public Library of Science: e1005985.</p>
</div>
<div id="ref-bozic2010accumulation">
<p>Bozic, Ivana, Tibor Antal, Hisashi Ohtsuki, Hannah Carter, Dewey Kim, Sining Chen, Rachel Karchin, Kenneth W Kinzler, Bert Vogelstein, and Martin A Nowak. 2010. “Accumulation of Driver and Passenger Mutations During Tumor Progression.” <em>Proceedings of the National Academy of Sciences</em> 107 (43). National Acad Sciences: 18545–50.</p>
</div>
<div id="ref-byrne2010dissecting">
<p>Byrne, Helen M. 2010. “Dissecting Cancer Through Mathematics: From the Cell to the Animal Model.” <em>Nature Reviews Cancer</em> 10 (3). Nature Publishing Group: 221–30.</p>
</div>
<div id="ref-calzone2018logical">
<p>Calzone, Laurence, Emmanuel Barillot, and Andrei Zinovyev. 2018. “Logical Versus Kinetic Modeling of Biological Networks: Applications in Cancer Research.” <em>Current Opinion in Chemical Engineering</em> 21. Elsevier: 22–31.</p>
</div>
<div id="ref-calzone2010mathematical">
<p>Calzone, Laurence, Laurent Tournier, Simon Fourquet, Denis Thieffry, Boris Zhivotovsky, Emmanuel Barillot, and Andrei Zinovyev. 2010. “Mathematical Modelling of Cell-Fate Decision in Response to Death Receptor Engagement.” <em>PLoS Computational Biology</em> 6 (3). Public Library of Science.</p>
</div>
<div id="ref-de2002modeling">
<p>De Jong, Hidde. 2002. “Modeling and Simulation of Genetic Regulatory Systems: A Literature Review.” <em>Journal of Computational Biology</em> 9 (1). Mary Ann Liebert, Inc.: 67–103.</p>
</div>
<div id="ref-fey2015signaling">
<p>Fey, Dirk, Melinda Halasz, Daniel Dreidax, Sean P Kennedy, Jordan F Hastings, Nora Rauch, Amaya Garcia Munoz, et al. 2015. “Signaling Pathway Models as Biomarkers: Patient-Specific Simulations of Jnk Activity Predict the Survival of Neuroblastoma Patients.” <em>Sci. Signal.</em> 8 (408). American Association for the Advancement of Science: ra130–ra130.</p>
</div>
<div id="ref-flobak2015discovery">
<p>Flobak, Åsmund, Anaïs Baudot, Elisabeth Remy, Liv Thommesen, Denis Thieffry, Martin Kuiper, and Astrid Lægreid. 2015. “Discovery of Drug Synergies in Gastric Cancer Cells Predicted by Logical Modeling.” <em>PLoS Computational Biology</em> 11 (8). Public Library of Science.</p>
</div>
<div id="ref-frohlich2018efficient">
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6). Elsevier: 567–79.</p>
</div>
<div id="ref-hector2012clinical">
<p>Hector, Suzanne, Markus Rehm, Jasmin Schmid, Joan Kehoe, Niamh McCawley, Patrick Dicker, Frank Murray, et al. 2012. “Clinical Application of a Systems Model of Apoptosis Execution for the Prediction of Colorectal Cancer Therapy Responses and Personalisation of Therapy.” <em>Gut</em> 61 (5). BMJ Publishing Group: 725–33.</p>
</div>
<div id="ref-helikar2008emergent">
<p>Helikar, Tomáš, John Konvalina, Jack Heidel, and Jim A Rogers. 2008. “Emergent Decision-Making in Biological Signal Transduction Networks.” <em>Proceedings of the National Academy of Sciences</em> 105 (6). National Acad Sciences: 1913–8.</p>
</div>
<div id="ref-henry2012cancer">
<p>Henry, N Lynn, and Daniel F Hayes. 2012. “Cancer Biomarkers.” <em>Molecular Oncology</em> 6 (2). Wiley Online Library: 140–46.</p>
</div>
<div id="ref-hornberg2006cancer">
<p>Hornberg, Jorrit J, Frank J Bruggeman, Hans V Westerhoff, and Jan Lankelma. 2006. “Cancer: A Systems Biology Disease.” <em>Biosystems</em> 83 (2-3). Elsevier: 81–90.</p>
</div>
<div id="ref-jastrzebski2018integrative">
<p>Jastrzebski, Katarzyna, Bram Thijssen, Roelof JC Kluin, Klaas de Lint, Ian J Majewski, Roderick L Beijersbergen, and Lodewyk FA Wessels. 2018. “Integrative Modeling Identifies Key Determinants of Inhibitor Sensitivity in Breast Cancer Cell Lines.” <em>Cancer Research</em> 78 (15). AACR: 4396–4410.</p>
</div>
<div id="ref-kanehisa2012kegg">
<p>Kanehisa, Minoru, Susumu Goto, Yoko Sato, Miho Furumichi, and Mao Tanabe. 2012. “KEGG for Integration and Interpretation of Large-Scale Molecular Data Sets.” <em>Nucleic Acids Research</em> 40 (D1). Oxford University Press: D109–D114.</p>
</div>
<div id="ref-kauffman1969homeostasis">
<p>Kauffman, Stuart. 1969. “Homeostasis and Differentiation in Random Genetic Control Networks.” <em>Nature</em> 224 (5215). Springer: 177–78.</p>
</div>
<div id="ref-khan2017unraveling">
<p>Khan, Faiz M, Stephan Marquardt, Shailendra K Gupta, Susanne Knoll, Ulf Schmitz, Alf Spitschak, David Engelmann, Julio Vera, Olaf Wolkenhauer, and Brigitte M Pützer. 2017. “Unraveling a Tumor Type-Specific Regulatory Core Underlying E2f1-Mediated Epithelial-Mesenchymal Transition to Predict Receptor Protein Signatures.” <em>Nature Communications</em> 8 (1). Nature Publishing Group: 1–15.</p>
</div>
<div id="ref-kitano2002computational">
<p>Kitano, Hiroaki. 2002. “Computational Systems Biology.” <em>Nature</em> 420 (6912). Nature Publishing Group: 206–10.</p>
</div>
<div id="ref-knijnenburg2016logic">
<p>Knijnenburg, Theo A, Gunnar W Klau, Francesco Iorio, Mathew J Garnett, Ultan McDermott, Ilya Shmulevich, and Lodewyk FA Wessels. 2016. “Logic Models to Predict Continuous Outputs Based on Binary Inputs with an Application to Personalized Cancer Therapy.” <em>Scientific Reports</em> 6 (1). Nature Publishing Group: 1–14.</p>
</div>
<div id="ref-knudson1971mutation">
<p>Knudson, Alfred G. 1971. “Mutation and Cancer: Statistical Study of Retinoblastoma.” <em>Proceedings of the National Academy of Sciences</em> 68 (4). National Acad Sciences: 820–23.</p>
</div>
<div id="ref-kreeger2010cancer">
<p>Kreeger, Pamela K, and Douglas A Lauffenburger. 2010. “Cancer Systems Biology: A Network Modeling Perspective.” <em>Carcinogenesis</em> 31 (1). Oxford University Press: 2–8.</p>
</div>
<div id="ref-le2015quantitative">
<p>Le Novere, Nicolas. 2015. “Quantitative and Logic Modelling of Molecular and Gene Networks.” <em>Nature Reviews Genetics</em> 16 (3). Nature Publishing Group: 146–58.</p>
</div>
<div id="ref-letort2019physiboss">
<p>Letort, Gaelle, Arnau Montagud, Gautier Stoll, Randy Heiland, Emmanuel Barillot, Paul Macklin, Andrei Zinovyev, and Laurence Calzone. 2019. “PhysiBoSS: A Multi-Scale Agent-Based Modelling Framework Integrating Physical Dimension and Cell Signalling.” <em>Bioinformatics</em> 35 (7). Oxford University Press: 1188–96.</p>
</div>
<div id="ref-markevich2004signaling">
<p>Markevich, Nick I, Jan B Hoek, and Boris N Kholodenko. 2004. “Signaling Switches and Bistability Arising from Multisite Phosphorylation in Protein Kinase Cascades.” <em>The Journal of Cell Biology</em> 164 (3). Rockefeller University Press: 353–59.</p>
</div>
<div id="ref-murphy2013activation">
<p>Murphy, Á C, Birgit Weyhenmeyer, Jasmin Schmid, Seán M Kilbride, Markus Rehm, Heinrich J Huber, C Senft, et al. 2013. “Activation of Executioner Caspases Is a Predictor of Progression-Free Survival in Glioblastoma Patients: A Systems Medicine Approach.” <em>Cell Death &amp; Disease</em> 4 (5). Nature Publishing Group: e629–e629.</p>
</div>
<div id="ref-nicolo2020machine">
<p>Nicolò, Chiara, Cynthia Périer, Melanie Prague, Carine Bellera, Gaëtan MacGrogan, Olivier Saut, and Sébastien Benzekry. 2020. “Machine Learning and Mechanistic Modeling for Prediction of Metastatic Relapse in Early-Stage Breast Cancer.” <em>JCO Clinical Cancer Informatics</em> 4. American Society of Clinical Oncology: 259–74.</p>
</div>
<div id="ref-oldenhuis2008prognostic">
<p>Oldenhuis, CNAM, SF Oosting, JA Gietema, and EGE De Vries. 2008. “Prognostic Versus Predictive Value of Biomarkers in Oncology.” <em>European Journal of Cancer</em> 44 (7). Elsevier: 946–53.</p>
</div>
<div id="ref-polynikis2009comparing">
<p>Polynikis, Athanasios, SJ Hogan, and Mario di Bernardo. 2009. “Comparing Different Ode Modelling Approaches for Gene Regulatory Networks.” <em>Journal of Theoretical Biology</em> 261 (4). Elsevier: 511–30.</p>
</div>
<div id="ref-powathil2015systems">
<p>Powathil, Gibin G, Maciej Swat, and Mark AJ Chaplain. 2015. “Systems Oncology: Towards Patient-Specific Treatment Regimes Informed by Multiscale Mathematical Modelling.” In <em>Seminars in Cancer Biology</em>, 30:13–20. Elsevier.</p>
</div>
<div id="ref-remy2015modeling">
<p>Remy, Elisabeth, Sandra Rebouissou, Claudine Chaouiya, Andrei Zinovyev, François Radvanyi, and Laurence Calzone. 2015. “A Modeling Approach to Explain Mutually Exclusive and Co-Occurring Genetic Alterations in Bladder Tumorigenesis.” <em>Cancer Research</em> 75 (19). AACR: 4042–52.</p>
</div>
<div id="ref-saez2011comparing">
<p>Saez-Rodriguez, Julio, Leonidas G Alexopoulos, MingSheng Zhang, Melody K Morris, Douglas A Lauffenburger, and Peter K Sorger. 2011a. “Comparing Signaling Networks Between Normal and Transformed Hepatocytes Using Discrete Logical Models.” <em>Cancer Research</em> 71 (16). AACR: 5400–5411.</p>
</div>
<div id="ref-salvucci2019machine">
<p>Salvucci, Manuela, Arman Rahman, Alexa J Resler, Girish M Udupi, Deborah A McNamara, Elaine W Kay, Pierre Laurent-Puig, et al. 2019. “A Machine Learning Platform to Optimize the Translation of Personalized Network Models to the Clinic.” <em>JCO Clinical Cancer Informatics</em> 3. American Society of Clinical Oncology: 1–17.</p>
</div>
<div id="ref-salvucci2017stepwise">
<p>Salvucci, Manuela, Maximilian L Würstle, Clare Morgan, Sarah Curry, Mattia Cremona, Andreas U Lindner, Orna Bacon, et al. 2017. “A Stepwise Integrated Approach to Personalized Risk Predictions in Stage Iii Colorectal Cancer.” <em>Clinical Cancer Research</em> 23 (5). AACR: 1200–1212.</p>
</div>
<div id="ref-sawyers2008cancer">
<p>Sawyers, Charles L. 2008. “The Cancer Biomarker Problem.” <em>Nature</em> 452 (7187). Nature Publishing Group: 548–52.</p>
</div>
<div id="ref-swanson2003virtual">
<p>Swanson, Kristin R, Carly Bridge, JD Murray, and Ellsworth C Alvord Jr. 2003. “Virtual and Real Brain Tumors: Using Mathematical Modeling to Quantify Glioma Growth and Invasion.” <em>Journal of the Neurological Sciences</em> 216 (1). Elsevier: 1–10.</p>
</div>
<div id="ref-thomas1973boolean">
<p>Thomas, René. 1973. “Boolean Formalization of Genetic Control Circuits.” <em>Journal of Theoretical Biology</em> 42 (3). Elsevier: 563–85.</p>
</div>
<div id="ref-trefois2015critical">
<p>Trefois, Christophe, Paul MA Antony, Jorge Goncalves, Alexander Skupin, and Rudi Balling. 2015. “Critical Transitions in Chronic Disease: Transferring Concepts from Ecology to Systems Medicine.” <em>Current Opinion in Biotechnology</em> 34. Elsevier: 48–55.</p>
</div>
<div id="ref-tyson2011dynamic">
<p>Tyson, John J, William T Baumann, Chun Chen, Anael Verdugo, Iman Tavassoly, Yue Wang, Louis M Weiner, and Robert Clarke. 2011. “Dynamic Modelling of Oestrogen Signalling and Cell Fate in Breast Cancer Cells.” <em>Nature Reviews Cancer</em> 11 (7). Nature Publishing Group: 523–32.</p>
</div>
<div id="ref-tyson2003sniffers">
<p>Tyson, John J, Katherine C Chen, and Bela Novak. 2003. “Sniffers, Buzzers, Toggles and Blinkers: Dynamics of Regulatory and Signaling Pathways in the Cell.” <em>Current Opinion in Cell Biology</em> 15 (2). Elsevier: 221–31.</p>
</div>
<div id="ref-villaverde2014reverse">
<p>Villaverde, Alejandro F, and Julio R Banga. 2014. “Reverse Engineering and Identification in Systems Biology: Strategies, Perspectives and Challenges.” <em>Journal of the Royal Society Interface</em> 11 (91). The Royal Society: 20130505.</p>
</div>
<div id="ref-zanudo2017network">
<p>Zañudo, Jorge Gómez Tejeda, Maurizio Scaltriti, and Réka Albert. 2017. “A Network Modeling Approach to Elucidate Drug Resistance Mechanisms and Predict Combinatorial Drug Treatments in Breast Cancer.” <em>Cancer Convergence</em> 1 (1). Springer: 5.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="cancer-as-deregulation-of-complex-machinery.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="logical-modeling-principles-and-data-integration.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
