<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Personalized logical models to study an interpret drug response | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Personalized logical models to study an interpret drug response | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Personalized logical models to study an interpret drug response | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="personalization-of-logical-models-method-and-prognostic-validation.html"/>
<link rel="next" href="information-flows-in-mechanistic-models-of-cancer.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="remerciements.html"><a href="remerciements.html"><i class="fa fa-check"></i>Remerciements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#introducing-the-diversity-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#tools-for-logical-modeling"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.1</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.2</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.3</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#comparison-of-the-mechanistic-approach-with-machine-learning-methods"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#limitations-and-perspectives"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information-in-information-out"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artifical-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artifical examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-inference-methods-to-leverage-data"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#limitations-and-perspectives-1"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="personalized-logical-models-to-study-an-interpret-drug-response" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Personalized logical models to study an interpret drug response</h1>
<blockquote>
<p><em>Il serait excellent que tout médecin ait la possibilité d’expérimenter un grand nombre de médicaments sur lui-même. Sa compréhension de leurs effets en serait tout autre.</em><br/> Mikhail Bulgakov (Morphine, 1927)</p>
</blockquote>
<p>Historically, all mechanistic models of molecular networks, and logical models in particular, have been widely used to study response to treatments <span class="citation">(Flobak et al. <a href="#ref-flobak2015discovery">2015</a>; Jastrzebski et al. <a href="#ref-jastrzebski2018integrative">2018</a>)</span>. Indeed, biological entities, many of which are prospective therapeutic targets, are explicitly represented in the model, making it possible to simulate their inhibition. This is what will be presented in this chapter using the personalized logical models described above. Can they be used to study the response of biological systems to perturbations, in this case the response of cell lines to gene or protein inhibitions? Compared to the numerous statistical models designed to predict the sensitivity of cell lines to treatements, what information do these personalized mechanistic models provide?</p>

<div class="summarybox">
<h4 id="scientific-content-3" class="unnumbered">Scientific content</h4>
<p>This chapter extends the method presented in the previous chapter to investigate drug response with personalized logical models. The first application to cell lines of all cancer types was presented orally at ISMB2020 in Basel but is not published.</p>
<p>The example about BRAF in melanomas and colorectal cancers is under review and the corresponding pre-print is available as <span class="citation">Béal et al. (<a href="#ref-beal2020personalized">2020</a>)</span>. In this joint work, only the construction of the generic logical model was mostly carried out by collaborators and will therefore be described more succinctly.</p>
<p>Finally, the work on prostate cancer presented in a third section will be submitted soon. It is also a joint work, in which my participation focused on the application of the PROFILE method.</p>
</div>
<p></p>
<div id="one-step-further-with-drugs" class="section level2">
<h2><span class="header-section-number">6.1</span> One step further with drugs</h2>
<p>One of the main clinical consequences of the underlying molecular complexity of cancers is the divergent response to treatment, even for <em>a priori</em> similar tumors. In the light of high-throughput sequencing data, the mechanisms governing these responses are somewhat better understood, for patients and especially for model organisms such as cell lines <span class="citation">(Heiser et al. <a href="#ref-heiser2012subtype">2012</a>; Garnett et al. <a href="#ref-garnett2012systematic">2012</a>)</span>. But beyond a few simple cases, the diversity of response biomarkers once again calls for <strong>holistic approaches</strong> to unravel the underlying mechanisms.</p>
<div id="modeling-response-to-cancer-treatments" class="section level3">
<h3><span class="header-section-number">6.1.1</span> Modeling response to cancer treatments</h3>
<p>To study these observed differences in drugs response in various cancers, some approaches based on mathematical modeling were developed to explore the complexity of differential drug sensitivities. A number of <strong>machine learning-based methods for predicting sensitivities</strong> have been proposed <span class="citation">(Costello et al. <a href="#ref-costello2014community">2014</a>)</span>, either without particular constraints or with varying degrees of prior knowledge; but they do not necessarily provide a mechanistic understanding of the response. Some other approaches focused on the description of the processes that might influence the response by integrating knowledge of the signaling pathways and their mechanisms and translated it into a mathematical model <span class="citation">(Eduati et al. <a href="#ref-eduati2017drug">2017</a>; Jastrzebski et al. <a href="#ref-jastrzebski2018integrative">2018</a>; Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>. The first step of this approach implies the construction of a network recapitulating knowledge of the interactions between selected biological entities (driver genes but also key genes of signaling pathways), extracted from the literature or from public pathway databases, or directly inferred from data <span class="citation">(Verny et al. <a href="#ref-verny2017learning">2017</a>)</span>. This static representation of the mechanisms is then translated into a dynamical mathematical model with the goal to not only understand the observed differences <span class="citation">(Jastrzebski et al. <a href="#ref-jastrzebski2018integrative">2018</a>)</span> but also to predict means to revert unexpected behaviours.</p>
<p>One way to address issues related to patient response to treatments is to <strong>fit these mechanistic models to the available data</strong>, and to train them on high-throughput cell-line specific perturbation data <span class="citation">(Eduati et al. <a href="#ref-eduati2017drug">2017</a>; Jastrzebski et al. <a href="#ref-jastrzebski2018integrative">2018</a>; Klinger et al. <a href="#ref-klinger2013network">2013</a>)</span>. These mechanistic models are then easier to interpret with regard to the main drivers of drug response. They also enable the <em>in silico</em> simulations of new designs such as combinations of drugs not present in the initial training data <span class="citation">(Fröhlich et al. <a href="#ref-frohlich2018efficient">2018</a>)</span>. However, these mechanistic models contain many parameters that need to be fitted or extracted from the literature. Some parsimonious mathematical formalisms have been developed to make up for the absence of either rich perturbation data to train the models or fully quantified kinetic or molecular data to derive the parameters directly from literature. One of these approaches is the logical modeling, which uses discrete variables governed by logical rules. Its explicit syntax facilitates the interpretation of mechanisms and drug response <span class="citation">(Zañudo, Scaltriti, and Albert <a href="#ref-zanudo2017network">2017</a>; Iorio et al. <a href="#ref-iorio2016landscape">2016</a>)</span> and despite its simplicity, semi-quantitative analyses have already been performed on complex systems including drug response studies <span class="citation">(Knijnenburg et al. <a href="#ref-knijnenburg2016logic">2016</a>; Eduati et al. <a href="#ref-eduati2020patient">2020</a>)</span>.</p>
</div>
<div id="an-application-of-personalized-logical-models" class="section level3">
<h3><span class="header-section-number">6.1.2</span> An application of personalized logical models</h3>
<p>But logical formalism has also shown its relevance regarding drug response in cases where the model is not automatically trained on data but simply constructed from literature or pathway databases and where biological experiments focus on a particular cell line <span class="citation">(Flobak et al. <a href="#ref-flobak2015discovery">2015</a>)</span>. The study is then restricted to one cell line only from which some data and parameters have been experimentally inferred. Using the PROFILE method, it is possible to generate personalized logical models associated with different cell lines and then use them to study the response to treatment. <strong>Since the models are not trained with perturbation data but simply specified/constrained by interpreting the molecular profiles, it is possible to do this with a rather limited amount of data</strong>.</p>
<div class="figure" style="text-align: center"><span id="fig:PROFILE-drug"></span>
<img src="fig/PROFILE-drug.png" alt="Schematic extension of PROFILE-personalized logical models to drug investigation. (A) Schematic representation of a logical model of cancer molecular networks, in particular the one described in appendix B.2 and used in the next subsection. (B) Sequential pipeline for drug response investigation with PROFILE, starting from a generic logical model, then transformed into several personalized models with different molecular profiles (correspondong to several cell lines); these models are finally simulated with a defined drug inhibition. (C) A possible analysis of the predictions of personalized models obtained from the generic model described in (A); the position of the personalized models in the PCA phenotype (or read-out) space of the model is studied as well as the impact of the treatment on this position." width="90%" />
<p class="caption">
Figure 6.1: <strong>Schematic extension of PROFILE-personalized logical models to drug investigation.</strong> (A) Schematic representation of a logical model of cancer molecular networks, in particular the one described in appendix <a href="about-logical-models.html#appendix-verlingue">B.2</a> and used in the next subsection. (B) Sequential pipeline for drug response investigation with PROFILE, starting from a generic logical model, then transformed into several personalized models with different molecular profiles (correspondong to several cell lines); these models are finally simulated with a defined drug inhibition. (C) A possible analysis of the predictions of personalized models obtained from the generic model described in (A); the position of the personalized models in the PCA phenotype (or read-out) space of the model is studied as well as the impact of the treatment on this position.
</p>
</div>

<p>The principles are summarized in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>. A generic model (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>A) is first transformed into as many personalized models as there are cell lines with an omics profile. These persnalized models are then simulated by adding the effect of a given treatment (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>B). The treatments that can be studied are generally targeted inhibitors. Generally speaking, one must be able to <strong>translate the mechanism of action of the treatment into the logical model</strong>. The impact of more systemic treatments such as chemotherapy or radiotherapy is more difficult to study with these methods, in any case with most of the logical models published to date, even if in theory, precise modeling of the pathways associated with these treatments (such as DNA repair) could contribute to this.</p>
<p>It is then possible to analyze the personalized scores for each cell line (asymptotic values of the phenotypic read-outs of the model) with or without the effect of treatment. If the model includes more than two phenotypes of interest, such as the one in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>A, one can visualize these behaviors in the PCA space of the personalized scores, as shown schematically in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>C. In this case the directions of the original phenotype features (<em>Proliferation</em>, <em>Apoptosis</em>, <em>Quiescence</em>) have been added in the PCA-transformed space in order to facilitate the interpretation of positions and drug-induced displacements. In this mock example, based on personalized models, treatment would promote a shift from proliferative to more apoptotic or quiescent behaviors, in particular in the red and green cell lines, which are <em>a priori</em> more sensitive to the treatment.</p>
</div>
<div id="a-pan-cancer-attempt" class="section level3">
<h3><span class="header-section-number">6.1.3</span> A pan-cancer attempt</h3>
<p>This versatile analysis framework was first applied during this thesis to a <strong>large pan-drug and pan-cancer analysis</strong>. On the basis of generic logical models such as those previously presented (see appendix <a href="about-logical-models.html#appendix-fumia">B.1</a> and <a href="about-logical-models.html#appendix-verlingue">B.2</a>), and in view of the abundance of available data (across cancer tissues and drugs such as in GDSC cell line dataset, see appendix <a href="about-logical-models.html#appendix-fumia">B.1</a>), there were no theoretical obstacle to such an analysis. Although the simulations were carried out without any problems, the analysis nevertheless proved extremely difficult to interpret. We will highlight the various problems encountered, propose an illustration and some perspectives that led to the work presented in the following section.</p>
<p>Based on the PROFILE methodology and GDSC data, hundreds of personalized models can be obtained, each corresponding to a cell line. For each of these personalized models, several dozen of potential drugs have a mechanism of action that can be mechanistically translated into the logical model. We thus obtain <strong>tens of thousands of “personalized model/drug” pairs that correspond to experimentally evaluated drug sensitivies</strong> (cf appendix <a href="appendix-datasets.html#appendix-GDSC">A.1.2</a> for details). Firstly, the comparison of simulated and experimental data is not straightforward. As the models are qualitative, it is necessary to carry out the validation in this spirit. The idea is not to predict sensitivity quantitatively, rather to verify their relative relevance. In the first place, do we recover the cell lines that are most sensitive to a given drug? With several hundred cell lines, it is difficult to make this reflection graphically as in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>C. More quantitative approaches, such as correlation, would require the <strong>definition of a precise sensitivity proxy in personalized models</strong>. Should we choose the personalised <em>Proliferation</em> score obtained with drug? Or the drug-induced displacement in the mechanistic model (the drug arrows in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>C)? Or is a combination of phenotypes used, if so which one? As for experimental metrics, which ones to choose, and what interpretations do they allow? Whatever the choice, dose-response AUC or IC50 (see details in the appendix <a href="appendix-datasets.html#appendix-GDSC">A.1.2</a>), a problem arises: can the sensitivities of a cell line to different drugs be compared? Such a comparison would allow the most clinically interesting questions of precision medicine to be asked: for a given molecular profile, can the model predict the best treatment to administer? However, AUCs are comparable for different drugs only if the concentration ranges tested are similar; and IC50s are extrapolated, sometimes well beyond the concentrations tested. Qualitative comparisons for a given drug therefore seem the most meaningful, as long as a relevant proxy in personalized models can be justified.</p>
<p>Aware of these difficulties, if one decides to do a correlation analysis, for each drug, of the personalized correlation scores with experimental sensitivities, one realizes that responses to <strong>certain drugs are correctly predicted when others are not correlated at all</strong>. But it is difficult to decide between two different interpretations: does this mean that correctly predicted drugs are well modeled and others are not? Or does it mean that some correlations appear to be better by chance because so many drugs have been modeled? A case study can be illustrated more precisely with the example shown in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-PCA">6.2</a>. In order to simplify the analysis presented schematically in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-drug">6.1</a>C, the 663 cell lines were averaged by cancer type (according to <a href="https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables/tcga-study-abbreviations">TCGA denominations</a>) and the drug-induced shifts are all represented from the origin in the PCA space. There is evidence that the effect of the drug on personalized models (using only mutations) tends to make them less proliferative and more apoptotic/quiescent (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-PCA">6.2</a>A). This shift is strongest for those types of cancer that are actually most sensitive to this inhibitor experimentally (<em>i.e.</em>, low AUC), such as skin cutaneous melanomas (SKCM) in particular, and colorectal (COAD/READ) or pancreatic (PAAD) cancers to a lesser extent. The ability of personalized models to explain this difference can be understood by a known underlying biological reality: the prevalence of BRAF or RAS mutations in these cancers. The three aforementioned cancers are thus very frequently mutated for one of the two genes (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-PCA">6.2</a>B). Then, the model translates the fact that these two genes are located just upstream of MAP2K1. It is therefore natural that an inhibition just downstream of these important mutations is particularly effective (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:PROFILE-PCA">6.2</a>C). In a case such as this, the relevance of the model can be explained and justified <em>a posteriori</em>. This analysis is much more difficult in the vast majority of cases, whether the correlations are apparently good or not.</p>
<div class="figure" style="text-align: center"><span id="fig:PROFILE-PCA"></span>
<img src="06-Drugs_files/figure-html/PROFILE-PCA-1.png" alt="PROFILE-generated models and sensitivites to MAP2K1 inhibitors averaged per cancer type. (A) Effects of MAP2K1 inhibitors on personalized logical models averaged per cancer types and represented in a normalized PCA space with super-imposed original phenotypes. (B) Proportion of BRAF- and RAS-mutated cell lines in some cancer types. (C) Zoom on the MAPK pathway of the logical model used." width="90%" />
<p class="caption">
Figure 6.2: <strong>PROFILE-generated models and sensitivites to MAP2K1 inhibitors averaged per cancer type.</strong> (A) Effects of MAP2K1 inhibitors on personalized logical models averaged per cancer types and represented in a normalized PCA space with super-imposed original phenotypes. (B) Proportion of BRAF- and RAS-mutated cell lines in some cancer types. (C) Zoom on the MAPK pathway of the logical model used.
</p>
</div>

<p>This example highlights a problem of scope. <strong>The fact that the method enables to study hundreds of cell lines and dozens of drugs does not mean that it is relevant in each case</strong>. The description of pathways in the model is more or less accurate. For example, a node at the model boundaries probably has many regulators missing. Is it then relevant to investigate the response of personalized models to its inhibition? It is therefore necessary to restrict the drugs studied. Similarly, even if the logical model summarizes many important pathways, it is probably unsuitable for certain cell lines or certain types of cancer with different etiologies. However, it is difficult to restrict the scope of the analysis in an unbiased way without having designed a model <em>de novo</em> for a specific purpose.</p>
<p>For all these reasons, it was decided to leave aside this naive, broad-spectrum approach in favour of starting from a more specific biological question and constructing the appropriate logical model.</p>
</div>
</div>
<div id="case-study-on-braf-in-melanoma-and-colorectal-cancers" class="section level2">
<h2><span class="header-section-number">6.2</span> Case study on BRAF in melanoma and colorectal cancers</h2>
<p>In order to address the limitations outlined in this exploratory analysis, we propose here a pipeline based on logical modeling and able to go from the formulation of a specific biological question to the validation of a mathematical model on pre-clinical data, in this case a set of cell lines, and the subsequent interpretation of potential resistance mechanisms (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-GA">6.3</a>). As before, <strong>one of the main points of differentiation with existing mechanistic approaches, is that this framework does not rely on any training of parameters but only on the automatic integration and interpretation of molecular features</strong>.</p>
<div class="figure" style="text-align: center"><span id="fig:BRAF-GA"></span>
<img src="fig/BRAF-GA.png" alt="BRAF modeling flowchart: from a biological question to validated personalized logical models. Logical models are written with MaBoSS, and the checking model procedure is therefore provided in the same formalism. Cell line data are taken from Cell Model Passports (Meer et al. 2019)." width="90%" />
<p class="caption">
Figure 6.3: <strong>BRAF modeling flowchart: from a biological question to validated personalized logical models.</strong> Logical models are written with MaBoSS, and the checking model procedure is therefore provided in the same formalism. Cell line data are taken from Cell Model Passports <span class="citation">(Meer et al. <a href="#ref-van2019cell">2019</a>)</span>.
</p>
</div>

<div id="biological-and-clinical-context" class="section level3">
<h3><span class="header-section-number">6.2.1</span> Biological and clinical context</h3>
<p>The construction of a mathematical model must be based first and foremost on a precise and specific biological question, at the origin of the design of the model. Here, we choose to explore the different responses to treatments in tumors from diverse cancers that bear the same mutation. A well-studied example of these variations is the BRAF mutation and especially its V600E substitution. BRAF is mutated in 40 to 70% of melanoma tumors and in 10% of colorectal tumors, each time composed almost entirely of V600E mutations <span class="citation">(Cantwell-Dorris, O’Leary, and Sheils <a href="#ref-cantwell2011brafv600e">2011</a>)</span>. In spite of these similarities, <strong>BRAF inhibition treatments have experienced opposite results with improved survival in patients with melanoma <span class="citation">(Chapman et al. <a href="#ref-chapman2011improved">2011</a>)</span> and significant resistance in colorectal cancers <span class="citation">(Kopetz et al. <a href="#ref-kopetz2010plx4032">2010</a>)</span></strong>, suggesting drastic mechanistic differences. Some subsequent studies have proposed context-based molecular explanations, often highlighting surrounding genes or signalling pathways, such as a feedback activation of EGFR <span class="citation">(Prahallad et al. <a href="#ref-prahallad2012unresponsiveness">2012</a>)</span> or other mechanisms <span class="citation">(Poulikakos et al. <a href="#ref-poulikakos2011raf">2011</a>; Sun et al. <a href="#ref-sun2014reversible">2014</a>)</span>. These various findings support the need for an integrative mechanistic model able to formalize and explain more systematically the differences in drug sensitivities depending on the molecular context. The purpose of the study we propose here is not to provide a comprehensive molecular description of the response but to verify that the existence and functionality of the suggested feedback loops around the signalling pathway in which BRAF is involved <span class="citation">(Prahallad et al. <a href="#ref-prahallad2012unresponsiveness">2012</a>)</span> may be a first hint towards these differences. For a more thorough study of these cancers, we refer to other works <span class="citation">(Eduati et al. <a href="#ref-eduati2017drug">2017</a>; Baur et al. <a href="#ref-baur2020connecting">2020</a>; Cho et al. <a href="#ref-cho2016attractor">2016</a>)</span>.</p>
</div>
<div id="a-logical-model-centred-on-braf" class="section level3">
<h3><span class="header-section-number">6.2.2</span> A logical model centred on BRAF</h3>
<p>A logical model summarizing the main molecular interactions at work in colorectal cancers and melanomas is thus built from the literature and completed with databases. As previously mentioned, the objective is to understand whether it is possible to model and explain differences in responses to BRAF inhibition in melanoma and colorectal cancer patients using the same regulatory network. <strong>The fact that the two cancers share the same network but differ from the alterations and expression of their genes constitutes our prior hypothesis</strong>. The focus of this model is put on two important signaling pathways involved in the mechanisms of resistance to BRAF inhibition which are the ERK1/2 MAPK and PI3K/AKT pathways <span class="citation">(Ursem, Atreya, and Van Loon <a href="#ref-ursem2018emerging">2018</a>; Rossi et al. <a href="#ref-rossi2019drug">2019</a>)</span>. The generic network presented in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-model">6.4</a> recapitulates the known interactions between the biological entities of the network that was first built from the literature, and then verified and completed with potential missing connections using SIGNOR database <span class="citation">(Perfetto et al. <a href="#ref-perfetto2016signor">2016</a>)</span>. More details and references about the model can be found in appendix <a href="about-logical-models.html#appendix-pantolini">B.3</a>. All in all, the logical model formalizes the knowledge compiled from different sources and highlights the role of SOX10, FOXD3, CRAF, PI3K, PTEN and of EGFR in resistance to anti-BRAF treatments.</p>
<div class="figure" style="text-align: center"><span id="fig:BRAF-model"></span>
<img src="fig/BRAF-model.png" alt="Logical model of signaling pathways around BRAF in colorectal andmelanoma cancers. Grey nodes represent input nodes, which may correspond to the environmental conditions. Square nodes represent multi-valued variable (MEK, ERK, p70 and Proliferation). Dark blue nodes accounts for families (several genes/entities for one node). Light blue node represents the phenotypic read-out of the model, i.e., Proliferation." width="90%" />
<p class="caption">
Figure 6.4: <strong>Logical model of signaling pathways around BRAF in colorectal andmelanoma cancers.</strong> Grey nodes represent input nodes, which may correspond to the environmental conditions. Square nodes represent multi-valued variable (MEK, ERK, p70 and Proliferation). Dark blue nodes accounts for families (several genes/entities for one node). Light blue node represents the phenotypic read-out of the model, <em>i.e.</em>, <em>Proliferation</em>.
</p>
</div>

<p>Once the structure of the model was defined, and before moving on to its personalization, its consistency with the literature was checked using a <strong>model-checking procedure</strong>. Indeed, due to the complexity of the system, properly taking into account the interactions between entities does not automatically guarantee that the model will reproduce certain dynamic behaviours. An example of a biological assertion may be the reactivation of the MAPK (mitogen-activated protein kinase) pathway through EGFR signal after BRAF inhibition in colorectal cancer <span class="citation">(Prahallad et al. <a href="#ref-prahallad2012unresponsiveness">2012</a>)</span>: it is possible to check whether a simulation of this situation with the model gives the same result or not. Because there are many such assertions and because it is useful to verify them as the model is built, automatic model-checking tools have been defined, based on the MaBoSS syntax and inspired by the Python <em>unittest</em> library. More details are provided in <span class="citation">Béal et al. (<a href="#ref-beal2020personalized">2020</a>)</span> and in a corresponding <a href="https://github.com/sysbio-curie/MaBoSS_test">GitHub repository</a>. The list of biological assertions used to validate the model is detailed in the appendix <a href="about-logical-models.html#appendix-pantolini">B.3</a>.</p>
</div>
<div id="cell-lines-data" class="section level3">
<h3><span class="header-section-number">6.2.3</span> Cell lines data</h3>
<p>The omics profiles of colorectal and melanoma cell lines are downloaded from Cell Model Passports portal <span class="citation">(Meer et al. <a href="#ref-van2019cell">2019</a>)</span>. 64 colorectal cancer (CRC) cell lines and 65 cutaneaous melanoma (CM) cell lines are listed in the database, with at least mutation or RNA-seq data (59 CM and 53 CRC with both mutations and RNA-seq data). These omics profiles are used to generate cell-line-specific logical models as described in PROFILE method (Figure <a href="personalization-of-logical-models-method-and-prognostic-validation.html#fig:PROFILE-abstract">5.1</a>). The prevalence of mutations and their combination for the two types of cancer can be seen in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-GDSC">6.5</a>A and is consistent with the clinical situation described above with melanomas more frequently BRAF-mutated and colorectal cancers more frequently RAS-mutated.</p>
<div class="figure" style="text-align: center"><span id="fig:BRAF-GDSC"></span>
<img src="06-Drugs_files/figure-html/BRAF-GDSC-1.png" alt="Descriptive analysis of cell lines for melanomas and colorectal cancers. (A) Number of cell lines for the four most frequently mutated genes and their combinations (plot from UpSetR package (Conway, Lex, and Gehlenborg 2017)). (B) Differential sensitivities to BRAF inhibition by the drug PLX-4720 (lower panel) or by CRISPR inhibition (upper panel), depending on BRAF mutational status and cancer type. Numbers of cell lines in each category are indicated. Note that high sensitivities correspond to low AUC and high scaled Bayesian factors." width="95%" />
<p class="caption">
Figure 6.5: <strong>Descriptive analysis of cell lines for melanomas and colorectal cancers.</strong> (A) Number of cell lines for the four most frequently mutated genes and their combinations (plot from UpSetR package <span class="citation">(Conway, Lex, and Gehlenborg <a href="#ref-conway2017upsetr">2017</a>)</span>). (B) Differential sensitivities to BRAF inhibition by the drug PLX-4720 (lower panel) or by CRISPR inhibition (upper panel), depending on BRAF mutational status and cancer type. Numbers of cell lines in each category are indicated. Note that high sensitivities correspond to low AUC and high scaled Bayesian factors.
</p>
</div>

<p>In order to validate the relevance of personalized models to explain differential sensitivities to drugs, some experimental screening datasets are used. <strong>Drug screening data</strong> are downloaded from the Genomics of Drug Sensitivity in Cancer (GDSC) dataset <span class="citation">(Yang et al. <a href="#ref-yang2012genomics">2012</a>)</span> which includes two BRAF inhibitors: PLX-4720 and Dabrafenib. The cell lines are treated with increasing concentration of drugs and the viability of the cell line relative to untreated control is measured. The dose-response relative viability curve is fitted and then used to compute the <strong>area under the dose-response curve (AUC)</strong> <span class="citation">(Vis et al. <a href="#ref-vis2016multilevel">2016</a>)</span>. AUC is a value between <span class="math inline">\(0\)</span> and <span class="math inline">\(1\)</span>: values close to <span class="math inline">\(1\)</span> mean that the relative viability has not been decreased, and lower values correspond to increased sensitivity to inhibitions (details in appendix <a href="appendix-datasets.html#appendix-GDSC">A.1.2</a>). The results obtained with the two drugs are very strongly correlated (Pearson correlation of <span class="math inline">\(0.91\)</span>) and the analyses presented here will therefore focus on only one of them, PLX-4720.</p>
<p>In a complementary way, some results of <strong>CRISPR/Cas9 screening</strong> are also downloaded from Cell Model Passports. This technology, which is described in more detail in the appendix <a href="appendix-datasets.html#appendix-CRISPR">A.1.3</a>, allows targeted inhibitions of certain genes. Two different datasets from Sanger Institute <span class="citation">(Behan et al. <a href="#ref-behan2019prioritization">2019</a>)</span> and Broad Institute <span class="citation">(Meyers et al. <a href="#ref-meyers2017computational">2017</a>)</span> are available. We use <strong>scaled Bayesian factors</strong> to assess the effect of CRISPR targeting of genes. These scores are computed based on the fold change distribution of single guide RNAs <span class="citation">(Hart and Moffat <a href="#ref-hart2016bagel">2016</a>)</span>. The highest values indicate that the targeted gene is essential to the cell fitness. The agreement between the two databases is good <span class="citation">(Dempster et al. <a href="#ref-dempster2019agreement">2019</a>)</span> but we choose to focus on the Broad database, which is more balanced in terms of the relative proportions of melanomas and colorectal cancers.</p>
<p>Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-GDSC">6.5</a>B illustrates both the relative quantities of cell lines for which drug or CRISPR screening data are available (depending on their BRAF status) as well as differences in sensitivity to BRAF inhibition. The greater sensitivity of BRAF-mutated melanomas compared to BRAF-mutated colorectal cancers is well observed for PLX-4720. However, the overlap in the distributions requires a deeper look into the data and a search for more precise explanations of the differences in sensitivity, including within each type of cancer. The finding appears to be similar for CRISPR despite a sample size that is too small; the higher average sensitivity of melanomas even extends to non-mutated BRAF.</p>
</div>
<div id="validation-of-personalized-models-using-crisprcas9-and-drug-screening" class="section level3">
<h3><span class="header-section-number">6.2.4</span> Validation of personalized models using CRISPR/Cas9 and drug screening</h3>
<p>The validation of personalized logical models using these screening data is done with the following rationale. First, the models are personalized using omics data from the cell lines. Then, two separate simulations are performed for each personalized model: one without the inhibition, the other by creating and activating a BRAF inhibitor to mimic the drug or CRISPR inhibition. The ratio of the <em>Proliferation</em> phenotype obtained with inhibition and without inhibition is the proxy used to be compared with the different screening metrics each of which is also standardized (AUC calculated on relative viability for drugs and Bayes factor computed from fold-changes and then scaled).</p>
<div id="diff-BRAF" class="section level4">
<h4><span class="header-section-number">6.2.4.1</span> Differential sensitivities to BRAF targeting explained by personalized logical models</h4>
<p>Once the logical model consistency has been validated, personalized models are generated for each cell line by integrating their interpreted genomic features directly as model constraints or parameters. <strong>Sensitivities to BRAF inhibition inferred from models are then compared to experimentally observed sensitivities</strong> (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>). In all the following analyses, we focus on three different personalization strategies using: only mutations as discrete personalization (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>A, upper row), only RNA as continuous personalization (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>A, middle row) or mutations combined with RNA (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>A, lower row). These choices reflect first of all the following <em>a priori</em>: mutations are much more drastic and permanent changes than RNA, whose expression levels are more subject to fluctuation and regulation. The objective is also to answer the following questions: What type of data is most likely to explain the differences in responses? Is it relevant to combine them? Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a> shows an example of the type of analyses possible with personalized models, zooming in more and more on the details from panel A to panel C.</p>
<div class="figure" style="text-align: center"><span id="fig:BRAF-results"></span>
<img src="fig/BRAF-results.png" alt="Validation of personalized models of BRAF inhibition with cell lines data. (A) Pearson correlations between normalized Proliferation scores from models and experimental sensitivities to BRAF inhibition (drug or CRISPR); only significant correlations are displayed. (B) Scatter plots with non-overlapping points corresponding to correlations of panel A for one drug (PLX-4720) and one CRISPR dataset (Broad) only. (C) Enlargement of one scatter plot in B (left) with the table describing the omics profiles used for each cell line to explore the response mechanisms (right); interactive version in Figure 6.7 or GitHub files." width="90%" />
<p class="caption">
Figure 6.6: <strong>Validation of personalized models of BRAF inhibition with cell lines data.</strong> (A) Pearson correlations between normalized <em>Proliferation</em> scores from models and experimental sensitivities to BRAF inhibition (drug or CRISPR); only significant correlations are displayed. (B) Scatter plots with non-overlapping points corresponding to correlations of panel A for one drug (PLX-4720) and one CRISPR dataset (Broad) only. (C) Enlargement of one scatter plot in B (left) with the table describing the omics profiles used for each cell line to explore the response mechanisms (right); interactive version in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-interactive">6.7</a> or <a href="https://github.com/sysbio-curie/PROFILE_BRAF_Model/blob/master/Analysis.html">GitHub files</a>.
</p>
</div>

<p>The first approach consists in using only mutations as discrete personalization (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>, A, upper row): the mutations identified in the dataset and that are present in the regulatory network are set to 1 for activating mutations and set to 0 for inactivating mutations. In this case, the <em>Proliferation</em> scores from personalized models significantly correlate with both BRAF drug inhibitors (PLX-4720 and Dabrafenib) and both CRISPR datasets (using Pearson correlations). Note that the opposite directions of the correlations for the drug and CRISPR datasets are due to the fact that cell lines sensitive to BRAF inhibition result in low AUCs, and high scaled Bayesian factors, respectively, and, if the models are relevant, to low standardized <em>Proliferation</em> scores. Looking more closely at the corresponding scatter plot for PLX-4720 (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>B, upper left), it can be seen that this correlation results from the <strong>model’s ability to use mutations’ information to recover the highest sensitivity of the BRAF-mutated cell lines</strong> that form an undifferentiated cluster on the left side. These cell lines are indeed relatively more sensitive than non-mutated BRAF cell lines. However, the integration of mutations alone does not explain the significant differences within this subgroup (AUC between 0.55 and 0.9). A very similar behaviour can be observed when comparing model simulations with CRISPR data (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>B, upper right).</p>
<p>Using only RNA data as continuous personalization (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>A and B, middle rows) is both less informative and more difficult to interpret. For continuous data such as RNA-sequencing data, we normalize the expression values and set both the initial conditions and the transition rates of the model variables to the corresponding values. Correlations with experimental BRAF inhibitions appear weaker and more uncertain. The key point, however, is that the <strong>combination of mutations and RNA, as depicted in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a> A and B lower rows, seems to be more relevant</strong>. This is partially true in quantitative terms but it is even easier to interpret in the corresponding scatter plots (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-interactive">6.7</a>). Comparing first the Broad CRISPR scatter plots using mutations only (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>B, upper right) and using both mutations and RNA (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>B, lower right), we can observe that non-responsive cell lines (scaled Bayesian factor below 0), grouped in the lower right corner and correctly predicted using only mutations stayed in the same area: these strong mutational phenotypes have not been displaced by the addition of RNA data. Other cell lines previously considered to be of intermediate sensitivity by the model (<em>e.g.</em>, COLO-678 or SK-MEL-2) were shifted to the right, consistent with the lack of sensitivity observed experimentally. Finally, BRAF-mutated cell lines, previously clustered in one single column on the left using only mutations (with normalized <em>Proliferation</em> scores around 0.5), have been moved in different directions. Many of the most sensitive cell lines (scaled Bayesian factor above 2) have been pushed to the left in accordance with the high sensitivities observed experimentally (<em>e.g.</em>, HT-29 or SK-MEL-24). It is even observed that the model corrected the position of the two BRAF mutated cell lines, but whose sensitivity is experimentally low (melanoma cell line HT-144 and colorectal cell line HT-55). Only one cell line (SK-MEL-30) has seen its positioning evolve counter-intuitively as a result of the addition of RNA in the personalization strategy: relatively sensitive to the inhibition of BRAF, it has, however, seen its standardised <em>Proliferation</em> score approach 1. All in all, this contribution of RNA data results in significant correlations even when restricted to BRAF-mutated cell lines only (<span class="math inline">\(R=0.69\)</span>, <span class="math inline">\(p.value=0.006\)</span>).</p>
<div class="figure" style="text-align: center"><span id="fig:BRAF-interactive"></span>
<div id="htmlwidget-126cce2504cee9260e66" style="width:90%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-126cce2504cee9260e66">{"x":{"data":[{"x":[0.657366812807227,0.50069866010391,0.476628245892952,0.954255457995634,0.474161179878852,0.50243290864733,0.696903856599674,0.667741069030352,0.475920157068063,0.464198412698413,0.67029610941442,0.463516006511123,0.657971785999468,0.482868874172185,0.462960179324894],"y":[4.32459783253377,3.49819817512565,3.46082419383759,-0.0688895464576371,-0.0134927580430537,4.10719566722206,0.112254534228113,-1.55755323289045,2.31809582877849,4.16717875883292,0.0387058411151215,4.23935672985528,2.31941524290884,4.93426920761054,5.35458281508404],"text":["A2058 (CM)<br />Mutations->MEK:1,BRAF:1,p53:0<br />RNA->ERK:0.34,p70:0.69,PDPK1:0.16,CRAF:0.58,","C32 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.43,p70:0.65,PDPK1:0.5,CRAF:0.77,","COLO-679 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.59,p70:0.63,PDPK1:0.16,CRAF:0.5,","COLO-792 (CM)<br />Mutations:None<br />RNA->ERK:0.62,p70:0.33,PDPK1:0.32,CRAF:0.77,","HT-144 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.46,p70:0.41,PDPK1:0.65,CRAF:0.92,","IGR-1 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.48,p70:0.46,PDPK1:0.32,CRAF:0.5,","IPC-298 (CM)<br />Mutations->RAS:1<br />RNA->ERK:0.65,p70:0.41,PDPK1:0.16,CRAF:0.5,","MEL-JUSO (CM)<br />Mutations->RAS:1<br />RNA->ERK:0.62,p70:0.63,PDPK1:0.32,CRAF:0.94,","RPMI-7951 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.37,p70:0.32,PDPK1:0.16,CRAF:0.72,","RVH-421 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.63,p70:0.24,PDPK1:0.65,CRAF:0.24,","SK-MEL-2 (CM)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.32,p70:0.26,PDPK1:0.32,CRAF:0.66,","SK-MEL-24 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.43,p70:0.41,PDPK1:0.32,CRAF:0.03,","SK-MEL-30 (CM)<br />Mutations->BRAF:1,RAS:1<br />RNA->ERK:0.44,p70:0.66,PDPK1:0.16,CRAF:0.58,","UACC-62 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.34,p70:0.5,PDPK1:0.06,CRAF:0.41,","WM793B (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.54,p70:0.67,PDPK1:0.76,CRAF:0.32,"],"frame":"Mutations","type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(180,117,98,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(180,117,98,1)"}},"hoveron":"points","name":"CM","legendgroup":"CM","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","visible":true},{"x":[0.93856453489481,0.66720128307939,1.01453205166008,0.4270615,0.494168694241687,1.01460460329316,0.655729028843609,1,0.67295179922985,0.4247765,0.690811535881958,1,0.675390572840166,0.663105888636667],"y":[-1.21982415420915,-1.94163726587743,-1.41212557900204,4.22554038953254,0.338178721390904,-1.44255283497272,0.45725570120034,-0.764980996447155,-1.46799285380722,3.63045427955743,-1.6300880286577,-0.660753136475747,-0.595467989996735,-0.986951714239596],"text":["C2BBe1 (CRC)<br />Mutations:None<br />RNA->ERK:0.67,p70:0.84,PDPK1:0.89,CRAF:0.85,","COLO-678 (CRC)<br />Mutations->RAS:1<br />RNA->ERK:0.25,p70:0.9,PDPK1:0.32,CRAF:0.58,","HT-115 (CRC)<br />Mutations->PI3K:1<br />RNA->ERK:0.5,p70:0.75,PDPK1:0.94,CRAF:0.9,","HT-29 (CRC)<br />Mutations->PI3K:1,BRAF:1,p53:0<br />RNA->ERK:0.46,p70:0.63,PDPK1:0.89,CRAF:0.03,","HT55 (CRC)<br />Mutations->BRAF:1<br />RNA->ERK:0.48,p70:0.58,PDPK1:0.94,CRAF:0.81,","KM12 (CRC)<br />Mutations->p53:0<br />RNA->ERK:0.12,p70:0.87,PDPK1:0.5,CRAF:0.5,","LoVo (CRC)<br />Mutations->RAS:1<br />RNA->ERK:0.29,p70:0.77,PDPK1:0.5,CRAF:0.66,","LS-180 (CRC)<br />Mutations->PI3K:1,RAS:1<br />RNA->ERK:0.53,p70:0.67,PDPK1:0.92,CRAF:0.58,","NCI-H747 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.38,p70:0.71,PDPK1:0.76,CRAF:0.41,","RKO (CRC)<br />Mutations->PI3K:1,BRAF:1<br />RNA->ERK:0.51,p70:0.58,PDPK1:0.5,CRAF:0.24,","SW1463 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.38,p70:0.5,PDPK1:0.89,CRAF:0.24,","SW48 (CRC)<br />Mutations->EGFR:1,PI3K:1,MEK:1<br />RNA->ERK:0.34,p70:0.54,PDPK1:0.76,CRAF:0.17,","SW620 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.39,p70:0.83,PDPK1:0.76,CRAF:0.32,","SW837 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.79,p70:0.63,PDPK1:0.84,CRAF:0.03,"],"frame":"Mutations","type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,40,38,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,40,38,1)"}},"hoveron":"points","name":"CRC","legendgroup":"CRC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","visible":true}],"layout":{"margin":{"t":28.7521793275218,"r":7.97011207970112,"b":45.4296388542964,"l":40.6475716064757},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.00552314976082763,1.11688691963919],"tickmode":"array","ticktext":["0.3","0.6","0.9"],"tickvals":[0.3,0.6,0.9],"categoryorder":"array","categoryarray":["0.3","0.6","0.9"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.98505603985056,"tickwidth":0.724555643609193,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":12.7521793275218},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(255,255,255,1)","gridwidth":0.724555643609193,"zeroline":false,"anchor":"y","title":{"text":"Proliferation scores from  personalized models","font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-2.3064482699255,5.71939381913211],"tickmode":"array","ticktext":["-2","0","2","4"],"tickvals":[-2,0,2,4],"categoryorder":"array","categoryarray":["-2","0","2","4"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.98505603985056,"tickwidth":0.724555643609193,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":12.7521793275218},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.724555643609193,"zeroline":false,"anchor":"x","title":{"text":"CRISPR screening metric<br />(scaled Bayesian factor)","font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(77,77,77,1)","width":0.929846409298464,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":2.06156048675734,"font":{"color":"rgba(0,0,0,1)","family":"","size":12.7521793275218}},"hovermode":"closest","barmode":"relative","sliders":[{"currentvalue":{"prefix":"Personalization with ","xanchor":"right","font":{"size":16,"color":"black"}},"steps":[{"method":"animate","args":[["Mutations"],{"transition":{"duration":500,"easing":"linear"},"frame":{"duration":500,"redraw":false},"mode":"immediate"}],"label":"Mutations","value":"Mutations"},{"method":"animate","args":[["Mutations & RNA"],{"transition":{"duration":500,"easing":"linear"},"frame":{"duration":500,"redraw":false},"mode":"immediate"}],"label":"Mutations & RNA","value":"Mutations & RNA"}],"visible":true,"pad":{"t":40},"lenmode":"fraction","len":0.5,"x":0.3,"y":-0.2}],"updatemenus":[{"type":"buttons","direction":"right","showactive":false,"y":0,"x":0,"yanchor":"top","xanchor":"right","pad":{"t":60,"r":5},"buttons":[{"label":"Play","method":"animate","args":[null,{"fromcurrent":true,"mode":"immediate","transition":{"duration":500,"easing":"linear"},"frame":{"duration":500,"redraw":false}}]}]}]},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"42757576f400":{"colour":{},"text":{},"frame":{},"x":{},"y":{},"type":"scatter"}},"cur_data":"42757576f400","visdat":{"42757576f400":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"frames":[{"name":"Mutations","data":[{"x":[0.657366812807227,0.50069866010391,0.476628245892952,0.954255457995634,0.474161179878852,0.50243290864733,0.696903856599674,0.667741069030352,0.475920157068063,0.464198412698413,0.67029610941442,0.463516006511123,0.657971785999468,0.482868874172185,0.462960179324894],"y":[4.32459783253377,3.49819817512565,3.46082419383759,-0.0688895464576371,-0.0134927580430537,4.10719566722206,0.112254534228113,-1.55755323289045,2.31809582877849,4.16717875883292,0.0387058411151215,4.23935672985528,2.31941524290884,4.93426920761054,5.35458281508404],"text":["A2058 (CM)<br />Mutations->MEK:1,BRAF:1,p53:0<br />RNA->ERK:0.34,p70:0.69,PDPK1:0.16,CRAF:0.58,","C32 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.43,p70:0.65,PDPK1:0.5,CRAF:0.77,","COLO-679 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.59,p70:0.63,PDPK1:0.16,CRAF:0.5,","COLO-792 (CM)<br />Mutations:None<br />RNA->ERK:0.62,p70:0.33,PDPK1:0.32,CRAF:0.77,","HT-144 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.46,p70:0.41,PDPK1:0.65,CRAF:0.92,","IGR-1 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.48,p70:0.46,PDPK1:0.32,CRAF:0.5,","IPC-298 (CM)<br />Mutations->RAS:1<br />RNA->ERK:0.65,p70:0.41,PDPK1:0.16,CRAF:0.5,","MEL-JUSO (CM)<br />Mutations->RAS:1<br />RNA->ERK:0.62,p70:0.63,PDPK1:0.32,CRAF:0.94,","RPMI-7951 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.37,p70:0.32,PDPK1:0.16,CRAF:0.72,","RVH-421 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.63,p70:0.24,PDPK1:0.65,CRAF:0.24,","SK-MEL-2 (CM)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.32,p70:0.26,PDPK1:0.32,CRAF:0.66,","SK-MEL-24 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.43,p70:0.41,PDPK1:0.32,CRAF:0.03,","SK-MEL-30 (CM)<br />Mutations->BRAF:1,RAS:1<br />RNA->ERK:0.44,p70:0.66,PDPK1:0.16,CRAF:0.58,","UACC-62 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.34,p70:0.5,PDPK1:0.06,CRAF:0.41,","WM793B (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.54,p70:0.67,PDPK1:0.76,CRAF:0.32,"],"frame":"Mutations","type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(180,117,98,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(180,117,98,1)"}},"hoveron":"points","name":"CM","legendgroup":"CM","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","visible":true},{"x":[0.93856453489481,0.66720128307939,1.01453205166008,0.4270615,0.494168694241687,1.01460460329316,0.655729028843609,1,0.67295179922985,0.4247765,0.690811535881958,1,0.675390572840166,0.663105888636667],"y":[-1.21982415420915,-1.94163726587743,-1.41212557900204,4.22554038953254,0.338178721390904,-1.44255283497272,0.45725570120034,-0.764980996447155,-1.46799285380722,3.63045427955743,-1.6300880286577,-0.660753136475747,-0.595467989996735,-0.986951714239596],"text":["C2BBe1 (CRC)<br />Mutations:None<br />RNA->ERK:0.67,p70:0.84,PDPK1:0.89,CRAF:0.85,","COLO-678 (CRC)<br />Mutations->RAS:1<br />RNA->ERK:0.25,p70:0.9,PDPK1:0.32,CRAF:0.58,","HT-115 (CRC)<br />Mutations->PI3K:1<br />RNA->ERK:0.5,p70:0.75,PDPK1:0.94,CRAF:0.9,","HT-29 (CRC)<br />Mutations->PI3K:1,BRAF:1,p53:0<br />RNA->ERK:0.46,p70:0.63,PDPK1:0.89,CRAF:0.03,","HT55 (CRC)<br />Mutations->BRAF:1<br />RNA->ERK:0.48,p70:0.58,PDPK1:0.94,CRAF:0.81,","KM12 (CRC)<br />Mutations->p53:0<br />RNA->ERK:0.12,p70:0.87,PDPK1:0.5,CRAF:0.5,","LoVo (CRC)<br />Mutations->RAS:1<br />RNA->ERK:0.29,p70:0.77,PDPK1:0.5,CRAF:0.66,","LS-180 (CRC)<br />Mutations->PI3K:1,RAS:1<br />RNA->ERK:0.53,p70:0.67,PDPK1:0.92,CRAF:0.58,","NCI-H747 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.38,p70:0.71,PDPK1:0.76,CRAF:0.41,","RKO (CRC)<br />Mutations->PI3K:1,BRAF:1<br />RNA->ERK:0.51,p70:0.58,PDPK1:0.5,CRAF:0.24,","SW1463 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.38,p70:0.5,PDPK1:0.89,CRAF:0.24,","SW48 (CRC)<br />Mutations->EGFR:1,PI3K:1,MEK:1<br />RNA->ERK:0.34,p70:0.54,PDPK1:0.76,CRAF:0.17,","SW620 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.39,p70:0.83,PDPK1:0.76,CRAF:0.32,","SW837 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.79,p70:0.63,PDPK1:0.84,CRAF:0.03,"],"frame":"Mutations","type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,40,38,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,40,38,1)"}},"hoveron":"points","name":"CRC","legendgroup":"CRC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","visible":true}],"traces":[0,1]},{"name":"Mutations & RNA","data":[{"x":[0.554264968315315,0.306613819447787,0.127922928691449,0.993383191499772,0.672902163864717,0.167602935246021,0.856659570168674,0.614623949787932,0.425356833361998,0.262431007208621,0.994373256974529,0.06084031020404,0.983421641109714,0.0560396847552988,0.410233397319945],"y":[4.32459783253377,3.49819817512565,3.46082419383759,-0.0688895464576371,-0.0134927580430537,4.10719566722206,0.112254534228113,-1.55755323289045,2.31809582877849,4.16717875883292,0.0387058411151215,4.23935672985528,2.31941524290884,4.93426920761054,5.35458281508404],"text":["A2058 (CM)<br />Mutations->MEK:1,BRAF:1,p53:0<br />RNA->ERK:0.34,p70:0.69,PDPK1:0.16,CRAF:0.58,","C32 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.43,p70:0.65,PDPK1:0.5,CRAF:0.77,","COLO-679 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.59,p70:0.63,PDPK1:0.16,CRAF:0.5,","COLO-792 (CM)<br />Mutations:None<br />RNA->ERK:0.62,p70:0.33,PDPK1:0.32,CRAF:0.77,","HT-144 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.46,p70:0.41,PDPK1:0.65,CRAF:0.92,","IGR-1 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.48,p70:0.46,PDPK1:0.32,CRAF:0.5,","IPC-298 (CM)<br />Mutations->RAS:1<br />RNA->ERK:0.65,p70:0.41,PDPK1:0.16,CRAF:0.5,","MEL-JUSO (CM)<br />Mutations->RAS:1<br />RNA->ERK:0.62,p70:0.63,PDPK1:0.32,CRAF:0.94,","RPMI-7951 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.37,p70:0.32,PDPK1:0.16,CRAF:0.72,","RVH-421 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.63,p70:0.24,PDPK1:0.65,CRAF:0.24,","SK-MEL-2 (CM)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.32,p70:0.26,PDPK1:0.32,CRAF:0.66,","SK-MEL-24 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.43,p70:0.41,PDPK1:0.32,CRAF:0.03,","SK-MEL-30 (CM)<br />Mutations->BRAF:1,RAS:1<br />RNA->ERK:0.44,p70:0.66,PDPK1:0.16,CRAF:0.58,","UACC-62 (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.34,p70:0.5,PDPK1:0.06,CRAF:0.41,","WM793B (CM)<br />Mutations->BRAF:1<br />RNA->ERK:0.54,p70:0.67,PDPK1:0.76,CRAF:0.32,"],"frame":"Mutations & RNA","type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(180,117,98,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(180,117,98,1)"}},"hoveron":"points","name":"CM","legendgroup":"CM","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","visible":true},{"x":[0.992928675654441,0.988588311937459,1.00692451268687,0.184126515944252,0.79771474442771,1.06637038464472,0.737506930303503,0.991633481240538,0.950149076786583,0.329139,0.885108245369963,0.998733493415474,0.886645982229888,0.821130000670041],"y":[-1.21982415420915,-1.94163726587743,-1.41212557900204,4.22554038953254,0.338178721390904,-1.44255283497272,0.45725570120034,-0.764980996447155,-1.46799285380722,3.63045427955743,-1.6300880286577,-0.660753136475747,-0.595467989996735,-0.986951714239596],"text":["C2BBe1 (CRC)<br />Mutations:None<br />RNA->ERK:0.67,p70:0.84,PDPK1:0.89,CRAF:0.85,","COLO-678 (CRC)<br />Mutations->RAS:1<br />RNA->ERK:0.25,p70:0.9,PDPK1:0.32,CRAF:0.58,","HT-115 (CRC)<br />Mutations->PI3K:1<br />RNA->ERK:0.5,p70:0.75,PDPK1:0.94,CRAF:0.9,","HT-29 (CRC)<br />Mutations->PI3K:1,BRAF:1,p53:0<br />RNA->ERK:0.46,p70:0.63,PDPK1:0.89,CRAF:0.03,","HT55 (CRC)<br />Mutations->BRAF:1<br />RNA->ERK:0.48,p70:0.58,PDPK1:0.94,CRAF:0.81,","KM12 (CRC)<br />Mutations->p53:0<br />RNA->ERK:0.12,p70:0.87,PDPK1:0.5,CRAF:0.5,","LoVo (CRC)<br />Mutations->RAS:1<br />RNA->ERK:0.29,p70:0.77,PDPK1:0.5,CRAF:0.66,","LS-180 (CRC)<br />Mutations->PI3K:1,RAS:1<br />RNA->ERK:0.53,p70:0.67,PDPK1:0.92,CRAF:0.58,","NCI-H747 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.38,p70:0.71,PDPK1:0.76,CRAF:0.41,","RKO (CRC)<br />Mutations->PI3K:1,BRAF:1<br />RNA->ERK:0.51,p70:0.58,PDPK1:0.5,CRAF:0.24,","SW1463 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.38,p70:0.5,PDPK1:0.89,CRAF:0.24,","SW48 (CRC)<br />Mutations->EGFR:1,PI3K:1,MEK:1<br />RNA->ERK:0.34,p70:0.54,PDPK1:0.76,CRAF:0.17,","SW620 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.39,p70:0.83,PDPK1:0.76,CRAF:0.32,","SW837 (CRC)<br />Mutations->RAS:1,p53:0<br />RNA->ERK:0.79,p70:0.63,PDPK1:0.84,CRAF:0.03,"],"frame":"Mutations & RNA","type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,40,38,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,40,38,1)"}},"hoveron":"points","name":"CRC","legendgroup":"CRC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","visible":true}],"traces":[0,1]}],"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.7: <strong>Multi-omics integration and enhanced value of RNA in addition to mutations.</strong> The successive observation of the two personalization strategies allows to visualize the impact of the RNA addition and to look at the corresponding omics sprofiles for the different points.
</p>
</div>

<p>A similar analysis can be made of the impact of adding RNA data to personalization when comparing with the experimental response to PLX-4720 (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>B, upper and lower left). Most of the non-sensitive cell lines (upper right corner) have not seen the behaviour of the personalized models change with RNA addition. However, the numerous BRAF-mutated cell lines previously grouped around standardized <em>Proliferation</em> scores of 0.5, are now better differentiated and their sensitivity predicted by personalized models has generally been revised towards lower scores (<em>i.e.</em>, higher sensitivity). Similar to the CRISPR data analysis, three sensitive cell lines have been shifted to the right and are misinterpreted by the model. As a result, the correlation restricted to BRAF-mutated cell lines is no longer significant (R=0.26, p.value=0.1).</p>
</div>
<div id="an-investigative-tool" class="section level4">
<h4><span class="header-section-number">6.2.4.2</span> An investigative tool</h4>
<p>These <strong>personalized models are not primarily intended to be predictive tools but rather used to reason and explore the possible mechanisms and specificities of each cell line, for example by studying the molecular alterations at the origin of the observed behaviour</strong> (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-interactive">6.7</a>). To continue on the previous examples, the two melanoma cell lines, HT-144 and SK-MEL-24, share the same mutational profiles but have very different sensitivities to BRAF targeting (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results">6.6</a>C). This inconsistency is partially corrected by the addition of the RNA data, which allows the model to take into account the difference in CRAF expression between the two cell lines. In fact, CRAF is a crucial node for the network since it is necessary for the reactivation of the MAPK pathway after BRAF inhibition. Therefore, the high sensitivity of SK-MEL-24 may be explained by its low CRAF expression level, which makes the reactivation of the MAPK pathway more difficult for this cell line. Conversely, in HT-144, the high level of CRAF expression allows the signal to flow properly through this pathway even after BRAF inhibition, thus making this cell line more resistant. The importance of CRAF expression is also evident in HT-29, a CRC BRAF mutated cell line with other important mutations (PI3K activation and p53 deletion). However, it remains sensitive to treatment, due to its very low level of CRAF expression.</p>
<p>Another interesting contribution of RNA appears in the melanoma cell line UACC-62, which is particularly sensitive to treatment. The model is able to correctly predict its response once RNA levels are integrated. In this case, the reason for sensitivity seems to be due to the low level of PDPK1, which makes it difficult to activate p70 and thus to trigger the resistance linked to PI3K/AKT pathway activation. Similarly, the CRC resistant cell line, HT55, which carries only the BRAF mutation, expresses high levels of PDPK1, in addition to high levels of CRAF, supporting the idea that the presence of both MAPK and PI3K/AKT pathways may confer resistance to BRAF inhibition treatments. We can also mention a cluster of RAS mutated cell lines, usually NRAS mutated for melanomas (<em>e.g.</em>, SK-MEL-2) and KRAS for colorectal cancers (<em>e.g.</em>, COLO-678), which are classified by the model as resistant. Interestingly, in these cell lines, a low level of CRAF is not enough to block the signal of the MAPK pathway, which is stronger in the model because of the simulation of the RAS mutation (RAS is set to <span class="math inline">\(1\)</span>). Only SK-MEL-30 appears to be incorrectly classified and is observed to be more sensitive than the other cell lines with a similar mutation profile. This could be due to the fact that our network is incomplete and not able to account for some alterations responsible for this cell line sensitivity. The problem may also come from the fact that this cell line contains a frameshift mutation of RPS6KB2 (p70 node) not referenced in OncoKB and therefore not included in the simulation.</p>
<div class="figure" style="text-align: center"><span id="fig:BRAF-results-add"></span>
<img src="06-Drugs_files/figure-html/BRAF-results-add-1.png" alt="Application of personalized models to other CRISPR targets. (A) Personalization strategies using either mutations only (as discrete data) or combined with RNA (as continuous data) with their corresponding scatter plots in panels B and C. (B) Scatter plot comparing normalized Proliferation scores of p53 inhibition in the models with experiment sensitivity of cell lines to TP53 CRISPR inhibition, indicating p53 mutational status as interpreted in the model. Pearson correlations and the corresponding p-values are shown. (C) Similar analysis as in panel B with PI3K model node and PIK3CA CRISPR inhibition." width="90%" />
<p class="caption">
Figure 6.8: <strong>Application of personalized models to other CRISPR targets.</strong> (A) Personalization strategies using either mutations only (as discrete data) or combined with RNA (as continuous data) with their corresponding scatter plots in panels B and C. (B) Scatter plot comparing normalized <em>Proliferation</em> scores of p53 inhibition in the models with experiment sensitivity of cell lines to TP53 CRISPR inhibition, indicating p53 mutational status as interpreted in the model. Pearson correlations and the corresponding p-values are shown. (C) Similar analysis as in panel B with PI3K model node and PIK3CA CRISPR inhibition.
</p>
</div>

<p>The versatility of the logical formalism makes it possible to test other node inhibitions as in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results-add">6.8</a>, but remains limited by the scope of the model. Since the present model has been designed around BRAF, its regulators have been carefully selected and implemented, which is not necessarily the case for other nodes of the model. Therefore, these personalized models can be used to study how comprehensive the descriptions of the regulation of other nodes or parts of the model are. Thus, model simulations show that response trends to TP53 inhibition are consistently recovered by the model (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results-add">6.8</a>B) but the simple regulation of p53 in the model results in coarse-grained patterns, although slightly improved by addition of RNA data. Similar analyses regarding the targeting of PIK3CA (in CRISPR data) simulated, in the model, by the inhibition of PI3K node, can be performed (Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-results-add">6.8</a>C). <strong>Low correlations are an indication highlighting the insufficient regulation of the node, probably confirming the scope issues raised in the pan-cancer-preliminary analysis</strong>.</p>
</div>
</div>
<div id="comparison-of-the-mechanistic-approach-with-machine-learning-methods" class="section level3">
<h3><span class="header-section-number">6.2.5</span> Comparison of the mechanistic approach with machine learning methods</h3>
<div class="figure" style="text-align: center"><span id="fig:BRAF-ML"></span>
<img src="06-Drugs_files/figure-html/BRAF-ML-1.png" alt="Random forests to predict and explain sensitivity to BRAF inhibition. (A) Performances of random forests for BRAF sensitivity prediction measured with percentage of explained variance; different learning task with unprocessed original data (thousands of genes), unprocessed original data for model-related genes only (tens of genes), and processed profiles of cell lines (tens of genes); \(n\) samples and \(p\) variables per learning task. (B) Variable importance for drug prediction only, with the 10 best variables with positive importance for each case." width="90%" />
<p class="caption">
Figure 6.9: <strong>Random forests to predict and explain sensitivity to BRAF inhibition.</strong> (A) Performances of random forests for BRAF sensitivity prediction measured with percentage of explained variance; different learning task with unprocessed original data (thousands of genes), unprocessed original data for model-related genes only (tens of genes), and processed profiles of cell lines (tens of genes); <span class="math inline">\(n\)</span> samples and <span class="math inline">\(p\)</span> variables per learning task. (B) Variable importance for drug prediction only, with the 10 best variables with positive importance for each case.
</p>
</div>

<p>In order to provide comparison elements unbiased by prior knowledge or by the construction of the model, we performed some simple machine learning algorithms. Random forests are used as an example of a machine learning approach to compare with mechanistic models and are implemented with <em>randomForestSRC</em> R package <span class="citation">(Breiman <a href="#ref-breiman2001random">2001</a><a href="#ref-breiman2001random">a</a>)</span>. Random forests can be seen as an aggregation of decision trees, each trained on a different training set formed by uniform sampling with replacement of the original cohort. Prediction performances are computed using out-of-the bag estimates for each individual (i.e, average estimate from trees that did not contain the individual in their bootstrap training sample) and summarized as percentage of variance explained by the random forest. In this case, random forests have been fitted with inputs (mutations and/or RNA data) and outputs (sensitivities to drug or CRISPR BRAF inhibition) similar to those of logical models and the corresponding predictive accuracies are reported in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-ML">6.9</a>A. The first insight concerns data processing. The percentages of variance explained by the models are similar (around 70% of explained variance for drug sensitivity prediction) in the following three cases: unprocessed original data (thousands of genes), unprocessed original data for model-related genes only (tens of genes), and processed profiles of cell lines (tens of genes). This supports the choice of a model with a small number of relevant genes, which appear to contain most of the information needed for prediction. Second, the absolute level of performance appears much lower for CRISPR (between 30 and 50%) probably suffering from the lower number of samples, especially in cases where the number of variables is the highest. This tends to <strong>reinforce the interest of mechanistic approaches that do not use any training on the data for smaller datasets, less suitable for learning</strong>. Finally, while mutations and RNA data seem to provide the same predictive power (especially for drugs), using the two together does not necessarily result in a better performance in this case.</p>
<p>It is also possible to compute the variable importance that assesses the contribution of variables to the overall performance. The solution adopted in this paper to measure it, and called VIMP in the package, consists in introducing random permutations between individuals for the values of a variable and quantifying the variation in performance resulting from this addition of noise. In the case of key variables for prediction, this perturbation will decrease the performance and will result in a high variable importance <span class="citation">(Ishwaran <a href="#ref-ishwaran2007variable">2007</a>)</span>. <strong>Variable importance in these different random forests are reported in Figure <a href="personalized-logical-models-to-study-an-interpret-drug-response.html#fig:BRAF-ML">6.9</a>B and are consistent with the analysis of mechanistic models</strong>. The mutational status of BRAF is definitely the most important variable followed by mutations in RAS or TP53. Concerning RNA levels, the most explanatory variables seem to be FOXD3 or PTEN, in line with the definition of the logical model.</p>
</div>
</div>
<div id="prostate-model" class="section level2">
<h2><span class="header-section-number">6.3</span> Application on prostate cancer study and challenges</h2>
<p>Before summarizing the potential and limitations of the PROFILE approaches described in this and the previous chapter, a final example may be mentioned. Indeed, another application of the PROFILE method, quite similar to the examples presented in the previous and this chapter, has been carried out on prostate cancer. Chronologically, this project was one of the first applications of the method. However, as this project was more collaborative than personal, the previous chapters have been illustrated by more exclusively personal work when they were equivalent. We will therefore only briefly mention here the differences and insights specific to this study.</p>
<p>First, a logical model specific to prostate cancer was developed by some collaborators (Pauline Traynard and Arnau Montagud) over a long period of time, resulting in a large and comprehensive model of 146 nodes, which is described in more detail in the appendix <a href="about-logical-models.html#appendix-montagud">B.4</a> and Figure <a href="about-logical-models.html#fig:Montagud">B.3</a>. Using the TCGA prostate cancer dataset (<a href="appendix-datasets.html#appendix-prostate">A.3.3</a>) prognostic validation of the model was first carried out, similarly to Figure <a href="personalization-of-logical-models-method-and-prognostic-validation.html#fig:PROFILE-METABRIC-Grade">5.7</a>, by comparing individualized scores of some phenotypes in the model (<em>i.e.</em>, <em>Proliferation</em>) with clinical markers, in this case Gleason score, a grading system specific to prostate cancer. The qualitative evolution of the personalized <em>Proliferation</em> scores is also qualitatively validated (predicted proliferating tumors are on average of higher grade) but, despite the specificity and magnitude of the model, much of the variability is not explained.</p>
<p>The use of cell line data was also explored using Cell Model Passports data, restricted to the 7 prostate cell lines. The size of the model then allows qualitative predictions to be made on the proliferative, apoptotic and metastatic qualities of the different lines. Except for proliferation, however, experimental validation of the relevance of these predictions is difficult using public data or the literature. But again, after these preliminary validations, the focus of the study was on treatment response with a slightly different rationale than in the previous example. Focusing on a particular cell line (LnCaP) and its corresponding personalized logical model, the idea is to <strong>simulate with the models all possible inhibitions or combinations of inhibitions in order to identify possible vulnerabilities or relevant treatment synergies</strong>. Experimental validation on the cell line was then carried out for certain genes that could be targeted depending on the existence of the treatments. The <strong>efficacy of certain inhibitions highlighted by the simulations, such as that of HSP90, was confirmed experimentally</strong> on this particular cell line. Despite the limitations of the approach in this application to prostate cancer, the study demonstrates the feasibility of the method for investigating the complexity of therapeutic responses and guiding experimental validation.</p>
</div>
<div id="limitations-and-perspectives" class="section level2">
<h2><span class="header-section-number">6.4</span> Limitations and perspectives</h2>
<p>The emergence of high-throughput data has made it easier to generate models for prognostic or diagnostic predictions in the field of cancer. The numerous lists of genes, biomarkers and predictors proposed have, however, often been difficult to interpret because of their sometimes uncertain clinical impact and little overlap between competing approaches <span class="citation">(Domany <a href="#ref-domany2014using">2014</a>)</span>. Methods that can be interpreted by design, which integrate <em>a priori</em> biological knowledge, therefore appear to be an interesting complement able to reconcile the omics data generated and the knowledge accumulated in the literature.</p>
<p>These benefits come at the cost of having <strong>accurate expert description of the problem</strong> to provide a relevant basis to the mechanistic models. This is particularly true in this work since the personalized models all derive from the same structure (the initial generic logical model) of which they are partially constrained versions. It is therefore necessary to have a generic model that is both sufficiently accurate and broad enough so that the data integration allows the expression of the singularities of each cell line. If this is not the case, the learning of logical rules or the use of ensemble modeling could be favoured, usually including perturbation time-series data <span class="citation">(Razzaq et al. <a href="#ref-razzaq2018computational">2018</a>)</span>. It should also be noted that, in the logical models presented here, the translation of biological knowledge into a logical rule is not necessarily deterministic and unambiguous. The choices here have been made based on the interpretation of the literature only. And the presence of certain outliers, <em>i.e.</em>, cell lines whose behaviour is not explained by the models, may indeed result from the limitations of the model, either in its scope (important genes not integrated), or in its definition (incorrect logical rules). More global or data-driven approaches to define the model would be possible but would require different training/validation steps and different sets of data.</p>
<p>The <strong>second key point is the omics data used</strong>. For practical reasons, we have focused on mutation and RNA data. The legitimacy of the former is not in doubt, but their interpretation is, on the other hand, a crucial point whose relevance must be systematically verified. The omission or over-interpretation of certain mutations can severely affect the behaviour of personalized models. Validation using sensitivity data provides a good indicator in this respect. However, the question is broader for RNA data: are they relevant data to be used to personalize models, <em>i.e.</em>, can they be considered as <strong>good proxies for node activity?</strong> The protein nature of many nodes in the model would encourage the use of protein level data instead, or even phosphorylation levels if they were available for these data. One perspective could even be to push personalization to the point of defining different types of data or even different personalization strategies for each node according to the knowledge of the mechanisms at work in the corresponding biological entity. A balance should then be found to allow a certain degree of automation in the code and to avoid overfitting.</p>
<p>Despite these limitations, the results described above support the <strong>importance of combining the integration of different types of data to better explain differences in drug sensitivities</strong>. There was no doubt about this position of principle in general <span class="citation">(Azuaje <a href="#ref-azuaje2017computational">2017</a>)</span>, and in particular in machine learning methods <span class="citation">(Costello et al. <a href="#ref-costello2014community">2014</a>; Aben et al. <a href="#ref-aben2016tandem">2016</a>)</span>. The technical implementation of these multi-omic integrations is nevertheless more difficult in mechanistic models where the relationships between the different types of data need to be more explicitly formulated <span class="citation">(Klinger et al. <a href="#ref-klinger2013network">2013</a>)</span>. The present work therefore reinforces the possibility and value of integrating different types of data in a mechanistic framework to improve relevance and interpretation and illustrates this by highlighting the value of RNA data in addition to mutation data in predicting the response of cell lines to BRAF inhibition. In addition, one piece of data that could be further exploited is that of the specific behaviour of the drugs or inhibitors studied, since for instance some BRAF inhibitors have affinities that vary according to mutations in the BRAF gene itself. The integration of truly precise data on the nature of the drug is nevertheless limited by logical formalism and is more often found in more flexible approaches, <em>e.g.</em>, in deep learning <span class="citation">(Manica et al. <a href="#ref-manica2019toward">2019</a>)</span>.</p>
<p>The application presented in this chapter, focused on BRAF inhibitors, made it possible to verify the good performance of the models through different types of data (drug or CRISPR/Cas9). However, the molecular profiles used to personalize the models were all derived from cell lines, reported in the same database <span class="citation">(Meer et al. <a href="#ref-van2019cell">2019</a>)</span>. It would be possible to use different types of data such as organoids, patient-derived xenografts (PDX), etc. The critical clinical question will then be: <strong>do the mechanisms highlighted for cell lines transfer easily to tumours in vivo?</strong> The ability to identify common reasons explaining the response to treatments has been studied by different statistical approaches with the aim of promoting translational medicine <span class="citation">(Mourragui et al. <a href="#ref-mourragui2019precise">2019</a>; Kim et al. <a href="#ref-kim2019genomic">2019</a>)</span>. The ability of personalized mechanistic models to follow this path remains to be explored.</p>
<p>To conclude, we provide a comprehensive pipeline from clinical question to a validated mechanistic model which uses different types of omics data and adapts to dozens of different cell lines. This work, which is <strong>based only on the interpretation of data and not on the training of the model</strong>, continues some previous work that has already demonstrated the value of mechanistic approaches to answer questions about response to treatment, especially using dynamic data <span class="citation">(Saez-Rodriguez and Blüthgen <a href="#ref-saez2020personalized">2020</a>)</span>, and sometimes about similar pathways <span class="citation">(Klinger et al. <a href="#ref-klinger2013network">2013</a>)</span>. In this context, our approach proves the interest of logical formalism to make use of scarce and static data facilitating application to a wide range of issues and datasets in a way that is sometimes complementary to learning-based approaches.</p>

</div>
</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-aben2016tandem">
<p>Aben, Nanne, Daniel J Vis, Magali Michaut, and Lodewyk Fa Wessels. 2016. “TANDEM: A Two-Stage Approach to Maximize Interpretability of Drug Response Models Based on Multiple Molecular Data Types.” <em>Bioinformatics</em> 32 (17). Oxford University Press: i413–i420.</p>
</div>
<div id="ref-azuaje2017computational">
<p>Azuaje, Francisco. 2017. “Computational Models for Predicting Drug Responses in Cancer Research.” <em>Briefings in Bioinformatics</em> 18 (5). Oxford University Press: 820–29.</p>
</div>
<div id="ref-baur2020connecting">
<p>Baur, Florentin, Sarah L Nietzer, Meik Kunz, Fabian Saal, Julian Jeromin, Stephanie Matschos, Michael Linnebacher, Heike Walles, Thomas Dandekar, and Gudrun Dandekar. 2020. “Connecting Cancer Pathways to Tumor Engines: A Stratification Tool for Colorectal Cancer Combining Human in Vitro Tissue Models with Boolean in Silico Models.” <em>Cancers</em> 12 (1). Multidisciplinary Digital Publishing Institute: 28.</p>
</div>
<div id="ref-behan2019prioritization">
<p>Behan, Fiona M, Francesco Iorio, Gabriele Picco, Emanuel Gonçalves, Charlotte M Beaver, Giorgia Migliardi, Rita Santos, et al. 2019. “Prioritization of Cancer Therapeutic Targets Using Crispr–Cas9 Screens.” <em>Nature</em> 568 (7753). Nature Publishing Group: 511.</p>
</div>
<div id="ref-beal2020personalized">
<p>Béal, Jonas, Lorenzo Pantolini, Vincent Noël, Emmanuel Barillot, and Laurence Calzone. 2020. “Personalized Logical Models to Investigate Cancer Response to Braf Treatments in Melanomas and Colorectal Cancers.” <em>BioRxiv</em>. Cold Spring Harbor Laboratory.</p>
</div>
<div id="ref-breiman2001random">
<p>Breiman, Leo. 2001a. “Random Forests.” <em>Machine Learning</em> 45 (1). Springer: 5–32.</p>
</div>
<div id="ref-cantwell2011brafv600e">
<p>Cantwell-Dorris, Emma R, John J O’Leary, and Orla M Sheils. 2011. “BRAFV600E: Implications for Carcinogenesis and Molecular Therapy.” <em>Molecular Cancer Therapeutics</em> 10 (3). AACR: 385–94.</p>
</div>
<div id="ref-chapman2011improved">
<p>Chapman, Paul B, Axel Hauschild, Caroline Robert, John B Haanen, Paolo Ascierto, James Larkin, Reinhard Dummer, et al. 2011. “Improved Survival with Vemurafenib in Melanoma with Braf V600e Mutation.” <em>New England Journal of Medicine</em> 364 (26). Mass Medical Soc: 2507–16.</p>
</div>
<div id="ref-cho2016attractor">
<p>Cho, Sung-Hwan, Sang-Min Park, Ho-Sung Lee, Hwang-Yeol Lee, and Kwang-Hyun Cho. 2016. “Attractor Landscape Analysis of Colorectal Tumorigenesis and Its Reversion.” <em>BMC Systems Biology</em> 10 (1). Springer: 96.</p>
</div>
<div id="ref-conway2017upsetr">
<p>Conway, Jake R, Alexander Lex, and Nils Gehlenborg. 2017. “UpSetR: An R Package for the Visualization of Intersecting Sets and Their Properties.” <em>Bioinformatics</em> 33 (18). Oxford University Press: 2938–40.</p>
</div>
<div id="ref-costello2014community">
<p>Costello, James C, Laura M Heiser, Elisabeth Georgii, Mehmet Gönen, Michael P Menden, Nicholas J Wang, Mukesh Bansal, et al. 2014. “A Community Effort to Assess and Improve Drug Sensitivity Prediction Algorithms.” <em>Nature Biotechnology</em> 32 (12). Nature Publishing Group: 1202.</p>
</div>
<div id="ref-dempster2019agreement">
<p>Dempster, Joshua M, Clare Pacini, Sasha Pantel, Fiona M Behan, Thomas Green, John Krill-Burger, Charlotte M Beaver, et al. 2019. “Agreement Between Two Large Pan-Cancer Crispr-Cas9 Gene Dependency Data Sets.” <em>Nature Communications</em> 10 (1). Nature Publishing Group: 1–14.</p>
</div>
<div id="ref-domany2014using">
<p>Domany, Eytan. 2014. “Using High-Throughput Transcriptomic Data for Prognosis: A Critical Overview and Perspectives.” <em>Cancer Research</em> 74 (17). AACR: 4612–21.</p>
</div>
<div id="ref-eduati2017drug">
<p>Eduati, Federica, Victoria Doldàn-Martelli, Bertram Klinger, Thomas Cokelaer, Anja Sieber, Fiona Kogera, Mathurin Dorel, Mathew J Garnett, Nils Blüthgen, and Julio Saez-Rodriguez. 2017. “Drug Resistance Mechanisms in Colorectal Cancer Dissected with Cell Type–specific Dynamic Logic Models.” <em>Cancer Research</em> 77 (12). AACR: 3364–75.</p>
</div>
<div id="ref-eduati2020patient">
<p>Eduati, Federica, Patricia Jaaks, Jessica Wappler, Thorsten Cramer, Christoph A Merten, Mathew J Garnett, and Julio Saez-Rodriguez. 2020. “Patient-Specific Logic Models of Signaling Pathways from Screenings on Cancer Biopsies to Prioritize Personalized Combination Therapies.” <em>Molecular Systems Biology</em> 16 (2). John Wiley &amp; Sons, Ltd.</p>
</div>
<div id="ref-flobak2015discovery">
<p>Flobak, Åsmund, Anaïs Baudot, Elisabeth Remy, Liv Thommesen, Denis Thieffry, Martin Kuiper, and Astrid Lægreid. 2015. “Discovery of Drug Synergies in Gastric Cancer Cells Predicted by Logical Modeling.” <em>PLoS Computational Biology</em> 11 (8). Public Library of Science.</p>
</div>
<div id="ref-frohlich2018efficient">
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6). Elsevier: 567–79.</p>
</div>
<div id="ref-garnett2012systematic">
<p>Garnett, Mathew J, Elena J Edelman, Sonja J Heidorn, Chris D Greenman, Anahita Dastur, King Wai Lau, Patricia Greninger, et al. 2012. “Systematic Identification of Genomic Markers of Drug Sensitivity in Cancer Cells.” <em>Nature</em> 483 (7391). Nature Publishing Group: 570–75.</p>
</div>
<div id="ref-hart2016bagel">
<p>Hart, Traver, and Jason Moffat. 2016. “BAGEL: A Computational Framework for Identifying Essential Genes from Pooled Library Screens.” <em>BMC Bioinformatics</em> 17 (1). Springer: 164.</p>
</div>
<div id="ref-heiser2012subtype">
<p>Heiser, Laura M, Anguraj Sadanandam, Wen-Lin Kuo, Stephen C Benz, Theodore C Goldstein, Sam Ng, William J Gibb, et al. 2012. “Subtype and Pathway Specific Responses to Anticancer Compounds in Breast Cancer.” <em>Proceedings of the National Academy of Sciences</em> 109 (8). National Acad Sciences: 2724–9.</p>
</div>
<div id="ref-iorio2016landscape">
<p>Iorio, Francesco, Theo A Knijnenburg, Daniel J Vis, Graham R Bignell, Michael P Menden, Michael Schubert, Nanne Aben, et al. 2016. “A Landscape of Pharmacogenomic Interactions in Cancer.” <em>Cell</em> 166 (3). Elsevier: 740–54.</p>
</div>
<div id="ref-ishwaran2007variable">
<p>Ishwaran, Hemant. 2007. “Variable Importance in Binary Regression Trees and Forests.” <em>Electronic Journal of Statistics</em> 1. The Institute of Mathematical Statistics; the Bernoulli Society: 519–37.</p>
</div>
<div id="ref-jastrzebski2018integrative">
<p>Jastrzebski, Katarzyna, Bram Thijssen, Roelof JC Kluin, Klaas de Lint, Ian J Majewski, Roderick L Beijersbergen, and Lodewyk FA Wessels. 2018. “Integrative Modeling Identifies Key Determinants of Inhibitor Sensitivity in Breast Cancer Cell Lines.” <em>Cancer Research</em> 78 (15). AACR: 4396–4410.</p>
</div>
<div id="ref-kim2019genomic">
<p>Kim, Yongsoo, Tycho Bismeijer, Wilbert Zwart, Lodewyk FA Wessels, and Daniel J Vis. 2019. “Genomic Data Integration by Won-Parafac Identifies Interpretable Factors for Predicting Drug-Sensitivity in Vivo.” <em>Nature Communications</em> 10 (1). Nature Publishing Group: 1–12.</p>
</div>
<div id="ref-klinger2013network">
<p>Klinger, Bertram, Anja Sieber, Raphaela Fritsche-Guenther, Franziska Witzel, Leanne Berry, Dirk Schumacher, Yibing Yan, et al. 2013. “Network Quantification of Egfr Signaling Unveils Potential for Targeted Combination Therapy.” <em>Molecular Systems Biology</em> 9 (1). John Wiley &amp; Sons, Ltd.</p>
</div>
<div id="ref-knijnenburg2016logic">
<p>Knijnenburg, Theo A, Gunnar W Klau, Francesco Iorio, Mathew J Garnett, Ultan McDermott, Ilya Shmulevich, and Lodewyk FA Wessels. 2016. “Logic Models to Predict Continuous Outputs Based on Binary Inputs with an Application to Personalized Cancer Therapy.” <em>Scientific Reports</em> 6 (1). Nature Publishing Group: 1–14.</p>
</div>
<div id="ref-kopetz2010plx4032">
<p>Kopetz, S, J Desai, E Chan, JR Hecht, PJ O’dwyer, RJ Lee, KB Nolop, and L Saltz. 2010. “PLX4032 in Metastatic Colorectal Cancer Patients with Mutant Braf Tumors.” <em>Journal of Clinical Oncology</em> 28 (15_suppl). American Society of Clinical Oncology: 3534–4.</p>
</div>
<div id="ref-manica2019toward">
<p>Manica, Matteo, Ali Oskooei, Jannis Born, Vigneshwari Subramanian, Julio Sáez-Rodríguez, and María Rodríguez Martínez. 2019. “Toward Explainable Anticancer Compound Sensitivity Prediction via Multimodal Attention-Based Convolutional Encoders.” <em>Molecular Pharmaceutics</em>. ACS Publications.</p>
</div>
<div id="ref-van2019cell">
<p>Meer, Dieudonne van der, Syd Barthorpe, Wanjuan Yang, Howard Lightfoot, Caitlin Hall, James Gilbert, Hayley E Francies, and Mathew J Garnett. 2019. “Cell Model Passports—a Hub for Clinical, Genetic and Functional Datasets of Preclinical Cancer Models.” <em>Nucleic Acids Research</em> 47 (D1). Oxford University Press: D923–D929.</p>
</div>
<div id="ref-meyers2017computational">
<p>Meyers, Robin M, Jordan G Bryan, James M McFarland, Barbara A Weir, Ann E Sizemore, Han Xu, Neekesh V Dharia, et al. 2017. “Computational Correction of Copy Number Effect Improves Specificity of Crispr–Cas9 Essentiality Screens in Cancer Cells.” <em>Nature Genetics</em> 49 (12). Nature Publishing Group: 1779–84.</p>
</div>
<div id="ref-mourragui2019precise">
<p>Mourragui, Soufiane, Marco Loog, Mark A Van De Wiel, Marcel JT Reinders, and Lodewyk FA Wessels. 2019. “PRECISE: A Domain Adaptation Approach to Transfer Predictors of Drug Response from Pre-Clinical Models to Tumors.” <em>Bioinformatics</em> 35 (14). Oxford University Press: i510–i519.</p>
</div>
<div id="ref-perfetto2016signor">
<p>Perfetto, Livia, Leonardo Briganti, Alberto Calderone, Andrea Cerquone Perpetuini, Marta Iannuccelli, Francesca Langone, Luana Licata, et al. 2016. “SIGNOR: A Database of Causal Relationships Between Biological Entities.” <em>Nucleic Acids Research</em> 44 (D1). Oxford University Press: D548–D554.</p>
</div>
<div id="ref-poulikakos2011raf">
<p>Poulikakos, Poulikos I, Yogindra Persaud, Manickam Janakiraman, Xiangju Kong, Charles Ng, Gatien Moriceau, Hubing Shi, et al. 2011. “RAF Inhibitor Resistance Is Mediated by Dimerization of Aberrantly Spliced Braf (V600e).” <em>Nature</em> 480 (7377). Nature Publishing Group: 387.</p>
</div>
<div id="ref-prahallad2012unresponsiveness">
<p>Prahallad, Anirudh, Chong Sun, Sidong Huang, Federica Di Nicolantonio, Ramon Salazar, Davide Zecchin, Roderick L Beijersbergen, Alberto Bardelli, and René Bernards. 2012. “Unresponsiveness of Colon Cancer to Braf (V600e) Inhibition Through Feedback Activation of Egfr.” <em>Nature</em> 483 (7387). Nature Publishing Group: 100–103.</p>
</div>
<div id="ref-razzaq2018computational">
<p>Razzaq, Misbah, Loïc Paulevé, Anne Siegel, Julio Saez-Rodriguez, Jérémie Bourdon, and Carito Guziolowski. 2018. “Computational Discovery of Dynamic Cell Line Specific Boolean Networks from Multiplex Time-Course Data.” <em>PLoS Computational Biology</em> 14 (10). Public Library of Science: e1006538.</p>
</div>
<div id="ref-rossi2019drug">
<p>Rossi, Alessandro, Michela Roberto, Martina Panebianco, Andrea Botticelli, Federica Mazzuca, and Paolo Marchetti. 2019. “Drug Resistance of Braf-Mutant Melanoma: Review of up-to-Date Mechanisms of Action and Promising Targeted Agents.” <em>European Journal of Pharmacology</em>. Elsevier, 172621.</p>
</div>
<div id="ref-saez2020personalized">
<p>Saez-Rodriguez, Julio, and Nils Blüthgen. 2020. “Personalized Signaling Models for Personalized Treatments.” <em>Molecular Systems Biology</em> 16 (1). John Wiley &amp; Sons, Ltd.</p>
</div>
<div id="ref-sun2014reversible">
<p>Sun, Chong, Liqin Wang, Sidong Huang, Guus JJE Heynen, Anirudh Prahallad, Caroline Robert, John Haanen, et al. 2014. “Reversible and Adaptive Resistance to Braf (V600e) Inhibition in Melanoma.” <em>Nature</em> 508 (7494). Nature Publishing Group: 118–22.</p>
</div>
<div id="ref-ursem2018emerging">
<p>Ursem, Carling, Chloe E Atreya, and Katherine Van Loon. 2018. “Emerging Treatment Options for Braf-Mutant Colorectal Cancer.” <em>Gastrointestinal Cancer: Targets and Therapy</em> 8. NIH Public Access: 13.</p>
</div>
<div id="ref-verny2017learning">
<p>Verny, Louis, Nadir Sella, Séverine Affeldt, Param Priya Singh, and Hervé Isambert. 2017. “Learning Causal Networks with Latent Variables from Multivariate Information in Genomic Data.” <em>PLOS Computational Biology</em> 13 (10): e1005662. doi:<a href="https://doi.org/10.1371/journal.pcbi.1005662">10.1371/journal.pcbi.1005662</a>.</p>
</div>
<div id="ref-vis2016multilevel">
<p>Vis, Daniel J, Lorenzo Bombardelli, Howard Lightfoot, Francesco Iorio, Mathew J Garnett, and Lodewyk FA Wessels. 2016. “Multilevel Models Improve Precision and Speed of Ic50 Estimates.” <em>Pharmacogenomics</em> 17 (7). Future Medicine: 691–700.</p>
</div>
<div id="ref-yang2012genomics">
<p>Yang, Wanjuan, Jorge Soares, Patricia Greninger, Elena J Edelman, Howard Lightfoot, Simon Forbes, Nidhi Bindal, et al. 2012. “Genomics of Drug Sensitivity in Cancer (Gdsc): A Resource for Therapeutic Biomarker Discovery in Cancer Cells.” <em>Nucleic Acids Research</em> 41 (D1). Oxford University Press: D955–D961.</p>
</div>
<div id="ref-zanudo2017network">
<p>Zañudo, Jorge Gómez Tejeda, Maurizio Scaltriti, and Réka Albert. 2017. “A Network Modeling Approach to Elucidate Drug Resistance Mechanisms and Predict Combinatorial Drug Treatments in Breast Cancer.” <em>Cancer Convergence</em> 1 (1). Springer: 5.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="personalization-of-logical-models-method-and-prognostic-validation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="information-flows-in-mechanistic-models-of-cancer.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
