<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="References | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="about-causality.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#introducing-the-diversity-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic-models"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive-models"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#tools-for-logical-modeling"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.1</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.2</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.3</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#validation-with-patient-data"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#comparison-of-the-mechanistic-approach-with-machine-learning-methods"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#limitations-and-perspectives"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#patient-derived-xenografts"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-causality.html"><a href="about-causality.html"><i class="fa fa-check"></i><b>C</b> About causality</a><ul>
<li class="chapter" data-level="C.1" data-path="about-causality.html"><a href="about-causality.html#theoretical-framework"><i class="fa fa-check"></i><b>C.1</b> Theoretical framework</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>

<!-- This manually sets the header for this unnumbered chapter -->
<p> <!-- config from papaja -->  </p>

<div id="refs" class="references">
<div>
<p>Aben, Nanne, Daniel J Vis, Magali Michaut, and Lodewyk Fa Wessels. 2016. “TANDEM: A Two-Stage Approach to Maximize Interpretability of Drug Response Models Based on Multiple Molecular Data Types.” <em>Bioinformatics</em> 32 (17). Oxford University Press: i413–i420.</p>
</div>
<div>
<p>Abeshouse, Adam, Jaeil Ahn, Rehan Akbani, Adrian Ally, Samirkumar Amin, Christopher D Andry, Matti Annala, et al. 2015. “The Molecular Taxonomy of Primary Prostate Cancer.” <em>Cell</em> 163 (4). Elsevier: 1011–25.</p>
</div>
<div>
<p>Abou-Jaoudé, Wassim, Pauline Traynard, Pedro T Monteiro, Julio Saez-Rodriguez, Tomáš Helikar, Denis Thieffry, and Claudine Chaouiya. 2016. “Logical Modeling and Dynamical Analysis of Cellular Networks.” <em>Frontiers in Genetics</em> 7. Frontiers: 94.</p>
</div>
<div>
<p>Adzhubei, Ivan A., Steffen Schmidt, Leonid Peshkin, Vasily E. Ramensky, Anna Gerasimova, Peer Bork, Alexey S. Kondrashov, and Shamil R. Sunyaev. 2010. “A Method and Server for Predicting Damaging Missense Mutations.” <em>Nature Methods</em> 7 (4): 248–49. doi:<a href="https://doi.org/10.1038/nmeth0410-248">10.1038/nmeth0410-248</a>.</p>
</div>
<div>
<p>Albert, István, Juilee Thakar, Song Li, Ranran Zhang, and Réka Albert. 2008. “Boolean Network Simulations for Life Scientists.” <em>Source Code for Biology and Medicine</em> 3 (1). BioMed Central: 16.</p>
</div>
<div>
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, Martin Raff, Keith Roberts, and Peter Walter. 2007. “Molecular Biology of the Cell. Garland Science.” <em>New York</em> 1392.</p>
</div>
<div>
<p>Altrock, Philipp M, Lin L Liu, and Franziska Michor. 2015. “The Mathematics of Cancer: Integrating Quantitative Models.” <em>Nature Reviews Cancer</em> 15 (12). Nature Publishing Group: 730–45.</p>
</div>
<div>
<p>Anderson, Alexander RA, and Vito Quaranta. 2008. “Integrative Mathematical Oncology.” <em>Nature Reviews Cancer</em> 8 (3). Nature Publishing Group: 227–34.</p>
</div>
<div>
<p>Araujo, Robyn P, and DL Sean McElwain. 2004. “A History of the Study of Solid Tumour Growth: The Contribution of Mathematical Modelling.” <em>Bulletin of Mathematical Biology</em> 66 (5). Elsevier: 1039–91.</p>
</div>
<div>
<p>Armitage, Peter, and Richard Doll. 1954. “The Age Distribution of Cancer and a Multi-Stage Theory of Carcinogenesis.” <em>British Journal of Cancer</em> 8 (1). Nature Publishing Group: 1.</p>
</div>
<div>
<p>Azuaje, Francisco. 2017. “Computational Models for Predicting Drug Responses in Cancer Research.” <em>Briefings in Bioinformatics</em> 18 (5). Oxford University Press: 820–29.</p>
</div>
<div>
<p>Bailer-Jones, Daniela M. 2002. “Scientists’ Thoughts on Scientific Models.” <em>Perspectives on Science</em> 10 (3). MIT Press: 275–301.</p>
</div>
<div>
<p>Baker, Ruth E, Jose-Maria Pena, Jayaratnam Jayamohan, and Antoine Jérusalem. 2018. “Mechanistic Models Versus Machine Learning, a Fight Worth Fighting for the Biological Community?” <em>Biology Letters</em> 14 (5). The Royal Society: 20170660.</p>
</div>
<div>
<p>Barabasi, Albert-Laszlo, and Zoltan N Oltvai. 2004. “Network Biology: Understanding the Cell’s Functional Organization.” <em>Nature Reviews Genetics</em> 5 (2). Nature Publishing Group: 101–13.</p>
</div>
<div>
<p>Barbolosi, Dominique, Joseph Ciccolini, Bruno Lacarelle, Fabrice Barlési, and Nicolas André. 2016. “Computational Oncology—mathematical Modelling of Drug Regimens for Precision Medicine.” <em>Nature Reviews Clinical Oncology</em> 13 (4). Nature Publishing Group: 242.</p>
</div>
<div>
<p>Barillot, Emmanuel, Laurence Calzone, Philippe Hupe, Jean-Philippe Vert, and Andrei Zinovyev. 2012. <em>Computational Systems Biology of Cancer</em>. CRC Press.</p>
</div>
<div>
<p>Baur, Florentin, Sarah L Nietzer, Meik Kunz, Fabian Saal, Julian Jeromin, Stephanie Matschos, Michael Linnebacher, Heike Walles, Thomas Dandekar, and Gudrun Dandekar. 2020. “Connecting Cancer Pathways to Tumor Engines: A Stratification Tool for Colorectal Cancer Combining Human in Vitro Tissue Models with Boolean in Silico Models.” <em>Cancers</em> 12 (1). Multidisciplinary Digital Publishing Institute: 28.</p>
</div>
<div>
<p>Behan, Fiona M, Francesco Iorio, Gabriele Picco, Emanuel Gonçalves, Charlotte M Beaver, Giorgia Migliardi, Rita Santos, et al. 2019. “Prioritization of Cancer Therapeutic Targets Using Crispr–Cas9 Screens.” <em>Nature</em> 568 (7753). Nature Publishing Group: 511.</p>
</div>
<div>
<p>Bellomo, Nicola, NK Li, and Ph K Maini. 2008. “On the Foundations of Cancer Modelling: Selected Topics, Speculations, and Perspectives.” <em>Mathematical Models and Methods in Applied Sciences</em> 18 (04). World Scientific: 593–646.</p>
</div>
<div>
<p>Benzekry, Sébastien, Clare Lamont, Afshin Beheshti, Amanda Tracz, John ML Ebos, Lynn Hlatky, and Philip Hahnfeldt. 2014. “Classical Mathematical Models for Description and Prediction of Experimental Tumor Growth.” <em>PLoS Comput Biol</em> 10 (8). Public Library of Science: e1003800.</p>
</div>
<div>
<p>Béal, Jonas, Arnau Montagud, Pauline Traynard, Emmanuel Barillot, and Laurence Calzone. 2019. “Personalization of Logical Models With Multi-Omics Data Allows Clinical Stratification of Patients.” <em>Frontiers in Physiology</em>. Frontiers Media S.A. doi:<a href="https://doi.org/10.3389/fphys.2018.01965">10.3389/fphys.2018.01965</a>.</p>
</div>
<div>
<p>Béal, Jonas, Lorenzo Pantolini, Vincent Noël, Emmanuel Barillot, and Laurence Calzone. 2020. “Personalized Logical Models to Investigate Cancer Response to Braf Treatments in Melanomas and Colorectal Cancers.” <em>BioRxiv</em>. Cold Spring Harbor Laboratory.</p>
</div>
<div>
<p>Béal, Jonas, Elizabeth Rémy, and Laurence Calzone. 2020. “Modélisation Logique et Données Omiques : De La Construction Des Modèles à La Médecine Personnalisée.” In <em>Approche Symbolique de La Modélisation et de L’analyse Des Systèmes Biologiques</em>, edited by Elisabeth Rémy and Cédric Lhoussaine. ISTE.</p>
</div>
<div>
<p>Bhalla, Upinder S, and Ravi Iyengar. 1999. “Emergent Properties of Networks of Biological Signaling Pathways.” <em>Science</em> 283 (5400). American Association for the Advancement of Science: 381–87.</p>
</div>
<div>
<p>Bianucci, Raffaella, Antonio Perciaccante, Philippe Charlier, Otto Appenzeller, and Donatella Lippi. 2018. “Earliest Evidence of Malignant Breast Cancer in Renaissance Paintings.” <em>The Lancet Oncology</em> 19 (2). Elsevier: 166–67.</p>
</div>
<div>
<p>Bilal, Erhan, Janusz Dutkowski, Justin Guinney, In Sock Jang, Benjamin A Logsdon, Gaurav Pandey, Benjamin A Sauerwine, et al. 2013. “Improving Breast Cancer Survival Analysis Through Competition-Based Multidimensional Modeling.” <em>PLoS Computational Biology</em> 9 (5). Public Library of Science.</p>
</div>
<div>
<p>Bouhaddou, Mehdi, Anne Marie Barrette, Alan D Stern, Rick J Koch, Matthew S DiStefano, Eric A Riesel, Luis C Santos, Annie L Tan, Alex E Mertz, and Marc R Birtwistle. 2018. “A Mechanistic Pan-Cancer Pathway Model Informed by Multi-Omics Data Interprets Stochastic Cell Fate Responses to Drugs and Mitogens.” <em>PLoS Computational Biology</em> 14 (3). Public Library of Science: e1005985.</p>
</div>
<div>
<p>Bozic, Ivana, Tibor Antal, Hisashi Ohtsuki, Hannah Carter, Dewey Kim, Sining Chen, Rachel Karchin, Kenneth W Kinzler, Bert Vogelstein, and Martin A Nowak. 2010. “Accumulation of Driver and Passenger Mutations During Tumor Progression.” <em>Proceedings of the National Academy of Sciences</em> 107 (43). National Acad Sciences: 18545–50.</p>
</div>
<div>
<p>Braithwaite, Peter Allen, and Dace Shugg. 1983. “Rembrandt’s Bathsheba: The Dark Shadow of the Left Breast.” <em>Annals of the Royal College of Surgeons of England</em> 65 (5). Royal College of Surgeons of England: 337.</p>
</div>
<div>
<p>Bray, Freddie, Jacques Ferlay, Isabelle Soerjomataram, Rebecca L Siegel, Lindsey A Torre, and Ahmedin Jemal. 2018. “Global Cancer Statistics 2018: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries.” <em>CA: A Cancer Journal for Clinicians</em> 68 (6). Wiley Online Library: 394–424.</p>
</div>
<div>
<p>Breiman, Leo. 2001a. “Random Forests.” <em>Machine Learning</em> 45 (1). Springer: 5–32.</p>
</div>
<div>
<p>———. 2001b. “Statistical Modeling: The Two Cultures (with Comments and a Rejoinder by the Author).” <em>Statistical Science</em> 16 (3). Institute of Mathematical Statistics: 199–231.</p>
</div>
<div>
<p>Byrne, Helen M. 2010. “Dissecting Cancer Through Mathematics: From the Cell to the Animal Model.” <em>Nature Reviews Cancer</em> 10 (3). Nature Publishing Group: 221–30.</p>
</div>
<div>
<p>Calzone, Laurence, Emmanuel Barillot, and Andrei Zinovyev. 2018. “Logical Versus Kinetic Modeling of Biological Networks: Applications in Cancer Research.” <em>Current Opinion in Chemical Engineering</em> 21. Elsevier: 22–31.</p>
</div>
<div>
<p>Calzone, Laurence, Laurent Tournier, Simon Fourquet, Denis Thieffry, Boris Zhivotovsky, Emmanuel Barillot, and Andrei Zinovyev. 2010. “Mathematical Modelling of Cell-Fate Decision in Response to Death Receptor Engagement.” <em>PLoS Computational Biology</em> 6 (3). Public Library of Science.</p>
</div>
<div>
<p>Cantwell-Dorris, Emma R, John J O’Leary, and Orla M Sheils. 2011. “BRAFV600E: Implications for Carcinogenesis and Molecular Therapy.” <em>Molecular Cancer Therapeutics</em> 10 (3). AACR: 385–94.</p>
</div>
<div>
<p>Chakravarty, Debyani, Jianjiong Gao, Sarah Phillips, Ritika Kundra, Hongxin Zhang, Jiaojiao Wang, Julia E. Rudolph, et al. 2017. “OncoKB: A Precision Oncology Knowledge Base.” <em>JCO Precision Oncology</em>, no. 1 (May): 1–16. doi:<a href="https://doi.org/10.1200/PO.17.00011">10.1200/PO.17.00011</a>.</p>
</div>
<div>
<p>Chapman, Paul B, Axel Hauschild, Caroline Robert, John B Haanen, Paolo Ascierto, James Larkin, Reinhard Dummer, et al. 2011. “Improved Survival with Vemurafenib in Melanoma with Braf V600e Mutation.” <em>New England Journal of Medicine</em> 364 (26). Mass Medical Soc: 2507–16.</p>
</div>
<div>
<p>Cho, Sung-Hwan, Sang-Min Park, Ho-Sung Lee, Hwang-Yeol Lee, and Kwang-Hyun Cho. 2016. “Attractor Landscape Analysis of Colorectal Tumorigenesis and Its Reversion.” <em>BMC Systems Biology</em> 10 (1). Springer: 96.</p>
</div>
<div>
<p>Cohen, David P. A., Loredana Martignetti, Sylvie Robine, Emmanuel Barillot, Andrei Zinovyev, and Laurence Calzone. 2015. “Mathematical Modelling of Molecular Pathways Enabling Tumour Cell Invasion and Migration.” <em>PLOS Computational Biology</em> 11 (11): e1004571. doi:<a href="https://doi.org/10.1371/journal.pcbi.1004571">10.1371/journal.pcbi.1004571</a>.</p>
</div>
<div>
<p>Cohrs, Randall J, Tyler Martin, Parviz Ghahramani, Luc Bidaut, Paul J Higgins, and Aamir Shahzad. 2015. “Translational Medicine Definition by the European Society for Translational Medicine.” Elsevier.</p>
</div>
<div>
<p>Collins. 2020. <em>The Collins English Dictionary</em>. HarperCollins. <a href="https://www.collinsdictionary.com/dictionary/english/model" class="uri">https://www.collinsdictionary.com/dictionary/english/model</a>.</p>
</div>
<div>
<p>Collombet, Samuel, Chris van Oevelen, Jose Luis Sardina Ortega, Wassim Abou-Jaoude, Bruno Di Stefano, Morgane Thomas-Chollier, Thomas Graf, and Denis Thieffry. 2017. “Logical Modeling of Lymphoid and Myeloid Cell Specification and Transdifferentiation.” <em>Proceedings of the National Academy of Sciences</em> 114 (23): 5792–9. doi:<a href="https://doi.org/10.1073/pnas.1610622114">10.1073/pnas.1610622114</a>.</p>
</div>
<div>
<p>Consortium, UniProt. 2019. “UniProt: A Worldwide Hub of Protein Knowledge.” <em>Nucleic Acids Research</em> 47 (D1). Oxford University Press: D506–D515.</p>
</div>
<div>
<p>Conway, Jake R, Alexander Lex, and Nils Gehlenborg. 2017. “UpSetR: An R Package for the Visualization of Intersecting Sets and Their Properties.” <em>Bioinformatics</em> 33 (18). Oxford University Press: 2938–40.</p>
</div>
<div>
<p>Corcoran, Ryan B, Hiromichi Ebi, Alexa B Turke, Erin M Coffee, Michiya Nishino, Alexandria P Cogdill, Ronald D Brown, et al. 2012. “EGFR-Mediated Reactivation of Mapk Signaling Contributes to Insensitivity of Braf-Mutant Colorectal Cancers to Raf Inhibition with Vemurafenib.” <em>Cancer Discovery</em> 2 (3). AACR: 227–35.</p>
</div>
<div>
<p>Cortes, Corinna, and Vladimir Vapnik. 1995. “Support-Vector Networks.” <em>Machine Learning</em> 20 (3). Springer: 273–97.</p>
</div>
<div>
<p>Costello, James C, Laura M Heiser, Elisabeth Georgii, Mehmet Gönen, Michael P Menden, Nicholas J Wang, Mukesh Bansal, et al. 2014. “A Community Effort to Assess and Improve Drug Sensitivity Prediction Algorithms.” <em>Nature Biotechnology</em> 32 (12). Nature Publishing Group: 1202.</p>
</div>
<div>
<p>Crick, Francis. 1970. “Central Dogma of Molecular Biology.” <em>Nature</em> 227 (5258). Nature Publishing Group: 561–63.</p>
</div>
<div>
<p>Curtis, Christina, Sohrab P. Shah, Suet-Feung Chin, Gulisa Turashvili, Oscar M. Rueda, Mark J. Dunning, Doug Speed, et al. 2012. “The Genomic and Transcriptomic Architecture of 2,000 Breast Tumours Reveals Novel Subgroups.” <em>Nature</em> 486 (7403): 346–52. doi:<a href="https://doi.org/10.1038/nature10983">10.1038/nature10983</a>.</p>
</div>
<div>
<p>Davies, Helen, Graham R Bignell, Charles Cox, Philip Stephens, Sarah Edkins, Sheila Clegg, Jon Teague, et al. 2002. “Mutations of the Braf Gene in Human Cancer.” <em>Nature</em> 417 (6892). Nature Publishing Group: 949–54.</p>
</div>
<div>
<p>De Jong, Hidde. 2002. “Modeling and Simulation of Genetic Regulatory Systems: A Literature Review.” <em>Journal of Computational Biology</em> 9 (1). Mary Ann Liebert, Inc.: 67–103.</p>
</div>
<div>
<p>Deisboeck, Thomas S, Le Zhang, Jeongah Yoon, and Jose Costa. 2009. “In Silico Cancer Modeling: Is It Ready for Prime Time?” <em>Nature Clinical Practice Oncology</em> 6 (1). Nature Publishing Group: 34–42.</p>
</div>
<div>
<p>Del Sol, Antonio, Rudi Balling, Lee Hood, and David Galas. 2010. “Diseases as Network Perturbations.” <em>Current Opinion in Biotechnology</em> 21 (4). Elsevier: 566–71.</p>
</div>
<div>
<p>Dempster, Joshua M, Clare Pacini, Sasha Pantel, Fiona M Behan, Thomas Green, John Krill-Burger, Charlotte M Beaver, et al. 2019. “Agreement Between Two Large Pan-Cancer Crispr-Cas9 Gene Dependency Data Sets.” <em>Nature Communications</em> 10 (1). Nature Publishing Group: 1–14.</p>
</div>
<div>
<p>Ding, Li, Matthew H Bailey, Eduard Porta-Pardo, Vesteinn Thorsson, Antonio Colaprico, Denis Bertrand, David L Gibbs, et al. 2018. “Perspective on Oncogenic Processes at the End of the Beginning of Cancer Genomics.” <em>Cell</em> 173 (2). Elsevier: 305–20.</p>
</div>
<div>
<p>Domany, Eytan. 2014. “Using High-Throughput Transcriptomic Data for Prognosis: A Critical Overview and Perspectives.” <em>Cancer Research</em> 74 (17). AACR: 4612–21.</p>
</div>
<div>
<p>Drier, Yotam, Michal Sheffer, and Eytan Domany. 2013. “Pathway-Based Personalized Analysis of Cancer.” <em>Proceedings of the National Academy of Sciences</em> 110 (16). National Acad Sciences: 6388–93.</p>
</div>
<div>
<p>Dulbecco, Renato. 1986. “A Turning Point in Cancer Research: Sequencing the Human Genome.” <em>Science</em> 231. American Association for the Advancement of Science: 1055–7.</p>
</div>
<div>
<p>Eduati, Federica, Victoria Doldàn-Martelli, Bertram Klinger, Thomas Cokelaer, Anja Sieber, Fiona Kogera, Mathurin Dorel, Mathew J Garnett, Nils Blüthgen, and Julio Saez-Rodriguez. 2017. “Drug Resistance Mechanisms in Colorectal Cancer Dissected with Cell Type–specific Dynamic Logic Models.” <em>Cancer Research</em> 77 (12). AACR: 3364–75.</p>
</div>
<div>
<p>Eduati, Federica, Patricia Jaaks, Jessica Wappler, Thorsten Cramer, Christoph A Merten, Mathew J Garnett, and Julio Saez-Rodriguez. 2020. “Patient-Specific Logic Models of Signaling Pathways from Screenings on Cancer Biopsies to Prioritize Personalized Combination Therapies.” <em>Molecular Systems Biology</em> 16 (2). John Wiley &amp; Sons, Ltd.</p>
</div>
<div>
<p>Edwin, Francis, Kimberly Anderson, Chunyi Ying, and Tarun B Patel. 2009. “Intermolecular Interactions of Sprouty Proteins and Their Implications in Development and Disease.” <em>Molecular Pharmacology</em> 76 (4). ASPET: 679–91.</p>
</div>
<div>
<p>Elati, Mohamed, Pierre Neuvial, Monique Bolotin-Fukuhara, Emmanuel Barillot, François Radvanyi, and Céline Rouveirol. 2007. “LICORN: Learning Cooperative Regulation Networks from Gene Expression Data.” <em>Bioinformatics</em> 23 (18): 2407–14. doi:<a href="https://doi.org/10.1093/bioinformatics/btm352">10.1093/bioinformatics/btm352</a>.</p>
</div>
<div>
<p>Fauré, Adrien, Aurélien Naldi, Claudine Chaouiya, and Denis Thieffry. 2006. “Dynamical Analysis of a Generic Boolean Model for the Control of the Mammalian Cell Cycle.” <em>Bioinformatics</em> 22 (14). Oxford University Press: e124–e131.</p>
</div>
<div>
<p>Ferranti, Dana, David Krane, and David Craft. 2017. “The Value of Prior Knowledge in Machine Learning of Complex Network Systems.” <em>Bioinformatics</em> 33 (22). Oxford University Press: 3610–8.</p>
</div>
<div>
<p>Fey, Dirk, Melinda Halasz, Daniel Dreidax, Sean P Kennedy, Jordan F Hastings, Nora Rauch, Amaya Garcia Munoz, et al. 2015. “Signaling Pathway Models as Biomarkers: Patient-Specific Simulations of Jnk Activity Predict the Survival of Neuroblastoma Patients.” <em>Sci. Signal.</em> 8 (408). American Association for the Advancement of Science: ra130–ra130.</p>
</div>
<div>
<p>Flake, Gary William. 1998. <em>The Computational Beauty of Nature: Computer Explorations of Fractals, Chaos, Complex Systems, and Adaptation</em>. MIT press.</p>
</div>
<div>
<p>Flobak, Åsmund, Anaïs Baudot, Elisabeth Remy, Liv Thommesen, Denis Thieffry, Martin Kuiper, and Astrid Lægreid. 2015. “Discovery of Drug Synergies in Gastric Cancer Cells Predicted by Logical Modeling.” <em>PLoS Computational Biology</em> 11 (8). Public Library of Science.</p>
</div>
<div>
<p>Fowler, Andrew Cadle, Anna C Fowler, and AC Fowler. 1997. <em>Mathematical Models in the Applied Sciences</em>. Vol. 17. Cambridge University Press.</p>
</div>
<div>
<p>Frigg, Roman, and Stephan Hartmann. 2020. “Models in Science.” In <em>The Stanford Encyclopedia of Philosophy</em>, edited by Edward N. Zalta, Spring 2020. <a href="https://plato.stanford.edu/archives/spr2020/entries/models-science/" class="uri">https://plato.stanford.edu/archives/spr2020/entries/models-science/</a>; Metaphysics Research Lab, Stanford University.</p>
</div>
<div>
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6). Elsevier: 567–79.</p>
</div>
<div>
<p>Fumia, Herman F, and Marcelo L Martins. 2013. “Boolean Network Model for Cancer Pathways: Predicting Carcinogenesis and Targeted Therapy Outcomes.” <em>PloS One</em> 8 (7). Public Library of Science.</p>
</div>
<div>
<p>Gao, Hui, Joshua M Korn, Stéphane Ferretti, John E Monahan, Youzhen Wang, Mallika Singh, Chao Zhang, et al. 2015. “High-Throughput Screening Using Patient-Derived Tumor Xenografts to Predict Clinical Trial Drug Response.” <em>Nature Medicine</em> 21 (11). Nature Publishing Group: 1318.</p>
</div>
<div>
<p>Gillespie, Daniel T. 1976. “A General Method for Numerically Simulating the Stochastic Time Evolution of Coupled Chemical Reactions.” <em>Journal of Computational Physics</em> 22 (4): 403–34. doi:<a href="https://doi.org/10.1016/0021-9991(76)90041-3">10.1016/0021-9991(76)90041-3</a>.</p>
</div>
<div>
<p>Gopal, YN Vashisht, Wanleng Deng, Scott E Woodman, Kakajan Komurov, Prahlad Ram, Paul D Smith, and Michael A Davies. 2010. “Basal and Treatment-Induced Activation of Akt Mediates Resistance to Cell Death by Azd6244 (Arry-142886) in Braf-Mutant Human Cutaneous Melanoma Cells.” <em>Cancer Research</em> 70 (21). AACR: 8736–47.</p>
</div>
<div>
<p>Grieco, Luca, Laurence Calzone, Isabelle Bernard-Pierrot, François Radvanyi, Brigitte Kahn-Perles, and Denis Thieffry. 2013. “Integrative Modelling of the Influence of Mapk Network on Cancer Cell Fate Decision.” <em>PLoS Computational Biology</em> 9 (10). Public Library of Science: e1003286.</p>
</div>
<div>
<p>Hahn, William C, Christopher M Counter, Ante S Lundberg, Roderick L Beijersbergen, Mary W Brooks, and Robert A Weinberg. 1999. “Creation of Human Tumour Cells with Defined Genetic Elements.” <em>Nature</em> 400 (6743). Nature Publishing Group: 464–68.</p>
</div>
<div>
<p>Hajdu, Steven I. 2011a. “A Note from History: Landmarks in History of Cancer, Part 1.” <em>Cancer</em> 117 (5). Wiley Online Library: 1097–1102.</p>
</div>
<div>
<p>———. 2011b. “A Note from History: Landmarks in History of Cancer, Part 2.” <em>Cancer</em> 117 (12). Wiley Online Library: 2811–20.</p>
</div>
<div>
<p>———. 2012a. “A Note from History: Landmarks in History of Cancer, Part 3.” <em>Cancer</em> 118 (4). Wiley Online Library: 1155–68.</p>
</div>
<div>
<p>———. 2012b. “A Note from History: Landmarks in History of Cancer, Part 4.” <em>Cancer</em> 118 (20). Wiley Online Library: 4914–28.</p>
</div>
<div>
<p>Hajdu, Steven I, and Farbod Darvishian. 2013. “A Note from History: Landmarks in History of Cancer, Part 5.” <em>Cancer</em> 119 (8). Wiley Online Library: 1450–66.</p>
</div>
<div>
<p>Hajdu, Steven I, and Manjunath Vadmal. 2013. “A Note from History: Landmarks in History of Cancer, Part 6.” <em>Cancer</em> 119 (23). Wiley Online Library: 4058–82.</p>
</div>
<div>
<p>Han, Shujun, Yibo Ren, Wangxiao He, Huadong Liu, Zhe Zhi, Xinliang Zhu, Tielin Yang, et al. 2018. “ERK-Mediated Phosphorylation Regulates Sox10 Sumoylation and Targets Expression in Mutant Braf Melanoma.” <em>Nature Communications</em> 9 (1). Nature Publishing Group: 1–14.</p>
</div>
<div>
<p>Hanahan, Douglas, and Robert A Weinberg. 2000. “The Hallmarks of Cancer.” <em>Cell</em> 100 (1). Elsevier: 57–70.</p>
</div>
<div>
<p>———. 2011. “Hallmarks of Cancer: The Next Generation.” <em>Cell</em> 144 (5). Elsevier: 646–74.</p>
</div>
<div>
<p>Hart, Traver, and Jason Moffat. 2016. “BAGEL: A Computational Framework for Identifying Essential Genes from Pooled Library Screens.” <em>BMC Bioinformatics</em> 17 (1). Springer: 164.</p>
</div>
<div>
<p>Hartigan, J. A., and P. M. Hartigan. 1985. “The Dip Test of Unimodality.” <em>The Annals of Statistics</em> 13 (1): 70–84. doi:<a href="https://doi.org/10.1214/aos/1176346577">10.1214/aos/1176346577</a>.</p>
</div>
<div>
<p>Hasin, Yehudit, Marcus Seldin, and Aldons Lusis. 2017. “Multi-Omics Approaches to Disease.” <em>Genome Biology</em> 18 (1). BioMed Central: 83.</p>
</div>
<div>
<p>Hastie, Trevor, Robert Tibshirani, and Jerome Friedman. 2009. <em>The Elements of Statistical Learning: Data Mining, Inference, and Prediction</em>. Springer Science &amp; Business Media.</p>
</div>
<div>
<p>Hector, Suzanne, Markus Rehm, Jasmin Schmid, Joan Kehoe, Niamh McCawley, Patrick Dicker, Frank Murray, et al. 2012. “Clinical Application of a Systems Model of Apoptosis Execution for the Prediction of Colorectal Cancer Therapy Responses and Personalisation of Therapy.” <em>Gut</em> 61 (5). BMJ Publishing Group: 725–33.</p>
</div>
<div>
<p>Heijblom, Michelle, Linda M Meijer, Ton G van Leeuwen, Wiendelt Steenbergen, and Srirang Manohar. 2014. “Monte Carlo Simulations Shed Light on Bathsheba’s Suspect Breast.” <em>Journal of Biophotonics</em> 7 (5). Wiley Online Library: 323–31.</p>
</div>
<div>
<p>Heiser, Laura M, Anguraj Sadanandam, Wen-Lin Kuo, Stephen C Benz, Theodore C Goldstein, Sam Ng, William J Gibb, et al. 2012. “Subtype and Pathway Specific Responses to Anticancer Compounds in Breast Cancer.” <em>Proceedings of the National Academy of Sciences</em> 109 (8). National Acad Sciences: 2724–9.</p>
</div>
<div>
<p>Helikar, Tomáš, John Konvalina, Jack Heidel, and Jim A Rogers. 2008. “Emergent Decision-Making in Biological Signal Transduction Networks.” <em>Proceedings of the National Academy of Sciences</em> 105 (6). National Acad Sciences: 1913–8.</p>
</div>
<div>
<p>Helikar, Tomáš, Bryan Kowal, Sean McClenathan, Mitchell Bruckner, Thaine Rowley, Alex Madrahimov, Ben Wicks, Manish Shrestha, Kahani Limbu, and Jim A Rogers. 2012. “The Cell Collective: Toward an Open and Collaborative Approach to Systems Biology.” <em>BMC Systems Biology</em> 6 (1). BioMed Central: 96.</p>
</div>
<div>
<p>Henry, N Lynn, and Daniel F Hayes. 2012. “Cancer Biomarkers.” <em>Molecular Oncology</em> 6 (2). Wiley Online Library: 140–46.</p>
</div>
<div>
<p>Hernán, MA, and JM Robins. 2020. “Causal Inference: What If.” <em>Boca Raton: Chapman &amp; Hill/CRC</em>.</p>
</div>
<div>
<p>Hewitt, C Gordon. 1917. “Conservation of Wild Life in Canada.” Nature Publishing Group.</p>
</div>
<div>
<p>Hidalgo, Manuel, Frederic Amant, Andrew V Biankin, Eva Budinská, Annette T Byrne, Carlos Caldas, Robert B Clarke, et al. 2014. “Patient-Derived Xenograft Models: An Emerging Platform for Translational Cancer Research.” <em>Cancer Discovery</em> 4 (9). AACR: 998–1013.</p>
</div>
<div>
<p>Hill, Steven M, Laura M Heiser, Thomas Cokelaer, Michael Unger, Nicole K Nesser, Daniel E Carlin, Yang Zhang, et al. 2016. “Inferring Causal Molecular Networks: Empirical Assessment Through a Community-Based Effort.” <em>Nature Methods</em> 13 (4). Nature Publishing Group: 310.</p>
</div>
<div>
<p>Hornberg, Jorrit J, Frank J Bruggeman, Hans V Westerhoff, and Jan Lankelma. 2006. “Cancer: A Systems Biology Disease.” <em>Biosystems</em> 83 (2-3). Elsevier: 81–90.</p>
</div>
<div>
<p>Huang, Sui, Ingemar Ernberg, and Stuart Kauffman. 2009. “Cancer Attractors: A Systems View of Tumors from a Gene Network Dynamics and Developmental Perspective” 20 (7). Elsevier: 869–76.</p>
</div>
<div>
<p>Iorio, Francesco, Theo A Knijnenburg, Daniel J Vis, Graham R Bignell, Michael P Menden, Michael Schubert, Nanne Aben, et al. 2016. “A Landscape of Pharmacogenomic Interactions in Cancer.” <em>Cell</em> 166 (3). Elsevier: 740–54.</p>
</div>
<div>
<p>Ishwaran, Hemant. 2007. “Variable Importance in Binary Regression Trees and Forests.” <em>Electronic Journal of Statistics</em> 1. The Institute of Mathematical Statistics; the Bernoulli Society: 519–37.</p>
</div>
<div>
<p>Jacob, François, and Jacques Monod. 1961. “Genetic Regulatory Mechanisms in the Synthesis of Proteins.” <em>Journal of Molecular Biology</em> 3 (3). Elsevier: 318–56.</p>
</div>
<div>
<p>Jastrzebski, Katarzyna, Bram Thijssen, Roelof JC Kluin, Klaas de Lint, Ian J Majewski, Roderick L Beijersbergen, and Lodewyk FA Wessels. 2018. “Integrative Modeling Identifies Key Determinants of Inhibitor Sensitivity in Breast Cancer Cell Lines.” <em>Cancer Research</em> 78 (15). AACR: 4396–4410.</p>
</div>
<div>
<p>Jemal, Ahmedin, Elizabeth M Ward, Christopher J Johnson, Kathleen A Cronin, Jiemin Ma, A Blythe Ryerson, Angela Mariotto, et al. 2017. “Annual Report to the Nation on the Status of Cancer, 1975–2014, Featuring Survival.” <em>JNCI: Journal of the National Cancer Institute</em> 109 (9). Oxford University Press: djx030.</p>
</div>
<div>
<p>Jones, Siân, Xiaosong Zhang, D Williams Parsons, Jimmy Cheng-Ho Lin, Rebecca J Leary, Philipp Angenendt, Parminder Mankoo, et al. 2008. “Core Signaling Pathways in Human Pancreatic Cancers Revealed by Global Genomic Analyses.” <em>Science</em> 321 (5897). American Association for the Advancement of Science: 1801–6.</p>
</div>
<div>
<p>Kanehisa, Minoru, Susumu Goto, Yoko Sato, Miho Furumichi, and Mao Tanabe. 2012. “KEGG for Integration and Interpretation of Large-Scale Molecular Data Sets.” <em>Nucleic Acids Research</em> 40 (D1). Oxford University Press: D109–D114.</p>
</div>
<div>
<p>Kauffman, Stuart. 1969. “Homeostasis and Differentiation in Random Genetic Control Networks.” <em>Nature</em> 224 (5215). Springer: 177–78.</p>
</div>
<div>
<p>Khan, Faiz M, Stephan Marquardt, Shailendra K Gupta, Susanne Knoll, Ulf Schmitz, Alf Spitschak, David Engelmann, Julio Vera, Olaf Wolkenhauer, and Brigitte M Pützer. 2017. “Unraveling a Tumor Type-Specific Regulatory Core Underlying E2f1-Mediated Epithelial-Mesenchymal Transition to Predict Receptor Protein Signatures.” <em>Nature Communications</em> 8 (1). Nature Publishing Group: 1–15.</p>
</div>
<div>
<p>Kitano, Hiroaki. 2002. “Computational Systems Biology.” <em>Nature</em> 420 (6912). Nature Publishing Group: 206–10.</p>
</div>
<div>
<p>Klarner, Hannes, Adam Streck, and Heike Siebert. 2016. “PyBoolNet: A Python Package for the Generation, Analysis and Visualization of Boolean Networks.” <em>Bioinformatics</em> 33 (5). Oxford University Press: 770–72.</p>
</div>
<div>
<p>Klinger, Bertram, Anja Sieber, Raphaela Fritsche-Guenther, Franziska Witzel, Leanne Berry, Dirk Schumacher, Yibing Yan, et al. 2013. “Network Quantification of Egfr Signaling Unveils Potential for Targeted Combination Therapy.” <em>Molecular Systems Biology</em> 9 (1). John Wiley &amp; Sons, Ltd.</p>
</div>
<div>
<p>Knijnenburg, Theo A, Gunnar W Klau, Francesco Iorio, Mathew J Garnett, Ultan McDermott, Ilya Shmulevich, and Lodewyk FA Wessels. 2016. “Logic Models to Predict Continuous Outputs Based on Binary Inputs with an Application to Personalized Cancer Therapy.” <em>Scientific Reports</em> 6 (1). Nature Publishing Group: 1–14.</p>
</div>
<div>
<p>Knudson, Alfred G. 1971. “Mutation and Cancer: Statistical Study of Retinoblastoma.” <em>Proceedings of the National Academy of Sciences</em> 68 (4). National Acad Sciences: 820–23.</p>
</div>
<div>
<p>Knuuttila, Tarja, and Andrea Loettgers. 2017. “Modelling as Indirect Representation? The Lotka–Volterra Model Revisited.” <em>The British Journal for the Philosophy of Science</em> 68 (4). Oxford University Press: 1007–36.</p>
</div>
<div>
<p>Kopetz, S, J Desai, E Chan, JR Hecht, PJ O’dwyer, RJ Lee, KB Nolop, and L Saltz. 2010. “PLX4032 in Metastatic Colorectal Cancer Patients with Mutant Braf Tumors.” <em>Journal of Clinical Oncology</em> 28 (15_suppl). American Society of Clinical Oncology: 3534–4.</p>
</div>
<div>
<p>Kreeger, Pamela K, and Douglas A Lauffenburger. 2010. “Cancer Systems Biology: A Network Modeling Perspective.” <em>Carcinogenesis</em> 31 (1). Oxford University Press: 2–8.</p>
</div>
<div>
<p>Kumar, Prateek, Steven Henikoff, and Pauline C. Ng. 2009. “Predicting the Effects of Coding Non-Synonymous Variants on Protein Function Using the SIFT Algorithm.” <em>Nature Protocols</em> 4 (7): 1073. doi:<a href="https://doi.org/10.1038/nprot.2009.86">10.1038/nprot.2009.86</a>.</p>
</div>
<div>
<p>Kuperstein, I, E Bonnet, HA Nguyen, D Cohen, E Viara, L Grieco, S Fourquet, et al. 2015. “Atlas of Cancer Signalling Network: A Systems Biology Resource for Integrative Analysis of Cancer Data with Google Maps.” <em>Oncogenesis</em> 4 (7). Nature Publishing Group: e160–e160.</p>
</div>
<div>
<p>Kurilov, Roman, Benjamin Haibe-Kains, and Benedikt Brors. 2020. “Assessment of Modelling Strategies for Drug Response Prediction in Cell Lines and Xenografts.” <em>Scientific Reports</em> 10 (1). Nature Publishing Group: 1–11.</p>
</div>
<div>
<p>Lake, David, Sonia AL Corrêa, and Jürgen Müller. 2016. “Negative Feedback Regulation of the Erk1/2 Mapk Pathway.” <em>Cellular and Molecular Life Sciences</em> 73 (23). Springer: 4397–4413.</p>
</div>
<div>
<p>Lander, Eric S. 2011. “Initial Impact of the Sequencing of the Human Genome.” <em>Nature</em> 470 (7333). Nature Publishing Group: 187–97.</p>
</div>
<div>
<p>Lander, Eric S, Lauren M Linton, Bruce Birren, Chad Nusbaum, Michael C Zody, Jennifer Baldwin, Keri Devon, et al. 2001. “Initial Sequencing and Analysis of the Human Genome.” Macmillan Publishers Ltd.</p>
</div>
<div>
<p>Le Novere, Nicolas. 2015. “Quantitative and Logic Modelling of Molecular and Gene Networks.” <em>Nature Reviews Genetics</em> 16 (3). Nature Publishing Group: 146–58.</p>
</div>
<div>
<p>Lefebvre, Celine, Presha Rajbhandari, Mariano J. Alvarez, Pradeep Bandaru, Wei Keat Lim, Mai Sato, Kai Wang, et al. 2010. “A Human B-Cell Interactome Identifies MYB and FOXM1 as Master Regulators of Proliferation in Germinal Centers.” <em>Molecular Systems Biology</em> 6 (June): 377. doi:<a href="https://doi.org/10.1038/msb.2010.31">10.1038/msb.2010.31</a>.</p>
</div>
<div>
<p>Lehr, Stefan, Jorg Kotzka, Haluk Avci, Albert Sickmann, Helmut E Meyer, Armin Herkner, and Dirk Muller-Wieland. 2004. “Identification of Major Erk-Related Phosphorylation Sites in Gab1.” <em>Biochemistry</em> 43 (38). ACS Publications: 12133–40.</p>
</div>
<div>
<p>Letort, Gaelle, Arnau Montagud, Gautier Stoll, Randy Heiland, Emmanuel Barillot, Paul Macklin, Andrei Zinovyev, and Laurence Calzone. 2019. “PhysiBoSS: A Multi-Scale Agent-Based Modelling Framework Integrating Physical Dimension and Cell Signalling.” <em>Bioinformatics</em> 35 (7). Oxford University Press: 1188–96.</p>
</div>
<div>
<p>Li, Kun, Qunfeng Guo, Jun Yang, Hui Chen, Kewen Hu, Juan Zhao, Shunxin Zheng, et al. 2017. “FOXD3 Is a Tumor Suppressor of Colon Cancer by Inhibiting Egfr-Ras-Raf-Mek-Erk Signal Pathway.” <em>Oncotarget</em> 8 (3). Impact Journals, LLC: 5048.</p>
</div>
<div>
<p>Liu, Jianfang, Tara Lichtenberg, Katherine A Hoadley, Laila M Poisson, Alexander J Lazar, Andrew D Cherniack, Albert J Kovatich, et al. 2018. “An Integrated Tcga Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome Analytics.” <em>Cell</em> 173 (2). Elsevier: 400–416.</p>
</div>
<div>
<p>Lotka, AJ. 1925. “Principles of Physical Biology.” <em>Baltimore: Waverly</em>.</p>
</div>
<div>
<p>Manica, Matteo, Ali Oskooei, Jannis Born, Vigneshwari Subramanian, Julio Sáez-Rodríguez, and María Rodríguez Martínez. 2019. “Toward Explainable Anticancer Compound Sensitivity Prediction via Multimodal Attention-Based Convolutional Encoders.” <em>Molecular Pharmaceutics</em>. ACS Publications.</p>
</div>
<div>
<p>Manzano, José Luís, Laura Layos, Cristina Bugés, María de los Llanos Gil, Laia Vila, Eva Martinez-Balibrea, and Anna Martínez-Cardús. 2016. “Resistant Mechanisms to Braf Inhibitors in Melanoma.” <em>Annals of Translational Medicine</em> 4 (12). AME Publications.</p>
</div>
<div>
<p>Margolin, Adam A., Ilya Nemenman, Katia Basso, Chris Wiggins, Gustavo Stolovitzky, Riccardo Dalla Favera, and Andrea Califano. 2006. “ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context.” <em>BMC Bioinformatics</em> 7 (1): S7. doi:<a href="https://doi.org/10.1186/1471-2105-7-S1-S7">10.1186/1471-2105-7-S1-S7</a>.</p>
</div>
<div>
<p>Markevich, Nick I, Jan B Hoek, and Boris N Kholodenko. 2004. “Signaling Switches and Bistability Arising from Multisite Phosphorylation in Protein Kinase Cascades.” <em>The Journal of Cell Biology</em> 164 (3). Rockefeller University Press: 353–59.</p>
</div>
<div>
<p>Martignetti, Loredana, Laurence Calzone, Eric Bonnet, Emmanuel Barillot, and Andrei Zinovyev. 2016. “ROMA: Representation and Quantification of Module Activity from Target Expression Data.” <em>Frontiers in Genetics</em> 7. Frontiers: 18.</p>
</div>
<div>
<p>Meer, Dieudonne van der, Syd Barthorpe, Wanjuan Yang, Howard Lightfoot, Caitlin Hall, James Gilbert, Hayley E Francies, and Mathew J Garnett. 2019. “Cell Model Passports—a Hub for Clinical, Genetic and Functional Datasets of Preclinical Cancer Models.” <em>Nucleic Acids Research</em> 47 (D1). Oxford University Press: D923–D929.</p>
</div>
<div>
<p>Meinshausen, Nicolai, Alain Hauser, Joris M Mooij, Jonas Peters, Philip Versteeg, and Peter Bühlmann. 2016. “Methods for Causal Inference from Gene Perturbation Experiments and Validation.” <em>Proceedings of the National Academy of Sciences</em> 113 (27). National Acad Sciences: 7361–8.</p>
</div>
<div>
<p>Mermel, Craig H., Steven E. Schumacher, Barbara Hill, Matthew L. Meyerson, Rameen Beroukhim, and Gad Getz. 2011. “GISTIC2.0 Facilitates Sensitive and Confident Localization of the Targets of Focal Somatic Copy-Number Alteration in Human Cancers.” <em>Genome Biology</em> 12 (April): R41. doi:<a href="https://doi.org/10.1186/gb-2011-12-4-r41">10.1186/gb-2011-12-4-r41</a>.</p>
</div>
<div>
<p>Meyers, Robin M, Jordan G Bryan, James M McFarland, Barbara A Weir, Ann E Sizemore, Han Xu, Neekesh V Dharia, et al. 2017. “Computational Correction of Copy Number Effect Improves Specificity of Crispr–Cas9 Essentiality Screens in Cancer Cells.” <em>Nature Genetics</em> 49 (12). Nature Publishing Group: 1779–84.</p>
</div>
<div>
<p>Meyerson, Matthew, Stacey Gabriel, and Gad Getz. 2010. “Advances in Understanding Cancer Genomes Through Second-Generation Sequencing.” <em>Nature Reviews Genetics</em> 11 (10). Nature Publishing Group: 685–96.</p>
</div>
<div>
<p>Miller, Iain, Mingwei Min, Chen Yang, Chengzhe Tian, Sara Gookin, Dylan Carter, and Sabrina L Spencer. 2018. “Ki67 Is a Graded Rather Than a Binary Marker of Proliferation Versus Quiescence.” <em>Cell Reports</em> 24 (5). Elsevier: 1105–12.</p>
</div>
<div>
<p>Montagud, Arnau, Pauline Traynard, Loredana Martignetti, Eric Bonnet, Emmanuel Barillot, Andrei Zinovyev, and Laurence Calzone. 2017. “Conceptual and Computational Framework for Logical Modelling of Biological Networks Deregulated in Diseases.” <em>Briefings in Bioinformatics</em>. doi:<a href="https://doi.org/10.1093/bib/bbx163">10.1093/bib/bbx163</a>.</p>
</div>
<div>
<p>Murphy, Á C, Birgit Weyhenmeyer, Jasmin Schmid, Seán M Kilbride, Markus Rehm, Heinrich J Huber, C Senft, et al. 2013. “Activation of Executioner Caspases Is a Predictor of Progression-Free Survival in Glioblastoma Patients: A Systems Medicine Approach.” <em>Cell Death &amp; Disease</em> 4 (5). Nature Publishing Group: e629–e629.</p>
</div>
<div>
<p>Müssel, Christoph, Martin Hopfensitz, and Hans A Kestler. 2010. “BoolNet?an R Package for Generation, Reconstruction and Analysis of Boolean Networks.” <em>Bioinformatics</em> 26 (10). Oxford University Press: 1378–80.</p>
</div>
<div>
<p>Naldi, Aurélien. 2018. “BioLQM: A Java Toolkit for the Manipulation and Conversion of Logical Qualitative Models of Biological Networks.” <em>Frontiers in Physiology</em> 9. Frontiers: 1605.</p>
</div>
<div>
<p>Naldi, Aurélien, Céline Hernandez, Wassim Abou-Jaoudé, Pedro T Monteiro, Claudine Chaouiya, and Denis Thieffry. 2018. “Logical Modeling and Analysis of Cellular Regulatory Networks with Ginsim 3.0.” <em>Frontiers in Physiology</em> 9. Frontiers Media SA.</p>
</div>
<div>
<p>Naldi, Aurélien, Céline Hernandez, Nicolas Levy, Gautier Stoll, Pedro T Monteiro, Claudine Chaouiya, Tomáš Helikar, et al. 2018. “The Colomoto Interactive Notebook: Accessible and Reproducible Computational Analyses for Qualitative Biological Networks.” <em>Frontiers in Physiology</em> 9. Frontiers: 680.</p>
</div>
<div>
<p>Navin, Nicholas E. 2015. “The First Five Years of Single-Cell Cancer Genomics and Beyond.” <em>Genome Research</em> 25 (10). Cold Spring Harbor Lab: 1499–1507.</p>
</div>
<div>
<p>Network, Cancer Genome Atlas Research, and others. 2008. “Comprehensive Genomic Characterization Defines Human Glioblastoma Genes and Core Pathways.” <em>Nature</em> 455 (7216). Nature Publishing Group: 1061.</p>
</div>
<div>
<p>Network, Cancer Genome Atlas, and others. 2012. “Comprehensive Molecular Portraits of Human Breast Tumours.” <em>Nature</em> 490 (7418). Nature Publishing Group: 61.</p>
</div>
<div>
<p>Nicolò, Chiara, Cynthia Périer, Melanie Prague, Carine Bellera, Gaëtan MacGrogan, Olivier Saut, and Sébastien Benzekry. 2020. “Machine Learning and Mechanistic Modeling for Prediction of Metastatic Relapse in Early-Stage Breast Cancer.” <em>JCO Clinical Cancer Informatics</em> 4. American Society of Clinical Oncology: 259–74.</p>
</div>
<div>
<p>Novère, Nicolas Le, Michael Hucka, Huaiyu Mi, Stuart Moodie, Falk Schreiber, Anatoly Sorokin, Emek Demir, et al. 2009. “The Systems Biology Graphical Notation.” <em>Nature Biotechnology</em> 27 (8): 735–41. doi:<a href="https://doi.org/10.1038/nbt.1558">10.1038/nbt.1558</a>.</p>
</div>
<div>
<p>Nowell, Peter C. 1976. “The Clonal Evolution of Tumor Cell Populations.” <em>Science</em> 194 (4260). American Association for the Advancement of Science: 23–28.</p>
</div>
<div>
<p>Oldenhuis, CNAM, SF Oosting, JA Gietema, and EGE De Vries. 2008. “Prognostic Versus Predictive Value of Biomarkers in Oncology.” <em>European Journal of Cancer</em> 44 (7). Elsevier: 946–53.</p>
</div>
<div>
<p>Ostrowski, M., L. Paulevé, T. Schaub, A. Siegel, and C. Guziolowski. 2016. “Boolean Network Identification from Perturbation Time Series Data Combining Dynamics Abstraction and Logic Programming.” <em>Biosystems</em>, Selected papers from the Computational Methods in Systems Biology 2015 conference, 149 (November): 139–53. doi:<a href="https://doi.org/10.1016/j.biosystems.2016.07.009">10.1016/j.biosystems.2016.07.009</a>.</p>
</div>
<div>
<p>Paulevé, Loïc. 2017. “Pint: A Static Analyzer for Transient Dynamics of Qualitative Networks with IPython Interface.” In <em>CMSB 2017 - 15th conference on Computational Methods for Systems Biology</em>, 10545:370–16. Lecture Notes in Computer Science. Springer. doi:<a href="https://doi.org/10.1007/978-3-319-67471-1\_20">10.1007/978-3-319-67471-1\_20</a>.</p>
</div>
<div>
<p>Pawson, T, and N Warner. 2007. “Oncogenic Re-Wiring of Cellular Signaling Pathways.” <em>Oncogene</em> 26 (9). Nature Publishing Group: 1268–75.</p>
</div>
<div>
<p>Pereira, Bernard, Suet-Feung Chin, Oscar M. Rueda, Hans-Kristian Moen Vollan, Elena Provenzano, Helen A. Bardwell, Michelle Pugh, et al. 2016. “The Somatic Mutation Profiles of 2,433 Breast Cancers Refines Their Genomic and Transcriptomic Landscapes.” <em>Nature Communications</em> 7 (May). doi:<a href="https://doi.org/10.1038/ncomms11479">10.1038/ncomms11479</a>.</p>
</div>
<div>
<p>Perfetto, Livia, Leonardo Briganti, Alberto Calderone, Andrea Cerquone Perpetuini, Marta Iannuccelli, Francesca Langone, Luana Licata, et al. 2016. “SIGNOR: A Database of Causal Relationships Between Biological Entities.” <em>Nucleic Acids Research</em> 44 (D1). Oxford University Press: D548–D554.</p>
</div>
<div>
<p>Perou, Charles M, Stefanie S Jeffrey, Matt Van De Rijn, Christian A Rees, Michael B Eisen, Douglas T Ross, Alexander Pergamenschikov, et al. 1999. “Distinctive Gene Expression Patterns in Human Mammary Epithelial Cells and Breast Cancers.” <em>Proceedings of the National Academy of Sciences</em> 96 (16). National Acad Sciences: 9212–7.</p>
</div>
<div>
<p>Perou, Charles M, Therese Sørlie, Michael B Eisen, Matt Van De Rijn, Stefanie S Jeffrey, Christian A Rees, Jonathan R Pollack, et al. 2000. “Molecular Portraits of Human Breast Tumours.” <em>Nature</em> 406 (6797). Nature Publishing Group: 747–52.</p>
</div>
<div>
<p>Polynikis, Athanasios, SJ Hogan, and Mario di Bernardo. 2009. “Comparing Different Ode Modelling Approaches for Gene Regulatory Networks.” <em>Journal of Theoretical Biology</em> 261 (4). Elsevier: 511–30.</p>
</div>
<div>
<p>Potochnik, Angela. 2017. <em>Idealization and the Aims of Science</em>. University of Chicago Press.</p>
</div>
<div>
<p>Poulikakos, Poulikos I, Yogindra Persaud, Manickam Janakiraman, Xiangju Kong, Charles Ng, Gatien Moriceau, Hubing Shi, et al. 2011. “RAF Inhibitor Resistance Is Mediated by Dimerization of Aberrantly Spliced Braf (V600e).” <em>Nature</em> 480 (7377). Nature Publishing Group: 387.</p>
</div>
<div>
<p>Powathil, Gibin G, Maciej Swat, and Mark AJ Chaplain. 2015. “Systems Oncology: Towards Patient-Specific Treatment Regimes Informed by Multiscale Mathematical Modelling.” In <em>Seminars in Cancer Biology</em>, 30:13–20. Elsevier.</p>
</div>
<div>
<p>Prahallad, Anirudh, Chong Sun, Sidong Huang, Federica Di Nicolantonio, Ramon Salazar, Davide Zecchin, Roderick L Beijersbergen, Alberto Bardelli, and René Bernards. 2012. “Unresponsiveness of Colon Cancer to Braf (V600e) Inhibition Through Feedback Activation of Egfr.” <em>Nature</em> 483 (7387). Nature Publishing Group: 100–103.</p>
</div>
<div>
<p>Razzaq, Misbah, Loïc Paulevé, Anne Siegel, Julio Saez-Rodriguez, Jérémie Bourdon, and Carito Guziolowski. 2018. “Computational Discovery of Dynamic Cell Line Specific Boolean Networks from Multiplex Time-Course Data.” <em>PLoS Computational Biology</em> 14 (10). Public Library of Science: e1006538.</p>
</div>
<div>
<p>Reddy, E Premkumar, Roberta K Reynolds, Eugenio Santos, and Mariano Barbacid. 1982. “A Point Mutation Is Responsible for the Acquisition of Transforming Properties by the T24 Human Bladder Carcinoma Oncogene.” <em>Nature</em> 300 (5888). Nature Publishing Group: 149–52.</p>
</div>
<div>
<p>Remy, Elisabeth, Sandra Rebouissou, Claudine Chaouiya, Andrei Zinovyev, François Radvanyi, and Laurence Calzone. 2015. “A Modeling Approach to Explain Mutually Exclusive and Co-Occurring Genetic Alterations in Bladder Tumorigenesis.” <em>Cancer Research</em> 75 (19). AACR: 4042–52.</p>
</div>
<div>
<p>Reuter, Jason A, Damek V Spacek, and Michael P Snyder. 2015. “High-Throughput Sequencing Technologies.” <em>Molecular Cell</em> 58 (4). Elsevier: 586–97.</p>
</div>
<div>
<p>Rosenblueth, Arturo, and Norbert Wiener. 1945. “The Role of Models in Science.” <em>Philosophy of Science</em> 12 (4). Williams; Wilkins Co.: 316–21.</p>
</div>
<div>
<p>Rossi, Alessandro, Michela Roberto, Martina Panebianco, Andrea Botticelli, Federica Mazzuca, and Paolo Marchetti. 2019. “Drug Resistance of Braf-Mutant Melanoma: Review of up-to-Date Mechanisms of Action and Promising Targeted Agents.” <em>European Journal of Pharmacology</em>. Elsevier, 172621.</p>
</div>
<div>
<p>Saez-Rodriguez, Julio, Leonidas G Alexopoulos, MingSheng Zhang, Melody K Morris, Douglas A Lauffenburger, and Peter K Sorger. 2011a. “Comparing Signaling Networks Between Normal and Transformed Hepatocytes Using Discrete Logical Models.” <em>Cancer Research</em> 71 (16). AACR: 5400–5411.</p>
</div>
<div>
<p>Saez-Rodriguez, Julio, Leonidas G. Alexopoulos, MingSheng Zhang, Melody K. Morris, Douglas A. Lauffenburger, and Peter K. Sorger. 2011b. “Comparing Signaling Networks Between Normal and Transformed Hepatocytes Using Discrete Logical Models.” <em>Cancer Research</em> 71 (16): 5400–5411. doi:<a href="https://doi.org/10.1158/0008-5472.CAN-10-4453">10.1158/0008-5472.CAN-10-4453</a>.</p>
</div>
<div>
<p>Saez-Rodriguez, Julio, and Nils Blüthgen. 2020. “Personalized Signaling Models for Personalized Treatments.” <em>Molecular Systems Biology</em> 16 (1). John Wiley &amp; Sons, Ltd.</p>
</div>
<div>
<p>Sahin, Özgür, Holger Fröhlich, Christian Löbke, Ulrike Korf, Sara Burmester, Meher Majety, Jens Mattern, et al. 2009. “Modeling Erbb Receptor-Regulated G1/S Transition to Find Novel Targets for de Novo Trastuzumab Resistance.” <em>BMC Systems Biology</em> 3 (1). BioMed Central: 1.</p>
</div>
<div>
<p>Salvucci, Manuela, Arman Rahman, Alexa J Resler, Girish M Udupi, Deborah A McNamara, Elaine W Kay, Pierre Laurent-Puig, et al. 2019. “A Machine Learning Platform to Optimize the Translation of Personalized Network Models to the Clinic.” <em>JCO Clinical Cancer Informatics</em> 3. American Society of Clinical Oncology: 1–17.</p>
</div>
<div>
<p>Salvucci, Manuela, Maximilian L Würstle, Clare Morgan, Sarah Curry, Mattia Cremona, Andreas U Lindner, Orna Bacon, et al. 2017. “A Stepwise Integrated Approach to Personalized Risk Predictions in Stage Iii Colorectal Cancer.” <em>Clinical Cancer Research</em> 23 (5). AACR: 1200–1212.</p>
</div>
<div>
<p>Salvucci, Manuela, Zaitun Zakaria, Steven Carberry, Amanda Tivnan, Volker Seifert, Donat Kögel, Brona M Murphy, and Jochen HM Prehn. 2019. “System-Based Approaches as Prognostic Tools for Glioblastoma.” <em>BMC Cancer</em> 19 (1). Springer: 1092.</p>
</div>
<div>
<p>Samuels, Yardena, Zhenghe Wang, Alberto Bardelli, Natalie Silliman, Janine Ptak, Steve Szabo, Hai Yan, et al. 2004. “High Frequency of Mutations of the Pik3ca Gene in Human Cancers.” <em>Science</em> 304 (5670). American Association for the Advancement of Science: 554–54.</p>
</div>
<div>
<p>Sanchez-Vega, Francisco, Marco Mina, Joshua Armenia, Walid K Chatila, Augustin Luna, Konnor C La, Sofia Dimitriadoy, et al. 2018. “Oncogenic Signaling Pathways in the Cancer Genome Atlas.” <em>Cell</em> 173 (2). Elsevier: 321–37.</p>
</div>
<div>
<p>Sawyers, Charles L. 2008. “The Cancer Biomarker Problem.” <em>Nature</em> 452 (7187). Nature Publishing Group: 548–52.</p>
</div>
<div>
<p>Stadler, Zsofia K, Kasmintan A Schrader, Joseph Vijai, Mark E Robson, and Kenneth Offit. 2014. “Cancer Genomics and Inherited Risk.” <em>Journal of Clinical Oncology</em> 32 (7). American Society of Clinical Oncology: 687.</p>
</div>
<div>
<p>Steinway, Steven Nathaniel, Jorge Gomez Tejeda Zañudo, Paul J Michel, David J Feith, Thomas P Loughran, and Reka Albert. 2015. “Combinatorial Interventions Inhibit Tgf<span class="math inline">\(\beta\)</span>-Driven Epithelial-to-Mesenchymal Transition and Support Hybrid Cellular Phenotypes.” <em>NPJ Systems Biology and Applications</em> 1. Nature Publishing Group: 15014.</p>
</div>
<div>
<p>Stoll, Gautier, Barthélémy Caron, Eric Viara, Aurélien Dugourd, Andrei Zinovyev, Aurélien Naldi, Guido Kroemer, Emmanuel Barillot, and Laurence Calzone. 2017. “MaBoSS 2.0: An Environment for Stochastic Boolean Modeling.” <em>Bioinformatics</em> 33 (14). Oxford University Press: 2226–8.</p>
</div>
<div>
<p>Stoll, Gautier, Eric Viara, Emmanuel Barillot, and Laurence Calzone. 2012. “Continuous Time Boolean Modeling for Biological Signaling: Application of Gillespie Algorithm.” <em>BMC Systems Biology</em> 6 (1). BioMed Central: 116.</p>
</div>
<div>
<p>Stratton, Michael R, Peter J Campbell, and P Andrew Futreal. 2009. “The Cancer Genome.” <em>Nature</em> 458 (7239). Nature Publishing Group: 719–24.</p>
</div>
<div>
<p>Swanson, Kristin R, Carly Bridge, JD Murray, and Ellsworth C Alvord Jr. 2003. “Virtual and Real Brain Tumors: Using Mathematical Modeling to Quantify Glioma Growth and Invasion.” <em>Journal of the Neurological Sciences</em> 216 (1). Elsevier: 1–10.</p>
</div>
<div>
<p>Terfve, Camille D. A., Edmund H. Wilkes, Pedro Casado, Pedro R. Cutillas, and Julio Saez-Rodriguez. 2015. “Large-Scale Models of Signal Propagation in Human Cells Derived from Discovery Phosphoproteomic Data.” <em>Nature Communications</em> 6 (September): 8033. doi:<a href="https://doi.org/10.1038/ncomms9033">10.1038/ncomms9033</a>.</p>
</div>
<div>
<p>Terfve, Camille, Thomas Cokelaer, David Henriques, Aidan MacNamara, Emanuel Goncalves, Melody K. Morris, Martijn van Iersel, Douglas A. Lauffenburger, and Julio Saez-Rodriguez. 2012. “CellNOptR: A Flexible Toolkit to Train Protein Signaling Networks to Data Using Multiple Logic Formalisms.” <em>BMC Systems Biology</em> 6 (1): 133. doi:<a href="https://doi.org/10.1186/1752-0509-6-133">10.1186/1752-0509-6-133</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E, Ali Naderi, Nuno L Barbosa-Morais, and Carlos Caldas. 2006. “PACK: Profile Analysis Using Clustering and Kurtosis to Find Molecular Classifiers in Cancer.” <em>Bioinformatics</em> 22 (18). Oxford University Press: 2269–75.</p>
</div>
<div>
<p>Thomas, René. 1973. “Boolean Formalization of Genetic Control Circuits.” <em>Journal of Theoretical Biology</em> 42 (3). Elsevier: 563–85.</p>
</div>
<div>
<p>Thomas, René, and Richard d’Ari. 1990. <em>Biological Feedback</em>. CRC press.</p>
</div>
<div>
<p>Tokheim, Collin J., Nickolas Papadopoulos, Kenneth W. Kinzler, Bert Vogelstein, and Rachel Karchin. 2016. “Evaluating the Evaluation of Cancer Driver Genes.” <em>Proceedings of the National Academy of Sciences</em> 113 (50): 14330–5. doi:<a href="https://doi.org/10.1073/pnas.1616440113">10.1073/pnas.1616440113</a>.</p>
</div>
<div>
<p>Tomasetti, Cristian, and Bert Vogelstein. 2015. “Variation in Cancer Risk Among Tissues Can Be Explained by the Number of Stem Cell Divisions.” <em>Science</em> 347 (6217). American Association for the Advancement of Science: 78–81.</p>
</div>
<div>
<p>Tomasetti, Cristian, Luigi Marchionni, Martin A Nowak, Giovanni Parmigiani, and Bert Vogelstein. 2015. “Only Three Driver Gene Mutations Are Required for the Development of Lung and Colorectal Cancers.” <em>Proceedings of the National Academy of Sciences</em> 112 (1). National Acad Sciences: 118–23.</p>
</div>
<div>
<p>Tomlins, Scott A, Daniel R Rhodes, Sven Perner, Saravana M Dhanasekaran, Rohit Mehra, Xiao-Wei Sun, Sooryanarayana Varambally, et al. 2005. “Recurrent Fusion of Tmprss2 and Ets Transcription Factor Genes in Prostate Cancer.” <em>Science</em> 310 (5748). American Association for the Advancement of Science: 644–48.</p>
</div>
<div>
<p>Trefois, Christophe, Paul MA Antony, Jorge Goncalves, Alexander Skupin, and Rudi Balling. 2015. “Critical Transitions in Chronic Disease: Transferring Concepts from Ecology to Systems Medicine.” <em>Current Opinion in Biotechnology</em> 34. Elsevier: 48–55.</p>
</div>
<div>
<p>Türei, Dénes, Tamás Korcsmáros, and Julio Saez-Rodriguez. 2016. “OmniPath: Guidelines and Gateway for Literature-Curated Signaling Pathway Resources.” <em>Nature Methods</em> 13 (12). Nature Publishing Group: 966.</p>
</div>
<div>
<p>Tyson, John J, William T Baumann, Chun Chen, Anael Verdugo, Iman Tavassoly, Yue Wang, Louis M Weiner, and Robert Clarke. 2011. “Dynamic Modelling of Oestrogen Signalling and Cell Fate in Breast Cancer Cells.” <em>Nature Reviews Cancer</em> 11 (7). Nature Publishing Group: 523–32.</p>
</div>
<div>
<p>Tyson, John J, Katherine C Chen, and Bela Novak. 2003. “Sniffers, Buzzers, Toggles and Blinkers: Dynamics of Regulatory and Signaling Pathways in the Cell.” <em>Current Opinion in Cell Biology</em> 15 (2). Elsevier: 221–31.</p>
</div>
<div>
<p>Ursem, Carling, Chloe E Atreya, and Katherine Van Loon. 2018. “Emerging Treatment Options for Braf-Mutant Colorectal Cancer.” <em>Gastrointestinal Cancer: Targets and Therapy</em> 8. NIH Public Access: 13.</p>
</div>
<div>
<p>Van’t Veer, Laura J, Hongyue Dai, Marc J Van De Vijver, Yudong D He, Augustinus AM Hart, Mao Mao, Hans L Peterse, et al. 2002. “Gene Expression Profiling Predicts Clinical Outcome of Breast Cancer.” <em>Nature</em> 415 (6871). Nature Publishing Group: 530–36.</p>
</div>
<div>
<p>Venet, David, Jacques E Dumont, Vincent Detours, and others. 2011. “Most Random Gene Expression Signatures Are Significantly Associated with Breast Cancer Outcome.” <em>PLoS Comput Biol</em> 7 (10): e1002240.</p>
</div>
<div>
<p>Venter, J Craig, Mark D Adams, Eugene W Myers, Peter W Li, Richard J Mural, Granger G Sutton, Hamilton O Smith, et al. 2001. “The Sequence of the Human Genome.” <em>Science</em> 291 (5507). American Association for the Advancement of Science: 1304–51.</p>
</div>
<div>
<p>Verlingue, Loic, Laurence Calzone, Maud Kamal, Nicolas Servant, Lisa Belin, Emmanuel Barillot, and Christophe Le Tourneau. 2016. “In Silico Prediction of the Clinical Response to the MTOR Inhibitor Everolimus Using a Boolean Model: Validation from a Cohort of the Shiva Trial.” AACR.</p>
</div>
<div>
<p>Verlingue, Loic, Aurélien Dugourd, Gautier Stoll, Emmanuel Barillot, Laurence Calzone, and Arturo Londoño-Vallejo. 2016. “A Comprehensive Approach to the Molecular Determinants of Lifespan Using a Boolean Model of Geroconversion.” <em>Aging Cell</em> 15 (6). Wiley Online Library: 1018–26.</p>
</div>
<div>
<p>Verny, Louis, Nadir Sella, Séverine Affeldt, Param Priya Singh, and Hervé Isambert. 2017. “Learning Causal Networks with Latent Variables from Multivariate Information in Genomic Data.” <em>PLOS Computational Biology</em> 13 (10): e1005662. doi:<a href="https://doi.org/10.1371/journal.pcbi.1005662">10.1371/journal.pcbi.1005662</a>.</p>
</div>
<div>
<p>Vert, Jean-Philippe, Jian Qiu, and William S. Noble. 2007. “A New Pairwise Kernel for Biological Network Inference with Support Vector Machines.” <em>BMC Bioinformatics</em> 8 (Suppl 10): S8.</p>
</div>
<div>
<p>Videla, Santiago, Julio Saez-Rodriguez, Carito Guziolowski, and Anne Siegel. 2017. “Caspo: A Toolbox for Automated Reasoning on the Response of Logical Signaling Networks Families.” <em>Bioinformatics</em> 33 (6): 947–50. doi:<a href="https://doi.org/10.1093/bioinformatics/btw738">10.1093/bioinformatics/btw738</a>.</p>
</div>
<div>
<p>Villaverde, Alejandro F, and Julio R Banga. 2014. “Reverse Engineering and Identification in Systems Biology: Strategies, Perspectives and Challenges.” <em>Journal of the Royal Society Interface</em> 11 (91). The Royal Society: 20130505.</p>
</div>
<div>
<p>Vis, Daniel J, Lorenzo Bombardelli, Howard Lightfoot, Francesco Iorio, Mathew J Garnett, and Lodewyk FA Wessels. 2016. “Multilevel Models Improve Precision and Speed of Ic50 Estimates.” <em>Pharmacogenomics</em> 17 (7). Future Medicine: 691–700.</p>
</div>
<div>
<p>Volterra, Vito. 1926. “Fluctuations in the Abundance of a Species Considered Mathematically.” Nature Publishing Group.</p>
</div>
<div>
<p>Vucic, Emily A, Kelsie L Thu, Keith Robison, Leszek A Rybaczyk, Raj Chari, Carlos E Alvarez, and Wan L Lam. 2012. “Translating Cancer ‘Omics’ to Improved Outcomes.” <em>Genome Research</em> 22 (2). Cold Spring Harbor Lab: 188–95.</p>
</div>
<div>
<p>Wan, Frederick YM. 2018. <em>Mathematical Models and Their Analysis</em>. Vol. 79. SIAM.</p>
</div>
<div>
<p>Wang, Jing, Sijin Wen, W. Fraser Symmans, Lajos Pusztai, and Kevin R. Coombes. 2009. “The Bimodality Index: A Criterion for Discovering and Ranking Bimodal Signatures from Cancer Gene Expression Profiling Data.” <em>Cancer Informatics</em> 7 (August): 199–216. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2730180/" class="uri">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2730180/</a>.</p>
</div>
<div>
<p>Watson, James D, and Francis HC Crick. 1953. “Molecular Structure of Nucleic Acids.” <em>Nature</em> 171 (4356): 737–38.</p>
</div>
<div>
<p>Weinberg, Robert A. 2013. <em>The Biology of Cancer</em>. Garland science.</p>
</div>
<div>
<p>Weinstein, Nathan, Luis Mendoza, Isidoro Gitler, and Jaime Klapp. 2017. “A Network Model to Explore the Effect of the Micro-Environment on Endothelial Cell Behavior During Angiogenesis.” <em>Frontiers in Physiology</em> 8. Frontiers: 960.</p>
</div>
<div>
<p>Wellbrock, Claudia, Maria Karasarides, and Richard Marais. 2004. “The Raf Proteins Take Centre Stage.” <em>Nature Reviews Molecular Cell Biology</em> 5 (11). Nature Publishing Group: 875–85.</p>
</div>
<div>
<p>Woodhouse, Steven, Nir Piterman, Christoph M Wintersteiger, Berthold Göttgens, and Jasmin Fisher. 2018. “SCNS: A Graphical Tool for Reconstructing Executable Regulatory Networks from Single-Cell Genomic Data.” <em>BMC Systems Biology</em> 12 (1). BioMed Central: 59.</p>
</div>
<div>
<p>Wroblewski, David, Branka Mijatov, Nethia Mohana-Kumaran, Fritz Lai, Stuart J Gallagher, Nikolas K Haass, Xu Dong Zhang, and Peter Hersey. 2013. “The Bh3-Mimetic Abt-737 Sensitizes Human Melanoma Cells to Apoptosis Induced by Selective Braf Inhibitors but Does Not Reverse Acquired Resistance.” <em>Carcinogenesis</em> 34 (2). Oxford University Press UK: 237–47.</p>
</div>
<div>
<p>Yang, Wanjuan, Jorge Soares, Patricia Greninger, Elena J Edelman, Howard Lightfoot, Simon Forbes, Nidhi Bindal, et al. 2012. “Genomics of Drug Sensitivity in Cancer (Gdsc): A Resource for Therapeutic Biomarker Discovery in Cancer Cells.” <em>Nucleic Acids Research</em> 41 (D1). Oxford University Press: D955–D961.</p>
</div>
<div>
<p>Zañudo, Jorge Gómez Tejeda, Maurizio Scaltriti, and Réka Albert. 2017. “A Network Modeling Approach to Elucidate Drug Resistance Mechanisms and Predict Combinatorial Drug Treatments in Breast Cancer.” <em>Cancer Convergence</em> 1 (1). Springer: 5.</p>
</div>
</div>
</div>






            </section>

          </div>
        </div>
      </div>
<a href="about-causality.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
