<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Scientific modeling: abstract the complexity | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</title>
  <meta name="description" content="This is my PhD thesis." />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Scientific modeling: abstract the complexity | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is my PhD thesis." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Scientific modeling: abstract the complexity | From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation" />
  
  <meta name="twitter:description" content="This is my PhD thesis." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="preface.html"/>
<link rel="next" href="cancer-as-deregulation-of-complex-machinery.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.9.2.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-1.52.2/plotly-latest.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Jonas BÉAL</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>PhD Thesis</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="resumé.html"><a href="resumé.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Cells and their models</b></span></li>
<li class="chapter" data-level="1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html"><i class="fa fa-check"></i><b>1</b> Scientific modeling: abstract the complexity</a><ul>
<li class="chapter" data-level="1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#what-is-a-model"><i class="fa fa-check"></i><b>1.1</b> What is a model?</a><ul>
<li class="chapter" data-level="1.1.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#in-your-own-words"><i class="fa fa-check"></i><b>1.1.1</b> In your own words</a></li>
<li class="chapter" data-level="1.1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#physical-world-and-world-of-ideas"><i class="fa fa-check"></i><b>1.1.2</b> Physical world and world of ideas</a></li>
<li class="chapter" data-level="1.1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#preview-about-cancer-models"><i class="fa fa-check"></i><b>1.1.3</b> Preview about cancer models</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#stat-mech"><i class="fa fa-check"></i><b>1.2</b> Statistics or mechanistic</a><ul>
<li class="chapter" data-level="1.2.1" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#the-inside-of-the-box"><i class="fa fa-check"></i><b>1.2.1</b> The inside of the box</a></li>
<li class="chapter" data-level="1.2.2" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#lotkasection"><i class="fa fa-check"></i><b>1.2.2</b> A tale of prey and predators</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="scientific-modeling-abstract-the-complexity.html"><a href="scientific-modeling-abstract-the-complexity.html#simplicity-is-the-ultimate-sophistication"><i class="fa fa-check"></i><b>1.3</b> Simplicity is the ultimate sophistication</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html"><i class="fa fa-check"></i><b>2</b> Cancer as deregulation of complex machinery</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#what-is-cancer"><i class="fa fa-check"></i><b>2.1</b> What is cancer?</a></li>
<li class="chapter" data-level="2.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#epidemio"><i class="fa fa-check"></i><b>2.2</b> Cancer from a distance: epidemiology and main figures</a></li>
<li class="chapter" data-level="2.3" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#molecular-biology"><i class="fa fa-check"></i><b>2.3</b> Basic molecular biology and cancer</a><ul>
<li class="chapter" data-level="2.3.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#central-dogma-and-core-principles"><i class="fa fa-check"></i><b>2.3.1</b> Central dogma and core principles</a></li>
<li class="chapter" data-level="2.3.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#a-rogue-machinery"><i class="fa fa-check"></i><b>2.3.2</b> A rogue machinery</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#the-new-era-of-genomics"><i class="fa fa-check"></i><b>2.4</b> The new era of genomics</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#from-sequencing-to-multi-omics-data"><i class="fa fa-check"></i><b>2.4.1</b> From sequencing to multi-omics data</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#state-of-the-art-of-cancer-data"><i class="fa fa-check"></i><b>2.4.2</b> State-of-the art of cancer data</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="cancer-as-deregulation-of-complex-machinery.html"><a href="cancer-as-deregulation-of-complex-machinery.html#data-and-beyond-from-genetic-to-network-disease"><i class="fa fa-check"></i><b>2.5</b> Data and beyond: from genetic to network disease</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html"><i class="fa fa-check"></i><b>3</b> Mechanistic modeling of cancer: from complex disease to systems biology</a><ul>
<li class="chapter" data-level="3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#introducing-the-diversity-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>3.1</b> Introducing the diversity of mechanistic models of cancer</a></li>
<li class="chapter" data-level="3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#cell-circuitry-and-the-need-for-cancer-systems-biology"><i class="fa fa-check"></i><b>3.2</b> Cell circuitry and the need for cancer systems biology</a></li>
<li class="chapter" data-level="3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#mechanistic-models-of-molecular-signaling"><i class="fa fa-check"></i><b>3.3</b> Mechanistic models of molecular signaling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#networks-and-data"><i class="fa fa-check"></i><b>3.3.1</b> Networks and data</a></li>
<li class="chapter" data-level="3.3.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#different-formalisms-for-different-applications"><i class="fa fa-check"></i><b>3.3.2</b> Different formalisms for different applications</a></li>
<li class="chapter" data-level="3.3.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#some-examples-of-complex-features"><i class="fa fa-check"></i><b>3.3.3</b> Some examples of complex features</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#from-mechanistic-models-to-clinical-impact"><i class="fa fa-check"></i><b>3.4</b> From mechanistic models to clinical impact?</a><ul>
<li class="chapter" data-level="3.4.1" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#a-new-class-of-biomarkers"><i class="fa fa-check"></i><b>3.4.1</b> A new class of biomarkers</a></li>
<li class="chapter" data-level="3.4.2" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#prognostic"><i class="fa fa-check"></i><b>3.4.2</b> Prognostic models</a></li>
<li class="chapter" data-level="3.4.3" data-path="mechanistic-cancer.html"><a href="mechanistic-cancer.html#predictive"><i class="fa fa-check"></i><b>3.4.3</b> Predictive models</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Personalized logical models of cancer</b></span></li>
<li class="chapter" data-level="4" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html"><i class="fa fa-check"></i><b>4</b> Logical modeling principles and data integration</a><ul>
<li class="chapter" data-level="4.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-modeling-paradigms-for-qualitative-description"><i class="fa fa-check"></i><b>4.1</b> Logical modeling paradigms for qualitative description</a><ul>
<li class="chapter" data-level="4.1.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#regulatory-graph-and-logical-rules"><i class="fa fa-check"></i><b>4.1.1</b> Regulatory graph and logical rules</a></li>
<li class="chapter" data-level="4.1.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#state-transition-graph-and-updates"><i class="fa fa-check"></i><b>4.1.2</b> State transition graph and updates</a></li>
<li class="chapter" data-level="4.1.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#tools-for-logical-modeling"><i class="fa fa-check"></i><b>4.1.3</b> Tools for logical modeling</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#maboss-section"><i class="fa fa-check"></i><b>4.2</b> The MaBoSS framework for logical modeling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#gillespie-algorithm"><i class="fa fa-check"></i><b>4.2.1</b> Gillespie algorithm</a></li>
<li class="chapter" data-level="4.2.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#a-stochastic-exploration-of-model-behaviours"><i class="fa fa-check"></i><b>4.2.2</b> A stochastic exploration of model behaviours</a></li>
<li class="chapter" data-level="4.2.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#from-theoretical-models-to-data-models"><i class="fa fa-check"></i><b>4.2.3</b> From theoretical models to data models?</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#logical-data-section"><i class="fa fa-check"></i><b>4.3</b> Data integration and semi-quantitative logical modeling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#build-the-regulatory-graph"><i class="fa fa-check"></i><b>4.3.1</b> Build the regulatory graph</a></li>
<li class="chapter" data-level="4.3.2" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#define-the-logical-rules"><i class="fa fa-check"></i><b>4.3.2</b> Define the logical rules</a></li>
<li class="chapter" data-level="4.3.3" data-path="logical-modeling-principles-and-data-integration.html"><a href="logical-modeling-principles-and-data-integration.html#validate-the-model"><i class="fa fa-check"></i><b>4.3.3</b> Validate the model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html"><i class="fa fa-check"></i><b>5</b> Personalization of logical models: method and prognostic validation</a><ul>
<li class="chapter" data-level="5.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#from-one-generic-model-to-data-specific-models-with-profile-method"><i class="fa fa-check"></i><b>5.1</b> From one generic model to data-specific models with PROFILE method</a><ul>
<li class="chapter" data-level="5.1.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#gathering-knowledge-and-data"><i class="fa fa-check"></i><b>5.1.1</b> Gathering knowledge and data</a></li>
<li class="chapter" data-level="5.1.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#adapting-patient-profiles-to-a-logical-model"><i class="fa fa-check"></i><b>5.1.2</b> Adapting patient profiles to a logical model</a></li>
<li class="chapter" data-level="5.1.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#personalizing-logical-models-with-patient-data"><i class="fa fa-check"></i><b>5.1.3</b> Personalizing logical models with patient data</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#an-integration-tool-for-high-dimensional-data"><i class="fa fa-check"></i><b>5.2</b> An integration tool for high-dimensional data?</a><ul>
<li class="chapter" data-level="5.2.1" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#PROFILE-CL"><i class="fa fa-check"></i><b>5.2.1</b> Biological relevance in cell lines</a></li>
<li class="chapter" data-level="5.2.2" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#validation-METABRIC"><i class="fa fa-check"></i><b>5.2.2</b> Validation with patient data</a></li>
<li class="chapter" data-level="5.2.3" data-path="personalization-of-logical-models-method-and-prognostic-validation.html"><a href="personalization-of-logical-models-method-and-prognostic-validation.html#perspectives"><i class="fa fa-check"></i><b>5.2.3</b> Perspectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html"><i class="fa fa-check"></i><b>6</b> Personalized logical models to study an interpret drug response</a><ul>
<li class="chapter" data-level="6.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#one-step-further-with-drugs"><i class="fa fa-check"></i><b>6.1</b> One step further with drugs</a><ul>
<li class="chapter" data-level="6.1.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#modeling-response-to-cancer-treatments"><i class="fa fa-check"></i><b>6.1.1</b> Modeling response to cancer treatments</a></li>
<li class="chapter" data-level="6.1.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#an-application-of-personalized-logical-models"><i class="fa fa-check"></i><b>6.1.2</b> An application of personalized logical models</a></li>
<li class="chapter" data-level="6.1.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-pan-cancer-attempt"><i class="fa fa-check"></i><b>6.1.3</b> A pan-cancer attempt</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#case-study-on-braf-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>6.2</b> Case study on BRAF in melanoma and colorectal cancers</a><ul>
<li class="chapter" data-level="6.2.1" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#biological-and-clinical-context"><i class="fa fa-check"></i><b>6.2.1</b> Biological and clinical context</a></li>
<li class="chapter" data-level="6.2.2" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#a-logical-model-centred-on-braf"><i class="fa fa-check"></i><b>6.2.2</b> A logical model centred on BRAF</a></li>
<li class="chapter" data-level="6.2.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#cell-lines-data"><i class="fa fa-check"></i><b>6.2.3</b> Cell lines data</a></li>
<li class="chapter" data-level="6.2.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#validation-of-personalized-models-using-crisprcas9-and-drug-screening"><i class="fa fa-check"></i><b>6.2.4</b> Validation of personalized models using CRISPR/Cas9 and drug screening</a></li>
<li class="chapter" data-level="6.2.5" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#comparison-of-the-mechanistic-approach-with-machine-learning-methods"><i class="fa fa-check"></i><b>6.2.5</b> Comparison of the mechanistic approach with machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#prostate-model"><i class="fa fa-check"></i><b>6.3</b> Application on prostate cancer study and challenges</a></li>
<li class="chapter" data-level="6.4" data-path="personalized-logical-models-to-study-an-interpret-drug-response.html"><a href="personalized-logical-models-to-study-an-interpret-drug-response.html#limitations-and-perspectives"><i class="fa fa-check"></i><b>6.4</b> Limitations and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Statistical quantification of the clinical impact of models</b></span></li>
<li class="chapter" data-level="7" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html"><i class="fa fa-check"></i><b>7</b> Information flows in mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#evaluation-of-models-as-biomarkers"><i class="fa fa-check"></i><b>7.1</b> Evaluation of models as biomarkers</a></li>
<li class="chapter" data-level="7.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#processing-of-biological-information"><i class="fa fa-check"></i><b>7.2</b> Processing of biological information</a><ul>
<li class="chapter" data-level="7.2.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#information-in-information-out"><i class="fa fa-check"></i><b>7.2.1</b> Information in, information out</a></li>
<li class="chapter" data-level="7.2.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#emergence-of-information-in-artifical-examples"><i class="fa fa-check"></i><b>7.2.2</b> Emergence of information in artifical examples</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#reanalysis-of-mechanistic-models-of-cancer"><i class="fa fa-check"></i><b>7.3</b> Reanalysis of mechanistic models of cancer</a><ul>
<li class="chapter" data-level="7.3.1" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#ode-model-of-jnk-pathway-by-fey2015signaling"><i class="fa fa-check"></i><b>7.3.1</b> ODE model of JNK pathway by <span class="citation">Fey et al. (<a href="#ref-fey2015signaling">2015</a>)</span></a></li>
<li class="chapter" data-level="7.3.2" data-path="information-flows-in-mechanistic-models-of-cancer.html"><a href="information-flows-in-mechanistic-models-of-cancer.html#personalized-logical-models-braf-inhibition-in-melanoma-and-colorectal-cancers"><i class="fa fa-check"></i><b>7.3.2</b> Personalized logical models: BRAF inhibition in melanoma and colorectal cancers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html"><i class="fa fa-check"></i><b>8</b> Clinical evidence generation and causal inference</a><ul>
<li class="chapter" data-level="8.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#clinical-trials-and-beyond"><i class="fa fa-check"></i><b>8.1</b> Clinical trials and beyond</a><ul>
<li class="chapter" data-level="8.1.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#randomized-clinical-trials-as-gold-standards"><i class="fa fa-check"></i><b>8.1.1</b> Randomized clinical trials as gold standards</a></li>
<li class="chapter" data-level="8.1.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#observational-data-and-confounding-factors"><i class="fa fa-check"></i><b>8.1.2</b> Observational data and confounding factors</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-inference-methods-to-leverage-data"><i class="fa fa-check"></i><b>8.2</b> Causal inference methods to leverage data</a><ul>
<li class="chapter" data-level="8.2.1" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#notations-in-potential-outcomes-framework"><i class="fa fa-check"></i><b>8.2.1</b> Notations in potential outcomes framework</a></li>
<li class="chapter" data-level="8.2.2" data-path="clinical-evidence-generation-and-causal-inference.html"><a href="clinical-evidence-generation-and-causal-inference.html#causal-identification-simple"><i class="fa fa-check"></i><b>8.2.2</b> Identification of causal effects</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="chapter-precision.html"><a href="chapter-precision.html"><i class="fa fa-check"></i><b>9</b> Causal inference for precision medicine</a><ul>
<li class="chapter" data-level="9.1" data-path="chapter-precision.html"><a href="chapter-precision.html#precision-medicine-in-oncology"><i class="fa fa-check"></i><b>9.1</b> Precision medicine in oncology</a><ul>
<li class="chapter" data-level="9.1.1" data-path="chapter-precision.html"><a href="chapter-precision.html#main-PDX"><i class="fa fa-check"></i><b>9.1.1</b> An illustration with patient-derived xenografts</a></li>
<li class="chapter" data-level="9.1.2" data-path="chapter-precision.html"><a href="chapter-precision.html#clinical-trials-and-treatment-algorithms"><i class="fa fa-check"></i><b>9.1.2</b> Clinical trials and treatment algorithms</a></li>
<li class="chapter" data-level="9.1.3" data-path="chapter-precision.html"><a href="chapter-precision.html#computational-models-to-assign-cancer-treatments"><i class="fa fa-check"></i><b>9.1.3</b> Computational models to assign cancer treatments</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="chapter-precision.html"><a href="chapter-precision.html#emulating-clinical-trials-to-evaluate-precision-medicine-algorithms"><i class="fa fa-check"></i><b>9.2</b> Emulating clinical trials to evaluate precision medicine algorithms</a><ul>
<li class="chapter" data-level="9.2.1" data-path="chapter-precision.html"><a href="chapter-precision.html#objectives-and-applications"><i class="fa fa-check"></i><b>9.2.1</b> Objectives and applications</a></li>
<li class="chapter" data-level="9.2.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-estimates"><i class="fa fa-check"></i><b>9.2.2</b> Target trials for precision medicine: definition of causal estimates</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-methods-and-precision-medicine"><i class="fa fa-check"></i><b>9.3</b> Causal inference methods and precision medicine</a><ul>
<li class="chapter" data-level="9.3.1" data-path="chapter-precision.html"><a href="chapter-precision.html#a-treatment-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.1</b> A treatment with multiple versions</a></li>
<li class="chapter" data-level="9.3.2" data-path="chapter-precision.html"><a href="chapter-precision.html#causal-inference-with-multiple-versions"><i class="fa fa-check"></i><b>9.3.2</b> Causal inference with multiple versions</a></li>
<li class="chapter" data-level="9.3.3" data-path="chapter-precision.html"><a href="chapter-precision.html#identification_causal_PM"><i class="fa fa-check"></i><b>9.3.3</b> Application to precision medicine</a></li>
<li class="chapter" data-level="9.3.4" data-path="chapter-precision.html"><a href="chapter-precision.html#PM-others"><i class="fa fa-check"></i><b>9.3.4</b> Alternative estimation methods</a></li>
<li class="chapter" data-level="9.3.5" data-path="chapter-precision.html"><a href="chapter-precision.html#code"><i class="fa fa-check"></i><b>9.3.5</b> Code</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-simulated-data"><i class="fa fa-check"></i><b>9.4</b> Application to simulated data</a><ul>
<li class="chapter" data-level="9.4.1" data-path="chapter-precision.html"><a href="chapter-precision.html#general-settings"><i class="fa fa-check"></i><b>9.4.1</b> General settings</a></li>
<li class="chapter" data-level="9.4.2" data-path="chapter-precision.html"><a href="chapter-precision.html#simulation-results"><i class="fa fa-check"></i><b>9.4.2</b> Simulation results</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="chapter-precision.html"><a href="chapter-precision.html#application-to-pdx"><i class="fa fa-check"></i><b>9.5</b> Application to PDX</a></li>
<li class="chapter" data-level="9.6" data-path="chapter-precision.html"><a href="chapter-precision.html#limitations-and-perspectives-1"><i class="fa fa-check"></i><b>9.6</b> Limitations and perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="appendix-datasets.html"><a href="appendix-datasets.html"><i class="fa fa-check"></i><b>A</b> About datasets</a><ul>
<li class="chapter" data-level="A.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-cl"><i class="fa fa-check"></i><b>A.1</b> Cell lines</a><ul>
<li class="chapter" data-level="A.1.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#omics-profiles"><i class="fa fa-check"></i><b>A.1.1</b> Omics profiles</a></li>
<li class="chapter" data-level="A.1.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-GDSC"><i class="fa fa-check"></i><b>A.1.2</b> Drug screenings</a></li>
<li class="chapter" data-level="A.1.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-CRISPR"><i class="fa fa-check"></i><b>A.1.3</b> CRISPR-Cas9 screening</a></li>
</ul></li>
<li class="chapter" data-level="A.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-PDX"><i class="fa fa-check"></i><b>A.2</b> Patient-derived xenografts</a><ul>
<li class="chapter" data-level="A.2.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#overview-of-pdx-data-from-gao2015high"><i class="fa fa-check"></i><b>A.2.1</b> Overview of PDX data from <span class="citation">Gao et al. (<a href="#ref-gao2015high">2015</a>)</span></a></li>
<li class="chapter" data-level="A.2.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#drug-response-metrics"><i class="fa fa-check"></i><b>A.2.2</b> Drug response metrics</a></li>
</ul></li>
<li class="chapter" data-level="A.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-datasets-patients"><i class="fa fa-check"></i><b>A.3</b> Patients</a><ul>
<li class="chapter" data-level="A.3.1" data-path="appendix-datasets.html"><a href="appendix-datasets.html#metabric"><i class="fa fa-check"></i><b>A.3.1</b> METABRIC</a></li>
<li class="chapter" data-level="A.3.2" data-path="appendix-datasets.html"><a href="appendix-datasets.html#tcga-breast-cancer"><i class="fa fa-check"></i><b>A.3.2</b> TCGA: Breast cancer</a></li>
<li class="chapter" data-level="A.3.3" data-path="appendix-datasets.html"><a href="appendix-datasets.html#appendix-prostate"><i class="fa fa-check"></i><b>A.3.3</b> TCGA: Prostate cancer</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="B" data-path="about-logical-models.html"><a href="about-logical-models.html"><i class="fa fa-check"></i><b>B</b> About logical models</a><ul>
<li class="chapter" data-level="B.1" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-fumia"><i class="fa fa-check"></i><b>B.1</b> Generic logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.2" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-verlingue"><i class="fa fa-check"></i><b>B.2</b> Extended logical model of cancer pathways</a></li>
<li class="chapter" data-level="B.3" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-pantolini"><i class="fa fa-check"></i><b>B.3</b> Logical model of BRAF pathways in melanoma and colorectal cancer</a></li>
<li class="chapter" data-level="B.4" data-path="about-logical-models.html"><a href="about-logical-models.html#appendix-montagud"><i class="fa fa-check"></i><b>B.4</b> Logical model of prostate cancer</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>C</b> About statistics</a><ul>
<li class="chapter" data-level="C.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-decomp"><i class="fa fa-check"></i><b>C.1</b> Decomposition of <span class="math inline">\(R^2\)</span></a></li>
<li class="chapter" data-level="C.2" data-path="about-statistics.html"><a href="about-statistics.html#causal-inference-with-multiple-versions-of-treatment"><i class="fa fa-check"></i><b>C.2</b> Causal inference with multiple versions of treatment</a><ul>
<li class="chapter" data-level="C.2.1" data-path="about-statistics.html"><a href="about-statistics.html#appendix-overall-treatment-effect"><i class="fa fa-check"></i><b>C.2.1</b> Overall treatment effect with multiple versions of treatment (equation @ref(eq:overall-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.2" data-path="about-statistics.html"><a href="about-statistics.html#appendix-distrib-treatment-effect"><i class="fa fa-check"></i><b>C.2.2</b> Treatment effect with predefined distributions of versions of treatment (equation @ref(eq:distrib-treatment-effect))</a></li>
<li class="chapter" data-level="C.2.3" data-path="about-statistics.html"><a href="about-statistics.html#appendix-IPW"><i class="fa fa-check"></i><b>C.2.3</b> Inverse probability of treatment weighted (IPW) estimators for precision medicine</a></li>
<li class="chapter" data-level="C.2.4" data-path="about-statistics.html"><a href="about-statistics.html#appendix-TMLE"><i class="fa fa-check"></i><b>C.2.4</b> TMLE</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">From the mechanistic modeling of signaling pathways in cancer to the interpretation of models and their contributions: clinical applications and statistical evaluation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="scientific-modeling-abstract-the-complexity" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Scientific modeling: abstract the complexity</h1>
<blockquote>
<p><em>Ce qui est simple est toujours faux. Ce qui ne l’est pas est inutilisable.</em><br />
Paul Valéry (Mauvaises pensées et autres, 1942)</p>
</blockquote>
<p>The notion of modeling is embedded in science, to the point that it has sometimes been used to define the very nature of scientific research. What is called a model can, however, correspond to very different realities which need to be defined before addressing the object of this thesis which will consist, if one wants to be mischievous, in analyzing models with other models. This semantic elucidation is all the more necessary as this thesis is interdisciplinary, suspended between systems biology and biostatistics. In order to convince the reader of the need for such a preamble, he is invited to ask a statistician and a biologist how they would define what a model is.</p>
<div class="figure" style="text-align: center"><span id="fig:orrery"></span>
<img src="fig/orrery.jpg" alt="A scientist and his model. Joseph Wright of Derby, A Philosopher Giving a Lecture at the Orrery (in which a lamp is put in place of the sun), c. 1763-65, oil on canvas, Derby Museums and Art Gallery" width="90%" />
<p class="caption">
Figure 1.1: <strong>A scientist and his model.</strong> Joseph Wright of Derby, <em>A Philosopher Giving a Lecture at the Orrery (in which a lamp is put in place of the sun)</em>, c. 1763-65, oil on canvas, Derby Museums and Art Gallery
</p>
</div>

<div id="what-is-a-model" class="section level2">
<h2><span class="header-section-number">1.1</span> What is a model?</h2>
<!-- Generic intro -->
<div id="in-your-own-words" class="section level3">
<h3><span class="header-section-number">1.1.1</span> In your own words</h3>
<p>A model is first of all an ambiguous object and a polysemous word. It therefore seems necessary to start with a semantic study. Among the many meanings and synonymous proposed by the dictionary (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:visual-thesaurus">1.2</a>), while some definitions are more related to art, several find echoes in scientific practice. It is sometimes a question of the physical representation of an object, often on a reduced scale as in Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:orrery">1.1</a>, and sometimes of a theoretical description intended to facilitate the understanding of the way in which a system works <span class="citation">(Collins <a href="#ref-dictionnarymodel">2020</a>)</span>. It is even sometimes an ideal to be reached and therefore an ambitious prospect for an introduction.</p>
<div class="figure" style="text-align: center"><span id="fig:visual-thesaurus"></span>
<img src="fig/visualThesaurus.png" alt="Network visualization of model thesaurus entries. Generated with the ‘Visual Thesaurus’ ressource" width="90%" />
<p class="caption">
Figure 1.2: <strong>Network visualization of <em>model</em> thesaurus entries.</strong> Generated with the <a href="https://www.visualthesaurus.com">‘Visual Thesaurus’</a> ressource
</p>
</div>

<p>The narrower perspective of the scientist does not reduce the completeness of the dictionary’s description to an unambiguous object <span class="citation">(Bailer-Jones <a href="#ref-bailer2002scientists">2002</a>)</span>. In an attempt to approach these multi-faceted objects that are the models, Daniela Bailer-Jones interviewed different scientists and asked them the same question: what is a model? Across the different profiles and fields of study, the answers vary but some patterns begin to emerge (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:interviews">1.3</a>). A model must capture the essence of the phenomenon being studied. Because it eludes, voluntarily or not, many details or complexity, it is by nature a simplification of the phenomenon. These limitations may restrict its validity to certain cases or suspend it to the fulfilment of some hypotheses. They are not necessarily predictive, but they must be able to generate new hypotheses, be tested and possibly questioned. Finally, and fundamentally, they <strong>must provide insights about the object of study and contribute to its understanding</strong>.</p>
<div class="figure" style="text-align: center"><span id="fig:interviews"></span>
<img src="01-Models_files/figure-html/interviews-1.png" alt="Scientists talk about their models: words cloud. Cloud of words summarizing the lexical fields used by scientists to talk about their models in dedicated interviews reported by Bailer-Jones (2002)." width="90%" />
<p class="caption">
Figure 1.3: <strong>Scientists talk about their models: words cloud.</strong> Cloud of words summarizing the lexical fields used by scientists to talk about their models in dedicated interviews reported by <span class="citation">Bailer-Jones (<a href="#ref-bailer2002scientists">2002</a>)</span>.
</p>
</div>

<p>These definitions circumscribe the <em>model</em> object, its use and its objectives, but they do not in any way describe its nature. And for good reason, because even if we agree on the described contours, the biodiversity of the models remains overwhelming for taxonomists:</p>
<blockquote>
<p><em>Probing models, phenomenological models, computational models, developmental models, explanatory models, impoverished models, testing models, idealized models, theoretical models, scale models, heuristic models, caricature models, exploratory models, didactic models, fantasy models, minimal models, toy models, imaginary models, mathematical models, mechanistic models, substitute models, iconic models, formal models, analogue models, and instrumental models are but some of the notions that are used to categorize models.</em><br />
<span class="citation">(Frigg and Hartmann <a href="#ref-frigg2020models">2020</a>)</span></p>
</blockquote>
</div>
<div id="physical-world-and-world-of-ideas" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Physical world and world of ideas</h3>
<p>Without claiming to be exhaustive, we can make a <strong>first simple dichotomy between physical/material and formal/intellectual models</strong> <span class="citation">(Rosenblueth and Wiener <a href="#ref-rosenblueth1945role">1945</a>)</span>. The former consists in replacing the object of study by another object, just as physical but nevertheless simpler or better known. These may be models involving a change of scale such as the simple miniature replica placed in a wind tunnel, or the metal double helix model used by Watson and Crick to visualize DNA. In all these cases the model allows to visualize the object of study (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:planets">1.4</a> A and B), to manipulate it and play with it to better understand or explain a phenomenon, just like the scientist with his orrery (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:orrery">1.1</a>). In the case of biology, there are mainly model organisms such as drosophila, zebrafish or mice, for example. We then benefit from the relative simplicity of their genomes, a shorter time scale or ethical differences, usually to elucidate mechanisms of interest in humans. Correspondence between the target system and its model can sometimes be more conceptual, such as that ones relying on mechanical–electrical analogies: a mechanical system (e.g. a spring-mass system) can sometimes be represented by an electric network (e.g. a RLC circuit with a resistor, a capacitor and an inductor).</p>
<div class="figure" style="text-align: center"><span id="fig:planets"></span>
<img src="01-Models_files/figure-html/planets-1.png" alt="Orrery, planets and models. Physical models of planetary motion, either geocentric (Armillary sphere from Plate LXXVII in Encyclopedia Britannica, 1771) or heliocentric in panel B (Bion, 1751, catalogue Bnf) and some geometric representations by Johannes Kepler in panel C (in Astronomia Nova, 1609)" width="90%" />
<p class="caption">
Figure 1.4: <strong>Orrery, planets and models</strong>. Physical models of planetary motion, either geocentric (Armillary sphere from <em>Plate LXXVII</em> in <a href="https://commons.wikimedia.org/wiki/File:EB1711_Armillary_Sphere.png"><em>Encyclopedia Britannica</em></a>, 1771) or heliocentric in panel B (Bion, 1751, <a href="https://gallica.bnf.fr/ark:/12148/btv1b2600252q/f8.item.r=Bion">catalogue Bnf</a>) and some geometric representations by Johannes Kepler in panel C (in <a href="https://commons.wikimedia.org/wiki/File:Kepler_astronomia_nova.jpg"><em>Astronomia Nova</em></a>, 1609)
</p>
</div>

<p>The model is then no longer simply a mimetic replica but is based on an intellectual equivalence: we are gradually moving into the realm of formal models <span class="citation">(Rosenblueth and Wiener <a href="#ref-rosenblueth1945role">1945</a>)</span>. These are of a more symbolic nature and they <strong>represent the original system with a set of logical or mathematical terms</strong>, describing the main driving forces or similar structural properties as geometrical models of planetary motions summarized by Kepler in Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:planets">1.4</a>C. Historically these models have often been expressed by sets of mathematical equations or relationships. Increasingly, these have been implemented by computer. Despite their sometimes less analytical and more numerical nature, many so-called computational models could also belong to this category of formal models. There are then many formalisms, discrete or continuous, deterministic or stochastic, based on differential equations or Boolean algebra <span class="citation">(Fowler, Fowler, and Fowler <a href="#ref-fowler1997mathematical">1997</a>)</span>. Despite their more abstract nature, they offer similar scientific services: it is possible to play with their parameters, specifications or boundary conditions in order to better understand the phenomenon. One can also imagine these formal models from a different perspective, which starts from the data in a bottom-up approach instead of starting from the phenomenon in a top-down analysis. These models will then often be called statistical models or models of data <span class="citation">(Frigg and Hartmann <a href="#ref-frigg2020models">2020</a>)</span>. This distinction will be further clarified in section <a href="scientific-modeling-abstract-the-complexity.html#stat-mech">1.2</a>.</p>
<p>To summarize and continue a little longer with the astronomical metaphor, the study of a particularly complex system (the solar system) can be broken down into a variety of different models. Physical and mechanical models such as armillary spheres (<a href="scientific-modeling-abstract-the-complexity.html#fig:planets">1.4</a>A and B) make it possible to touch the object of study. In addition, we can observe the evolution of models which, when confronted with data, have progressed from a geocentric to a heliocentric representation to get closer to the current state of knowledge. Sometimes, models with more formal representations are used to give substance to ideas and hypotheses (<a href="scientific-modeling-abstract-the-complexity.html#fig:planets">1.4</a>C). One of the most conceptual forms is then the mathematical language and one can thus consider that the previously mentioned astronomical models find their culmination in Kepler’s equations about orbits, areas and periods that describe the elliptical motion of the planets. We refer to them today as Kepler’s laws. The model has become a law and therefore a paragon of mathematical modeling <span class="citation">(Wan <a href="#ref-wan2018mathematical">2018</a>)</span>.</p>
</div>
<div id="preview-about-cancer-models" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Preview about cancer models</h3>
<p>As we get closer to the subject of our study, and in order to illustrate these definitions more concretely, we can take an interest in the meaning of the word <em>model</em> in the context of cancer research. For this, we restrict our corpus to scientific articles found when searching for “cancer model” in the PubMed article database. Among these, we look at the occurrences of the word <em>model</em> and the sentences in which it is included. This cancer-related context of model is represented as a tree in Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:pubmed-tree">1.5</a>. Some of the distinctions already mentioned can be found here. The <em>mouse</em> and <em>xenograft</em> models, which will be discussed later in this thesis, represent some of the most common physical models in cancer studies. These are animal models in which the occurrence and mechanisms of cancer, usually induced by the biologist, are studied. On the other hand, <em>prediction</em>, <em>prognostic</em> or <em>risk score</em> models refer to formal models and borrow from statistical language.</p>
<div class="figure" style="text-align: center"><span id="fig:pubmed-tree"></span>
<img src="fig/pubmed-tree.png" alt="Tree visualization of model semantic context in cancer-related literature Generated with the ‘PubTrees’ tool by Ed Sperr, and based on most relevant PubMed entries for “cancer model” search." width="90%" />
<p class="caption">
Figure 1.5: <strong>Tree visualization of <em>model</em> semantic context in cancer-related literature</strong> Generated with the <a href="https://esperr.github.io/pub-trees/">‘PubTrees’</a> tool by Ed Sperr, and based on most relevant PubMed entries for “cancer model” search.
</p>
</div>

<p>Another way to classify cancer models may be to group them into the following categories: <em>in vivo</em>, <em>in vitro</em> and <em>in silico</em>. The first two clearly belong to the physical models but one uses whole living organisms (e.g. a human tumor implanted in an immunodeficient mouse) and the other separates the living from its organism in order to place it in a controlled environment (e.g. tumor cells in growth medium in a Petri dish). <strong>In the thesis, data from both <em>in vivo</em> and <em>in vitro</em> models will be used. However, unless otherwise stated, a model will always refer to a representation <em>in silico</em>.</strong> This third category, however, contains a very wide variety of models <span class="citation">(Deisboeck et al. <a href="#ref-deisboeck2009silico">2009</a>)</span>, to which we will come back in chapter <a href="mechanistic-cancer.html#mechanistic-cancer">3</a>. A final ambiguity about the nature of the formal models used in this thesis needs to be clarified beforehand.</p>
</div>
</div>
<div id="stat-mech" class="section level2">
<h2><span class="header-section-number">1.2</span> Statistics or mechanistic</h2>
<p>A rather frequent metaphor is to compare formal models to black boxes that take in input <span class="math inline">\(X\)</span> predictors, or independent variables, and output response variable(s) <span class="math inline">\(Y\)</span>, also named dependent variables. The models then split into two categories (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>) depending on the answer to the question: are you modeling the inside of the box or not?</p>
<div id="the-inside-of-the-box" class="section level3">
<h3><span class="header-section-number">1.2.1</span> The inside of the box</h3>
<p>The purpose of this section is to present in a schematic, and therefore somewhat caricatural, manner the two competing formal modeling approaches that will be used in this thesis and that we will call mechanistic modeling and statistical modeling. Assuming the unambiguous nature of the predictors and outputs we can imagine that the natural process consists in defining the result Y from the inputs X according to a function of a completely unknown form (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>A).</p>
<div class="figure" style="text-align: center"><span id="fig:boxes"></span>
<img src="fig/boxes.png" alt="Different modeling strategies. (A) Data generation from predictors \(X\) to response \(Y\) in the natural phenomenon. (B) Mechanistic modeling defining mechanisms of data generation inside the box. (C) Statistical modeling finding the function \(f\) that gives the best predictions. Adapted from Breiman (2001b)." width="60%" />
<p class="caption">
Figure 1.6: <strong>Different modeling strategies.</strong> (A) Data generation from predictors <span class="math inline">\(X\)</span> to response <span class="math inline">\(Y\)</span> in the natural phenomenon. (B) Mechanistic modeling defining mechanisms of data generation inside the box. (C) Statistical modeling finding the function <span class="math inline">\(f\)</span> that gives the best predictions. Adapted from <span class="citation">Breiman (<a href="#ref-breiman2001statistical">2001</a><a href="#ref-breiman2001statistical">b</a>)</span>.
</p>
</div>

<p>The first modeling approach, that we will call <strong>mechanistic</strong>, consists in <strong>building the box by imitating what we think is the process of data generation</strong>, or in other words, by representing the mechanisms at work (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>B). This integration of prior knowledge can take different forms. In this thesis it will often come back to presupposing certain relations between entities according to what is known about their behaviour. <span class="math inline">\(X_1\)</span> which acts on <span class="math inline">\(X_3\)</span> may correspond to the action of one biological entity on another, supposedly unidirectional; just as the joint action of <span class="math inline">\(X_2\)</span> and <span class="math inline">\(X_3\)</span> may reflect a known synergy in the expression of genes or the action of proteins. Mathematically this is expressed here with a perfectly deterministic model defined <em>a priori</em>. All in all, in a purely mechanistic approach, the nature of the relations between entities should be linked to biological processes and the parameters in the model all have biological definitions in such a way that it could even be considered to measure them directly. For example, the coefficient <span class="math inline">\(2\)</span> multiplying <span class="math inline">\(X_2X_3\)</span> can correspond to a stoichiometric coefficient or a reaction constant which have a theoretical justification or are accessible by experimentation. In some fields of literature these models are sometimes called mathematical models because they propose a mathematical translation of a phenomenon, which does not start from the data in a bottom-up approach but rather from a top-down theoretical framework. In this thesis we will adhere to the <em>mechanistic model</em> name, which is more transparent and less ambiguous compared to other approaches also based on mathematics, without necessarily the other characteristics described above.</p>
<p>The second approach, often called <strong>statistical modeling</strong>, or sometimes machine learning depending on the precise context and objective, does not necessarily seek to reproduce the natural process of data generation but to <strong>find the function allowing the best prediction of <span class="math inline">\(Y\)</span> from <span class="math inline">\(X\)</span></strong> (Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>C). Pushed to the limit, they are an “idealized version of the data we gain from immediate observation” <span class="citation">(Frigg and Hartmann <a href="#ref-frigg2020models">2020</a>)</span>, thus providing a phenomenological description. The methods and algorithms used are then intended to be sufficiently flexible and to make the fewest possible assumptions about the relationships between variables or the distribution of data. Without listing them exhaustively, the approaches such as simple linear regressions or more complex support vector machines <span class="citation">(Cortes and Vapnik <a href="#ref-cortes1995support">1995</a>)</span> or random forests <span class="citation">(Breiman <a href="#ref-breiman2001random">2001</a><a href="#ref-breiman2001random">a</a>)</span>, which will sometimes be mentioned in this thesis, fall into this category which contains many others <span class="citation">(Hastie, Tibshirani, and Friedman <a href="#ref-hastie2009elements">2009</a>)</span>.</p>
<p>Several discrepancies result from this difference in nature between mechanistic and statistical models, some of which are summarized in the Table <a href="scientific-modeling-abstract-the-complexity.html#tab:mechstat">1.1</a>. In a somewhat schematic way, we can say that the mechanistic model first asks the question of <em>how</em> and then looks at the result for the output. The <strong>notion of causality is intrinsic to the definition of the model</strong>. Conversely, the statistical model first tries to approach the Y and then possibly analyses what can be deduced from it, regarding the importance of the variables or their relationships in a <em>post hoc</em> approach <span class="citation">(Ishwaran <a href="#ref-ishwaran2007variable">2007</a>; Manica et al. <a href="#ref-manica2019toward">2019</a>)</span>. The causality is then not a by-product of the algorithm and must be evaluated with dedicated frameworks <span class="citation">(Hernán and Robins <a href="#ref-hernan2020causal">2020</a>)</span>. The greater flexibility of statistical methods makes it possible to better accept the heterogeneity of the variables, but this is generally done at the cost of a larger number of parameters and therefore requires more data. Moreover, statistical models can be considered as inductive, since they are able to use already generated data to identify patterns in it. Conversely, mechanistic models are more deductive and they can theoretically allow to extrapolate beyond the original data or knowledge used to build the model <span class="citation">(Baker et al. <a href="#ref-baker2018mechanistic">2018</a>)</span>. Finally, the most relevant way of assessing the value or adequacy of these models may be quite different. A statistical model is measured by its ability to predict output in a validation dataset different from the one used to train its parameters. The mechanistic model will also be evaluated on its capacity to approach the data but also to order it, to give a meaning. If its pure predictive performance is generally inferior, <strong>how can the value of understanding be assessed?</strong> This question will be one of the threads of the dissertation.</p>
<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:mechstat">Table 1.1: </span><strong>Some pros and cons for mechanistic and statistical modeling</strong>. Adapted from <span class="citation">Baker et al. (<a href="#ref-baker2018mechanistic">2018</a>)</span>.
</caption>
<thead>
<tr>
<th style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;text-align: center;">
Mechanistic modeling
</th>
<th style="text-align:left;font-weight: bold;color: white !important;background-color: #808080 !important;text-align: center;">
Statistical modeling
</th>
</tr>
</thead>
<tbody>
<tr grouplength="1">
<td colspan="2" style="border-bottom: 1px solid;">
<strong>Definition</strong>
</td>
</tr>
<tr>
<td style="text-align:left;border-right:1px solid; padding-left: 2em;" indentlevel="1">
Seeks to establish a mechanistic relationship between inputs and outputs
</td>
<td style="text-align:left;">
Seeks to establish statistical relationships between inputs and outputs
</td>
</tr>
<tr grouplength="5">
<td colspan="2" style="border-bottom: 1px solid;">
<strong>Pros and cons</strong>
</td>
</tr>
<tr>
<td style="text-align:left;border-right:1px solid; padding-left: 2em;" indentlevel="1">
Presupposes and investigates causal links between the variables
</td>
<td style="text-align:left;">
Looks for patterns and establishes correlations between variables
</td>
</tr>
<tr>
<td style="text-align:left;border-right:1px solid; padding-left: 2em;" indentlevel="1">
Capable of handling small datasets
</td>
<td style="text-align:left;">
Requires large datasets
</td>
</tr>
<tr>
<td style="text-align:left;border-right:1px solid; padding-left: 2em;" indentlevel="1">
Once validated, can be used as a predictive tool in new situations possibly difficult to access through experimentation
</td>
<td style="text-align:left;">
Can only make predictions that relate to patterns within the data supplied
</td>
</tr>
<tr>
<td style="text-align:left;border-right:1px solid; padding-left: 2em;" indentlevel="1">
Difficult to accurately incorporate information from multiple space and time scales due to constrained specifications
</td>
<td style="text-align:left;">
Can tackle problems with multiple space and time scales thanks to flexible specifications
</td>
</tr>
<tr>
<td style="text-align:left;border-right:1px solid; padding-left: 2em;" indentlevel="1">
Evaluated on closeness to data and ability to make sense of it
</td>
<td style="text-align:left;">
Evaluated based on predictive performance
</td>
</tr>
</tbody>
</table>

<p>Mechanistic and statistical models are not perfectly exclusive and rather form the two ends of a spectrum. The definitions and classification of some examples is therefore still partly personal and arbitrary. For instance, the example in <a href="scientific-modeling-abstract-the-complexity.html#fig:boxes">1.6</a>B can be transformed into a model with a more ambiguous status:</p>
<p><span class="math display">\[logit(P[Y=1])=\beta_1X_1 + \beta_{23}X_2X_3\]</span></p>
<p>This model is deliberately ambiguous. As a logistic model, it is therefore naturally defined as a statistical model. But the definition of the interaction between <span class="math inline">\(X_2\)</span> and <span class="math inline">\(X_3\)</span> denotes a mechanistic presupposition. The very choice of a logistic and therefore parametric model could also result from a knowledge of the phenomenon, even if in practice it is often a default choice for a binary output. Finally, the nature of the parameters <span class="math inline">\(\beta_{1}\)</span> and <span class="math inline">\(\beta_{23}\)</span> is likely to change the interpretation of the model. If they are deduced from the data and therefore optimized to fit Y as well as possible, one will think of a statistical model whose specification is nevertheless based on knowledge of the phenomenon. On the other hand, one could imagine that these parameters are taken from the biochemistry literature or other data. The model will then be more mechanistic. The boundary between these models is further blurred by the different possibilities of combining these approaches and making them complementary <span class="citation">(Baker et al. <a href="#ref-baker2018mechanistic">2018</a>; Salvucci, Rahman, et al. <a href="#ref-salvucci2019machine">2019</a>)</span>.</p>
</div>
<div id="lotkasection" class="section level3">
<h3><span class="header-section-number">1.2.2</span> A tale of prey and predators</h3>
<p>The following is a final general illustration of the concepts and procedures introduced with respect to statistical and mechanistic models through a famous and characteristic example: the Lotka-Volterra model of interactions between prey and predators. This model was, like many students, my first encounter with what could be called mathematical biology. The Italian mathematician Vito Volterra states this system for the first time studying the unexpected characteristics of fish populations in the Adriatic Sea after the First World War. Interestingly, Alfred Lotka, an American physicist deduced the exact same system independantly, starting from very generic process of redistribution of matter among the several components derived from law of mass action <span class="citation">(Knuuttila and Loettgers <a href="#ref-knuuttila2017modelling">2017</a>)</span>. A detailed description of their works and historical formulation can be found in original articles <span class="citation">(Lotka <a href="#ref-lotka1925principles">1925</a>; Volterra <a href="#ref-volterra1926fluctuations">1926</a>)</span> or dedicated reviews <span class="citation">(Knuuttila and Loettgers <a href="#ref-knuuttila2017modelling">2017</a>)</span>.</p>
<p>The general objective is to understand the evolution of the populations of a species of prey and its predator, reasonably isolated from outside intervention. Here we will use Canada lynx (<em>Lynx canadensis</em>) and snowshow hare (<em>Lepus americanus</em>) populations for which an illustrative data set exists <span class="citation">(Hewitt <a href="#ref-hewitt1917conservation">1917</a>)</span>. In fact, commercial records listing the quantities of furs sold by trappers to the Canadian Hudson Bay Company may represent a proxy for the populations of these two species as represented in Figure <a href="scientific-modeling-abstract-the-complexity.html#fig:lotka">1.7</a>A. Denoting the population of lynx <span class="math inline">\(L(t)\)</span> and the population of hare <span class="math inline">\(H(t)\)</span> it can be hypothesized that prey, in the absence of predators, would increase in population, while predators on their own would decline in the absence of prey. A prey/predator interaction term can then be added, which will positively impact predators and negatively impact prey. The system can then be formalized with the following differential questions with all coefficients <span class="math inline">\(a_1, a_2, b_1, b_2 &gt;0\)</span>:</p>
<p><span class="math display">\[\dfrac{dH}{dt}=a_1H-a_2HL\]</span></p>
<p><span class="math display">\[\dfrac{dL}{dt}=-b_1L+b_2HL\]</span></p>
<p><span class="math inline">\(a_1H\)</span> represents the growth rate of the hare population (prey), i.e., the population grows in proportion to the population itself according to usual birth modeling. The main losses of hares are due to predation by lynx, as represented with a negative coefficient in the <span class="math inline">\(-a_2HT\)</span> term. It is therefore assumed that a fixed percentage of prey-predator encounters will result in the death of the prey. Conversely, it is assumed that the growth of the lynx population depends primarily on the availability of food for all lynxes, summarized in the <span class="math inline">\(b_2HL\)</span> term. In the absence of hares, the lynx population decreases, as denoted by the coefficient <span class="math inline">\(-b_1L\)</span>. Important features of mechanistic models are illustrated here: the equations are based on <em>a priori</em> knowledge or assumptions about the structure of the problem and the parameters of the model can be interpreted. <span class="math inline">\(a_1\)</span>, for example, could correspond to the frequency of litters among hares and the number of offspring per litter.</p>
<div class="figure" style="text-align: center"><span id="fig:lotka"></span>
<img src="01-Models_files/figure-html/lotka-1.png" alt="Some analyses around Lotka-Volterra model of a prey-predator system. (A) Evolution of lynx and hares populations based on Hudson Bay Company data about fur pelts. (B) and (C) Linear regression for estimation of parameters. (D) Evolution of lynx and hare populations as predicted by the model based on inferred parameters and initial conditions." width="90%" />
<p class="caption">
Figure 1.7: <strong>Some analyses around Lotka-Volterra model of a prey-predator system</strong>. (A) Evolution of lynx and hares populations based on Hudson Bay Company data about fur pelts. (B) and (C) Linear regression for estimation of parameters. (D) Evolution of lynx and hare populations as predicted by the model based on inferred parameters and initial conditions.
</p>
</div>

<p>This being said, the structure of the model having been defined <em>a priori</em>, it remains to determine its parameters. Two options would theoretically be possible: to propose values based on the interpretation of the parameters and ecological knowledge, or to fit the model to the data in order to find the best parameters. For the sake of simplicity, and because this example has only a pedagogical value in this presentation, we propose to determine them approximately using the following Taylor-based approximation:</p>
<p><span class="math display">\[\dfrac{1}{y(t)} \dfrac{dy}{dt} \simeq \dfrac{1}{y(t)} \dfrac{y(t+1)-y(t-1)}{2}\]</span></p>
<p>By applying this approximation to the two equations of the differential system and plotting the corresponding linear regressions (Figures <a href="scientific-modeling-abstract-the-complexity.html#fig:lotka">1.7</a>B and C), we can obtain an evaluation of the parameters such as <span class="math inline">\(a_1=0.82\)</span>, <span class="math inline">\(a_2=0.0298\)</span>, <span class="math inline">\(b_1=0.509\)</span>, <span class="math inline">\(b_2=0.0129\)</span>. By matching the initial conditions to the data, the differential system can then be fully determined and solved numerically (Figures <a href="scientific-modeling-abstract-the-complexity.html#fig:lotka">1.7</a>D). Comparison of data and modeling provides a good illustration of the virtues and weaknesses of a mechanistic model. Firstly, based on explicit and interpretable hypotheses, the model was able to recover the cyclical behaviour and dependencies between the two species: the increase in the lynx population always seems to be preceded by the increase in the hare population. However, the amplitude of the oscillations and their periods are not exactly those observed in the data. This may be related to approximations in the evaluation of parameters, random variation in the data or, of course, simplifications or errors in the structure of the model itself.</p>
<p>Besides, if one tries to carry out a statistical modeling of these data, it is very likely that it is possible to approach the curve of populations evolution much closer, especially for the hares. But should it be expressed simply as a function of time or should a joint modeling be proposed? The nature of the causal link between prey and predators will be extremely difficult to establish without strong hypotheses such as those of the mechanistic model. On the other hand, if populations in later years had to be predicted as accurately as possible, it is likely that a sufficiently well-trained statistical model would perform better. Finally, and this is a fundamental difference, the <strong>mechanistic model enables to test cases or hypotheses that go beyond the scope of the data</strong>. Quite simply, by playing with the variables or parameters of the model, we can predict the exponential decrease of predators in the absence of prey and the exponential growth of prey in the absence of predator. More generally, it is also possible to study analytically or numerically the bifurcation points of the system in order to determine the families of behaviours according to the relative values of the parameters <span class="citation">(Flake <a href="#ref-flake1998computational">1998</a>)</span>. It is not possible to infer these new or hypothetical behaviours directly from the data of the statistical model. This is theoretically possible on the basis of the mechanistic model, provided that it is sufficiently relevant and that its operating hypotheses cover the cases under investigation. Now that the value of mechanistic models has been illustrated in a fairly theoretical example, all that remains is to explore in the next chapters how they can be built and used in the context of cancer.</p>
</div>
</div>
<div id="simplicity-is-the-ultimate-sophistication" class="section level2">
<h2><span class="header-section-number">1.3</span> Simplicity is the ultimate sophistication</h2>
<p>Before concluding this modeling introduction, it is important to highlight one of the most important points already introduced in a concise manner by the poet Paul Valéry at the beginning of this chapter. <strong>Whatever its nature, a model is always a simplified representation of reality and by extension is always wrong to a certain extent</strong>. This is a generally well-accepted fact, but it is crucial to understand the implications for the modeler. This simplification is not a collateral effect but an intrinsic feature of any model:</p>
<blockquote>
<p><em>No substantial part of the universe is so simple that it can be grasped and controlled without abstraction. Abstraction consists in replacing the part of the universe under consideration by a model of similar but simpler structure. Models, formal and intellectual on the one hand, or material on the other, are thus a central necessity of scientific procedure.</em><br />
<span class="citation">(Rosenblueth and Wiener <a href="#ref-rosenblueth1945role">1945</a>)</span></p>
</blockquote>
<p>Therefore, a model exists only because we are not able to deal directly with the phenomenon and simplification is a necessity to make it more tractable <span class="citation">(Potochnik <a href="#ref-potochnik2017idealization">2017</a>)</span>. This simplification appeared many times in the studies of frictionless planes or theoretically isolated systems, in a totally deliberate strategy. However, this idealization can be viewed in several ways <span class="citation">(Weisberg <a href="#ref-weisberg2007three">2007</a>)</span>. One of them, called Aristotelian or minimal idealization, is to eliminate all the properties of an object that we think are not relevant to the problem in question. This amounts to lying by omission or making assumptions of insignificance by focusing on key causal factors only <span class="citation">(Frigg and Hartmann <a href="#ref-frigg2020models">2020</a>)</span>. We therefore refer to the <em>a priori</em> idea that we have of the phenomenon. The other idealization, called Galilean, is to deliberately distort the theory to make it tractable as explicited by Galileo himself:</p>
<blockquote>
<p><em>We are trying to investigate what would happen to moveables very diverse in weight, in a medium quite devoid of resistance, so that the whole difference of speed existing between these moveables would have to be referred to inequality of weight alone. Since we lack such a space, let us (instead) observe what happens in the thinnest and least resistant media, comparing this with what happens in others less thin and more resistant.</em></p>
</blockquote>
<p>This fairly pragmatic approach should make it possible to evolve iteratively, reducing distortions as and when possible. This could involve the addition of other species or human intervention into the Lotka-Volterra system described above. A three-species Lotka-Volterra model can however become chaotic <span class="citation">(Flake <a href="#ref-flake1998computational">1998</a>)</span>, and therefore extremely difficult to use and interpret, thus underlining the importance of simplifying the model.</p>
<p>We will have the opportunity to come back to the idealizations made in the course of the cancer models but it is already possible to give some orientations. The biologist who seeks to study cancer using cell lines or animal models is clearly part of Galileo’s lineage. The mathematical or <em>in silico</em> modeler has a more balanced profile. The design of qualitative mechanistic models based on prior knowledge, which is the core of the second part of the thesis, is more akin to minimal idealization, which seeks to highlight the salient features of a system. The Galilean consistins in studying mathematically tractable systems was also important. To take the example of prey-predator interactions, a differential system with more variables quickly becomes impossible to solve by hand. The development of more and more powerful computers has apparently pushed back the limits of the computationally tractable systems and thus of Galilean idealization. However, this is always necessary, for example in high-dimensional statistical approaches (thousands of variables) where the modelers decide to consider the variables independently while neglecting their interactions.</p>
<p>Because of the complexity of the phenomena, simplification is therefore a necessity. The objective then should not necessarily be to make the model more complex, but to <strong>match its level of simplification with its assumptions and objectives</strong>. Faced with the temptation of the author of the model, or his reviewer, to always extend and complicate the model, it could be replied with Lewis Carrol words<a href="#fn2" class="footnoteRef" id="fnref2"><sup>2</sup></a>:</p>
<blockquote>
<p><em>“That’s another thing we’ve learned from your Nation,” said Mein Herr, “map-making. But we’ve carried it much further than you. What do you consider the largest map that would be really useful?”</em><br />
<em>“About six inches to the mile.”</em><br />
<em>“Only six inches!” exclaimed Mein Herr. “We very soon got to six yards to the mile. Then we tried a hundred yards to the mile. And then came the grandest idea of all! We actually made a map of the country, on the scale of a mile to the mile!”</em><br />
<em>“Have you used it much?” I enquired.</em><br />
<em>“It has never been spread out, yet,” said Mein Herr: “the farmers objected: they said it would cover the whole country, and shut out the sunlight! So we now use the country itself, as its own map, and I assure you it does nearly as well.”</em><br />
Lewis Carroll, <em>Sylvie and Bruno</em> (1893)</p>
</blockquote>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-bailer2002scientists">
<p>Bailer-Jones, Daniela M. 2002. “Scientists’ Thoughts on Scientific Models.” <em>Perspectives on Science</em> 10 (3). MIT Press: 275–301.</p>
</div>
<div id="ref-baker2018mechanistic">
<p>Baker, Ruth E, Jose-Maria Pena, Jayaratnam Jayamohan, and Antoine Jérusalem. 2018. “Mechanistic Models Versus Machine Learning, a Fight Worth Fighting for the Biological Community?” <em>Biology Letters</em> 14 (5). The Royal Society: 20170660.</p>
</div>
<div id="ref-breiman2001random">
<p>Breiman, Leo. 2001a. “Random Forests.” <em>Machine Learning</em> 45 (1). Springer: 5–32.</p>
</div>
<div id="ref-breiman2001statistical">
<p>Breiman, Leo. 2001b. “Statistical Modeling: The Two Cultures (with Comments and a Rejoinder by the Author).” <em>Statistical Science</em> 16 (3). Institute of Mathematical Statistics: 199–231.</p>
</div>
<div id="ref-dictionnarymodel">
<p>Collins. 2020. <em>The Collins English Dictionary</em>. HarperCollins.</p>
</div>
<div id="ref-cortes1995support">
<p>Cortes, Corinna, and Vladimir Vapnik. 1995. “Support-Vector Networks.” <em>Machine Learning</em> 20 (3). Springer: 273–97.</p>
</div>
<div id="ref-deisboeck2009silico">
<p>Deisboeck, Thomas S, Le Zhang, Jeongah Yoon, and Jose Costa. 2009. “In Silico Cancer Modeling: Is It Ready for Prime Time?” <em>Nature Clinical Practice Oncology</em> 6 (1). Nature Publishing Group: 34–42.</p>
</div>
<div id="ref-flake1998computational">
<p>Flake, Gary William. 1998. <em>The Computational Beauty of Nature: Computer Explorations of Fractals, Chaos, Complex Systems, and Adaptation</em>. MIT press.</p>
</div>
<div id="ref-fowler1997mathematical">
<p>Fowler, Andrew Cadle, Anna C Fowler, and AC Fowler. 1997. <em>Mathematical Models in the Applied Sciences</em>. Vol. 17. Cambridge University Press.</p>
</div>
<div id="ref-frigg2020models">
<p>Frigg, Roman, and Stephan Hartmann. 2020. “Models in Science.” In <em>The Stanford Encyclopedia of Philosophy</em>, edited by Edward N. Zalta, Spring 2020. <a href="https://plato.stanford.edu/archives/spr2020/entries/models-science/" class="uri">https://plato.stanford.edu/archives/spr2020/entries/models-science/</a>; Metaphysics Research Lab, Stanford University.</p>
</div>
<div id="ref-hastie2009elements">
<p>Hastie, Trevor, Robert Tibshirani, and Jerome Friedman. 2009. <em>The Elements of Statistical Learning: Data Mining, Inference, and Prediction</em>. Springer Science &amp; Business Media.</p>
</div>
<div id="ref-hernan2020causal">
<p>Hernán, MA, and JM Robins. 2020. “Causal Inference: What If.” <em>Boca Raton: Chapman &amp; Hill/CRC</em>.</p>
</div>
<div id="ref-hewitt1917conservation">
<p>Hewitt, C Gordon. 1917. “Conservation of Wild Life in Canada.” Nature Publishing Group.</p>
</div>
<div id="ref-ishwaran2007variable">
<p>Ishwaran, Hemant. 2007. “Variable Importance in Binary Regression Trees and Forests.” <em>Electronic Journal of Statistics</em> 1. The Institute of Mathematical Statistics; the Bernoulli Society: 519–37.</p>
</div>
<div id="ref-knuuttila2017modelling">
<p>Knuuttila, Tarja, and Andrea Loettgers. 2017. “Modelling as Indirect Representation? The Lotka–Volterra Model Revisited.” <em>The British Journal for the Philosophy of Science</em> 68 (4). Oxford University Press: 1007–36.</p>
</div>
<div id="ref-lotka1925principles">
<p>Lotka, AJ. 1925. “Principles of Physical Biology.” <em>Baltimore: Waverly</em>.</p>
</div>
<div id="ref-manica2019toward">
<p>Manica, Matteo, Ali Oskooei, Jannis Born, Vigneshwari Subramanian, Julio Sáez-Rodríguez, and María Rodríguez Martínez. 2019. “Toward Explainable Anticancer Compound Sensitivity Prediction via Multimodal Attention-Based Convolutional Encoders.” <em>Molecular Pharmaceutics</em>. ACS Publications.</p>
</div>
<div id="ref-potochnik2017idealization">
<p>Potochnik, Angela. 2017. <em>Idealization and the Aims of Science</em>. University of Chicago Press.</p>
</div>
<div id="ref-rosenblueth1945role">
<p>Rosenblueth, Arturo, and Norbert Wiener. 1945. “The Role of Models in Science.” <em>Philosophy of Science</em> 12 (4). Williams; Wilkins Co.: 316–21.</p>
</div>
<div id="ref-salvucci2019machine">
<p>Salvucci, Manuela, Arman Rahman, Alexa J Resler, Girish M Udupi, Deborah A McNamara, Elaine W Kay, Pierre Laurent-Puig, et al. 2019. “A Machine Learning Platform to Optimize the Translation of Personalized Network Models to the Clinic.” <em>JCO Clinical Cancer Informatics</em> 3. American Society of Clinical Oncology: 1–17.</p>
</div>
<div id="ref-volterra1926fluctuations">
<p>Volterra, Vito. 1926. “Fluctuations in the Abundance of a Species Considered Mathematically.” Nature Publishing Group.</p>
</div>
<div id="ref-wan2018mathematical">
<p>Wan, Frederick YM. 2018. <em>Mathematical Models and Their Analysis</em>. Vol. 79. SIAM.</p>
</div>
<div id="ref-weisberg2007three">
<p>Weisberg, Michael. 2007. “Three Kinds of Idealization.” <em>The Journal of Philosophy</em> 104 (12): 639–59.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="2">
<li id="fn2"><p>More concisely stated by <span class="citation">Rosenblueth and Wiener (<a href="#ref-rosenblueth1945role">1945</a>)</span>: “best material model for a cat is another cat, or preferably the same cat.”<a href="scientific-modeling-abstract-the-complexity.html#fnref2">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="preface.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="cancer-as-deregulation-of-complex-machinery.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["PhdThesis.pdf"],
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
